{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "code",
      "source": [
        "%cd data-for-clinical-note-summarizer"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "mnIzExLggVH4",
        "outputId": "651a712a-8efa-4944-84aa-0b644eee8deb"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/data-for-clinical-note-summarizer\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "oxJ5mgqqexgw",
        "outputId": "e9dc97d7-6a1a-4ff4-8e70-7fe323f26c8c"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2024-12-17 02:45:34--  ftp://ftp.ncbi.nlm.nih.gov/pub/pmc/oa_bulk/oa_noncomm/xml/oa_noncomm_xml.PMC001xxxxxx.baseline.2024-06-18.tar.gz\n",
            "           => ‘oa_noncomm_xml.PMC001xxxxxx.baseline.2024-06-18.tar.gz’\n",
            "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.10, 130.14.250.11, ...\n",
            "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
            "Logging in as anonymous ... Logged in!\n",
            "==> SYST ... done.    ==> PWD ... done.\n",
            "==> TYPE I ... done.  ==> CWD (1) /pub/pmc/oa_bulk/oa_noncomm/xml ... done.\n",
            "==> SIZE oa_noncomm_xml.PMC001xxxxxx.baseline.2024-06-18.tar.gz ... 29943293\n",
            "==> PASV ... done.    ==> RETR oa_noncomm_xml.PMC001xxxxxx.baseline.2024-06-18.tar.gz ... done.\n",
            "Length: 29943293 (29M) (unauthoritative)\n",
            "\n",
            "oa_noncomm_xml.PMC0 100%[===================>]  28.56M  33.5MB/s    in 0.9s    \n",
            "\n",
            "2024-12-17 02:45:35 (33.5 MB/s) - ‘oa_noncomm_xml.PMC001xxxxxx.baseline.2024-06-18.tar.gz’ saved [29943293]\n",
            "\n",
            "--2024-12-17 02:45:35--  ftp://ftp.ncbi.nlm.nih.gov/pub/pmc/oa_bulk/oa_noncomm/xml/oa_noncomm_xml.PMC002xxxxxx.baseline.2024-06-18.tar.gz\n",
            "           => ‘oa_noncomm_xml.PMC002xxxxxx.baseline.2024-06-18.tar.gz’\n",
            "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.10, 130.14.250.11, ...\n",
            "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
            "Logging in as anonymous ... Logged in!\n",
            "==> SYST ... done.    ==> PWD ... done.\n",
            "==> TYPE I ... done.  ==> CWD (1) /pub/pmc/oa_bulk/oa_noncomm/xml ... done.\n",
            "==> SIZE oa_noncomm_xml.PMC002xxxxxx.baseline.2024-06-18.tar.gz ... 681529552\n",
            "==> PASV ... done.    ==> RETR oa_noncomm_xml.PMC002xxxxxx.baseline.2024-06-18.tar.gz ... done.\n",
            "Length: 681529552 (650M) (unauthoritative)\n",
            "\n",
            "oa_noncomm_xml.PMC0 100%[===================>] 649.96M  97.5MB/s    in 10s     \n",
            "\n",
            "2024-12-17 02:45:45 (64.7 MB/s) - ‘oa_noncomm_xml.PMC002xxxxxx.baseline.2024-06-18.tar.gz’ saved [681529552]\n",
            "\n",
            "--2024-12-17 02:45:45--  ftp://ftp.ncbi.nlm.nih.gov/pub/pmc/oa_bulk/oa_noncomm/xml/oa_noncomm_xml.PMC003xxxxxx.baseline.2024-06-18.tar.gz\n",
            "           => ‘oa_noncomm_xml.PMC003xxxxxx.baseline.2024-06-18.tar.gz’\n",
            "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.10, 130.14.250.11, ...\n",
            "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
            "Logging in as anonymous ... Logged in!\n",
            "==> SYST ... done.    ==> PWD ... done.\n",
            "==> TYPE I ... done.  ==> CWD (1) /pub/pmc/oa_bulk/oa_noncomm/xml ... done.\n",
            "==> SIZE oa_noncomm_xml.PMC003xxxxxx.baseline.2024-06-18.tar.gz ... 1680010978\n",
            "==> PASV ... done.    ==> RETR oa_noncomm_xml.PMC003xxxxxx.baseline.2024-06-18.tar.gz ... done.\n",
            "Length: 1680010978 (1.6G) (unauthoritative)\n",
            "\n",
            "oa_noncomm_xml.PMC0 100%[===================>]   1.56G  73.6MB/s    in 18s     \n",
            "\n",
            "2024-12-17 02:46:04 (88.8 MB/s) - ‘oa_noncomm_xml.PMC003xxxxxx.baseline.2024-06-18.tar.gz’ saved [1680010978]\n",
            "\n"
          ]
        }
      ],
      "source": [
        "!wget ftp://ftp.ncbi.nlm.nih.gov/pub/pmc/oa_bulk/oa_noncomm/xml/oa_noncomm_xml.PMC001xxxxxx.baseline.2024-06-18.tar.gz\n",
        "!wget ftp://ftp.ncbi.nlm.nih.gov/pub/pmc/oa_bulk/oa_noncomm/xml/oa_noncomm_xml.PMC002xxxxxx.baseline.2024-06-18.tar.gz\n",
        "!wget ftp://ftp.ncbi.nlm.nih.gov/pub/pmc/oa_bulk/oa_noncomm/xml/oa_noncomm_xml.PMC003xxxxxx.baseline.2024-06-18.tar.gz\n"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from google.colab import drive\n",
        "drive.mount('/content/drive')"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "SspicacTfdvx",
        "outputId": "0b6fa874-f5ba-45d0-d106-b9ec95482167"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Mounted at /content/drive\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!cp -r /content/data-for-clinical-note-summarizer /content/drive/MyDrive/data-for-clinical-note-summarizer/"
      ],
      "metadata": {
        "id": "SnCbKkHRftfA"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!mkdir /content/drive/MyDrive/data-for-clinical-note-summarizer"
      ],
      "metadata": {
        "id": "fYuBcduUgkO8"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!mkdir extracted_xml_files\n",
        "!tar -xzvf /content/data-for-clinical-note-summarizer/oa_noncomm_xml.PMC001xxxxxx.baseline.2024-06-18.tar.gz -C extracted_xml_files\n",
        "!tar -xzvf /content/data-for-clinical-note-summarizer/oa_noncomm_xml.PMC002xxxxxx.baseline.2024-06-18.tar.gz -C extracted_xml_files\n",
        "!tar -xzvf /content/data-for-clinical-note-summarizer/oa_noncomm_xml.PMC003xxxxxx.baseline.2024-06-18.tar.gz -C extracted_xml_files\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "EWmTfDWJog4N",
        "outputId": "a2e99994-a851-4479-9159-11dbadae1092"
      },
      "execution_count": 10,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[1;30;43mStreaming output truncated to the last 5000 lines.\u001b[0m\n",
            "PMC003xxxxxx/PMC3962322.xml\n",
            "PMC003xxxxxx/PMC3962505.xml\n",
            "PMC003xxxxxx/PMC3962506.xml\n",
            "PMC003xxxxxx/PMC3962507.xml\n",
            "PMC003xxxxxx/PMC3962508.xml\n",
            "PMC003xxxxxx/PMC3962515.xml\n",
            "PMC003xxxxxx/PMC3962527.xml\n",
            "PMC003xxxxxx/PMC3962688.xml\n",
            "PMC003xxxxxx/PMC3962689.xml\n",
            "PMC003xxxxxx/PMC3962727.xml\n",
            "PMC003xxxxxx/PMC3962728.xml\n",
            "PMC003xxxxxx/PMC3962741.xml\n",
            "PMC003xxxxxx/PMC3962742.xml\n",
            "PMC003xxxxxx/PMC3962744.xml\n",
            "PMC003xxxxxx/PMC3962983.xml\n",
            "PMC003xxxxxx/PMC3963030.xml\n",
            "PMC003xxxxxx/PMC3963031.xml\n",
            "PMC003xxxxxx/PMC3963032.xml\n",
            "PMC003xxxxxx/PMC3963033.xml\n",
            "PMC003xxxxxx/PMC3963034.xml\n",
            "PMC003xxxxxx/PMC3963035.xml\n",
            "PMC003xxxxxx/PMC3963064.xml\n",
            "PMC003xxxxxx/PMC3963065.xml\n",
            "PMC003xxxxxx/PMC3963066.xml\n",
            "PMC003xxxxxx/PMC3963068.xml\n",
            "PMC003xxxxxx/PMC3963069.xml\n",
            "PMC003xxxxxx/PMC3963070.xml\n",
            "PMC003xxxxxx/PMC3963071.xml\n",
            "PMC003xxxxxx/PMC3963072.xml\n",
            "PMC003xxxxxx/PMC3963073.xml\n",
            "PMC003xxxxxx/PMC3963074.xml\n",
            "PMC003xxxxxx/PMC3963075.xml\n",
            "PMC003xxxxxx/PMC3963076.xml\n",
            "PMC003xxxxxx/PMC3963077.xml\n",
            "PMC003xxxxxx/PMC3963079.xml\n",
            "PMC003xxxxxx/PMC3963080.xml\n",
            "PMC003xxxxxx/PMC3963081.xml\n",
            "PMC003xxxxxx/PMC3963082.xml\n",
            "PMC003xxxxxx/PMC3963083.xml\n",
            "PMC003xxxxxx/PMC3963085.xml\n",
            "PMC003xxxxxx/PMC3963086.xml\n",
            "PMC003xxxxxx/PMC3963087.xml\n",
            "PMC003xxxxxx/PMC3963088.xml\n",
            "PMC003xxxxxx/PMC3963089.xml\n",
            "PMC003xxxxxx/PMC3963090.xml\n",
            "PMC003xxxxxx/PMC3963091.xml\n",
            "PMC003xxxxxx/PMC3963092.xml\n",
            "PMC003xxxxxx/PMC3963093.xml\n",
            "PMC003xxxxxx/PMC3963094.xml\n",
            "PMC003xxxxxx/PMC3963096.xml\n",
            "PMC003xxxxxx/PMC3963097.xml\n",
            "PMC003xxxxxx/PMC3963098.xml\n",
            "PMC003xxxxxx/PMC3963099.xml\n",
            "PMC003xxxxxx/PMC3963101.xml\n",
            "PMC003xxxxxx/PMC3963102.xml\n",
            "PMC003xxxxxx/PMC3963103.xml\n",
            "PMC003xxxxxx/PMC3963104.xml\n",
            "PMC003xxxxxx/PMC3963121.xml\n",
            "PMC003xxxxxx/PMC3963127.xml\n",
            "PMC003xxxxxx/PMC3963129.xml\n",
            "PMC003xxxxxx/PMC3963133.xml\n",
            "PMC003xxxxxx/PMC3963151.xml\n",
            "PMC003xxxxxx/PMC3963164.xml\n",
            "PMC003xxxxxx/PMC3963178.xml\n",
            "PMC003xxxxxx/PMC3963191.xml\n",
            "PMC003xxxxxx/PMC3963192.xml\n",
            "PMC003xxxxxx/PMC3963193.xml\n",
            "PMC003xxxxxx/PMC3963194.xml\n",
            "PMC003xxxxxx/PMC3963195.xml\n",
            "PMC003xxxxxx/PMC3963196.xml\n",
            "PMC003xxxxxx/PMC3963197.xml\n",
            "PMC003xxxxxx/PMC3963198.xml\n",
            "PMC003xxxxxx/PMC3963199.xml\n",
            "PMC003xxxxxx/PMC3963200.xml\n",
            "PMC003xxxxxx/PMC3963201.xml\n",
            "PMC003xxxxxx/PMC3963202.xml\n",
            "PMC003xxxxxx/PMC3963203.xml\n",
            "PMC003xxxxxx/PMC3963204.xml\n",
            "PMC003xxxxxx/PMC3963205.xml\n",
            "PMC003xxxxxx/PMC3963206.xml\n",
            "PMC003xxxxxx/PMC3963207.xml\n",
            "PMC003xxxxxx/PMC3963208.xml\n",
            "PMC003xxxxxx/PMC3963209.xml\n",
            "PMC003xxxxxx/PMC3963256.xml\n",
            "PMC003xxxxxx/PMC3963281.xml\n",
            "PMC003xxxxxx/PMC3963293.xml\n",
            "PMC003xxxxxx/PMC3963294.xml\n",
            "PMC003xxxxxx/PMC3963295.xml\n",
            "PMC003xxxxxx/PMC3963296.xml\n",
            "PMC003xxxxxx/PMC3963297.xml\n",
            "PMC003xxxxxx/PMC3963298.xml\n",
            "PMC003xxxxxx/PMC3963299.xml\n",
            "PMC003xxxxxx/PMC3963300.xml\n",
            "PMC003xxxxxx/PMC3963301.xml\n",
            "PMC003xxxxxx/PMC3963302.xml\n",
            "PMC003xxxxxx/PMC3963303.xml\n",
            "PMC003xxxxxx/PMC3963304.xml\n",
            "PMC003xxxxxx/PMC3963305.xml\n",
            "PMC003xxxxxx/PMC3963306.xml\n",
            "PMC003xxxxxx/PMC3963307.xml\n",
            "PMC003xxxxxx/PMC3963308.xml\n",
            "PMC003xxxxxx/PMC3963309.xml\n",
            "PMC003xxxxxx/PMC3963310.xml\n",
            "PMC003xxxxxx/PMC3963311.xml\n",
            "PMC003xxxxxx/PMC3963312.xml\n",
            "PMC003xxxxxx/PMC3963313.xml\n",
            "PMC003xxxxxx/PMC3963314.xml\n",
            "PMC003xxxxxx/PMC3963316.xml\n",
            "PMC003xxxxxx/PMC3963317.xml\n",
            "PMC003xxxxxx/PMC3963318.xml\n",
            "PMC003xxxxxx/PMC3963319.xml\n",
            "PMC003xxxxxx/PMC3963320.xml\n",
            "PMC003xxxxxx/PMC3963321.xml\n",
            "PMC003xxxxxx/PMC3963322.xml\n",
            "PMC003xxxxxx/PMC3963323.xml\n",
            "PMC003xxxxxx/PMC3963324.xml\n",
            "PMC003xxxxxx/PMC3963325.xml\n",
            "PMC003xxxxxx/PMC3963326.xml\n",
            "PMC003xxxxxx/PMC3963327.xml\n",
            "PMC003xxxxxx/PMC3963328.xml\n",
            "PMC003xxxxxx/PMC3963329.xml\n",
            "PMC003xxxxxx/PMC3963330.xml\n",
            "PMC003xxxxxx/PMC3963331.xml\n",
            "PMC003xxxxxx/PMC3963332.xml\n",
            "PMC003xxxxxx/PMC3963333.xml\n",
            "PMC003xxxxxx/PMC3963334.xml\n",
            "PMC003xxxxxx/PMC3963335.xml\n",
            "PMC003xxxxxx/PMC3963336.xml\n",
            "PMC003xxxxxx/PMC3963337.xml\n",
            "PMC003xxxxxx/PMC3963338.xml\n",
            "PMC003xxxxxx/PMC3963339.xml\n",
            "PMC003xxxxxx/PMC3963340.xml\n",
            "PMC003xxxxxx/PMC3963341.xml\n",
            "PMC003xxxxxx/PMC3963342.xml\n",
            "PMC003xxxxxx/PMC3963343.xml\n",
            "PMC003xxxxxx/PMC3963344.xml\n",
            "PMC003xxxxxx/PMC3963345.xml\n",
            "PMC003xxxxxx/PMC3963346.xml\n",
            "PMC003xxxxxx/PMC3963347.xml\n",
            "PMC003xxxxxx/PMC3963348.xml\n",
            "PMC003xxxxxx/PMC3963349.xml\n",
            "PMC003xxxxxx/PMC3963350.xml\n",
            "PMC003xxxxxx/PMC3963351.xml\n",
            "PMC003xxxxxx/PMC3963352.xml\n",
            "PMC003xxxxxx/PMC3963353.xml\n",
            "PMC003xxxxxx/PMC3963354.xml\n",
            "PMC003xxxxxx/PMC3963355.xml\n",
            "PMC003xxxxxx/PMC3963356.xml\n",
            "PMC003xxxxxx/PMC3963357.xml\n",
            "PMC003xxxxxx/PMC3963358.xml\n",
            "PMC003xxxxxx/PMC3963367.xml\n",
            "PMC003xxxxxx/PMC3963385.xml\n",
            "PMC003xxxxxx/PMC3963413.xml\n",
            "PMC003xxxxxx/PMC3963422.xml\n",
            "PMC003xxxxxx/PMC3963458.xml\n",
            "PMC003xxxxxx/PMC3963515.xml\n",
            "PMC003xxxxxx/PMC3963526.xml\n",
            "PMC003xxxxxx/PMC3963527.xml\n",
            "PMC003xxxxxx/PMC3963528.xml\n",
            "PMC003xxxxxx/PMC3963529.xml\n",
            "PMC003xxxxxx/PMC3963530.xml\n",
            "PMC003xxxxxx/PMC3963531.xml\n",
            "PMC003xxxxxx/PMC3963532.xml\n",
            "PMC003xxxxxx/PMC3963533.xml\n",
            "PMC003xxxxxx/PMC3963534.xml\n",
            "PMC003xxxxxx/PMC3963535.xml\n",
            "PMC003xxxxxx/PMC3963536.xml\n",
            "PMC003xxxxxx/PMC3963537.xml\n",
            "PMC003xxxxxx/PMC3963538.xml\n",
            "PMC003xxxxxx/PMC3963539.xml\n",
            "PMC003xxxxxx/PMC3963540.xml\n",
            "PMC003xxxxxx/PMC3963541.xml\n",
            "PMC003xxxxxx/PMC3963542.xml\n",
            "PMC003xxxxxx/PMC3963543.xml\n",
            "PMC003xxxxxx/PMC3963544.xml\n",
            "PMC003xxxxxx/PMC3963545.xml\n",
            "PMC003xxxxxx/PMC3963546.xml\n",
            "PMC003xxxxxx/PMC3963547.xml\n",
            "PMC003xxxxxx/PMC3963548.xml\n",
            "PMC003xxxxxx/PMC3963549.xml\n",
            "PMC003xxxxxx/PMC3963550.xml\n",
            "PMC003xxxxxx/PMC3963552.xml\n",
            "PMC003xxxxxx/PMC3963553.xml\n",
            "PMC003xxxxxx/PMC3963555.xml\n",
            "PMC003xxxxxx/PMC3963556.xml\n",
            "PMC003xxxxxx/PMC3963557.xml\n",
            "PMC003xxxxxx/PMC3963558.xml\n",
            "PMC003xxxxxx/PMC3963559.xml\n",
            "PMC003xxxxxx/PMC3963598.xml\n",
            "PMC003xxxxxx/PMC3963599.xml\n",
            "PMC003xxxxxx/PMC3963600.xml\n",
            "PMC003xxxxxx/PMC3963818.xml\n",
            "PMC003xxxxxx/PMC3964020.xml\n",
            "PMC003xxxxxx/PMC3964021.xml\n",
            "PMC003xxxxxx/PMC3964022.xml\n",
            "PMC003xxxxxx/PMC3964023.xml\n",
            "PMC003xxxxxx/PMC3964024.xml\n",
            "PMC003xxxxxx/PMC3964025.xml\n",
            "PMC003xxxxxx/PMC3964026.xml\n",
            "PMC003xxxxxx/PMC3964027.xml\n",
            "PMC003xxxxxx/PMC3964028.xml\n",
            "PMC003xxxxxx/PMC3964029.xml\n",
            "PMC003xxxxxx/PMC3964030.xml\n",
            "PMC003xxxxxx/PMC3964031.xml\n",
            "PMC003xxxxxx/PMC3964032.xml\n",
            "PMC003xxxxxx/PMC3964033.xml\n",
            "PMC003xxxxxx/PMC3964034.xml\n",
            "PMC003xxxxxx/PMC3964154.xml\n",
            "PMC003xxxxxx/PMC3964155.xml\n",
            "PMC003xxxxxx/PMC3964156.xml\n",
            "PMC003xxxxxx/PMC3964157.xml\n",
            "PMC003xxxxxx/PMC3964158.xml\n",
            "PMC003xxxxxx/PMC3964159.xml\n",
            "PMC003xxxxxx/PMC3964160.xml\n",
            "PMC003xxxxxx/PMC3964161.xml\n",
            "PMC003xxxxxx/PMC3964162.xml\n",
            "PMC003xxxxxx/PMC3964163.xml\n",
            "PMC003xxxxxx/PMC3964164.xml\n",
            "PMC003xxxxxx/PMC3964165.xml\n",
            "PMC003xxxxxx/PMC3964166.xml\n",
            "PMC003xxxxxx/PMC3964167.xml\n",
            "PMC003xxxxxx/PMC3964168.xml\n",
            "PMC003xxxxxx/PMC3964169.xml\n",
            "PMC003xxxxxx/PMC3964170.xml\n",
            "PMC003xxxxxx/PMC3964171.xml\n",
            "PMC003xxxxxx/PMC3964172.xml\n",
            "PMC003xxxxxx/PMC3964173.xml\n",
            "PMC003xxxxxx/PMC3964174.xml\n",
            "PMC003xxxxxx/PMC3964175.xml\n",
            "PMC003xxxxxx/PMC3964201.xml\n",
            "PMC003xxxxxx/PMC3964261.xml\n",
            "PMC003xxxxxx/PMC3964262.xml\n",
            "PMC003xxxxxx/PMC3964263.xml\n",
            "PMC003xxxxxx/PMC3964264.xml\n",
            "PMC003xxxxxx/PMC3964265.xml\n",
            "PMC003xxxxxx/PMC3964266.xml\n",
            "PMC003xxxxxx/PMC3964267.xml\n",
            "PMC003xxxxxx/PMC3964268.xml\n",
            "PMC003xxxxxx/PMC3964269.xml\n",
            "PMC003xxxxxx/PMC3964270.xml\n",
            "PMC003xxxxxx/PMC3964271.xml\n",
            "PMC003xxxxxx/PMC3964272.xml\n",
            "PMC003xxxxxx/PMC3964273.xml\n",
            "PMC003xxxxxx/PMC3964274.xml\n",
            "PMC003xxxxxx/PMC3964275.xml\n",
            "PMC003xxxxxx/PMC3964276.xml\n",
            "PMC003xxxxxx/PMC3964277.xml\n",
            "PMC003xxxxxx/PMC3964278.xml\n",
            "PMC003xxxxxx/PMC3964279.xml\n",
            "PMC003xxxxxx/PMC3964280.xml\n",
            "PMC003xxxxxx/PMC3964281.xml\n",
            "PMC003xxxxxx/PMC3964282.xml\n",
            "PMC003xxxxxx/PMC3964283.xml\n",
            "PMC003xxxxxx/PMC3964284.xml\n",
            "PMC003xxxxxx/PMC3964285.xml\n",
            "PMC003xxxxxx/PMC3964286.xml\n",
            "PMC003xxxxxx/PMC3964287.xml\n",
            "PMC003xxxxxx/PMC3964288.xml\n",
            "PMC003xxxxxx/PMC3964289.xml\n",
            "PMC003xxxxxx/PMC3964290.xml\n",
            "PMC003xxxxxx/PMC3964291.xml\n",
            "PMC003xxxxxx/PMC3964292.xml\n",
            "PMC003xxxxxx/PMC3964293.xml\n",
            "PMC003xxxxxx/PMC3964295.xml\n",
            "PMC003xxxxxx/PMC3964296.xml\n",
            "PMC003xxxxxx/PMC3964297.xml\n",
            "PMC003xxxxxx/PMC3964298.xml\n",
            "PMC003xxxxxx/PMC3964312.xml\n",
            "PMC003xxxxxx/PMC3964313.xml\n",
            "PMC003xxxxxx/PMC3964314.xml\n",
            "PMC003xxxxxx/PMC3964315.xml\n",
            "PMC003xxxxxx/PMC3964316.xml\n",
            "PMC003xxxxxx/PMC3964344.xml\n",
            "PMC003xxxxxx/PMC3964441.xml\n",
            "PMC003xxxxxx/PMC3964442.xml\n",
            "PMC003xxxxxx/PMC3964443.xml\n",
            "PMC003xxxxxx/PMC3964444.xml\n",
            "PMC003xxxxxx/PMC3964445.xml\n",
            "PMC003xxxxxx/PMC3964446.xml\n",
            "PMC003xxxxxx/PMC3964484.xml\n",
            "PMC003xxxxxx/PMC3964485.xml\n",
            "PMC003xxxxxx/PMC3964486.xml\n",
            "PMC003xxxxxx/PMC3964487.xml\n",
            "PMC003xxxxxx/PMC3964488.xml\n",
            "PMC003xxxxxx/PMC3964489.xml\n",
            "PMC003xxxxxx/PMC3964490.xml\n",
            "PMC003xxxxxx/PMC3964491.xml\n",
            "PMC003xxxxxx/PMC3964492.xml\n",
            "PMC003xxxxxx/PMC3964493.xml\n",
            "PMC003xxxxxx/PMC3964494.xml\n",
            "PMC003xxxxxx/PMC3964495.xml\n",
            "PMC003xxxxxx/PMC3964496.xml\n",
            "PMC003xxxxxx/PMC3964497.xml\n",
            "PMC003xxxxxx/PMC3964498.xml\n",
            "PMC003xxxxxx/PMC3964499.xml\n",
            "PMC003xxxxxx/PMC3964500.xml\n",
            "PMC003xxxxxx/PMC3964501.xml\n",
            "PMC003xxxxxx/PMC3964502.xml\n",
            "PMC003xxxxxx/PMC3964503.xml\n",
            "PMC003xxxxxx/PMC3964504.xml\n",
            "PMC003xxxxxx/PMC3964505.xml\n",
            "PMC003xxxxxx/PMC3964506.xml\n",
            "PMC003xxxxxx/PMC3964507.xml\n",
            "PMC003xxxxxx/PMC3964508.xml\n",
            "PMC003xxxxxx/PMC3964509.xml\n",
            "PMC003xxxxxx/PMC3964510.xml\n",
            "PMC003xxxxxx/PMC3964511.xml\n",
            "PMC003xxxxxx/PMC3964512.xml\n",
            "PMC003xxxxxx/PMC3964513.xml\n",
            "PMC003xxxxxx/PMC3964514.xml\n",
            "PMC003xxxxxx/PMC3964515.xml\n",
            "PMC003xxxxxx/PMC3964516.xml\n",
            "PMC003xxxxxx/PMC3964518.xml\n",
            "PMC003xxxxxx/PMC3964519.xml\n",
            "PMC003xxxxxx/PMC3964520.xml\n",
            "PMC003xxxxxx/PMC3964521.xml\n",
            "PMC003xxxxxx/PMC3964559.xml\n",
            "PMC003xxxxxx/PMC3964593.xml\n",
            "PMC003xxxxxx/PMC3964742.xml\n",
            "PMC003xxxxxx/PMC3964743.xml\n",
            "PMC003xxxxxx/PMC3964744.xml\n",
            "PMC003xxxxxx/PMC3964745.xml\n",
            "PMC003xxxxxx/PMC3964746.xml\n",
            "PMC003xxxxxx/PMC3964747.xml\n",
            "PMC003xxxxxx/PMC3964748.xml\n",
            "PMC003xxxxxx/PMC3964749.xml\n",
            "PMC003xxxxxx/PMC3964750.xml\n",
            "PMC003xxxxxx/PMC3964852.xml\n",
            "PMC003xxxxxx/PMC3964853.xml\n",
            "PMC003xxxxxx/PMC3964856.xml\n",
            "PMC003xxxxxx/PMC3964905.xml\n",
            "PMC003xxxxxx/PMC3964906.xml\n",
            "PMC003xxxxxx/PMC3964907.xml\n",
            "PMC003xxxxxx/PMC3964908.xml\n",
            "PMC003xxxxxx/PMC3964909.xml\n",
            "PMC003xxxxxx/PMC3964910.xml\n",
            "PMC003xxxxxx/PMC3964911.xml\n",
            "PMC003xxxxxx/PMC3964912.xml\n",
            "PMC003xxxxxx/PMC3964913.xml\n",
            "PMC003xxxxxx/PMC3964914.xml\n",
            "PMC003xxxxxx/PMC3964915.xml\n",
            "PMC003xxxxxx/PMC3964916.xml\n",
            "PMC003xxxxxx/PMC3964917.xml\n",
            "PMC003xxxxxx/PMC3964918.xml\n",
            "PMC003xxxxxx/PMC3964919.xml\n",
            "PMC003xxxxxx/PMC3964938.xml\n",
            "PMC003xxxxxx/PMC3964940.xml\n",
            "PMC003xxxxxx/PMC3964942.xml\n",
            "PMC003xxxxxx/PMC3964945.xml\n",
            "PMC003xxxxxx/PMC3964949.xml\n",
            "PMC003xxxxxx/PMC3964952.xml\n",
            "PMC003xxxxxx/PMC3964953.xml\n",
            "PMC003xxxxxx/PMC3964954.xml\n",
            "PMC003xxxxxx/PMC3964956.xml\n",
            "PMC003xxxxxx/PMC3964958.xml\n",
            "PMC003xxxxxx/PMC3964959.xml\n",
            "PMC003xxxxxx/PMC3964961.xml\n",
            "PMC003xxxxxx/PMC3964970.xml\n",
            "PMC003xxxxxx/PMC3964971.xml\n",
            "PMC003xxxxxx/PMC3964974.xml\n",
            "PMC003xxxxxx/PMC3964976.xml\n",
            "PMC003xxxxxx/PMC3964978.xml\n",
            "PMC003xxxxxx/PMC3964980.xml\n",
            "PMC003xxxxxx/PMC3964981.xml\n",
            "PMC003xxxxxx/PMC3964982.xml\n",
            "PMC003xxxxxx/PMC3964988.xml\n",
            "PMC003xxxxxx/PMC3964990.xml\n",
            "PMC003xxxxxx/PMC3964997.xml\n",
            "PMC003xxxxxx/PMC3964998.xml\n",
            "PMC003xxxxxx/PMC3964999.xml\n",
            "PMC003xxxxxx/PMC3965000.xml\n",
            "PMC003xxxxxx/PMC3965020.xml\n",
            "PMC003xxxxxx/PMC3965023.xml\n",
            "PMC003xxxxxx/PMC3965028.xml\n",
            "PMC003xxxxxx/PMC3965029.xml\n",
            "PMC003xxxxxx/PMC3965034.xml\n",
            "PMC003xxxxxx/PMC3965040.xml\n",
            "PMC003xxxxxx/PMC3965041.xml\n",
            "PMC003xxxxxx/PMC3965042.xml\n",
            "PMC003xxxxxx/PMC3965045.xml\n",
            "PMC003xxxxxx/PMC3965046.xml\n",
            "PMC003xxxxxx/PMC3965048.xml\n",
            "PMC003xxxxxx/PMC3965054.xml\n",
            "PMC003xxxxxx/PMC3965056.xml\n",
            "PMC003xxxxxx/PMC3965058.xml\n",
            "PMC003xxxxxx/PMC3965064.xml\n",
            "PMC003xxxxxx/PMC3965074.xml\n",
            "PMC003xxxxxx/PMC3965076.xml\n",
            "PMC003xxxxxx/PMC3965081.xml\n",
            "PMC003xxxxxx/PMC3965084.xml\n",
            "PMC003xxxxxx/PMC3965086.xml\n",
            "PMC003xxxxxx/PMC3965092.xml\n",
            "PMC003xxxxxx/PMC3965105.xml\n",
            "PMC003xxxxxx/PMC3965106.xml\n",
            "PMC003xxxxxx/PMC3965107.xml\n",
            "PMC003xxxxxx/PMC3965111.xml\n",
            "PMC003xxxxxx/PMC3965112.xml\n",
            "PMC003xxxxxx/PMC3965114.xml\n",
            "PMC003xxxxxx/PMC3965115.xml\n",
            "PMC003xxxxxx/PMC3965119.xml\n",
            "PMC003xxxxxx/PMC3965120.xml\n",
            "PMC003xxxxxx/PMC3965122.xml\n",
            "PMC003xxxxxx/PMC3965124.xml\n",
            "PMC003xxxxxx/PMC3965271.xml\n",
            "PMC003xxxxxx/PMC3965319.xml\n",
            "PMC003xxxxxx/PMC3965357.xml\n",
            "PMC003xxxxxx/PMC3965358.xml\n",
            "PMC003xxxxxx/PMC3965359.xml\n",
            "PMC003xxxxxx/PMC3965362.xml\n",
            "PMC003xxxxxx/PMC3965364.xml\n",
            "PMC003xxxxxx/PMC3965567.xml\n",
            "PMC003xxxxxx/PMC3965692.xml\n",
            "PMC003xxxxxx/PMC3965693.xml\n",
            "PMC003xxxxxx/PMC3965694.xml\n",
            "PMC003xxxxxx/PMC3965695.xml\n",
            "PMC003xxxxxx/PMC3965696.xml\n",
            "PMC003xxxxxx/PMC3965697.xml\n",
            "PMC003xxxxxx/PMC3965698.xml\n",
            "PMC003xxxxxx/PMC3965699.xml\n",
            "PMC003xxxxxx/PMC3965700.xml\n",
            "PMC003xxxxxx/PMC3965701.xml\n",
            "PMC003xxxxxx/PMC3965702.xml\n",
            "PMC003xxxxxx/PMC3965703.xml\n",
            "PMC003xxxxxx/PMC3965704.xml\n",
            "PMC003xxxxxx/PMC3965705.xml\n",
            "PMC003xxxxxx/PMC3965706.xml\n",
            "PMC003xxxxxx/PMC3965707.xml\n",
            "PMC003xxxxxx/PMC3965712.xml\n",
            "PMC003xxxxxx/PMC3965713.xml\n",
            "PMC003xxxxxx/PMC3965714.xml\n",
            "PMC003xxxxxx/PMC3965751.xml\n",
            "PMC003xxxxxx/PMC3965794.xml\n",
            "PMC003xxxxxx/PMC3965795.xml\n",
            "PMC003xxxxxx/PMC3965796.xml\n",
            "PMC003xxxxxx/PMC3965797.xml\n",
            "PMC003xxxxxx/PMC3965798.xml\n",
            "PMC003xxxxxx/PMC3965799.xml\n",
            "PMC003xxxxxx/PMC3965800.xml\n",
            "PMC003xxxxxx/PMC3965801.xml\n",
            "PMC003xxxxxx/PMC3965802.xml\n",
            "PMC003xxxxxx/PMC3965835.xml\n",
            "PMC003xxxxxx/PMC3965837.xml\n",
            "PMC003xxxxxx/PMC3965839.xml\n",
            "PMC003xxxxxx/PMC3965840.xml\n",
            "PMC003xxxxxx/PMC3965848.xml\n",
            "PMC003xxxxxx/PMC3966031.xml\n",
            "PMC003xxxxxx/PMC3966032.xml\n",
            "PMC003xxxxxx/PMC3966033.xml\n",
            "PMC003xxxxxx/PMC3966034.xml\n",
            "PMC003xxxxxx/PMC3966036.xml\n",
            "PMC003xxxxxx/PMC3966037.xml\n",
            "PMC003xxxxxx/PMC3966038.xml\n",
            "PMC003xxxxxx/PMC3966040.xml\n",
            "PMC003xxxxxx/PMC3966041.xml\n",
            "PMC003xxxxxx/PMC3966043.xml\n",
            "PMC003xxxxxx/PMC3966044.xml\n",
            "PMC003xxxxxx/PMC3966045.xml\n",
            "PMC003xxxxxx/PMC3966046.xml\n",
            "PMC003xxxxxx/PMC3966048.xml\n",
            "PMC003xxxxxx/PMC3966050.xml\n",
            "PMC003xxxxxx/PMC3966051.xml\n",
            "PMC003xxxxxx/PMC3966052.xml\n",
            "PMC003xxxxxx/PMC3966053.xml\n",
            "PMC003xxxxxx/PMC3966054.xml\n",
            "PMC003xxxxxx/PMC3966055.xml\n",
            "PMC003xxxxxx/PMC3966056.xml\n",
            "PMC003xxxxxx/PMC3966057.xml\n",
            "PMC003xxxxxx/PMC3966080.xml\n",
            "PMC003xxxxxx/PMC3966090.xml\n",
            "PMC003xxxxxx/PMC3966091.xml\n",
            "PMC003xxxxxx/PMC3966092.xml\n",
            "PMC003xxxxxx/PMC3966093.xml\n",
            "PMC003xxxxxx/PMC3966094.xml\n",
            "PMC003xxxxxx/PMC3966095.xml\n",
            "PMC003xxxxxx/PMC3966096.xml\n",
            "PMC003xxxxxx/PMC3966097.xml\n",
            "PMC003xxxxxx/PMC3966098.xml\n",
            "PMC003xxxxxx/PMC3966099.xml\n",
            "PMC003xxxxxx/PMC3966100.xml\n",
            "PMC003xxxxxx/PMC3966116.xml\n",
            "PMC003xxxxxx/PMC3966117.xml\n",
            "PMC003xxxxxx/PMC3966143.xml\n",
            "PMC003xxxxxx/PMC3966144.xml\n",
            "PMC003xxxxxx/PMC3966145.xml\n",
            "PMC003xxxxxx/PMC3966146.xml\n",
            "PMC003xxxxxx/PMC3966147.xml\n",
            "PMC003xxxxxx/PMC3966215.xml\n",
            "PMC003xxxxxx/PMC3966261.xml\n",
            "PMC003xxxxxx/PMC3966283.xml\n",
            "PMC003xxxxxx/PMC3966285.xml\n",
            "PMC003xxxxxx/PMC3966435.xml\n",
            "PMC003xxxxxx/PMC3966436.xml\n",
            "PMC003xxxxxx/PMC3966437.xml\n",
            "PMC003xxxxxx/PMC3966438.xml\n",
            "PMC003xxxxxx/PMC3966439.xml\n",
            "PMC003xxxxxx/PMC3966440.xml\n",
            "PMC003xxxxxx/PMC3966441.xml\n",
            "PMC003xxxxxx/PMC3966442.xml\n",
            "PMC003xxxxxx/PMC3966443.xml\n",
            "PMC003xxxxxx/PMC3966444.xml\n",
            "PMC003xxxxxx/PMC3966445.xml\n",
            "PMC003xxxxxx/PMC3966446.xml\n",
            "PMC003xxxxxx/PMC3966447.xml\n",
            "PMC003xxxxxx/PMC3966448.xml\n",
            "PMC003xxxxxx/PMC3966449.xml\n",
            "PMC003xxxxxx/PMC3966450.xml\n",
            "PMC003xxxxxx/PMC3966451.xml\n",
            "PMC003xxxxxx/PMC3966452.xml\n",
            "PMC003xxxxxx/PMC3966453.xml\n",
            "PMC003xxxxxx/PMC3966454.xml\n",
            "PMC003xxxxxx/PMC3966455.xml\n",
            "PMC003xxxxxx/PMC3966456.xml\n",
            "PMC003xxxxxx/PMC3966457.xml\n",
            "PMC003xxxxxx/PMC3966458.xml\n",
            "PMC003xxxxxx/PMC3966459.xml\n",
            "PMC003xxxxxx/PMC3966460.xml\n",
            "PMC003xxxxxx/PMC3966461.xml\n",
            "PMC003xxxxxx/PMC3966462.xml\n",
            "PMC003xxxxxx/PMC3966463.xml\n",
            "PMC003xxxxxx/PMC3966464.xml\n",
            "PMC003xxxxxx/PMC3966510.xml\n",
            "PMC003xxxxxx/PMC3966515.xml\n",
            "PMC003xxxxxx/PMC3966536.xml\n",
            "PMC003xxxxxx/PMC3966537.xml\n",
            "PMC003xxxxxx/PMC3966538.xml\n",
            "PMC003xxxxxx/PMC3966539.xml\n",
            "PMC003xxxxxx/PMC3966540.xml\n",
            "PMC003xxxxxx/PMC3966541.xml\n",
            "PMC003xxxxxx/PMC3966542.xml\n",
            "PMC003xxxxxx/PMC3966543.xml\n",
            "PMC003xxxxxx/PMC3966627.xml\n",
            "PMC003xxxxxx/PMC3966913.xml\n",
            "PMC003xxxxxx/PMC3966914.xml\n",
            "PMC003xxxxxx/PMC3966915.xml\n",
            "PMC003xxxxxx/PMC3966916.xml\n",
            "PMC003xxxxxx/PMC3966917.xml\n",
            "PMC003xxxxxx/PMC3966921.xml\n",
            "PMC003xxxxxx/PMC3966922.xml\n",
            "PMC003xxxxxx/PMC3966924.xml\n",
            "PMC003xxxxxx/PMC3967014.xml\n",
            "PMC003xxxxxx/PMC3967015.xml\n",
            "PMC003xxxxxx/PMC3967016.xml\n",
            "PMC003xxxxxx/PMC3967044.xml\n",
            "PMC003xxxxxx/PMC3967046.xml\n",
            "PMC003xxxxxx/PMC3967047.xml\n",
            "PMC003xxxxxx/PMC3967048.xml\n",
            "PMC003xxxxxx/PMC3967049.xml\n",
            "PMC003xxxxxx/PMC3967050.xml\n",
            "PMC003xxxxxx/PMC3967051.xml\n",
            "PMC003xxxxxx/PMC3967052.xml\n",
            "PMC003xxxxxx/PMC3967056.xml\n",
            "PMC003xxxxxx/PMC3967066.xml\n",
            "PMC003xxxxxx/PMC3967068.xml\n",
            "PMC003xxxxxx/PMC3967069.xml\n",
            "PMC003xxxxxx/PMC3967070.xml\n",
            "PMC003xxxxxx/PMC3967072.xml\n",
            "PMC003xxxxxx/PMC3967075.xml\n",
            "PMC003xxxxxx/PMC3967079.xml\n",
            "PMC003xxxxxx/PMC3967080.xml\n",
            "PMC003xxxxxx/PMC3967081.xml\n",
            "PMC003xxxxxx/PMC3967107.xml\n",
            "PMC003xxxxxx/PMC3967147.xml\n",
            "PMC003xxxxxx/PMC3967148.xml\n",
            "PMC003xxxxxx/PMC3967149.xml\n",
            "PMC003xxxxxx/PMC3967150.xml\n",
            "PMC003xxxxxx/PMC3967151.xml\n",
            "PMC003xxxxxx/PMC3967152.xml\n",
            "PMC003xxxxxx/PMC3967197.xml\n",
            "PMC003xxxxxx/PMC3967199.xml\n",
            "PMC003xxxxxx/PMC3967201.xml\n",
            "PMC003xxxxxx/PMC3967202.xml\n",
            "PMC003xxxxxx/PMC3967249.xml\n",
            "PMC003xxxxxx/PMC3967250.xml\n",
            "PMC003xxxxxx/PMC3967251.xml\n",
            "PMC003xxxxxx/PMC3967252.xml\n",
            "PMC003xxxxxx/PMC3967253.xml\n",
            "PMC003xxxxxx/PMC3967254.xml\n",
            "PMC003xxxxxx/PMC3967255.xml\n",
            "PMC003xxxxxx/PMC3967256.xml\n",
            "PMC003xxxxxx/PMC3967257.xml\n",
            "PMC003xxxxxx/PMC3967258.xml\n",
            "PMC003xxxxxx/PMC3967259.xml\n",
            "PMC003xxxxxx/PMC3967260.xml\n",
            "PMC003xxxxxx/PMC3967261.xml\n",
            "PMC003xxxxxx/PMC3967262.xml\n",
            "PMC003xxxxxx/PMC3967263.xml\n",
            "PMC003xxxxxx/PMC3967264.xml\n",
            "PMC003xxxxxx/PMC3967265.xml\n",
            "PMC003xxxxxx/PMC3967266.xml\n",
            "PMC003xxxxxx/PMC3967267.xml\n",
            "PMC003xxxxxx/PMC3967268.xml\n",
            "PMC003xxxxxx/PMC3967286.xml\n",
            "PMC003xxxxxx/PMC3967302.xml\n",
            "PMC003xxxxxx/PMC3967303.xml\n",
            "PMC003xxxxxx/PMC3967304.xml\n",
            "PMC003xxxxxx/PMC3967305.xml\n",
            "PMC003xxxxxx/PMC3967306.xml\n",
            "PMC003xxxxxx/PMC3967307.xml\n",
            "PMC003xxxxxx/PMC3967308.xml\n",
            "PMC003xxxxxx/PMC3967309.xml\n",
            "PMC003xxxxxx/PMC3967310.xml\n",
            "PMC003xxxxxx/PMC3967311.xml\n",
            "PMC003xxxxxx/PMC3967312.xml\n",
            "PMC003xxxxxx/PMC3967313.xml\n",
            "PMC003xxxxxx/PMC3967314.xml\n",
            "PMC003xxxxxx/PMC3967315.xml\n",
            "PMC003xxxxxx/PMC3967316.xml\n",
            "PMC003xxxxxx/PMC3967317.xml\n",
            "PMC003xxxxxx/PMC3967318.xml\n",
            "PMC003xxxxxx/PMC3967319.xml\n",
            "PMC003xxxxxx/PMC3967320.xml\n",
            "PMC003xxxxxx/PMC3967321.xml\n",
            "PMC003xxxxxx/PMC3967326.xml\n",
            "PMC003xxxxxx/PMC3967327.xml\n",
            "PMC003xxxxxx/PMC3967328.xml\n",
            "PMC003xxxxxx/PMC3967329.xml\n",
            "PMC003xxxxxx/PMC3967330.xml\n",
            "PMC003xxxxxx/PMC3967331.xml\n",
            "PMC003xxxxxx/PMC3967332.xml\n",
            "PMC003xxxxxx/PMC3967333.xml\n",
            "PMC003xxxxxx/PMC3967334.xml\n",
            "PMC003xxxxxx/PMC3967335.xml\n",
            "PMC003xxxxxx/PMC3967412.xml\n",
            "PMC003xxxxxx/PMC3967422.xml\n",
            "PMC003xxxxxx/PMC3967423.xml\n",
            "PMC003xxxxxx/PMC3967424.xml\n",
            "PMC003xxxxxx/PMC3967425.xml\n",
            "PMC003xxxxxx/PMC3967426.xml\n",
            "PMC003xxxxxx/PMC3967427.xml\n",
            "PMC003xxxxxx/PMC3967428.xml\n",
            "PMC003xxxxxx/PMC3967429.xml\n",
            "PMC003xxxxxx/PMC3967450.xml\n",
            "PMC003xxxxxx/PMC3967451.xml\n",
            "PMC003xxxxxx/PMC3967452.xml\n",
            "PMC003xxxxxx/PMC3967453.xml\n",
            "PMC003xxxxxx/PMC3967454.xml\n",
            "PMC003xxxxxx/PMC3967455.xml\n",
            "PMC003xxxxxx/PMC3967456.xml\n",
            "PMC003xxxxxx/PMC3967457.xml\n",
            "PMC003xxxxxx/PMC3967516.xml\n",
            "PMC003xxxxxx/PMC3967517.xml\n",
            "PMC003xxxxxx/PMC3967521.xml\n",
            "PMC003xxxxxx/PMC3967522.xml\n",
            "PMC003xxxxxx/PMC3967523.xml\n",
            "PMC003xxxxxx/PMC3967524.xml\n",
            "PMC003xxxxxx/PMC3967525.xml\n",
            "PMC003xxxxxx/PMC3967555.xml\n",
            "PMC003xxxxxx/PMC3967557.xml\n",
            "PMC003xxxxxx/PMC3967563.xml\n",
            "PMC003xxxxxx/PMC3967566.xml\n",
            "PMC003xxxxxx/PMC3967631.xml\n",
            "PMC003xxxxxx/PMC3967831.xml\n",
            "PMC003xxxxxx/PMC3967833.xml\n",
            "PMC003xxxxxx/PMC3967967.xml\n",
            "PMC003xxxxxx/PMC3967968.xml\n",
            "PMC003xxxxxx/PMC3967969.xml\n",
            "PMC003xxxxxx/PMC3967970.xml\n",
            "PMC003xxxxxx/PMC3967971.xml\n",
            "PMC003xxxxxx/PMC3967972.xml\n",
            "PMC003xxxxxx/PMC3967973.xml\n",
            "PMC003xxxxxx/PMC3967974.xml\n",
            "PMC003xxxxxx/PMC3967975.xml\n",
            "PMC003xxxxxx/PMC3967976.xml\n",
            "PMC003xxxxxx/PMC3967977.xml\n",
            "PMC003xxxxxx/PMC3967978.xml\n",
            "PMC003xxxxxx/PMC3967979.xml\n",
            "PMC003xxxxxx/PMC3967980.xml\n",
            "PMC003xxxxxx/PMC3967981.xml\n",
            "PMC003xxxxxx/PMC3967982.xml\n",
            "PMC003xxxxxx/PMC3968078.xml\n",
            "PMC003xxxxxx/PMC3968079.xml\n",
            "PMC003xxxxxx/PMC3968080.xml\n",
            "PMC003xxxxxx/PMC3968081.xml\n",
            "PMC003xxxxxx/PMC3968082.xml\n",
            "PMC003xxxxxx/PMC3968083.xml\n",
            "PMC003xxxxxx/PMC3968084.xml\n",
            "PMC003xxxxxx/PMC3968085.xml\n",
            "PMC003xxxxxx/PMC3968086.xml\n",
            "PMC003xxxxxx/PMC3968087.xml\n",
            "PMC003xxxxxx/PMC3968088.xml\n",
            "PMC003xxxxxx/PMC3968089.xml\n",
            "PMC003xxxxxx/PMC3968251.xml\n",
            "PMC003xxxxxx/PMC3968252.xml\n",
            "PMC003xxxxxx/PMC3968253.xml\n",
            "PMC003xxxxxx/PMC3968254.xml\n",
            "PMC003xxxxxx/PMC3968255.xml\n",
            "PMC003xxxxxx/PMC3968256.xml\n",
            "PMC003xxxxxx/PMC3968257.xml\n",
            "PMC003xxxxxx/PMC3968258.xml\n",
            "PMC003xxxxxx/PMC3968261.xml\n",
            "PMC003xxxxxx/PMC3968262.xml\n",
            "PMC003xxxxxx/PMC3968263.xml\n",
            "PMC003xxxxxx/PMC3968264.xml\n",
            "PMC003xxxxxx/PMC3968265.xml\n",
            "PMC003xxxxxx/PMC3968266.xml\n",
            "PMC003xxxxxx/PMC3968267.xml\n",
            "PMC003xxxxxx/PMC3968268.xml\n",
            "PMC003xxxxxx/PMC3968269.xml\n",
            "PMC003xxxxxx/PMC3968270.xml\n",
            "PMC003xxxxxx/PMC3968271.xml\n",
            "PMC003xxxxxx/PMC3968275.xml\n",
            "PMC003xxxxxx/PMC3968297.xml\n",
            "PMC003xxxxxx/PMC3968402.xml\n",
            "PMC003xxxxxx/PMC3968431.xml\n",
            "PMC003xxxxxx/PMC3968432.xml\n",
            "PMC003xxxxxx/PMC3968433.xml\n",
            "PMC003xxxxxx/PMC3968434.xml\n",
            "PMC003xxxxxx/PMC3968435.xml\n",
            "PMC003xxxxxx/PMC3968436.xml\n",
            "PMC003xxxxxx/PMC3968437.xml\n",
            "PMC003xxxxxx/PMC3968438.xml\n",
            "PMC003xxxxxx/PMC3968439.xml\n",
            "PMC003xxxxxx/PMC3968447.xml\n",
            "PMC003xxxxxx/PMC3968448.xml\n",
            "PMC003xxxxxx/PMC3968449.xml\n",
            "PMC003xxxxxx/PMC3968450.xml\n",
            "PMC003xxxxxx/PMC3968451.xml\n",
            "PMC003xxxxxx/PMC3968452.xml\n",
            "PMC003xxxxxx/PMC3968454.xml\n",
            "PMC003xxxxxx/PMC3968455.xml\n",
            "PMC003xxxxxx/PMC3968456.xml\n",
            "PMC003xxxxxx/PMC3968457.xml\n",
            "PMC003xxxxxx/PMC3968458.xml\n",
            "PMC003xxxxxx/PMC3968459.xml\n",
            "PMC003xxxxxx/PMC3968460.xml\n",
            "PMC003xxxxxx/PMC3968461.xml\n",
            "PMC003xxxxxx/PMC3968484.xml\n",
            "PMC003xxxxxx/PMC3968485.xml\n",
            "PMC003xxxxxx/PMC3968486.xml\n",
            "PMC003xxxxxx/PMC3968487.xml\n",
            "PMC003xxxxxx/PMC3968488.xml\n",
            "PMC003xxxxxx/PMC3968515.xml\n",
            "PMC003xxxxxx/PMC3968519.xml\n",
            "PMC003xxxxxx/PMC3968539.xml\n",
            "PMC003xxxxxx/PMC3968548.xml\n",
            "PMC003xxxxxx/PMC3968549.xml\n",
            "PMC003xxxxxx/PMC3968565.xml\n",
            "PMC003xxxxxx/PMC3968566.xml\n",
            "PMC003xxxxxx/PMC3968567.xml\n",
            "PMC003xxxxxx/PMC3968568.xml\n",
            "PMC003xxxxxx/PMC3968569.xml\n",
            "PMC003xxxxxx/PMC3968570.xml\n",
            "PMC003xxxxxx/PMC3968571.xml\n",
            "PMC003xxxxxx/PMC3968572.xml\n",
            "PMC003xxxxxx/PMC3968573.xml\n",
            "PMC003xxxxxx/PMC3968574.xml\n",
            "PMC003xxxxxx/PMC3968575.xml\n",
            "PMC003xxxxxx/PMC3968576.xml\n",
            "PMC003xxxxxx/PMC3968577.xml\n",
            "PMC003xxxxxx/PMC3968578.xml\n",
            "PMC003xxxxxx/PMC3968579.xml\n",
            "PMC003xxxxxx/PMC3968580.xml\n",
            "PMC003xxxxxx/PMC3968581.xml\n",
            "PMC003xxxxxx/PMC3968582.xml\n",
            "PMC003xxxxxx/PMC3968583.xml\n",
            "PMC003xxxxxx/PMC3968584.xml\n",
            "PMC003xxxxxx/PMC3968585.xml\n",
            "PMC003xxxxxx/PMC3968586.xml\n",
            "PMC003xxxxxx/PMC3968587.xml\n",
            "PMC003xxxxxx/PMC3968597.xml\n",
            "PMC003xxxxxx/PMC3968598.xml\n",
            "PMC003xxxxxx/PMC3968599.xml\n",
            "PMC003xxxxxx/PMC3968600.xml\n",
            "PMC003xxxxxx/PMC3968601.xml\n",
            "PMC003xxxxxx/PMC3968602.xml\n",
            "PMC003xxxxxx/PMC3968603.xml\n",
            "PMC003xxxxxx/PMC3968604.xml\n",
            "PMC003xxxxxx/PMC3968605.xml\n",
            "PMC003xxxxxx/PMC3968606.xml\n",
            "PMC003xxxxxx/PMC3968607.xml\n",
            "PMC003xxxxxx/PMC3968608.xml\n",
            "PMC003xxxxxx/PMC3968609.xml\n",
            "PMC003xxxxxx/PMC3968610.xml\n",
            "PMC003xxxxxx/PMC3968611.xml\n",
            "PMC003xxxxxx/PMC3968612.xml\n",
            "PMC003xxxxxx/PMC3968613.xml\n",
            "PMC003xxxxxx/PMC3968614.xml\n",
            "PMC003xxxxxx/PMC3968615.xml\n",
            "PMC003xxxxxx/PMC3968616.xml\n",
            "PMC003xxxxxx/PMC3968617.xml\n",
            "PMC003xxxxxx/PMC3968618.xml\n",
            "PMC003xxxxxx/PMC3968619.xml\n",
            "PMC003xxxxxx/PMC3968620.xml\n",
            "PMC003xxxxxx/PMC3968621.xml\n",
            "PMC003xxxxxx/PMC3968622.xml\n",
            "PMC003xxxxxx/PMC3968623.xml\n",
            "PMC003xxxxxx/PMC3968624.xml\n",
            "PMC003xxxxxx/PMC3968625.xml\n",
            "PMC003xxxxxx/PMC3968626.xml\n",
            "PMC003xxxxxx/PMC3968627.xml\n",
            "PMC003xxxxxx/PMC3968628.xml\n",
            "PMC003xxxxxx/PMC3968629.xml\n",
            "PMC003xxxxxx/PMC3968630.xml\n",
            "PMC003xxxxxx/PMC3968631.xml\n",
            "PMC003xxxxxx/PMC3968632.xml\n",
            "PMC003xxxxxx/PMC3968633.xml\n",
            "PMC003xxxxxx/PMC3968634.xml\n",
            "PMC003xxxxxx/PMC3968635.xml\n",
            "PMC003xxxxxx/PMC3968636.xml\n",
            "PMC003xxxxxx/PMC3968637.xml\n",
            "PMC003xxxxxx/PMC3968638.xml\n",
            "PMC003xxxxxx/PMC3968639.xml\n",
            "PMC003xxxxxx/PMC3968640.xml\n",
            "PMC003xxxxxx/PMC3968641.xml\n",
            "PMC003xxxxxx/PMC3968642.xml\n",
            "PMC003xxxxxx/PMC3968643.xml\n",
            "PMC003xxxxxx/PMC3968644.xml\n",
            "PMC003xxxxxx/PMC3968645.xml\n",
            "PMC003xxxxxx/PMC3968646.xml\n",
            "PMC003xxxxxx/PMC3968647.xml\n",
            "PMC003xxxxxx/PMC3968648.xml\n",
            "PMC003xxxxxx/PMC3968649.xml\n",
            "PMC003xxxxxx/PMC3968650.xml\n",
            "PMC003xxxxxx/PMC3968651.xml\n",
            "PMC003xxxxxx/PMC3968652.xml\n",
            "PMC003xxxxxx/PMC3968653.xml\n",
            "PMC003xxxxxx/PMC3968654.xml\n",
            "PMC003xxxxxx/PMC3968655.xml\n",
            "PMC003xxxxxx/PMC3968656.xml\n",
            "PMC003xxxxxx/PMC3968657.xml\n",
            "PMC003xxxxxx/PMC3968658.xml\n",
            "PMC003xxxxxx/PMC3968659.xml\n",
            "PMC003xxxxxx/PMC3968660.xml\n",
            "PMC003xxxxxx/PMC3968661.xml\n",
            "PMC003xxxxxx/PMC3968662.xml\n",
            "PMC003xxxxxx/PMC3968663.xml\n",
            "PMC003xxxxxx/PMC3968664.xml\n",
            "PMC003xxxxxx/PMC3968665.xml\n",
            "PMC003xxxxxx/PMC3968666.xml\n",
            "PMC003xxxxxx/PMC3968667.xml\n",
            "PMC003xxxxxx/PMC3968668.xml\n",
            "PMC003xxxxxx/PMC3968669.xml\n",
            "PMC003xxxxxx/PMC3968670.xml\n",
            "PMC003xxxxxx/PMC3968671.xml\n",
            "PMC003xxxxxx/PMC3968672.xml\n",
            "PMC003xxxxxx/PMC3968673.xml\n",
            "PMC003xxxxxx/PMC3968674.xml\n",
            "PMC003xxxxxx/PMC3968675.xml\n",
            "PMC003xxxxxx/PMC3968676.xml\n",
            "PMC003xxxxxx/PMC3968677.xml\n",
            "PMC003xxxxxx/PMC3968692.xml\n",
            "PMC003xxxxxx/PMC3968693.xml\n",
            "PMC003xxxxxx/PMC3968694.xml\n",
            "PMC003xxxxxx/PMC3968695.xml\n",
            "PMC003xxxxxx/PMC3968696.xml\n",
            "PMC003xxxxxx/PMC3968697.xml\n",
            "PMC003xxxxxx/PMC3968698.xml\n",
            "PMC003xxxxxx/PMC3968699.xml\n",
            "PMC003xxxxxx/PMC3968700.xml\n",
            "PMC003xxxxxx/PMC3968701.xml\n",
            "PMC003xxxxxx/PMC3968702.xml\n",
            "PMC003xxxxxx/PMC3968703.xml\n",
            "PMC003xxxxxx/PMC3968704.xml\n",
            "PMC003xxxxxx/PMC3968705.xml\n",
            "PMC003xxxxxx/PMC3968706.xml\n",
            "PMC003xxxxxx/PMC3968707.xml\n",
            "PMC003xxxxxx/PMC3968708.xml\n",
            "PMC003xxxxxx/PMC3968709.xml\n",
            "PMC003xxxxxx/PMC3968710.xml\n",
            "PMC003xxxxxx/PMC3968711.xml\n",
            "PMC003xxxxxx/PMC3968712.xml\n",
            "PMC003xxxxxx/PMC3968713.xml\n",
            "PMC003xxxxxx/PMC3968714.xml\n",
            "PMC003xxxxxx/PMC3968715.xml\n",
            "PMC003xxxxxx/PMC3968716.xml\n",
            "PMC003xxxxxx/PMC3968717.xml\n",
            "PMC003xxxxxx/PMC3968718.xml\n",
            "PMC003xxxxxx/PMC3968719.xml\n",
            "PMC003xxxxxx/PMC3968720.xml\n",
            "PMC003xxxxxx/PMC3968721.xml\n",
            "PMC003xxxxxx/PMC3968722.xml\n",
            "PMC003xxxxxx/PMC3968723.xml\n",
            "PMC003xxxxxx/PMC3968724.xml\n",
            "PMC003xxxxxx/PMC3968725.xml\n",
            "PMC003xxxxxx/PMC3968726.xml\n",
            "PMC003xxxxxx/PMC3968727.xml\n",
            "PMC003xxxxxx/PMC3968728.xml\n",
            "PMC003xxxxxx/PMC3968729.xml\n",
            "PMC003xxxxxx/PMC3968730.xml\n",
            "PMC003xxxxxx/PMC3968731.xml\n",
            "PMC003xxxxxx/PMC3968732.xml\n",
            "PMC003xxxxxx/PMC3968733.xml\n",
            "PMC003xxxxxx/PMC3968734.xml\n",
            "PMC003xxxxxx/PMC3968735.xml\n",
            "PMC003xxxxxx/PMC3968736.xml\n",
            "PMC003xxxxxx/PMC3968737.xml\n",
            "PMC003xxxxxx/PMC3968738.xml\n",
            "PMC003xxxxxx/PMC3968739.xml\n",
            "PMC003xxxxxx/PMC3968740.xml\n",
            "PMC003xxxxxx/PMC3968741.xml\n",
            "PMC003xxxxxx/PMC3968742.xml\n",
            "PMC003xxxxxx/PMC3968864.xml\n",
            "PMC003xxxxxx/PMC3968881.xml\n",
            "PMC003xxxxxx/PMC3968882.xml\n",
            "PMC003xxxxxx/PMC3968890.xml\n",
            "PMC003xxxxxx/PMC3968911.xml\n",
            "PMC003xxxxxx/PMC3968912.xml\n",
            "PMC003xxxxxx/PMC3968920.xml\n",
            "PMC003xxxxxx/PMC3968958.xml\n",
            "PMC003xxxxxx/PMC3968991.xml\n",
            "PMC003xxxxxx/PMC3969038.xml\n",
            "PMC003xxxxxx/PMC3969039.xml\n",
            "PMC003xxxxxx/PMC3969040.xml\n",
            "PMC003xxxxxx/PMC3969041.xml\n",
            "PMC003xxxxxx/PMC3969042.xml\n",
            "PMC003xxxxxx/PMC3969043.xml\n",
            "PMC003xxxxxx/PMC3969044.xml\n",
            "PMC003xxxxxx/PMC3969045.xml\n",
            "PMC003xxxxxx/PMC3969046.xml\n",
            "PMC003xxxxxx/PMC3969047.xml\n",
            "PMC003xxxxxx/PMC3969048.xml\n",
            "PMC003xxxxxx/PMC3969049.xml\n",
            "PMC003xxxxxx/PMC3969148.xml\n",
            "PMC003xxxxxx/PMC3969149.xml\n",
            "PMC003xxxxxx/PMC3969152.xml\n",
            "PMC003xxxxxx/PMC3969153.xml\n",
            "PMC003xxxxxx/PMC3969154.xml\n",
            "PMC003xxxxxx/PMC3969155.xml\n",
            "PMC003xxxxxx/PMC3969156.xml\n",
            "PMC003xxxxxx/PMC3969226.xml\n",
            "PMC003xxxxxx/PMC3969227.xml\n",
            "PMC003xxxxxx/PMC3969228.xml\n",
            "PMC003xxxxxx/PMC3969251.xml\n",
            "PMC003xxxxxx/PMC3969252.xml\n",
            "PMC003xxxxxx/PMC3969253.xml\n",
            "PMC003xxxxxx/PMC3969254.xml\n",
            "PMC003xxxxxx/PMC3969255.xml\n",
            "PMC003xxxxxx/PMC3969256.xml\n",
            "PMC003xxxxxx/PMC3969257.xml\n",
            "PMC003xxxxxx/PMC3969258.xml\n",
            "PMC003xxxxxx/PMC3969260.xml\n",
            "PMC003xxxxxx/PMC3969267.xml\n",
            "PMC003xxxxxx/PMC3969273.xml\n",
            "PMC003xxxxxx/PMC3969286.xml\n",
            "PMC003xxxxxx/PMC3969288.xml\n",
            "PMC003xxxxxx/PMC3969301.xml\n",
            "PMC003xxxxxx/PMC3969303.xml\n",
            "PMC003xxxxxx/PMC3969304.xml\n",
            "PMC003xxxxxx/PMC3969306.xml\n",
            "PMC003xxxxxx/PMC3969338.xml\n",
            "PMC003xxxxxx/PMC3969339.xml\n",
            "PMC003xxxxxx/PMC3969340.xml\n",
            "PMC003xxxxxx/PMC3969341.xml\n",
            "PMC003xxxxxx/PMC3969342.xml\n",
            "PMC003xxxxxx/PMC3969343.xml\n",
            "PMC003xxxxxx/PMC3969344.xml\n",
            "PMC003xxxxxx/PMC3969345.xml\n",
            "PMC003xxxxxx/PMC3969346.xml\n",
            "PMC003xxxxxx/PMC3969347.xml\n",
            "PMC003xxxxxx/PMC3969513.xml\n",
            "PMC003xxxxxx/PMC3969516.xml\n",
            "PMC003xxxxxx/PMC3969517.xml\n",
            "PMC003xxxxxx/PMC3969519.xml\n",
            "PMC003xxxxxx/PMC3969523.xml\n",
            "PMC003xxxxxx/PMC3969524.xml\n",
            "PMC003xxxxxx/PMC3969527.xml\n",
            "PMC003xxxxxx/PMC3969553.xml\n",
            "PMC003xxxxxx/PMC3969554.xml\n",
            "PMC003xxxxxx/PMC3969555.xml\n",
            "PMC003xxxxxx/PMC3969558.xml\n",
            "PMC003xxxxxx/PMC3969568.xml\n",
            "PMC003xxxxxx/PMC3969569.xml\n",
            "PMC003xxxxxx/PMC3969574.xml\n",
            "PMC003xxxxxx/PMC3969575.xml\n",
            "PMC003xxxxxx/PMC3969577.xml\n",
            "PMC003xxxxxx/PMC3969580.xml\n",
            "PMC003xxxxxx/PMC3969583.xml\n",
            "PMC003xxxxxx/PMC3969587.xml\n",
            "PMC003xxxxxx/PMC3969601.xml\n",
            "PMC003xxxxxx/PMC3969602.xml\n",
            "PMC003xxxxxx/PMC3969603.xml\n",
            "PMC003xxxxxx/PMC3969604.xml\n",
            "PMC003xxxxxx/PMC3969605.xml\n",
            "PMC003xxxxxx/PMC3969606.xml\n",
            "PMC003xxxxxx/PMC3969607.xml\n",
            "PMC003xxxxxx/PMC3969608.xml\n",
            "PMC003xxxxxx/PMC3969609.xml\n",
            "PMC003xxxxxx/PMC3969625.xml\n",
            "PMC003xxxxxx/PMC3969626.xml\n",
            "PMC003xxxxxx/PMC3969627.xml\n",
            "PMC003xxxxxx/PMC3969628.xml\n",
            "PMC003xxxxxx/PMC3969629.xml\n",
            "PMC003xxxxxx/PMC3969630.xml\n",
            "PMC003xxxxxx/PMC3969631.xml\n",
            "PMC003xxxxxx/PMC3969634.xml\n",
            "PMC003xxxxxx/PMC3969635.xml\n",
            "PMC003xxxxxx/PMC3969636.xml\n",
            "PMC003xxxxxx/PMC3969637.xml\n",
            "PMC003xxxxxx/PMC3969638.xml\n",
            "PMC003xxxxxx/PMC3969639.xml\n",
            "PMC003xxxxxx/PMC3969640.xml\n",
            "PMC003xxxxxx/PMC3969641.xml\n",
            "PMC003xxxxxx/PMC3969642.xml\n",
            "PMC003xxxxxx/PMC3969643.xml\n",
            "PMC003xxxxxx/PMC3969644.xml\n",
            "PMC003xxxxxx/PMC3969645.xml\n",
            "PMC003xxxxxx/PMC3969646.xml\n",
            "PMC003xxxxxx/PMC3969647.xml\n",
            "PMC003xxxxxx/PMC3969648.xml\n",
            "PMC003xxxxxx/PMC3969649.xml\n",
            "PMC003xxxxxx/PMC3969650.xml\n",
            "PMC003xxxxxx/PMC3969651.xml\n",
            "PMC003xxxxxx/PMC3969652.xml\n",
            "PMC003xxxxxx/PMC3969653.xml\n",
            "PMC003xxxxxx/PMC3969654.xml\n",
            "PMC003xxxxxx/PMC3969655.xml\n",
            "PMC003xxxxxx/PMC3969656.xml\n",
            "PMC003xxxxxx/PMC3969657.xml\n",
            "PMC003xxxxxx/PMC3969658.xml\n",
            "PMC003xxxxxx/PMC3969659.xml\n",
            "PMC003xxxxxx/PMC3969660.xml\n",
            "PMC003xxxxxx/PMC3969661.xml\n",
            "PMC003xxxxxx/PMC3969662.xml\n",
            "PMC003xxxxxx/PMC3969663.xml\n",
            "PMC003xxxxxx/PMC3969664.xml\n",
            "PMC003xxxxxx/PMC3969665.xml\n",
            "PMC003xxxxxx/PMC3969666.xml\n",
            "PMC003xxxxxx/PMC3969667.xml\n",
            "PMC003xxxxxx/PMC3969668.xml\n",
            "PMC003xxxxxx/PMC3969669.xml\n",
            "PMC003xxxxxx/PMC3969670.xml\n",
            "PMC003xxxxxx/PMC3969671.xml\n",
            "PMC003xxxxxx/PMC3969672.xml\n",
            "PMC003xxxxxx/PMC3969673.xml\n",
            "PMC003xxxxxx/PMC3969674.xml\n",
            "PMC003xxxxxx/PMC3969675.xml\n",
            "PMC003xxxxxx/PMC3969676.xml\n",
            "PMC003xxxxxx/PMC3969677.xml\n",
            "PMC003xxxxxx/PMC3969678.xml\n",
            "PMC003xxxxxx/PMC3969679.xml\n",
            "PMC003xxxxxx/PMC3969680.xml\n",
            "PMC003xxxxxx/PMC3969681.xml\n",
            "PMC003xxxxxx/PMC3969682.xml\n",
            "PMC003xxxxxx/PMC3969683.xml\n",
            "PMC003xxxxxx/PMC3969684.xml\n",
            "PMC003xxxxxx/PMC3969685.xml\n",
            "PMC003xxxxxx/PMC3969686.xml\n",
            "PMC003xxxxxx/PMC3969687.xml\n",
            "PMC003xxxxxx/PMC3969688.xml\n",
            "PMC003xxxxxx/PMC3969689.xml\n",
            "PMC003xxxxxx/PMC3969690.xml\n",
            "PMC003xxxxxx/PMC3969691.xml\n",
            "PMC003xxxxxx/PMC3969692.xml\n",
            "PMC003xxxxxx/PMC3969693.xml\n",
            "PMC003xxxxxx/PMC3969694.xml\n",
            "PMC003xxxxxx/PMC3969695.xml\n",
            "PMC003xxxxxx/PMC3969696.xml\n",
            "PMC003xxxxxx/PMC3969697.xml\n",
            "PMC003xxxxxx/PMC3969698.xml\n",
            "PMC003xxxxxx/PMC3969699.xml\n",
            "PMC003xxxxxx/PMC3969700.xml\n",
            "PMC003xxxxxx/PMC3969701.xml\n",
            "PMC003xxxxxx/PMC3969702.xml\n",
            "PMC003xxxxxx/PMC3969703.xml\n",
            "PMC003xxxxxx/PMC3969704.xml\n",
            "PMC003xxxxxx/PMC3969705.xml\n",
            "PMC003xxxxxx/PMC3969706.xml\n",
            "PMC003xxxxxx/PMC3969707.xml\n",
            "PMC003xxxxxx/PMC3969708.xml\n",
            "PMC003xxxxxx/PMC3969709.xml\n",
            "PMC003xxxxxx/PMC3969717.xml\n",
            "PMC003xxxxxx/PMC3969752.xml\n",
            "PMC003xxxxxx/PMC3969794.xml\n",
            "PMC003xxxxxx/PMC3969827.xml\n",
            "PMC003xxxxxx/PMC3969828.xml\n",
            "PMC003xxxxxx/PMC3969829.xml\n",
            "PMC003xxxxxx/PMC3969830.xml\n",
            "PMC003xxxxxx/PMC3969831.xml\n",
            "PMC003xxxxxx/PMC3969832.xml\n",
            "PMC003xxxxxx/PMC3969833.xml\n",
            "PMC003xxxxxx/PMC3969834.xml\n",
            "PMC003xxxxxx/PMC3969835.xml\n",
            "PMC003xxxxxx/PMC3969889.xml\n",
            "PMC003xxxxxx/PMC3969890.xml\n",
            "PMC003xxxxxx/PMC3969897.xml\n",
            "PMC003xxxxxx/PMC3969900.xml\n",
            "PMC003xxxxxx/PMC3969901.xml\n",
            "PMC003xxxxxx/PMC3970091.xml\n",
            "PMC003xxxxxx/PMC3970092.xml\n",
            "PMC003xxxxxx/PMC3970093.xml\n",
            "PMC003xxxxxx/PMC3970094.xml\n",
            "PMC003xxxxxx/PMC3970095.xml\n",
            "PMC003xxxxxx/PMC3970096.xml\n",
            "PMC003xxxxxx/PMC3970097.xml\n",
            "PMC003xxxxxx/PMC3970098.xml\n",
            "PMC003xxxxxx/PMC3970099.xml\n",
            "PMC003xxxxxx/PMC3970100.xml\n",
            "PMC003xxxxxx/PMC3970101.xml\n",
            "PMC003xxxxxx/PMC3970102.xml\n",
            "PMC003xxxxxx/PMC3970103.xml\n",
            "PMC003xxxxxx/PMC3970104.xml\n",
            "PMC003xxxxxx/PMC3970105.xml\n",
            "PMC003xxxxxx/PMC3970106.xml\n",
            "PMC003xxxxxx/PMC3970107.xml\n",
            "PMC003xxxxxx/PMC3970108.xml\n",
            "PMC003xxxxxx/PMC3970119.xml\n",
            "PMC003xxxxxx/PMC3970120.xml\n",
            "PMC003xxxxxx/PMC3970121.xml\n",
            "PMC003xxxxxx/PMC3970123.xml\n",
            "PMC003xxxxxx/PMC3970124.xml\n",
            "PMC003xxxxxx/PMC3970125.xml\n",
            "PMC003xxxxxx/PMC3970127.xml\n",
            "PMC003xxxxxx/PMC3970128.xml\n",
            "PMC003xxxxxx/PMC3970130.xml\n",
            "PMC003xxxxxx/PMC3970131.xml\n",
            "PMC003xxxxxx/PMC3970181.xml\n",
            "PMC003xxxxxx/PMC3970189.xml\n",
            "PMC003xxxxxx/PMC3970190.xml\n",
            "PMC003xxxxxx/PMC3970191.xml\n",
            "PMC003xxxxxx/PMC3970192.xml\n",
            "PMC003xxxxxx/PMC3970223.xml\n",
            "PMC003xxxxxx/PMC3970224.xml\n",
            "PMC003xxxxxx/PMC3970225.xml\n",
            "PMC003xxxxxx/PMC3970226.xml\n",
            "PMC003xxxxxx/PMC3970227.xml\n",
            "PMC003xxxxxx/PMC3970228.xml\n",
            "PMC003xxxxxx/PMC3970229.xml\n",
            "PMC003xxxxxx/PMC3970230.xml\n",
            "PMC003xxxxxx/PMC3970231.xml\n",
            "PMC003xxxxxx/PMC3970232.xml\n",
            "PMC003xxxxxx/PMC3970233.xml\n",
            "PMC003xxxxxx/PMC3970270.xml\n",
            "PMC003xxxxxx/PMC3970271.xml\n",
            "PMC003xxxxxx/PMC3970272.xml\n",
            "PMC003xxxxxx/PMC3970273.xml\n",
            "PMC003xxxxxx/PMC3970274.xml\n",
            "PMC003xxxxxx/PMC3970275.xml\n",
            "PMC003xxxxxx/PMC3970276.xml\n",
            "PMC003xxxxxx/PMC3970277.xml\n",
            "PMC003xxxxxx/PMC3970278.xml\n",
            "PMC003xxxxxx/PMC3970279.xml\n",
            "PMC003xxxxxx/PMC3970280.xml\n",
            "PMC003xxxxxx/PMC3970281.xml\n",
            "PMC003xxxxxx/PMC3970282.xml\n",
            "PMC003xxxxxx/PMC3970283.xml\n",
            "PMC003xxxxxx/PMC3970284.xml\n",
            "PMC003xxxxxx/PMC3970285.xml\n",
            "PMC003xxxxxx/PMC3970286.xml\n",
            "PMC003xxxxxx/PMC3970291.xml\n",
            "PMC003xxxxxx/PMC3970292.xml\n",
            "PMC003xxxxxx/PMC3970293.xml\n",
            "PMC003xxxxxx/PMC3970294.xml\n",
            "PMC003xxxxxx/PMC3970295.xml\n",
            "PMC003xxxxxx/PMC3970296.xml\n",
            "PMC003xxxxxx/PMC3970297.xml\n",
            "PMC003xxxxxx/PMC3970298.xml\n",
            "PMC003xxxxxx/PMC3970299.xml\n",
            "PMC003xxxxxx/PMC3970300.xml\n",
            "PMC003xxxxxx/PMC3970301.xml\n",
            "PMC003xxxxxx/PMC3970302.xml\n",
            "PMC003xxxxxx/PMC3970303.xml\n",
            "PMC003xxxxxx/PMC3970304.xml\n",
            "PMC003xxxxxx/PMC3970305.xml\n",
            "PMC003xxxxxx/PMC3970306.xml\n",
            "PMC003xxxxxx/PMC3970307.xml\n",
            "PMC003xxxxxx/PMC3970308.xml\n",
            "PMC003xxxxxx/PMC3970309.xml\n",
            "PMC003xxxxxx/PMC3970310.xml\n",
            "PMC003xxxxxx/PMC3970311.xml\n",
            "PMC003xxxxxx/PMC3970312.xml\n",
            "PMC003xxxxxx/PMC3970313.xml\n",
            "PMC003xxxxxx/PMC3970341.xml\n",
            "PMC003xxxxxx/PMC3970352.xml\n",
            "PMC003xxxxxx/PMC3970371.xml\n",
            "PMC003xxxxxx/PMC3970372.xml\n",
            "PMC003xxxxxx/PMC3970373.xml\n",
            "PMC003xxxxxx/PMC3970374.xml\n",
            "PMC003xxxxxx/PMC3970375.xml\n",
            "PMC003xxxxxx/PMC3970376.xml\n",
            "PMC003xxxxxx/PMC3970377.xml\n",
            "PMC003xxxxxx/PMC3970378.xml\n",
            "PMC003xxxxxx/PMC3970379.xml\n",
            "PMC003xxxxxx/PMC3970381.xml\n",
            "PMC003xxxxxx/PMC3970382.xml\n",
            "PMC003xxxxxx/PMC3970383.xml\n",
            "PMC003xxxxxx/PMC3970384.xml\n",
            "PMC003xxxxxx/PMC3970385.xml\n",
            "PMC003xxxxxx/PMC3970386.xml\n",
            "PMC003xxxxxx/PMC3970387.xml\n",
            "PMC003xxxxxx/PMC3970388.xml\n",
            "PMC003xxxxxx/PMC3970389.xml\n",
            "PMC003xxxxxx/PMC3970390.xml\n",
            "PMC003xxxxxx/PMC3970391.xml\n",
            "PMC003xxxxxx/PMC3970392.xml\n",
            "PMC003xxxxxx/PMC3970393.xml\n",
            "PMC003xxxxxx/PMC3970394.xml\n",
            "PMC003xxxxxx/PMC3970565.xml\n",
            "PMC003xxxxxx/PMC3970566.xml\n",
            "PMC003xxxxxx/PMC3970567.xml\n",
            "PMC003xxxxxx/PMC3970583.xml\n",
            "PMC003xxxxxx/PMC3970588.xml\n",
            "PMC003xxxxxx/PMC3970589.xml\n",
            "PMC003xxxxxx/PMC3970590.xml\n",
            "PMC003xxxxxx/PMC3970591.xml\n",
            "PMC003xxxxxx/PMC3970592.xml\n",
            "PMC003xxxxxx/PMC3970593.xml\n",
            "PMC003xxxxxx/PMC3970594.xml\n",
            "PMC003xxxxxx/PMC3970595.xml\n",
            "PMC003xxxxxx/PMC3970596.xml\n",
            "PMC003xxxxxx/PMC3970597.xml\n",
            "PMC003xxxxxx/PMC3970598.xml\n",
            "PMC003xxxxxx/PMC3970599.xml\n",
            "PMC003xxxxxx/PMC3970600.xml\n",
            "PMC003xxxxxx/PMC3970601.xml\n",
            "PMC003xxxxxx/PMC3970602.xml\n",
            "PMC003xxxxxx/PMC3970603.xml\n",
            "PMC003xxxxxx/PMC3970604.xml\n",
            "PMC003xxxxxx/PMC3970605.xml\n",
            "PMC003xxxxxx/PMC3970606.xml\n",
            "PMC003xxxxxx/PMC3970607.xml\n",
            "PMC003xxxxxx/PMC3970608.xml\n",
            "PMC003xxxxxx/PMC3970609.xml\n",
            "PMC003xxxxxx/PMC3970610.xml\n",
            "PMC003xxxxxx/PMC3970611.xml\n",
            "PMC003xxxxxx/PMC3970612.xml\n",
            "PMC003xxxxxx/PMC3970613.xml\n",
            "PMC003xxxxxx/PMC3970614.xml\n",
            "PMC003xxxxxx/PMC3970615.xml\n",
            "PMC003xxxxxx/PMC3970616.xml\n",
            "PMC003xxxxxx/PMC3970617.xml\n",
            "PMC003xxxxxx/PMC3970618.xml\n",
            "PMC003xxxxxx/PMC3970619.xml\n",
            "PMC003xxxxxx/PMC3970620.xml\n",
            "PMC003xxxxxx/PMC3970621.xml\n",
            "PMC003xxxxxx/PMC3970622.xml\n",
            "PMC003xxxxxx/PMC3970623.xml\n",
            "PMC003xxxxxx/PMC3970624.xml\n",
            "PMC003xxxxxx/PMC3970625.xml\n",
            "PMC003xxxxxx/PMC3970626.xml\n",
            "PMC003xxxxxx/PMC3970627.xml\n",
            "PMC003xxxxxx/PMC3970628.xml\n",
            "PMC003xxxxxx/PMC3970629.xml\n",
            "PMC003xxxxxx/PMC3970630.xml\n",
            "PMC003xxxxxx/PMC3970631.xml\n",
            "PMC003xxxxxx/PMC3970632.xml\n",
            "PMC003xxxxxx/PMC3970633.xml\n",
            "PMC003xxxxxx/PMC3970634.xml\n",
            "PMC003xxxxxx/PMC3970635.xml\n",
            "PMC003xxxxxx/PMC3970636.xml\n",
            "PMC003xxxxxx/PMC3970637.xml\n",
            "PMC003xxxxxx/PMC3970638.xml\n",
            "PMC003xxxxxx/PMC3970639.xml\n",
            "PMC003xxxxxx/PMC3970640.xml\n",
            "PMC003xxxxxx/PMC3970641.xml\n",
            "PMC003xxxxxx/PMC3970643.xml\n",
            "PMC003xxxxxx/PMC3970644.xml\n",
            "PMC003xxxxxx/PMC3970645.xml\n",
            "PMC003xxxxxx/PMC3970646.xml\n",
            "PMC003xxxxxx/PMC3970647.xml\n",
            "PMC003xxxxxx/PMC3970649.xml\n",
            "PMC003xxxxxx/PMC3970650.xml\n",
            "PMC003xxxxxx/PMC3970651.xml\n",
            "PMC003xxxxxx/PMC3970652.xml\n",
            "PMC003xxxxxx/PMC3970653.xml\n",
            "PMC003xxxxxx/PMC3970654.xml\n",
            "PMC003xxxxxx/PMC3970678.xml\n",
            "PMC003xxxxxx/PMC3970679.xml\n",
            "PMC003xxxxxx/PMC3970680.xml\n",
            "PMC003xxxxxx/PMC3970681.xml\n",
            "PMC003xxxxxx/PMC3970682.xml\n",
            "PMC003xxxxxx/PMC3970683.xml\n",
            "PMC003xxxxxx/PMC3970684.xml\n",
            "PMC003xxxxxx/PMC3970685.xml\n",
            "PMC003xxxxxx/PMC3970686.xml\n",
            "PMC003xxxxxx/PMC3970687.xml\n",
            "PMC003xxxxxx/PMC3970688.xml\n",
            "PMC003xxxxxx/PMC3970689.xml\n",
            "PMC003xxxxxx/PMC3970690.xml\n",
            "PMC003xxxxxx/PMC3970691.xml\n",
            "PMC003xxxxxx/PMC3970692.xml\n",
            "PMC003xxxxxx/PMC3970693.xml\n",
            "PMC003xxxxxx/PMC3970694.xml\n",
            "PMC003xxxxxx/PMC3970695.xml\n",
            "PMC003xxxxxx/PMC3970696.xml\n",
            "PMC003xxxxxx/PMC3970697.xml\n",
            "PMC003xxxxxx/PMC3970699.xml\n",
            "PMC003xxxxxx/PMC3970719.xml\n",
            "PMC003xxxxxx/PMC3970720.xml\n",
            "PMC003xxxxxx/PMC3970721.xml\n",
            "PMC003xxxxxx/PMC3970722.xml\n",
            "PMC003xxxxxx/PMC3970763.xml\n",
            "PMC003xxxxxx/PMC3970764.xml\n",
            "PMC003xxxxxx/PMC3970901.xml\n",
            "PMC003xxxxxx/PMC3970902.xml\n",
            "PMC003xxxxxx/PMC3970903.xml\n",
            "PMC003xxxxxx/PMC3970915.xml\n",
            "PMC003xxxxxx/PMC3970916.xml\n",
            "PMC003xxxxxx/PMC3970917.xml\n",
            "PMC003xxxxxx/PMC3970918.xml\n",
            "PMC003xxxxxx/PMC3970919.xml\n",
            "PMC003xxxxxx/PMC3970920.xml\n",
            "PMC003xxxxxx/PMC3970921.xml\n",
            "PMC003xxxxxx/PMC3970922.xml\n",
            "PMC003xxxxxx/PMC3970940.xml\n",
            "PMC003xxxxxx/PMC3970941.xml\n",
            "PMC003xxxxxx/PMC3970942.xml\n",
            "PMC003xxxxxx/PMC3970943.xml\n",
            "PMC003xxxxxx/PMC3970944.xml\n",
            "PMC003xxxxxx/PMC3970945.xml\n",
            "PMC003xxxxxx/PMC3970946.xml\n",
            "PMC003xxxxxx/PMC3970947.xml\n",
            "PMC003xxxxxx/PMC3970948.xml\n",
            "PMC003xxxxxx/PMC3970949.xml\n",
            "PMC003xxxxxx/PMC3970950.xml\n",
            "PMC003xxxxxx/PMC3970951.xml\n",
            "PMC003xxxxxx/PMC3970983.xml\n",
            "PMC003xxxxxx/PMC3970984.xml\n",
            "PMC003xxxxxx/PMC3970985.xml\n",
            "PMC003xxxxxx/PMC3970986.xml\n",
            "PMC003xxxxxx/PMC3970987.xml\n",
            "PMC003xxxxxx/PMC3970988.xml\n",
            "PMC003xxxxxx/PMC3970989.xml\n",
            "PMC003xxxxxx/PMC3971119.xml\n",
            "PMC003xxxxxx/PMC3971120.xml\n",
            "PMC003xxxxxx/PMC3971124.xml\n",
            "PMC003xxxxxx/PMC3971125.xml\n",
            "PMC003xxxxxx/PMC3971126.xml\n",
            "PMC003xxxxxx/PMC3971127.xml\n",
            "PMC003xxxxxx/PMC3971128.xml\n",
            "PMC003xxxxxx/PMC3971129.xml\n",
            "PMC003xxxxxx/PMC3971130.xml\n",
            "PMC003xxxxxx/PMC3971131.xml\n",
            "PMC003xxxxxx/PMC3971151.xml\n",
            "PMC003xxxxxx/PMC3971257.xml\n",
            "PMC003xxxxxx/PMC3971353.xml\n",
            "PMC003xxxxxx/PMC3971354.xml\n",
            "PMC003xxxxxx/PMC3971355.xml\n",
            "PMC003xxxxxx/PMC3971356.xml\n",
            "PMC003xxxxxx/PMC3971357.xml\n",
            "PMC003xxxxxx/PMC3971358.xml\n",
            "PMC003xxxxxx/PMC3971359.xml\n",
            "PMC003xxxxxx/PMC3971360.xml\n",
            "PMC003xxxxxx/PMC3971361.xml\n",
            "PMC003xxxxxx/PMC3971362.xml\n",
            "PMC003xxxxxx/PMC3971363.xml\n",
            "PMC003xxxxxx/PMC3971365.xml\n",
            "PMC003xxxxxx/PMC3971366.xml\n",
            "PMC003xxxxxx/PMC3971367.xml\n",
            "PMC003xxxxxx/PMC3971383.xml\n",
            "PMC003xxxxxx/PMC3971384.xml\n",
            "PMC003xxxxxx/PMC3971394.xml\n",
            "PMC003xxxxxx/PMC3971395.xml\n",
            "PMC003xxxxxx/PMC3971397.xml\n",
            "PMC003xxxxxx/PMC3971399.xml\n",
            "PMC003xxxxxx/PMC3971400.xml\n",
            "PMC003xxxxxx/PMC3971402.xml\n",
            "PMC003xxxxxx/PMC3971403.xml\n",
            "PMC003xxxxxx/PMC3971412.xml\n",
            "PMC003xxxxxx/PMC3971413.xml\n",
            "PMC003xxxxxx/PMC3971418.xml\n",
            "PMC003xxxxxx/PMC3971419.xml\n",
            "PMC003xxxxxx/PMC3971420.xml\n",
            "PMC003xxxxxx/PMC3971421.xml\n",
            "PMC003xxxxxx/PMC3971424.xml\n",
            "PMC003xxxxxx/PMC3971425.xml\n",
            "PMC003xxxxxx/PMC3971449.xml\n",
            "PMC003xxxxxx/PMC3971450.xml\n",
            "PMC003xxxxxx/PMC3971460.xml\n",
            "PMC003xxxxxx/PMC3971462.xml\n",
            "PMC003xxxxxx/PMC3971464.xml\n",
            "PMC003xxxxxx/PMC3971465.xml\n",
            "PMC003xxxxxx/PMC3971484.xml\n",
            "PMC003xxxxxx/PMC3971579.xml\n",
            "PMC003xxxxxx/PMC3971580.xml\n",
            "PMC003xxxxxx/PMC3971581.xml\n",
            "PMC003xxxxxx/PMC3971583.xml\n",
            "PMC003xxxxxx/PMC3971584.xml\n",
            "PMC003xxxxxx/PMC3971587.xml\n",
            "PMC003xxxxxx/PMC3971589.xml\n",
            "PMC003xxxxxx/PMC3971590.xml\n",
            "PMC003xxxxxx/PMC3971591.xml\n",
            "PMC003xxxxxx/PMC3971596.xml\n",
            "PMC003xxxxxx/PMC3971597.xml\n",
            "PMC003xxxxxx/PMC3971601.xml\n",
            "PMC003xxxxxx/PMC3971655.xml\n",
            "PMC003xxxxxx/PMC3971656.xml\n",
            "PMC003xxxxxx/PMC3971657.xml\n",
            "PMC003xxxxxx/PMC3971658.xml\n",
            "PMC003xxxxxx/PMC3971659.xml\n",
            "PMC003xxxxxx/PMC3971660.xml\n",
            "PMC003xxxxxx/PMC3971661.xml\n",
            "PMC003xxxxxx/PMC3971662.xml\n",
            "PMC003xxxxxx/PMC3971663.xml\n",
            "PMC003xxxxxx/PMC3971737.xml\n",
            "PMC003xxxxxx/PMC3971738.xml\n",
            "PMC003xxxxxx/PMC3971739.xml\n",
            "PMC003xxxxxx/PMC3971740.xml\n",
            "PMC003xxxxxx/PMC3971741.xml\n",
            "PMC003xxxxxx/PMC3971742.xml\n",
            "PMC003xxxxxx/PMC3971743.xml\n",
            "PMC003xxxxxx/PMC3971744.xml\n",
            "PMC003xxxxxx/PMC3971745.xml\n",
            "PMC003xxxxxx/PMC3971746.xml\n",
            "PMC003xxxxxx/PMC3971747.xml\n",
            "PMC003xxxxxx/PMC3971748.xml\n",
            "PMC003xxxxxx/PMC3971749.xml\n",
            "PMC003xxxxxx/PMC3971750.xml\n",
            "PMC003xxxxxx/PMC3971751.xml\n",
            "PMC003xxxxxx/PMC3971752.xml\n",
            "PMC003xxxxxx/PMC3971753.xml\n",
            "PMC003xxxxxx/PMC3971754.xml\n",
            "PMC003xxxxxx/PMC3971762.xml\n",
            "PMC003xxxxxx/PMC3971769.xml\n",
            "PMC003xxxxxx/PMC3971770.xml\n",
            "PMC003xxxxxx/PMC3971771.xml\n",
            "PMC003xxxxxx/PMC3971791.xml\n",
            "PMC003xxxxxx/PMC3971804.xml\n",
            "PMC003xxxxxx/PMC3971909.xml\n",
            "PMC003xxxxxx/PMC3971910.xml\n",
            "PMC003xxxxxx/PMC3971911.xml\n",
            "PMC003xxxxxx/PMC3971912.xml\n",
            "PMC003xxxxxx/PMC3971913.xml\n",
            "PMC003xxxxxx/PMC3971933.xml\n",
            "PMC003xxxxxx/PMC3971934.xml\n",
            "PMC003xxxxxx/PMC3971935.xml\n",
            "PMC003xxxxxx/PMC3971936.xml\n",
            "PMC003xxxxxx/PMC3971937.xml\n",
            "PMC003xxxxxx/PMC3971938.xml\n",
            "PMC003xxxxxx/PMC3971939.xml\n",
            "PMC003xxxxxx/PMC3971940.xml\n",
            "PMC003xxxxxx/PMC3971941.xml\n",
            "PMC003xxxxxx/PMC3971942.xml\n",
            "PMC003xxxxxx/PMC3971943.xml\n",
            "PMC003xxxxxx/PMC3971944.xml\n",
            "PMC003xxxxxx/PMC3971945.xml\n",
            "PMC003xxxxxx/PMC3971946.xml\n",
            "PMC003xxxxxx/PMC3971947.xml\n",
            "PMC003xxxxxx/PMC3972023.xml\n",
            "PMC003xxxxxx/PMC3972024.xml\n",
            "PMC003xxxxxx/PMC3972025.xml\n",
            "PMC003xxxxxx/PMC3972026.xml\n",
            "PMC003xxxxxx/PMC3972027.xml\n",
            "PMC003xxxxxx/PMC3972028.xml\n",
            "PMC003xxxxxx/PMC3972064.xml\n",
            "PMC003xxxxxx/PMC3972400.xml\n",
            "PMC003xxxxxx/PMC3972401.xml\n",
            "PMC003xxxxxx/PMC3972402.xml\n",
            "PMC003xxxxxx/PMC3972403.xml\n",
            "PMC003xxxxxx/PMC3972404.xml\n",
            "PMC003xxxxxx/PMC3972405.xml\n",
            "PMC003xxxxxx/PMC3972406.xml\n",
            "PMC003xxxxxx/PMC3972407.xml\n",
            "PMC003xxxxxx/PMC3972408.xml\n",
            "PMC003xxxxxx/PMC3972409.xml\n",
            "PMC003xxxxxx/PMC3972410.xml\n",
            "PMC003xxxxxx/PMC3972411.xml\n",
            "PMC003xxxxxx/PMC3972412.xml\n",
            "PMC003xxxxxx/PMC3972414.xml\n",
            "PMC003xxxxxx/PMC3972415.xml\n",
            "PMC003xxxxxx/PMC3972416.xml\n",
            "PMC003xxxxxx/PMC3972417.xml\n",
            "PMC003xxxxxx/PMC3972437.xml\n",
            "PMC003xxxxxx/PMC3972438.xml\n",
            "PMC003xxxxxx/PMC3972498.xml\n",
            "PMC003xxxxxx/PMC3972499.xml\n",
            "PMC003xxxxxx/PMC3972500.xml\n",
            "PMC003xxxxxx/PMC3972501.xml\n",
            "PMC003xxxxxx/PMC3972503.xml\n",
            "PMC003xxxxxx/PMC3972504.xml\n",
            "PMC003xxxxxx/PMC3972505.xml\n",
            "PMC003xxxxxx/PMC3972506.xml\n",
            "PMC003xxxxxx/PMC3972507.xml\n",
            "PMC003xxxxxx/PMC3972508.xml\n",
            "PMC003xxxxxx/PMC3972509.xml\n",
            "PMC003xxxxxx/PMC3972632.xml\n",
            "PMC003xxxxxx/PMC3972633.xml\n",
            "PMC003xxxxxx/PMC3972689.xml\n",
            "PMC003xxxxxx/PMC3972691.xml\n",
            "PMC003xxxxxx/PMC3972692.xml\n",
            "PMC003xxxxxx/PMC3972693.xml\n",
            "PMC003xxxxxx/PMC3972697.xml\n",
            "PMC003xxxxxx/PMC3972698.xml\n",
            "PMC003xxxxxx/PMC3972699.xml\n",
            "PMC003xxxxxx/PMC3972701.xml\n",
            "PMC003xxxxxx/PMC3972702.xml\n",
            "PMC003xxxxxx/PMC3972703.xml\n",
            "PMC003xxxxxx/PMC3972706.xml\n",
            "PMC003xxxxxx/PMC3972783.xml\n",
            "PMC003xxxxxx/PMC3972784.xml\n",
            "PMC003xxxxxx/PMC3972785.xml\n",
            "PMC003xxxxxx/PMC3972787.xml\n",
            "PMC003xxxxxx/PMC3972788.xml\n",
            "PMC003xxxxxx/PMC3972789.xml\n",
            "PMC003xxxxxx/PMC3972790.xml\n",
            "PMC003xxxxxx/PMC3972791.xml\n",
            "PMC003xxxxxx/PMC3972792.xml\n",
            "PMC003xxxxxx/PMC3972794.xml\n",
            "PMC003xxxxxx/PMC3972997.xml\n",
            "PMC003xxxxxx/PMC3973038.xml\n",
            "PMC003xxxxxx/PMC3973039.xml\n",
            "PMC003xxxxxx/PMC3973041.xml\n",
            "PMC003xxxxxx/PMC3973043.xml\n",
            "PMC003xxxxxx/PMC3973070.xml\n",
            "PMC003xxxxxx/PMC3973071.xml\n",
            "PMC003xxxxxx/PMC3973121.xml\n",
            "PMC003xxxxxx/PMC3973124.xml\n",
            "PMC003xxxxxx/PMC3973125.xml\n",
            "PMC003xxxxxx/PMC3973126.xml\n",
            "PMC003xxxxxx/PMC3973192.xml\n",
            "PMC003xxxxxx/PMC3973193.xml\n",
            "PMC003xxxxxx/PMC3973194.xml\n",
            "PMC003xxxxxx/PMC3973195.xml\n",
            "PMC003xxxxxx/PMC3973196.xml\n",
            "PMC003xxxxxx/PMC3973197.xml\n",
            "PMC003xxxxxx/PMC3973198.xml\n",
            "PMC003xxxxxx/PMC3973199.xml\n",
            "PMC003xxxxxx/PMC3973200.xml\n",
            "PMC003xxxxxx/PMC3973201.xml\n",
            "PMC003xxxxxx/PMC3973202.xml\n",
            "PMC003xxxxxx/PMC3973203.xml\n",
            "PMC003xxxxxx/PMC3973204.xml\n",
            "PMC003xxxxxx/PMC3973205.xml\n",
            "PMC003xxxxxx/PMC3973206.xml\n",
            "PMC003xxxxxx/PMC3973207.xml\n",
            "PMC003xxxxxx/PMC3973208.xml\n",
            "PMC003xxxxxx/PMC3973209.xml\n",
            "PMC003xxxxxx/PMC3973210.xml\n",
            "PMC003xxxxxx/PMC3973211.xml\n",
            "PMC003xxxxxx/PMC3973212.xml\n",
            "PMC003xxxxxx/PMC3973213.xml\n",
            "PMC003xxxxxx/PMC3973214.xml\n",
            "PMC003xxxxxx/PMC3973215.xml\n",
            "PMC003xxxxxx/PMC3973217.xml\n",
            "PMC003xxxxxx/PMC3973218.xml\n",
            "PMC003xxxxxx/PMC3973219.xml\n",
            "PMC003xxxxxx/PMC3973220.xml\n",
            "PMC003xxxxxx/PMC3973221.xml\n",
            "PMC003xxxxxx/PMC3973222.xml\n",
            "PMC003xxxxxx/PMC3973223.xml\n",
            "PMC003xxxxxx/PMC3973224.xml\n",
            "PMC003xxxxxx/PMC3973226.xml\n",
            "PMC003xxxxxx/PMC3973227.xml\n",
            "PMC003xxxxxx/PMC3973228.xml\n",
            "PMC003xxxxxx/PMC3973229.xml\n",
            "PMC003xxxxxx/PMC3973230.xml\n",
            "PMC003xxxxxx/PMC3973231.xml\n",
            "PMC003xxxxxx/PMC3973232.xml\n",
            "PMC003xxxxxx/PMC3973233.xml\n",
            "PMC003xxxxxx/PMC3973234.xml\n",
            "PMC003xxxxxx/PMC3973235.xml\n",
            "PMC003xxxxxx/PMC3973236.xml\n",
            "PMC003xxxxxx/PMC3973237.xml\n",
            "PMC003xxxxxx/PMC3973238.xml\n",
            "PMC003xxxxxx/PMC3973239.xml\n",
            "PMC003xxxxxx/PMC3973240.xml\n",
            "PMC003xxxxxx/PMC3973241.xml\n",
            "PMC003xxxxxx/PMC3973242.xml\n",
            "PMC003xxxxxx/PMC3973244.xml\n",
            "PMC003xxxxxx/PMC3973245.xml\n",
            "PMC003xxxxxx/PMC3973246.xml\n",
            "PMC003xxxxxx/PMC3973247.xml\n",
            "PMC003xxxxxx/PMC3973248.xml\n",
            "PMC003xxxxxx/PMC3973284.xml\n",
            "PMC003xxxxxx/PMC3973285.xml\n",
            "PMC003xxxxxx/PMC3973287.xml\n",
            "PMC003xxxxxx/PMC3973288.xml\n",
            "PMC003xxxxxx/PMC3973289.xml\n",
            "PMC003xxxxxx/PMC3973290.xml\n",
            "PMC003xxxxxx/PMC3973291.xml\n",
            "PMC003xxxxxx/PMC3973292.xml\n",
            "PMC003xxxxxx/PMC3973295.xml\n",
            "PMC003xxxxxx/PMC3973296.xml\n",
            "PMC003xxxxxx/PMC3973300.xml\n",
            "PMC003xxxxxx/PMC3973301.xml\n",
            "PMC003xxxxxx/PMC3973305.xml\n",
            "PMC003xxxxxx/PMC3973307.xml\n",
            "PMC003xxxxxx/PMC3973309.xml\n",
            "PMC003xxxxxx/PMC3973310.xml\n",
            "PMC003xxxxxx/PMC3973311.xml\n",
            "PMC003xxxxxx/PMC3973313.xml\n",
            "PMC003xxxxxx/PMC3973315.xml\n",
            "PMC003xxxxxx/PMC3973318.xml\n",
            "PMC003xxxxxx/PMC3973321.xml\n",
            "PMC003xxxxxx/PMC3973322.xml\n",
            "PMC003xxxxxx/PMC3973323.xml\n",
            "PMC003xxxxxx/PMC3973328.xml\n",
            "PMC003xxxxxx/PMC3973329.xml\n",
            "PMC003xxxxxx/PMC3973331.xml\n",
            "PMC003xxxxxx/PMC3973332.xml\n",
            "PMC003xxxxxx/PMC3973333.xml\n",
            "PMC003xxxxxx/PMC3973335.xml\n",
            "PMC003xxxxxx/PMC3973337.xml\n",
            "PMC003xxxxxx/PMC3973339.xml\n",
            "PMC003xxxxxx/PMC3973340.xml\n",
            "PMC003xxxxxx/PMC3973341.xml\n",
            "PMC003xxxxxx/PMC3973343.xml\n",
            "PMC003xxxxxx/PMC3973347.xml\n",
            "PMC003xxxxxx/PMC3973446.xml\n",
            "PMC003xxxxxx/PMC3973452.xml\n",
            "PMC003xxxxxx/PMC3973453.xml\n",
            "PMC003xxxxxx/PMC3973454.xml\n",
            "PMC003xxxxxx/PMC3973455.xml\n",
            "PMC003xxxxxx/PMC3973456.xml\n",
            "PMC003xxxxxx/PMC3973457.xml\n",
            "PMC003xxxxxx/PMC3973458.xml\n",
            "PMC003xxxxxx/PMC3973459.xml\n",
            "PMC003xxxxxx/PMC3973460.xml\n",
            "PMC003xxxxxx/PMC3973461.xml\n",
            "PMC003xxxxxx/PMC3973462.xml\n",
            "PMC003xxxxxx/PMC3973463.xml\n",
            "PMC003xxxxxx/PMC3973464.xml\n",
            "PMC003xxxxxx/PMC3973465.xml\n",
            "PMC003xxxxxx/PMC3973466.xml\n",
            "PMC003xxxxxx/PMC3973467.xml\n",
            "PMC003xxxxxx/PMC3973468.xml\n",
            "PMC003xxxxxx/PMC3973469.xml\n",
            "PMC003xxxxxx/PMC3973470.xml\n",
            "PMC003xxxxxx/PMC3973471.xml\n",
            "PMC003xxxxxx/PMC3973472.xml\n",
            "PMC003xxxxxx/PMC3973473.xml\n",
            "PMC003xxxxxx/PMC3973474.xml\n",
            "PMC003xxxxxx/PMC3973659.xml\n",
            "PMC003xxxxxx/PMC3973734.xml\n",
            "PMC003xxxxxx/PMC3973735.xml\n",
            "PMC003xxxxxx/PMC3973752.xml\n",
            "PMC003xxxxxx/PMC3973753.xml\n",
            "PMC003xxxxxx/PMC3973754.xml\n",
            "PMC003xxxxxx/PMC3973755.xml\n",
            "PMC003xxxxxx/PMC3973756.xml\n",
            "PMC003xxxxxx/PMC3973757.xml\n",
            "PMC003xxxxxx/PMC3973758.xml\n",
            "PMC003xxxxxx/PMC3973759.xml\n",
            "PMC003xxxxxx/PMC3973760.xml\n",
            "PMC003xxxxxx/PMC3973761.xml\n",
            "PMC003xxxxxx/PMC3973762.xml\n",
            "PMC003xxxxxx/PMC3973763.xml\n",
            "PMC003xxxxxx/PMC3973764.xml\n",
            "PMC003xxxxxx/PMC3973765.xml\n",
            "PMC003xxxxxx/PMC3973766.xml\n",
            "PMC003xxxxxx/PMC3973767.xml\n",
            "PMC003xxxxxx/PMC3973768.xml\n",
            "PMC003xxxxxx/PMC3973769.xml\n",
            "PMC003xxxxxx/PMC3973770.xml\n",
            "PMC003xxxxxx/PMC3973771.xml\n",
            "PMC003xxxxxx/PMC3973805.xml\n",
            "PMC003xxxxxx/PMC3973806.xml\n",
            "PMC003xxxxxx/PMC3973807.xml\n",
            "PMC003xxxxxx/PMC3973808.xml\n",
            "PMC003xxxxxx/PMC3973809.xml\n",
            "PMC003xxxxxx/PMC3973810.xml\n",
            "PMC003xxxxxx/PMC3973811.xml\n",
            "PMC003xxxxxx/PMC3973824.xml\n",
            "PMC003xxxxxx/PMC3973934.xml\n",
            "PMC003xxxxxx/PMC3973952.xml\n",
            "PMC003xxxxxx/PMC3973953.xml\n",
            "PMC003xxxxxx/PMC3974034.xml\n",
            "PMC003xxxxxx/PMC3974035.xml\n",
            "PMC003xxxxxx/PMC3974036.xml\n",
            "PMC003xxxxxx/PMC3974071.xml\n",
            "PMC003xxxxxx/PMC3974072.xml\n",
            "PMC003xxxxxx/PMC3974073.xml\n",
            "PMC003xxxxxx/PMC3974074.xml\n",
            "PMC003xxxxxx/PMC3974075.xml\n",
            "PMC003xxxxxx/PMC3974076.xml\n",
            "PMC003xxxxxx/PMC3974077.xml\n",
            "PMC003xxxxxx/PMC3974078.xml\n",
            "PMC003xxxxxx/PMC3974079.xml\n",
            "PMC003xxxxxx/PMC3974080.xml\n",
            "PMC003xxxxxx/PMC3974081.xml\n",
            "PMC003xxxxxx/PMC3974082.xml\n",
            "PMC003xxxxxx/PMC3974083.xml\n",
            "PMC003xxxxxx/PMC3974084.xml\n",
            "PMC003xxxxxx/PMC3974085.xml\n",
            "PMC003xxxxxx/PMC3974086.xml\n",
            "PMC003xxxxxx/PMC3974087.xml\n",
            "PMC003xxxxxx/PMC3974088.xml\n",
            "PMC003xxxxxx/PMC3974089.xml\n",
            "PMC003xxxxxx/PMC3974090.xml\n",
            "PMC003xxxxxx/PMC3974091.xml\n",
            "PMC003xxxxxx/PMC3974092.xml\n",
            "PMC003xxxxxx/PMC3974093.xml\n",
            "PMC003xxxxxx/PMC3974094.xml\n",
            "PMC003xxxxxx/PMC3974095.xml\n",
            "PMC003xxxxxx/PMC3974096.xml\n",
            "PMC003xxxxxx/PMC3974097.xml\n",
            "PMC003xxxxxx/PMC3974098.xml\n",
            "PMC003xxxxxx/PMC3974099.xml\n",
            "PMC003xxxxxx/PMC3974130.xml\n",
            "PMC003xxxxxx/PMC3974131.xml\n",
            "PMC003xxxxxx/PMC3974132.xml\n",
            "PMC003xxxxxx/PMC3974135.xml\n",
            "PMC003xxxxxx/PMC3974136.xml\n",
            "PMC003xxxxxx/PMC3974137.xml\n",
            "PMC003xxxxxx/PMC3974138.xml\n",
            "PMC003xxxxxx/PMC3974139.xml\n",
            "PMC003xxxxxx/PMC3974142.xml\n",
            "PMC003xxxxxx/PMC3974143.xml\n",
            "PMC003xxxxxx/PMC3974169.xml\n",
            "PMC003xxxxxx/PMC3974176.xml\n",
            "PMC003xxxxxx/PMC3974178.xml\n",
            "PMC003xxxxxx/PMC3974179.xml\n",
            "PMC003xxxxxx/PMC3974213.xml\n",
            "PMC003xxxxxx/PMC3974215.xml\n",
            "PMC003xxxxxx/PMC3974216.xml\n",
            "PMC003xxxxxx/PMC3974219.xml\n",
            "PMC003xxxxxx/PMC3974221.xml\n",
            "PMC003xxxxxx/PMC3974308.xml\n",
            "PMC003xxxxxx/PMC3974309.xml\n",
            "PMC003xxxxxx/PMC3974310.xml\n",
            "PMC003xxxxxx/PMC3974311.xml\n",
            "PMC003xxxxxx/PMC3974312.xml\n",
            "PMC003xxxxxx/PMC3974313.xml\n",
            "PMC003xxxxxx/PMC3974314.xml\n",
            "PMC003xxxxxx/PMC3974315.xml\n",
            "PMC003xxxxxx/PMC3974316.xml\n",
            "PMC003xxxxxx/PMC3974317.xml\n",
            "PMC003xxxxxx/PMC3974318.xml\n",
            "PMC003xxxxxx/PMC3974319.xml\n",
            "PMC003xxxxxx/PMC3974320.xml\n",
            "PMC003xxxxxx/PMC3974321.xml\n",
            "PMC003xxxxxx/PMC3974322.xml\n",
            "PMC003xxxxxx/PMC3974323.xml\n",
            "PMC003xxxxxx/PMC3974324.xml\n",
            "PMC003xxxxxx/PMC3974325.xml\n",
            "PMC003xxxxxx/PMC3974326.xml\n",
            "PMC003xxxxxx/PMC3974327.xml\n",
            "PMC003xxxxxx/PMC3974328.xml\n",
            "PMC003xxxxxx/PMC3974329.xml\n",
            "PMC003xxxxxx/PMC3974330.xml\n",
            "PMC003xxxxxx/PMC3974338.xml\n",
            "PMC003xxxxxx/PMC3974462.xml\n",
            "PMC003xxxxxx/PMC3974463.xml\n",
            "PMC003xxxxxx/PMC3974464.xml\n",
            "PMC003xxxxxx/PMC3974465.xml\n",
            "PMC003xxxxxx/PMC3974466.xml\n",
            "PMC003xxxxxx/PMC3974467.xml\n",
            "PMC003xxxxxx/PMC3974468.xml\n",
            "PMC003xxxxxx/PMC3974469.xml\n",
            "PMC003xxxxxx/PMC3974470.xml\n",
            "PMC003xxxxxx/PMC3974471.xml\n",
            "PMC003xxxxxx/PMC3974472.xml\n",
            "PMC003xxxxxx/PMC3974473.xml\n",
            "PMC003xxxxxx/PMC3974474.xml\n",
            "PMC003xxxxxx/PMC3974475.xml\n",
            "PMC003xxxxxx/PMC3974476.xml\n",
            "PMC003xxxxxx/PMC3974477.xml\n",
            "PMC003xxxxxx/PMC3974478.xml\n",
            "PMC003xxxxxx/PMC3974479.xml\n",
            "PMC003xxxxxx/PMC3974480.xml\n",
            "PMC003xxxxxx/PMC3974481.xml\n",
            "PMC003xxxxxx/PMC3974482.xml\n",
            "PMC003xxxxxx/PMC3974483.xml\n",
            "PMC003xxxxxx/PMC3974484.xml\n",
            "PMC003xxxxxx/PMC3974485.xml\n",
            "PMC003xxxxxx/PMC3974486.xml\n",
            "PMC003xxxxxx/PMC3974487.xml\n",
            "PMC003xxxxxx/PMC3974488.xml\n",
            "PMC003xxxxxx/PMC3974489.xml\n",
            "PMC003xxxxxx/PMC3974490.xml\n",
            "PMC003xxxxxx/PMC3974491.xml\n",
            "PMC003xxxxxx/PMC3974492.xml\n",
            "PMC003xxxxxx/PMC3974493.xml\n",
            "PMC003xxxxxx/PMC3974494.xml\n",
            "PMC003xxxxxx/PMC3974495.xml\n",
            "PMC003xxxxxx/PMC3974496.xml\n",
            "PMC003xxxxxx/PMC3974497.xml\n",
            "PMC003xxxxxx/PMC3974498.xml\n",
            "PMC003xxxxxx/PMC3974499.xml\n",
            "PMC003xxxxxx/PMC3974567.xml\n",
            "PMC003xxxxxx/PMC3974568.xml\n",
            "PMC003xxxxxx/PMC3974569.xml\n",
            "PMC003xxxxxx/PMC3974570.xml\n",
            "PMC003xxxxxx/PMC3974571.xml\n",
            "PMC003xxxxxx/PMC3974572.xml\n",
            "PMC003xxxxxx/PMC3974687.xml\n",
            "PMC003xxxxxx/PMC3974688.xml\n",
            "PMC003xxxxxx/PMC3974689.xml\n",
            "PMC003xxxxxx/PMC3974690.xml\n",
            "PMC003xxxxxx/PMC3974691.xml\n",
            "PMC003xxxxxx/PMC3974692.xml\n",
            "PMC003xxxxxx/PMC3974693.xml\n",
            "PMC003xxxxxx/PMC3974694.xml\n",
            "PMC003xxxxxx/PMC3974695.xml\n",
            "PMC003xxxxxx/PMC3974696.xml\n",
            "PMC003xxxxxx/PMC3974697.xml\n",
            "PMC003xxxxxx/PMC3974823.xml\n",
            "PMC003xxxxxx/PMC3974881.xml\n",
            "PMC003xxxxxx/PMC3974887.xml\n",
            "PMC003xxxxxx/PMC3974895.xml\n",
            "PMC003xxxxxx/PMC3974896.xml\n",
            "PMC003xxxxxx/PMC3974950.xml\n",
            "PMC003xxxxxx/PMC3974951.xml\n",
            "PMC003xxxxxx/PMC3974952.xml\n",
            "PMC003xxxxxx/PMC3974953.xml\n",
            "PMC003xxxxxx/PMC3974954.xml\n",
            "PMC003xxxxxx/PMC3974955.xml\n",
            "PMC003xxxxxx/PMC3974956.xml\n",
            "PMC003xxxxxx/PMC3974957.xml\n",
            "PMC003xxxxxx/PMC3974958.xml\n",
            "PMC003xxxxxx/PMC3974959.xml\n",
            "PMC003xxxxxx/PMC3974960.xml\n",
            "PMC003xxxxxx/PMC3974961.xml\n",
            "PMC003xxxxxx/PMC3974962.xml\n",
            "PMC003xxxxxx/PMC3974963.xml\n",
            "PMC003xxxxxx/PMC3974964.xml\n",
            "PMC003xxxxxx/PMC3974965.xml\n",
            "PMC003xxxxxx/PMC3974966.xml\n",
            "PMC003xxxxxx/PMC3975055.xml\n",
            "PMC003xxxxxx/PMC3975056.xml\n",
            "PMC003xxxxxx/PMC3975057.xml\n",
            "PMC003xxxxxx/PMC3975058.xml\n",
            "PMC003xxxxxx/PMC3975059.xml\n",
            "PMC003xxxxxx/PMC3975060.xml\n",
            "PMC003xxxxxx/PMC3975061.xml\n",
            "PMC003xxxxxx/PMC3975062.xml\n",
            "PMC003xxxxxx/PMC3975063.xml\n",
            "PMC003xxxxxx/PMC3975064.xml\n",
            "PMC003xxxxxx/PMC3975065.xml\n",
            "PMC003xxxxxx/PMC3975066.xml\n",
            "PMC003xxxxxx/PMC3975067.xml\n",
            "PMC003xxxxxx/PMC3975068.xml\n",
            "PMC003xxxxxx/PMC3975070.xml\n",
            "PMC003xxxxxx/PMC3975083.xml\n",
            "PMC003xxxxxx/PMC3975086.xml\n",
            "PMC003xxxxxx/PMC3975089.xml\n",
            "PMC003xxxxxx/PMC3975172.xml\n",
            "PMC003xxxxxx/PMC3975173.xml\n",
            "PMC003xxxxxx/PMC3975174.xml\n",
            "PMC003xxxxxx/PMC3975175.xml\n",
            "PMC003xxxxxx/PMC3975182.xml\n",
            "PMC003xxxxxx/PMC3975183.xml\n",
            "PMC003xxxxxx/PMC3975184.xml\n",
            "PMC003xxxxxx/PMC3975185.xml\n",
            "PMC003xxxxxx/PMC3975186.xml\n",
            "PMC003xxxxxx/PMC3975187.xml\n",
            "PMC003xxxxxx/PMC3975188.xml\n",
            "PMC003xxxxxx/PMC3975192.xml\n",
            "PMC003xxxxxx/PMC3975193.xml\n",
            "PMC003xxxxxx/PMC3975194.xml\n",
            "PMC003xxxxxx/PMC3975195.xml\n",
            "PMC003xxxxxx/PMC3975196.xml\n",
            "PMC003xxxxxx/PMC3975197.xml\n",
            "PMC003xxxxxx/PMC3975198.xml\n",
            "PMC003xxxxxx/PMC3975199.xml\n",
            "PMC003xxxxxx/PMC3975200.xml\n",
            "PMC003xxxxxx/PMC3975201.xml\n",
            "PMC003xxxxxx/PMC3975202.xml\n",
            "PMC003xxxxxx/PMC3975203.xml\n",
            "PMC003xxxxxx/PMC3975204.xml\n",
            "PMC003xxxxxx/PMC3975205.xml\n",
            "PMC003xxxxxx/PMC3975206.xml\n",
            "PMC003xxxxxx/PMC3975207.xml\n",
            "PMC003xxxxxx/PMC3975208.xml\n",
            "PMC003xxxxxx/PMC3975209.xml\n",
            "PMC003xxxxxx/PMC3975210.xml\n",
            "PMC003xxxxxx/PMC3975211.xml\n",
            "PMC003xxxxxx/PMC3975220.xml\n",
            "PMC003xxxxxx/PMC3975221.xml\n",
            "PMC003xxxxxx/PMC3975222.xml\n",
            "PMC003xxxxxx/PMC3975223.xml\n",
            "PMC003xxxxxx/PMC3975234.xml\n",
            "PMC003xxxxxx/PMC3975236.xml\n",
            "PMC003xxxxxx/PMC3975238.xml\n",
            "PMC003xxxxxx/PMC3975239.xml\n",
            "PMC003xxxxxx/PMC3975321.xml\n",
            "PMC003xxxxxx/PMC3975322.xml\n",
            "PMC003xxxxxx/PMC3975454.xml\n",
            "PMC003xxxxxx/PMC3975469.xml\n",
            "PMC003xxxxxx/PMC3975470.xml\n",
            "PMC003xxxxxx/PMC3975471.xml\n",
            "PMC003xxxxxx/PMC3975472.xml\n",
            "PMC003xxxxxx/PMC3975473.xml\n",
            "PMC003xxxxxx/PMC3975474.xml\n",
            "PMC003xxxxxx/PMC3975475.xml\n",
            "PMC003xxxxxx/PMC3975476.xml\n",
            "PMC003xxxxxx/PMC3975477.xml\n",
            "PMC003xxxxxx/PMC3975478.xml\n",
            "PMC003xxxxxx/PMC3975479.xml\n",
            "PMC003xxxxxx/PMC3975480.xml\n",
            "PMC003xxxxxx/PMC3975500.xml\n",
            "PMC003xxxxxx/PMC3975505.xml\n",
            "PMC003xxxxxx/PMC3975623.xml\n",
            "PMC003xxxxxx/PMC3975632.xml\n",
            "PMC003xxxxxx/PMC3975633.xml\n",
            "PMC003xxxxxx/PMC3975634.xml\n",
            "PMC003xxxxxx/PMC3975720.xml\n",
            "PMC003xxxxxx/PMC3975721.xml\n",
            "PMC003xxxxxx/PMC3975722.xml\n",
            "PMC003xxxxxx/PMC3975723.xml\n",
            "PMC003xxxxxx/PMC3975724.xml\n",
            "PMC003xxxxxx/PMC3975735.xml\n",
            "PMC003xxxxxx/PMC3975738.xml\n",
            "PMC003xxxxxx/PMC3975739.xml\n",
            "PMC003xxxxxx/PMC3975740.xml\n",
            "PMC003xxxxxx/PMC3975741.xml\n",
            "PMC003xxxxxx/PMC3975742.xml\n",
            "PMC003xxxxxx/PMC3975743.xml\n",
            "PMC003xxxxxx/PMC3975744.xml\n",
            "PMC003xxxxxx/PMC3975745.xml\n",
            "PMC003xxxxxx/PMC3975746.xml\n",
            "PMC003xxxxxx/PMC3975747.xml\n",
            "PMC003xxxxxx/PMC3975748.xml\n",
            "PMC003xxxxxx/PMC3975749.xml\n",
            "PMC003xxxxxx/PMC3975750.xml\n",
            "PMC003xxxxxx/PMC3975751.xml\n",
            "PMC003xxxxxx/PMC3975752.xml\n",
            "PMC003xxxxxx/PMC3975753.xml\n",
            "PMC003xxxxxx/PMC3975754.xml\n",
            "PMC003xxxxxx/PMC3975755.xml\n",
            "PMC003xxxxxx/PMC3975756.xml\n",
            "PMC003xxxxxx/PMC3975757.xml\n",
            "PMC003xxxxxx/PMC3975758.xml\n",
            "PMC003xxxxxx/PMC3975759.xml\n",
            "PMC003xxxxxx/PMC3975760.xml\n",
            "PMC003xxxxxx/PMC3975762.xml\n",
            "PMC003xxxxxx/PMC3975763.xml\n",
            "PMC003xxxxxx/PMC3975764.xml\n",
            "PMC003xxxxxx/PMC3975765.xml\n",
            "PMC003xxxxxx/PMC3975766.xml\n",
            "PMC003xxxxxx/PMC3975767.xml\n",
            "PMC003xxxxxx/PMC3975786.xml\n",
            "PMC003xxxxxx/PMC3975787.xml\n",
            "PMC003xxxxxx/PMC3975810.xml\n",
            "PMC003xxxxxx/PMC3975811.xml\n",
            "PMC003xxxxxx/PMC3975812.xml\n",
            "PMC003xxxxxx/PMC3975813.xml\n",
            "PMC003xxxxxx/PMC3975825.xml\n",
            "PMC003xxxxxx/PMC3975925.xml\n",
            "PMC003xxxxxx/PMC3975996.xml\n",
            "PMC003xxxxxx/PMC3975997.xml\n",
            "PMC003xxxxxx/PMC3975998.xml\n",
            "PMC003xxxxxx/PMC3975999.xml\n",
            "PMC003xxxxxx/PMC3976000.xml\n",
            "PMC003xxxxxx/PMC3976001.xml\n",
            "PMC003xxxxxx/PMC3976002.xml\n",
            "PMC003xxxxxx/PMC3976003.xml\n",
            "PMC003xxxxxx/PMC3976004.xml\n",
            "PMC003xxxxxx/PMC3976005.xml\n",
            "PMC003xxxxxx/PMC3976006.xml\n",
            "PMC003xxxxxx/PMC3976007.xml\n",
            "PMC003xxxxxx/PMC3976008.xml\n",
            "PMC003xxxxxx/PMC3976009.xml\n",
            "PMC003xxxxxx/PMC3976010.xml\n",
            "PMC003xxxxxx/PMC3976011.xml\n",
            "PMC003xxxxxx/PMC3976012.xml\n",
            "PMC003xxxxxx/PMC3976013.xml\n",
            "PMC003xxxxxx/PMC3976014.xml\n",
            "PMC003xxxxxx/PMC3976015.xml\n",
            "PMC003xxxxxx/PMC3976016.xml\n",
            "PMC003xxxxxx/PMC3976017.xml\n",
            "PMC003xxxxxx/PMC3976018.xml\n",
            "PMC003xxxxxx/PMC3976019.xml\n",
            "PMC003xxxxxx/PMC3976020.xml\n",
            "PMC003xxxxxx/PMC3976021.xml\n",
            "PMC003xxxxxx/PMC3976022.xml\n",
            "PMC003xxxxxx/PMC3976023.xml\n",
            "PMC003xxxxxx/PMC3976024.xml\n",
            "PMC003xxxxxx/PMC3976025.xml\n",
            "PMC003xxxxxx/PMC3976026.xml\n",
            "PMC003xxxxxx/PMC3976027.xml\n",
            "PMC003xxxxxx/PMC3976028.xml\n",
            "PMC003xxxxxx/PMC3976029.xml\n",
            "PMC003xxxxxx/PMC3976205.xml\n",
            "PMC003xxxxxx/PMC3976206.xml\n",
            "PMC003xxxxxx/PMC3976207.xml\n",
            "PMC003xxxxxx/PMC3976208.xml\n",
            "PMC003xxxxxx/PMC3976209.xml\n",
            "PMC003xxxxxx/PMC3976210.xml\n",
            "PMC003xxxxxx/PMC3976211.xml\n",
            "PMC003xxxxxx/PMC3976234.xml\n",
            "PMC003xxxxxx/PMC3976235.xml\n",
            "PMC003xxxxxx/PMC3976236.xml\n",
            "PMC003xxxxxx/PMC3976237.xml\n",
            "PMC003xxxxxx/PMC3976238.xml\n",
            "PMC003xxxxxx/PMC3976239.xml\n",
            "PMC003xxxxxx/PMC3976285.xml\n",
            "PMC003xxxxxx/PMC3976505.xml\n",
            "PMC003xxxxxx/PMC3976506.xml\n",
            "PMC003xxxxxx/PMC3976508.xml\n",
            "PMC003xxxxxx/PMC3976509.xml\n",
            "PMC003xxxxxx/PMC3976517.xml\n",
            "PMC003xxxxxx/PMC3976519.xml\n",
            "PMC003xxxxxx/PMC3976532.xml\n",
            "PMC003xxxxxx/PMC3976533.xml\n",
            "PMC003xxxxxx/PMC3976534.xml\n",
            "PMC003xxxxxx/PMC3976611.xml\n",
            "PMC003xxxxxx/PMC3976684.xml\n",
            "PMC003xxxxxx/PMC3976685.xml\n",
            "PMC003xxxxxx/PMC3976686.xml\n",
            "PMC003xxxxxx/PMC3976687.xml\n",
            "PMC003xxxxxx/PMC3976688.xml\n",
            "PMC003xxxxxx/PMC3976689.xml\n",
            "PMC003xxxxxx/PMC3976690.xml\n",
            "PMC003xxxxxx/PMC3976691.xml\n",
            "PMC003xxxxxx/PMC3976742.xml\n",
            "PMC003xxxxxx/PMC3976843.xml\n",
            "PMC003xxxxxx/PMC3976970.xml\n",
            "PMC003xxxxxx/PMC3976973.xml\n",
            "PMC003xxxxxx/PMC3976978.xml\n",
            "PMC003xxxxxx/PMC3976979.xml\n",
            "PMC003xxxxxx/PMC3976981.xml\n",
            "PMC003xxxxxx/PMC3976982.xml\n",
            "PMC003xxxxxx/PMC3976984.xml\n",
            "PMC003xxxxxx/PMC3977028.xml\n",
            "PMC003xxxxxx/PMC3977126.xml\n",
            "PMC003xxxxxx/PMC3977127.xml\n",
            "PMC003xxxxxx/PMC3977128.xml\n",
            "PMC003xxxxxx/PMC3977129.xml\n",
            "PMC003xxxxxx/PMC3977130.xml\n",
            "PMC003xxxxxx/PMC3977131.xml\n",
            "PMC003xxxxxx/PMC3977134.xml\n",
            "PMC003xxxxxx/PMC3977146.xml\n",
            "PMC003xxxxxx/PMC3977153.xml\n",
            "PMC003xxxxxx/PMC3977154.xml\n",
            "PMC003xxxxxx/PMC3977155.xml\n",
            "PMC003xxxxxx/PMC3977156.xml\n",
            "PMC003xxxxxx/PMC3977157.xml\n",
            "PMC003xxxxxx/PMC3977158.xml\n",
            "PMC003xxxxxx/PMC3977159.xml\n",
            "PMC003xxxxxx/PMC3977160.xml\n",
            "PMC003xxxxxx/PMC3977161.xml\n",
            "PMC003xxxxxx/PMC3977162.xml\n",
            "PMC003xxxxxx/PMC3977163.xml\n",
            "PMC003xxxxxx/PMC3977164.xml\n",
            "PMC003xxxxxx/PMC3977165.xml\n",
            "PMC003xxxxxx/PMC3977166.xml\n",
            "PMC003xxxxxx/PMC3977167.xml\n",
            "PMC003xxxxxx/PMC3977168.xml\n",
            "PMC003xxxxxx/PMC3977169.xml\n",
            "PMC003xxxxxx/PMC3977170.xml\n",
            "PMC003xxxxxx/PMC3977171.xml\n",
            "PMC003xxxxxx/PMC3977172.xml\n",
            "PMC003xxxxxx/PMC3977173.xml\n",
            "PMC003xxxxxx/PMC3977206.xml\n",
            "PMC003xxxxxx/PMC3977207.xml\n",
            "PMC003xxxxxx/PMC3977208.xml\n",
            "PMC003xxxxxx/PMC3977209.xml\n",
            "PMC003xxxxxx/PMC3977210.xml\n",
            "PMC003xxxxxx/PMC3977211.xml\n",
            "PMC003xxxxxx/PMC3977212.xml\n",
            "PMC003xxxxxx/PMC3977213.xml\n",
            "PMC003xxxxxx/PMC3977214.xml\n",
            "PMC003xxxxxx/PMC3977221.xml\n",
            "PMC003xxxxxx/PMC3977222.xml\n",
            "PMC003xxxxxx/PMC3977226.xml\n",
            "PMC003xxxxxx/PMC3977227.xml\n",
            "PMC003xxxxxx/PMC3977228.xml\n",
            "PMC003xxxxxx/PMC3977319.xml\n",
            "PMC003xxxxxx/PMC3977322.xml\n",
            "PMC003xxxxxx/PMC3977323.xml\n",
            "PMC003xxxxxx/PMC3977324.xml\n",
            "PMC003xxxxxx/PMC3977325.xml\n",
            "PMC003xxxxxx/PMC3977326.xml\n",
            "PMC003xxxxxx/PMC3977327.xml\n",
            "PMC003xxxxxx/PMC3977328.xml\n",
            "PMC003xxxxxx/PMC3977329.xml\n",
            "PMC003xxxxxx/PMC3977330.xml\n",
            "PMC003xxxxxx/PMC3977331.xml\n",
            "PMC003xxxxxx/PMC3977332.xml\n",
            "PMC003xxxxxx/PMC3977333.xml\n",
            "PMC003xxxxxx/PMC3977334.xml\n",
            "PMC003xxxxxx/PMC3977335.xml\n",
            "PMC003xxxxxx/PMC3977349.xml\n",
            "PMC003xxxxxx/PMC3977350.xml\n",
            "PMC003xxxxxx/PMC3977351.xml\n",
            "PMC003xxxxxx/PMC3977352.xml\n",
            "PMC003xxxxxx/PMC3977353.xml\n",
            "PMC003xxxxxx/PMC3977354.xml\n",
            "PMC003xxxxxx/PMC3977355.xml\n",
            "PMC003xxxxxx/PMC3977356.xml\n",
            "PMC003xxxxxx/PMC3977357.xml\n",
            "PMC003xxxxxx/PMC3977358.xml\n",
            "PMC003xxxxxx/PMC3977359.xml\n",
            "PMC003xxxxxx/PMC3977360.xml\n",
            "PMC003xxxxxx/PMC3977361.xml\n",
            "PMC003xxxxxx/PMC3977362.xml\n",
            "PMC003xxxxxx/PMC3977363.xml\n",
            "PMC003xxxxxx/PMC3977365.xml\n",
            "PMC003xxxxxx/PMC3977366.xml\n",
            "PMC003xxxxxx/PMC3977367.xml\n",
            "PMC003xxxxxx/PMC3977368.xml\n",
            "PMC003xxxxxx/PMC3977369.xml\n",
            "PMC003xxxxxx/PMC3977370.xml\n",
            "PMC003xxxxxx/PMC3977371.xml\n",
            "PMC003xxxxxx/PMC3977372.xml\n",
            "PMC003xxxxxx/PMC3977373.xml\n",
            "PMC003xxxxxx/PMC3977374.xml\n",
            "PMC003xxxxxx/PMC3977375.xml\n",
            "PMC003xxxxxx/PMC3977376.xml\n",
            "PMC003xxxxxx/PMC3977377.xml\n",
            "PMC003xxxxxx/PMC3977378.xml\n",
            "PMC003xxxxxx/PMC3977379.xml\n",
            "PMC003xxxxxx/PMC3977380.xml\n",
            "PMC003xxxxxx/PMC3977381.xml\n",
            "PMC003xxxxxx/PMC3977382.xml\n",
            "PMC003xxxxxx/PMC3977383.xml\n",
            "PMC003xxxxxx/PMC3977384.xml\n",
            "PMC003xxxxxx/PMC3977385.xml\n",
            "PMC003xxxxxx/PMC3977386.xml\n",
            "PMC003xxxxxx/PMC3977387.xml\n",
            "PMC003xxxxxx/PMC3977390.xml\n",
            "PMC003xxxxxx/PMC3977391.xml\n",
            "PMC003xxxxxx/PMC3977392.xml\n",
            "PMC003xxxxxx/PMC3977393.xml\n",
            "PMC003xxxxxx/PMC3977394.xml\n",
            "PMC003xxxxxx/PMC3977395.xml\n",
            "PMC003xxxxxx/PMC3977396.xml\n",
            "PMC003xxxxxx/PMC3977397.xml\n",
            "PMC003xxxxxx/PMC3977398.xml\n",
            "PMC003xxxxxx/PMC3977399.xml\n",
            "PMC003xxxxxx/PMC3977400.xml\n",
            "PMC003xxxxxx/PMC3977401.xml\n",
            "PMC003xxxxxx/PMC3977402.xml\n",
            "PMC003xxxxxx/PMC3977403.xml\n",
            "PMC003xxxxxx/PMC3977404.xml\n",
            "PMC003xxxxxx/PMC3977405.xml\n",
            "PMC003xxxxxx/PMC3977406.xml\n",
            "PMC003xxxxxx/PMC3977407.xml\n",
            "PMC003xxxxxx/PMC3977408.xml\n",
            "PMC003xxxxxx/PMC3977409.xml\n",
            "PMC003xxxxxx/PMC3977410.xml\n",
            "PMC003xxxxxx/PMC3977411.xml\n",
            "PMC003xxxxxx/PMC3977412.xml\n",
            "PMC003xxxxxx/PMC3977413.xml\n",
            "PMC003xxxxxx/PMC3977414.xml\n",
            "PMC003xxxxxx/PMC3977456.xml\n",
            "PMC003xxxxxx/PMC3977457.xml\n",
            "PMC003xxxxxx/PMC3977458.xml\n",
            "PMC003xxxxxx/PMC3977499.xml\n",
            "PMC003xxxxxx/PMC3977553.xml\n",
            "PMC003xxxxxx/PMC3977554.xml\n",
            "PMC003xxxxxx/PMC3977606.xml\n",
            "PMC003xxxxxx/PMC3977616.xml\n",
            "PMC003xxxxxx/PMC3977619.xml\n",
            "PMC003xxxxxx/PMC3977621.xml\n",
            "PMC003xxxxxx/PMC3977623.xml\n",
            "PMC003xxxxxx/PMC3977624.xml\n",
            "PMC003xxxxxx/PMC3977911.xml\n",
            "PMC003xxxxxx/PMC3977913.xml\n",
            "PMC003xxxxxx/PMC3977915.xml\n",
            "PMC003xxxxxx/PMC3977918.xml\n",
            "PMC003xxxxxx/PMC3977919.xml\n",
            "PMC003xxxxxx/PMC3977920.xml\n",
            "PMC003xxxxxx/PMC3977921.xml\n",
            "PMC003xxxxxx/PMC3978099.xml\n",
            "PMC003xxxxxx/PMC3978100.xml\n",
            "PMC003xxxxxx/PMC3978101.xml\n",
            "PMC003xxxxxx/PMC3978102.xml\n",
            "PMC003xxxxxx/PMC3978103.xml\n",
            "PMC003xxxxxx/PMC3978104.xml\n",
            "PMC003xxxxxx/PMC3978105.xml\n",
            "PMC003xxxxxx/PMC3978106.xml\n",
            "PMC003xxxxxx/PMC3978107.xml\n",
            "PMC003xxxxxx/PMC3978108.xml\n",
            "PMC003xxxxxx/PMC3978109.xml\n",
            "PMC003xxxxxx/PMC3978110.xml\n",
            "PMC003xxxxxx/PMC3978184.xml\n",
            "PMC003xxxxxx/PMC3978185.xml\n",
            "PMC003xxxxxx/PMC3978186.xml\n",
            "PMC003xxxxxx/PMC3978187.xml\n",
            "PMC003xxxxxx/PMC3978188.xml\n",
            "PMC003xxxxxx/PMC3978189.xml\n",
            "PMC003xxxxxx/PMC3978190.xml\n",
            "PMC003xxxxxx/PMC3978191.xml\n",
            "PMC003xxxxxx/PMC3978197.xml\n",
            "PMC003xxxxxx/PMC3978269.xml\n",
            "PMC003xxxxxx/PMC3978270.xml\n",
            "PMC003xxxxxx/PMC3978271.xml\n",
            "PMC003xxxxxx/PMC3978272.xml\n",
            "PMC003xxxxxx/PMC3978273.xml\n",
            "PMC003xxxxxx/PMC3978274.xml\n",
            "PMC003xxxxxx/PMC3978275.xml\n",
            "PMC003xxxxxx/PMC3978276.xml\n",
            "PMC003xxxxxx/PMC3978277.xml\n",
            "PMC003xxxxxx/PMC3978278.xml\n",
            "PMC003xxxxxx/PMC3978279.xml\n",
            "PMC003xxxxxx/PMC3978280.xml\n",
            "PMC003xxxxxx/PMC3978281.xml\n",
            "PMC003xxxxxx/PMC3978304.xml\n",
            "PMC003xxxxxx/PMC3978497.xml\n",
            "PMC003xxxxxx/PMC3978498.xml\n",
            "PMC003xxxxxx/PMC3978499.xml\n",
            "PMC003xxxxxx/PMC3978500.xml\n",
            "PMC003xxxxxx/PMC3978501.xml\n",
            "PMC003xxxxxx/PMC3978502.xml\n",
            "PMC003xxxxxx/PMC3978503.xml\n",
            "PMC003xxxxxx/PMC3978666.xml\n",
            "PMC003xxxxxx/PMC3978790.xml\n",
            "PMC003xxxxxx/PMC3978813.xml\n",
            "PMC003xxxxxx/PMC3978819.xml\n",
            "PMC003xxxxxx/PMC3978830.xml\n",
            "PMC003xxxxxx/PMC3978896.xml\n",
            "PMC003xxxxxx/PMC3978932.xml\n",
            "PMC003xxxxxx/PMC3978933.xml\n",
            "PMC003xxxxxx/PMC3978934.xml\n",
            "PMC003xxxxxx/PMC3978935.xml\n",
            "PMC003xxxxxx/PMC3978936.xml\n",
            "PMC003xxxxxx/PMC3978937.xml\n",
            "PMC003xxxxxx/PMC3978938.xml\n",
            "PMC003xxxxxx/PMC3978939.xml\n",
            "PMC003xxxxxx/PMC3978940.xml\n",
            "PMC003xxxxxx/PMC3978941.xml\n",
            "PMC003xxxxxx/PMC3978942.xml\n",
            "PMC003xxxxxx/PMC3978952.xml\n",
            "PMC003xxxxxx/PMC3978953.xml\n",
            "PMC003xxxxxx/PMC3978954.xml\n",
            "PMC003xxxxxx/PMC3978955.xml\n",
            "PMC003xxxxxx/PMC3978956.xml\n",
            "PMC003xxxxxx/PMC3978957.xml\n",
            "PMC003xxxxxx/PMC3978958.xml\n",
            "PMC003xxxxxx/PMC3978959.xml\n",
            "PMC003xxxxxx/PMC3978960.xml\n",
            "PMC003xxxxxx/PMC3978961.xml\n",
            "PMC003xxxxxx/PMC3978962.xml\n",
            "PMC003xxxxxx/PMC3978963.xml\n",
            "PMC003xxxxxx/PMC3978964.xml\n",
            "PMC003xxxxxx/PMC3978965.xml\n",
            "PMC003xxxxxx/PMC3978966.xml\n",
            "PMC003xxxxxx/PMC3978967.xml\n",
            "PMC003xxxxxx/PMC3978968.xml\n",
            "PMC003xxxxxx/PMC3978969.xml\n",
            "PMC003xxxxxx/PMC3978970.xml\n",
            "PMC003xxxxxx/PMC3978971.xml\n",
            "PMC003xxxxxx/PMC3978972.xml\n",
            "PMC003xxxxxx/PMC3978973.xml\n",
            "PMC003xxxxxx/PMC3978974.xml\n",
            "PMC003xxxxxx/PMC3978975.xml\n",
            "PMC003xxxxxx/PMC3978976.xml\n",
            "PMC003xxxxxx/PMC3978977.xml\n",
            "PMC003xxxxxx/PMC3978978.xml\n",
            "PMC003xxxxxx/PMC3978979.xml\n",
            "PMC003xxxxxx/PMC3978980.xml\n",
            "PMC003xxxxxx/PMC3978981.xml\n",
            "PMC003xxxxxx/PMC3978982.xml\n",
            "PMC003xxxxxx/PMC3978983.xml\n",
            "PMC003xxxxxx/PMC3978984.xml\n",
            "PMC003xxxxxx/PMC3978985.xml\n",
            "PMC003xxxxxx/PMC3978986.xml\n",
            "PMC003xxxxxx/PMC3978987.xml\n",
            "PMC003xxxxxx/PMC3978988.xml\n",
            "PMC003xxxxxx/PMC3978989.xml\n",
            "PMC003xxxxxx/PMC3978990.xml\n",
            "PMC003xxxxxx/PMC3978991.xml\n",
            "PMC003xxxxxx/PMC3978992.xml\n",
            "PMC003xxxxxx/PMC3978993.xml\n",
            "PMC003xxxxxx/PMC3979107.xml\n",
            "PMC003xxxxxx/PMC3979112.xml\n",
            "PMC003xxxxxx/PMC3979113.xml\n",
            "PMC003xxxxxx/PMC3979172.xml\n",
            "PMC003xxxxxx/PMC3979173.xml\n",
            "PMC003xxxxxx/PMC3979174.xml\n",
            "PMC003xxxxxx/PMC3979175.xml\n",
            "PMC003xxxxxx/PMC3979176.xml\n",
            "PMC003xxxxxx/PMC3979177.xml\n",
            "PMC003xxxxxx/PMC3979178.xml\n",
            "PMC003xxxxxx/PMC3979179.xml\n",
            "PMC003xxxxxx/PMC3979180.xml\n",
            "PMC003xxxxxx/PMC3979181.xml\n",
            "PMC003xxxxxx/PMC3979182.xml\n",
            "PMC003xxxxxx/PMC3979183.xml\n",
            "PMC003xxxxxx/PMC3979184.xml\n",
            "PMC003xxxxxx/PMC3979185.xml\n",
            "PMC003xxxxxx/PMC3979186.xml\n",
            "PMC003xxxxxx/PMC3979187.xml\n",
            "PMC003xxxxxx/PMC3979188.xml\n",
            "PMC003xxxxxx/PMC3979189.xml\n",
            "PMC003xxxxxx/PMC3979190.xml\n",
            "PMC003xxxxxx/PMC3979191.xml\n",
            "PMC003xxxxxx/PMC3979192.xml\n",
            "PMC003xxxxxx/PMC3979193.xml\n",
            "PMC003xxxxxx/PMC3979194.xml\n",
            "PMC003xxxxxx/PMC3979195.xml\n",
            "PMC003xxxxxx/PMC3979196.xml\n",
            "PMC003xxxxxx/PMC3979197.xml\n",
            "PMC003xxxxxx/PMC3979198.xml\n",
            "PMC003xxxxxx/PMC3979199.xml\n",
            "PMC003xxxxxx/PMC3979200.xml\n",
            "PMC003xxxxxx/PMC3979201.xml\n",
            "PMC003xxxxxx/PMC3979202.xml\n",
            "PMC003xxxxxx/PMC3979203.xml\n",
            "PMC003xxxxxx/PMC3979204.xml\n",
            "PMC003xxxxxx/PMC3979205.xml\n",
            "PMC003xxxxxx/PMC3979206.xml\n",
            "PMC003xxxxxx/PMC3979207.xml\n",
            "PMC003xxxxxx/PMC3979208.xml\n",
            "PMC003xxxxxx/PMC3979209.xml\n",
            "PMC003xxxxxx/PMC3979210.xml\n",
            "PMC003xxxxxx/PMC3979211.xml\n",
            "PMC003xxxxxx/PMC3979212.xml\n",
            "PMC003xxxxxx/PMC3979213.xml\n",
            "PMC003xxxxxx/PMC3979214.xml\n",
            "PMC003xxxxxx/PMC3979215.xml\n",
            "PMC003xxxxxx/PMC3979216.xml\n",
            "PMC003xxxxxx/PMC3979217.xml\n",
            "PMC003xxxxxx/PMC3979218.xml\n",
            "PMC003xxxxxx/PMC3979219.xml\n",
            "PMC003xxxxxx/PMC3979220.xml\n",
            "PMC003xxxxxx/PMC3979221.xml\n",
            "PMC003xxxxxx/PMC3979222.xml\n",
            "PMC003xxxxxx/PMC3979223.xml\n",
            "PMC003xxxxxx/PMC3979224.xml\n",
            "PMC003xxxxxx/PMC3979225.xml\n",
            "PMC003xxxxxx/PMC3979226.xml\n",
            "PMC003xxxxxx/PMC3979227.xml\n",
            "PMC003xxxxxx/PMC3979228.xml\n",
            "PMC003xxxxxx/PMC3979229.xml\n",
            "PMC003xxxxxx/PMC3979230.xml\n",
            "PMC003xxxxxx/PMC3979231.xml\n",
            "PMC003xxxxxx/PMC3979235.xml\n",
            "PMC003xxxxxx/PMC3979236.xml\n",
            "PMC003xxxxxx/PMC3979237.xml\n",
            "PMC003xxxxxx/PMC3979238.xml\n",
            "PMC003xxxxxx/PMC3979239.xml\n",
            "PMC003xxxxxx/PMC3979240.xml\n",
            "PMC003xxxxxx/PMC3979241.xml\n",
            "PMC003xxxxxx/PMC3979242.xml\n",
            "PMC003xxxxxx/PMC3979243.xml\n",
            "PMC003xxxxxx/PMC3979244.xml\n",
            "PMC003xxxxxx/PMC3979245.xml\n",
            "PMC003xxxxxx/PMC3979246.xml\n",
            "PMC003xxxxxx/PMC3979247.xml\n",
            "PMC003xxxxxx/PMC3979248.xml\n",
            "PMC003xxxxxx/PMC3979249.xml\n",
            "PMC003xxxxxx/PMC3979250.xml\n",
            "PMC003xxxxxx/PMC3979251.xml\n",
            "PMC003xxxxxx/PMC3979252.xml\n",
            "PMC003xxxxxx/PMC3979253.xml\n",
            "PMC003xxxxxx/PMC3979254.xml\n",
            "PMC003xxxxxx/PMC3979255.xml\n",
            "PMC003xxxxxx/PMC3979256.xml\n",
            "PMC003xxxxxx/PMC3979257.xml\n",
            "PMC003xxxxxx/PMC3979258.xml\n",
            "PMC003xxxxxx/PMC3979259.xml\n",
            "PMC003xxxxxx/PMC3979260.xml\n",
            "PMC003xxxxxx/PMC3979261.xml\n",
            "PMC003xxxxxx/PMC3979262.xml\n",
            "PMC003xxxxxx/PMC3979263.xml\n",
            "PMC003xxxxxx/PMC3979264.xml\n",
            "PMC003xxxxxx/PMC3979265.xml\n",
            "PMC003xxxxxx/PMC3979266.xml\n",
            "PMC003xxxxxx/PMC3979267.xml\n",
            "PMC003xxxxxx/PMC3979268.xml\n",
            "PMC003xxxxxx/PMC3979269.xml\n",
            "PMC003xxxxxx/PMC3979270.xml\n",
            "PMC003xxxxxx/PMC3979271.xml\n",
            "PMC003xxxxxx/PMC3979272.xml\n",
            "PMC003xxxxxx/PMC3979273.xml\n",
            "PMC003xxxxxx/PMC3979274.xml\n",
            "PMC003xxxxxx/PMC3979342.xml\n",
            "PMC003xxxxxx/PMC3979453.xml\n",
            "PMC003xxxxxx/PMC3979498.xml\n",
            "PMC003xxxxxx/PMC3979573.xml\n",
            "PMC003xxxxxx/PMC3979574.xml\n",
            "PMC003xxxxxx/PMC3979686.xml\n",
            "PMC003xxxxxx/PMC3979687.xml\n",
            "PMC003xxxxxx/PMC3979688.xml\n",
            "PMC003xxxxxx/PMC3979689.xml\n",
            "PMC003xxxxxx/PMC3979690.xml\n",
            "PMC003xxxxxx/PMC3979691.xml\n",
            "PMC003xxxxxx/PMC3979692.xml\n",
            "PMC003xxxxxx/PMC3979693.xml\n",
            "PMC003xxxxxx/PMC3979694.xml\n",
            "PMC003xxxxxx/PMC3979695.xml\n",
            "PMC003xxxxxx/PMC3979779.xml\n",
            "PMC003xxxxxx/PMC3979780.xml\n",
            "PMC003xxxxxx/PMC3979781.xml\n",
            "PMC003xxxxxx/PMC3979782.xml\n",
            "PMC003xxxxxx/PMC3979783.xml\n",
            "PMC003xxxxxx/PMC3979784.xml\n",
            "PMC003xxxxxx/PMC3979785.xml\n",
            "PMC003xxxxxx/PMC3979786.xml\n",
            "PMC003xxxxxx/PMC3979787.xml\n",
            "PMC003xxxxxx/PMC3979788.xml\n",
            "PMC003xxxxxx/PMC3979789.xml\n",
            "PMC003xxxxxx/PMC3979790.xml\n",
            "PMC003xxxxxx/PMC3979791.xml\n",
            "PMC003xxxxxx/PMC3979792.xml\n",
            "PMC003xxxxxx/PMC3979793.xml\n",
            "PMC003xxxxxx/PMC3979794.xml\n",
            "PMC003xxxxxx/PMC3979795.xml\n",
            "PMC003xxxxxx/PMC3979796.xml\n",
            "PMC003xxxxxx/PMC3979797.xml\n",
            "PMC003xxxxxx/PMC3979798.xml\n",
            "PMC003xxxxxx/PMC3979799.xml\n",
            "PMC003xxxxxx/PMC3979800.xml\n",
            "PMC003xxxxxx/PMC3979801.xml\n",
            "PMC003xxxxxx/PMC3979802.xml\n",
            "PMC003xxxxxx/PMC3979810.xml\n",
            "PMC003xxxxxx/PMC3979823.xml\n",
            "PMC003xxxxxx/PMC3979827.xml\n",
            "PMC003xxxxxx/PMC3979828.xml\n",
            "PMC003xxxxxx/PMC3979829.xml\n",
            "PMC003xxxxxx/PMC3979830.xml\n",
            "PMC003xxxxxx/PMC3979862.xml\n",
            "PMC003xxxxxx/PMC3979866.xml\n",
            "PMC003xxxxxx/PMC3979867.xml\n",
            "PMC003xxxxxx/PMC3979868.xml\n",
            "PMC003xxxxxx/PMC3979872.xml\n",
            "PMC003xxxxxx/PMC3979873.xml\n",
            "PMC003xxxxxx/PMC3979877.xml\n",
            "PMC003xxxxxx/PMC3979884.xml\n",
            "PMC003xxxxxx/PMC3979886.xml\n",
            "PMC003xxxxxx/PMC3979887.xml\n",
            "PMC003xxxxxx/PMC3979888.xml\n",
            "PMC003xxxxxx/PMC3979889.xml\n",
            "PMC003xxxxxx/PMC3979895.xml\n",
            "PMC003xxxxxx/PMC3979896.xml\n",
            "PMC003xxxxxx/PMC3979897.xml\n",
            "PMC003xxxxxx/PMC3979898.xml\n",
            "PMC003xxxxxx/PMC3979899.xml\n",
            "PMC003xxxxxx/PMC3979900.xml\n",
            "PMC003xxxxxx/PMC3979901.xml\n",
            "PMC003xxxxxx/PMC3979902.xml\n",
            "PMC003xxxxxx/PMC3979903.xml\n",
            "PMC003xxxxxx/PMC3979904.xml\n",
            "PMC003xxxxxx/PMC3979905.xml\n",
            "PMC003xxxxxx/PMC3979908.xml\n",
            "PMC003xxxxxx/PMC3979909.xml\n",
            "PMC003xxxxxx/PMC3979910.xml\n",
            "PMC003xxxxxx/PMC3979916.xml\n",
            "PMC003xxxxxx/PMC3979936.xml\n",
            "PMC003xxxxxx/PMC3979937.xml\n",
            "PMC003xxxxxx/PMC3979938.xml\n",
            "PMC003xxxxxx/PMC3979939.xml\n",
            "PMC003xxxxxx/PMC3979940.xml\n",
            "PMC003xxxxxx/PMC3979941.xml\n",
            "PMC003xxxxxx/PMC3979942.xml\n",
            "PMC003xxxxxx/PMC3979943.xml\n",
            "PMC003xxxxxx/PMC3979944.xml\n",
            "PMC003xxxxxx/PMC3979945.xml\n",
            "PMC003xxxxxx/PMC3979946.xml\n",
            "PMC003xxxxxx/PMC3979947.xml\n",
            "PMC003xxxxxx/PMC3979948.xml\n",
            "PMC003xxxxxx/PMC3979949.xml\n",
            "PMC003xxxxxx/PMC3979950.xml\n",
            "PMC003xxxxxx/PMC3979951.xml\n",
            "PMC003xxxxxx/PMC3979952.xml\n",
            "PMC003xxxxxx/PMC3979953.xml\n",
            "PMC003xxxxxx/PMC3979954.xml\n",
            "PMC003xxxxxx/PMC3979955.xml\n",
            "PMC003xxxxxx/PMC3979956.xml\n",
            "PMC003xxxxxx/PMC3979957.xml\n",
            "PMC003xxxxxx/PMC3979958.xml\n",
            "PMC003xxxxxx/PMC3979959.xml\n",
            "PMC003xxxxxx/PMC3979960.xml\n",
            "PMC003xxxxxx/PMC3979987.xml\n",
            "PMC003xxxxxx/PMC3979988.xml\n",
            "PMC003xxxxxx/PMC3979989.xml\n",
            "PMC003xxxxxx/PMC3979990.xml\n",
            "PMC003xxxxxx/PMC3979991.xml\n",
            "PMC003xxxxxx/PMC3979992.xml\n",
            "PMC003xxxxxx/PMC3979993.xml\n",
            "PMC003xxxxxx/PMC3979994.xml\n",
            "PMC003xxxxxx/PMC3979995.xml\n",
            "PMC003xxxxxx/PMC3979996.xml\n",
            "PMC003xxxxxx/PMC3979997.xml\n",
            "PMC003xxxxxx/PMC3979998.xml\n",
            "PMC003xxxxxx/PMC3980145.xml\n",
            "PMC003xxxxxx/PMC3980146.xml\n",
            "PMC003xxxxxx/PMC3980147.xml\n",
            "PMC003xxxxxx/PMC3980148.xml\n",
            "PMC003xxxxxx/PMC3980151.xml\n",
            "PMC003xxxxxx/PMC3980152.xml\n",
            "PMC003xxxxxx/PMC3980153.xml\n",
            "PMC003xxxxxx/PMC3980154.xml\n",
            "PMC003xxxxxx/PMC3980155.xml\n",
            "PMC003xxxxxx/PMC3980156.xml\n",
            "PMC003xxxxxx/PMC3980157.xml\n",
            "PMC003xxxxxx/PMC3980158.xml\n",
            "PMC003xxxxxx/PMC3980159.xml\n",
            "PMC003xxxxxx/PMC3980160.xml\n",
            "PMC003xxxxxx/PMC3980203.xml\n",
            "PMC003xxxxxx/PMC3980204.xml\n",
            "PMC003xxxxxx/PMC3980205.xml\n",
            "PMC003xxxxxx/PMC3980206.xml\n",
            "PMC003xxxxxx/PMC3980207.xml\n",
            "PMC003xxxxxx/PMC3980208.xml\n",
            "PMC003xxxxxx/PMC3980209.xml\n",
            "PMC003xxxxxx/PMC3980210.xml\n",
            "PMC003xxxxxx/PMC3980211.xml\n",
            "PMC003xxxxxx/PMC3980212.xml\n",
            "PMC003xxxxxx/PMC3980213.xml\n",
            "PMC003xxxxxx/PMC3980214.xml\n",
            "PMC003xxxxxx/PMC3980215.xml\n",
            "PMC003xxxxxx/PMC3980217.xml\n",
            "PMC003xxxxxx/PMC3980218.xml\n",
            "PMC003xxxxxx/PMC3980219.xml\n",
            "PMC003xxxxxx/PMC3980220.xml\n",
            "PMC003xxxxxx/PMC3980222.xml\n",
            "PMC003xxxxxx/PMC3980223.xml\n",
            "PMC003xxxxxx/PMC3980224.xml\n",
            "PMC003xxxxxx/PMC3980225.xml\n",
            "PMC003xxxxxx/PMC3980226.xml\n",
            "PMC003xxxxxx/PMC3980227.xml\n",
            "PMC003xxxxxx/PMC3980228.xml\n",
            "PMC003xxxxxx/PMC3980229.xml\n",
            "PMC003xxxxxx/PMC3980230.xml\n",
            "PMC003xxxxxx/PMC3980318.xml\n",
            "PMC003xxxxxx/PMC3980335.xml\n",
            "PMC003xxxxxx/PMC3980375.xml\n",
            "PMC003xxxxxx/PMC3980414.xml\n",
            "PMC003xxxxxx/PMC3980419.xml\n",
            "PMC003xxxxxx/PMC3980420.xml\n",
            "PMC003xxxxxx/PMC3980508.xml\n",
            "PMC003xxxxxx/PMC3980510.xml\n",
            "PMC003xxxxxx/PMC3980513.xml\n",
            "PMC003xxxxxx/PMC3980515.xml\n",
            "PMC003xxxxxx/PMC3980516.xml\n",
            "PMC003xxxxxx/PMC3980517.xml\n",
            "PMC003xxxxxx/PMC3980519.xml\n",
            "PMC003xxxxxx/PMC3980543.xml\n",
            "PMC003xxxxxx/PMC3980544.xml\n",
            "PMC003xxxxxx/PMC3980545.xml\n",
            "PMC003xxxxxx/PMC3980546.xml\n",
            "PMC003xxxxxx/PMC3980547.xml\n",
            "PMC003xxxxxx/PMC3980548.xml\n",
            "PMC003xxxxxx/PMC3980549.xml\n",
            "PMC003xxxxxx/PMC3980550.xml\n",
            "PMC003xxxxxx/PMC3980551.xml\n",
            "PMC003xxxxxx/PMC3980552.xml\n",
            "PMC003xxxxxx/PMC3980554.xml\n",
            "PMC003xxxxxx/PMC3980555.xml\n",
            "PMC003xxxxxx/PMC3980556.xml\n",
            "PMC003xxxxxx/PMC3980557.xml\n",
            "PMC003xxxxxx/PMC3980558.xml\n",
            "PMC003xxxxxx/PMC3980559.xml\n",
            "PMC003xxxxxx/PMC3980560.xml\n",
            "PMC003xxxxxx/PMC3980561.xml\n",
            "PMC003xxxxxx/PMC3980562.xml\n",
            "PMC003xxxxxx/PMC3980563.xml\n",
            "PMC003xxxxxx/PMC3980564.xml\n",
            "PMC003xxxxxx/PMC3980789.xml\n",
            "PMC003xxxxxx/PMC3980797.xml\n",
            "PMC003xxxxxx/PMC3980806.xml\n",
            "PMC003xxxxxx/PMC3980950.xml\n",
            "PMC003xxxxxx/PMC3981105.xml\n",
            "PMC003xxxxxx/PMC3981176.xml\n",
            "PMC003xxxxxx/PMC3981223.xml\n",
            "PMC003xxxxxx/PMC3981224.xml\n",
            "PMC003xxxxxx/PMC3981225.xml\n",
            "PMC003xxxxxx/PMC3981226.xml\n",
            "PMC003xxxxxx/PMC3981227.xml\n",
            "PMC003xxxxxx/PMC3981228.xml\n",
            "PMC003xxxxxx/PMC3981229.xml\n",
            "PMC003xxxxxx/PMC3981230.xml\n",
            "PMC003xxxxxx/PMC3981231.xml\n",
            "PMC003xxxxxx/PMC3981232.xml\n",
            "PMC003xxxxxx/PMC3981233.xml\n",
            "PMC003xxxxxx/PMC3981234.xml\n",
            "PMC003xxxxxx/PMC3981235.xml\n",
            "PMC003xxxxxx/PMC3981236.xml\n",
            "PMC003xxxxxx/PMC3981258.xml\n",
            "PMC003xxxxxx/PMC3981259.xml\n",
            "PMC003xxxxxx/PMC3981260.xml\n",
            "PMC003xxxxxx/PMC3981261.xml\n",
            "PMC003xxxxxx/PMC3981262.xml\n",
            "PMC003xxxxxx/PMC3981263.xml\n",
            "PMC003xxxxxx/PMC3981264.xml\n",
            "PMC003xxxxxx/PMC3981265.xml\n",
            "PMC003xxxxxx/PMC3981266.xml\n",
            "PMC003xxxxxx/PMC3981267.xml\n",
            "PMC003xxxxxx/PMC3981268.xml\n",
            "PMC003xxxxxx/PMC3981269.xml\n",
            "PMC003xxxxxx/PMC3981270.xml\n",
            "PMC003xxxxxx/PMC3981271.xml\n",
            "PMC003xxxxxx/PMC3981272.xml\n",
            "PMC003xxxxxx/PMC3981273.xml\n",
            "PMC003xxxxxx/PMC3981274.xml\n",
            "PMC003xxxxxx/PMC3981275.xml\n",
            "PMC003xxxxxx/PMC3981833.xml\n",
            "PMC003xxxxxx/PMC3981834.xml\n",
            "PMC003xxxxxx/PMC3981975.xml\n",
            "PMC003xxxxxx/PMC3981976.xml\n",
            "PMC003xxxxxx/PMC3981977.xml\n",
            "PMC003xxxxxx/PMC3981978.xml\n",
            "PMC003xxxxxx/PMC3981979.xml\n",
            "PMC003xxxxxx/PMC3981980.xml\n",
            "PMC003xxxxxx/PMC3981981.xml\n",
            "PMC003xxxxxx/PMC3981982.xml\n",
            "PMC003xxxxxx/PMC3981983.xml\n",
            "PMC003xxxxxx/PMC3981984.xml\n",
            "PMC003xxxxxx/PMC3981985.xml\n",
            "PMC003xxxxxx/PMC3981986.xml\n",
            "PMC003xxxxxx/PMC3981987.xml\n",
            "PMC003xxxxxx/PMC3981988.xml\n",
            "PMC003xxxxxx/PMC3981989.xml\n",
            "PMC003xxxxxx/PMC3981990.xml\n",
            "PMC003xxxxxx/PMC3982035.xml\n",
            "PMC003xxxxxx/PMC3982038.xml\n",
            "PMC003xxxxxx/PMC3982039.xml\n",
            "PMC003xxxxxx/PMC3982040.xml\n",
            "PMC003xxxxxx/PMC3982041.xml\n",
            "PMC003xxxxxx/PMC3982042.xml\n",
            "PMC003xxxxxx/PMC3982118.xml\n",
            "PMC003xxxxxx/PMC3982122.xml\n",
            "PMC003xxxxxx/PMC3982123.xml\n",
            "PMC003xxxxxx/PMC3982128.xml\n",
            "PMC003xxxxxx/PMC3982129.xml\n",
            "PMC003xxxxxx/PMC3982130.xml\n",
            "PMC003xxxxxx/PMC3982131.xml\n",
            "PMC003xxxxxx/PMC3982132.xml\n",
            "PMC003xxxxxx/PMC3982133.xml\n",
            "PMC003xxxxxx/PMC3982134.xml\n",
            "PMC003xxxxxx/PMC3982135.xml\n",
            "PMC003xxxxxx/PMC3982154.xml\n",
            "PMC003xxxxxx/PMC3982156.xml\n",
            "PMC003xxxxxx/PMC3982158.xml\n",
            "PMC003xxxxxx/PMC3982163.xml\n",
            "PMC003xxxxxx/PMC3982164.xml\n",
            "PMC003xxxxxx/PMC3982166.xml\n",
            "PMC003xxxxxx/PMC3982167.xml\n",
            "PMC003xxxxxx/PMC3982168.xml\n",
            "PMC003xxxxxx/PMC3982169.xml\n",
            "PMC003xxxxxx/PMC3982170.xml\n",
            "PMC003xxxxxx/PMC3982171.xml\n",
            "PMC003xxxxxx/PMC3982172.xml\n",
            "PMC003xxxxxx/PMC3982173.xml\n",
            "PMC003xxxxxx/PMC3982174.xml\n",
            "PMC003xxxxxx/PMC3982175.xml\n",
            "PMC003xxxxxx/PMC3982176.xml\n",
            "PMC003xxxxxx/PMC3982177.xml\n",
            "PMC003xxxxxx/PMC3982178.xml\n",
            "PMC003xxxxxx/PMC3982179.xml\n",
            "PMC003xxxxxx/PMC3982180.xml\n",
            "PMC003xxxxxx/PMC3982181.xml\n",
            "PMC003xxxxxx/PMC3982182.xml\n",
            "PMC003xxxxxx/PMC3982196.xml\n",
            "PMC003xxxxxx/PMC3982213.xml\n",
            "PMC003xxxxxx/PMC3982236.xml\n",
            "PMC003xxxxxx/PMC3982237.xml\n",
            "PMC003xxxxxx/PMC3982238.xml\n",
            "PMC003xxxxxx/PMC3982239.xml\n",
            "PMC003xxxxxx/PMC3982240.xml\n",
            "PMC003xxxxxx/PMC3982241.xml\n",
            "PMC003xxxxxx/PMC3982242.xml\n",
            "PMC003xxxxxx/PMC3982243.xml\n",
            "PMC003xxxxxx/PMC3982325.xml\n",
            "PMC003xxxxxx/PMC3982330.xml\n",
            "PMC003xxxxxx/PMC3982331.xml\n",
            "PMC003xxxxxx/PMC3982332.xml\n",
            "PMC003xxxxxx/PMC3982333.xml\n",
            "PMC003xxxxxx/PMC3982334.xml\n",
            "PMC003xxxxxx/PMC3982335.xml\n",
            "PMC003xxxxxx/PMC3982336.xml\n",
            "PMC003xxxxxx/PMC3982337.xml\n",
            "PMC003xxxxxx/PMC3982338.xml\n",
            "PMC003xxxxxx/PMC3982339.xml\n",
            "PMC003xxxxxx/PMC3982340.xml\n",
            "PMC003xxxxxx/PMC3982341.xml\n",
            "PMC003xxxxxx/PMC3982342.xml\n",
            "PMC003xxxxxx/PMC3982343.xml\n",
            "PMC003xxxxxx/PMC3982344.xml\n",
            "PMC003xxxxxx/PMC3982345.xml\n",
            "PMC003xxxxxx/PMC3982346.xml\n",
            "PMC003xxxxxx/PMC3982347.xml\n",
            "PMC003xxxxxx/PMC3982348.xml\n",
            "PMC003xxxxxx/PMC3982349.xml\n",
            "PMC003xxxxxx/PMC3982350.xml\n",
            "PMC003xxxxxx/PMC3982351.xml\n",
            "PMC003xxxxxx/PMC3982352.xml\n",
            "PMC003xxxxxx/PMC3982353.xml\n",
            "PMC003xxxxxx/PMC3982354.xml\n",
            "PMC003xxxxxx/PMC3982355.xml\n",
            "PMC003xxxxxx/PMC3982356.xml\n",
            "PMC003xxxxxx/PMC3982357.xml\n",
            "PMC003xxxxxx/PMC3982358.xml\n",
            "PMC003xxxxxx/PMC3982359.xml\n",
            "PMC003xxxxxx/PMC3982360.xml\n",
            "PMC003xxxxxx/PMC3982361.xml\n",
            "PMC003xxxxxx/PMC3982362.xml\n",
            "PMC003xxxxxx/PMC3982363.xml\n",
            "PMC003xxxxxx/PMC3982364.xml\n",
            "PMC003xxxxxx/PMC3982365.xml\n",
            "PMC003xxxxxx/PMC3982366.xml\n",
            "PMC003xxxxxx/PMC3982367.xml\n",
            "PMC003xxxxxx/PMC3982368.xml\n",
            "PMC003xxxxxx/PMC3982369.xml\n",
            "PMC003xxxxxx/PMC3982370.xml\n",
            "PMC003xxxxxx/PMC3982371.xml\n",
            "PMC003xxxxxx/PMC3982372.xml\n",
            "PMC003xxxxxx/PMC3982373.xml\n",
            "PMC003xxxxxx/PMC3982374.xml\n",
            "PMC003xxxxxx/PMC3982375.xml\n",
            "PMC003xxxxxx/PMC3982376.xml\n",
            "PMC003xxxxxx/PMC3982377.xml\n",
            "PMC003xxxxxx/PMC3982378.xml\n",
            "PMC003xxxxxx/PMC3982486.xml\n",
            "PMC003xxxxxx/PMC3982499.xml\n",
            "PMC003xxxxxx/PMC3982619.xml\n",
            "PMC003xxxxxx/PMC3982624.xml\n",
            "PMC003xxxxxx/PMC3982625.xml\n",
            "PMC003xxxxxx/PMC3982626.xml\n",
            "PMC003xxxxxx/PMC3982627.xml\n",
            "PMC003xxxxxx/PMC3982628.xml\n",
            "PMC003xxxxxx/PMC3982629.xml\n",
            "PMC003xxxxxx/PMC3982630.xml\n",
            "PMC003xxxxxx/PMC3982631.xml\n",
            "PMC003xxxxxx/PMC3982632.xml\n",
            "PMC003xxxxxx/PMC3982633.xml\n",
            "PMC003xxxxxx/PMC3982634.xml\n",
            "PMC003xxxxxx/PMC3982635.xml\n",
            "PMC003xxxxxx/PMC3982636.xml\n",
            "PMC003xxxxxx/PMC3982637.xml\n",
            "PMC003xxxxxx/PMC3982638.xml\n",
            "PMC003xxxxxx/PMC3982639.xml\n",
            "PMC003xxxxxx/PMC3982640.xml\n",
            "PMC003xxxxxx/PMC3982641.xml\n",
            "PMC003xxxxxx/PMC3982642.xml\n",
            "PMC003xxxxxx/PMC3982643.xml\n",
            "PMC003xxxxxx/PMC3982644.xml\n",
            "PMC003xxxxxx/PMC3982645.xml\n",
            "PMC003xxxxxx/PMC3982646.xml\n",
            "PMC003xxxxxx/PMC3982647.xml\n",
            "PMC003xxxxxx/PMC3982718.xml\n",
            "PMC003xxxxxx/PMC3982781.xml\n",
            "PMC003xxxxxx/PMC3982805.xml\n",
            "PMC003xxxxxx/PMC3982806.xml\n",
            "PMC003xxxxxx/PMC3982807.xml\n",
            "PMC003xxxxxx/PMC3982808.xml\n",
            "PMC003xxxxxx/PMC3982809.xml\n",
            "PMC003xxxxxx/PMC3982810.xml\n",
            "PMC003xxxxxx/PMC3982811.xml\n",
            "PMC003xxxxxx/PMC3982812.xml\n",
            "PMC003xxxxxx/PMC3982813.xml\n",
            "PMC003xxxxxx/PMC3982814.xml\n",
            "PMC003xxxxxx/PMC3982815.xml\n",
            "PMC003xxxxxx/PMC3982816.xml\n",
            "PMC003xxxxxx/PMC3982817.xml\n",
            "PMC003xxxxxx/PMC3982818.xml\n",
            "PMC003xxxxxx/PMC3982819.xml\n",
            "PMC003xxxxxx/PMC3982820.xml\n",
            "PMC003xxxxxx/PMC3982821.xml\n",
            "PMC003xxxxxx/PMC3982822.xml\n",
            "PMC003xxxxxx/PMC3982823.xml\n",
            "PMC003xxxxxx/PMC3982824.xml\n",
            "PMC003xxxxxx/PMC3982825.xml\n",
            "PMC003xxxxxx/PMC3982826.xml\n",
            "PMC003xxxxxx/PMC3982827.xml\n",
            "PMC003xxxxxx/PMC3982828.xml\n",
            "PMC003xxxxxx/PMC3982829.xml\n",
            "PMC003xxxxxx/PMC3982830.xml\n",
            "PMC003xxxxxx/PMC3982837.xml\n",
            "PMC003xxxxxx/PMC3982849.xml\n",
            "PMC003xxxxxx/PMC3982906.xml\n",
            "PMC003xxxxxx/PMC3982969.xml\n",
            "PMC003xxxxxx/PMC3982970.xml\n",
            "PMC003xxxxxx/PMC3982973.xml\n",
            "PMC003xxxxxx/PMC3982974.xml\n",
            "PMC003xxxxxx/PMC3982977.xml\n",
            "PMC003xxxxxx/PMC3982978.xml\n",
            "PMC003xxxxxx/PMC3982982.xml\n",
            "PMC003xxxxxx/PMC3982986.xml\n",
            "PMC003xxxxxx/PMC3982987.xml\n",
            "PMC003xxxxxx/PMC3982988.xml\n",
            "PMC003xxxxxx/PMC3982989.xml\n",
            "PMC003xxxxxx/PMC3982990.xml\n",
            "PMC003xxxxxx/PMC3982991.xml\n",
            "PMC003xxxxxx/PMC3982992.xml\n",
            "PMC003xxxxxx/PMC3982993.xml\n",
            "PMC003xxxxxx/PMC3982994.xml\n",
            "PMC003xxxxxx/PMC3982995.xml\n",
            "PMC003xxxxxx/PMC3982996.xml\n",
            "PMC003xxxxxx/PMC3982997.xml\n",
            "PMC003xxxxxx/PMC3982998.xml\n",
            "PMC003xxxxxx/PMC3982999.xml\n",
            "PMC003xxxxxx/PMC3983000.xml\n",
            "PMC003xxxxxx/PMC3983001.xml\n",
            "PMC003xxxxxx/PMC3983002.xml\n",
            "PMC003xxxxxx/PMC3983003.xml\n",
            "PMC003xxxxxx/PMC3983009.xml\n",
            "PMC003xxxxxx/PMC3983010.xml\n",
            "PMC003xxxxxx/PMC3983015.xml\n",
            "PMC003xxxxxx/PMC3983019.xml\n",
            "PMC003xxxxxx/PMC3983023.xml\n",
            "PMC003xxxxxx/PMC3983024.xml\n",
            "PMC003xxxxxx/PMC3983027.xml\n",
            "PMC003xxxxxx/PMC3983075.xml\n",
            "PMC003xxxxxx/PMC3983076.xml\n",
            "PMC003xxxxxx/PMC3983077.xml\n",
            "PMC003xxxxxx/PMC3983078.xml\n",
            "PMC003xxxxxx/PMC3983411.xml\n",
            "PMC003xxxxxx/PMC3983412.xml\n",
            "PMC003xxxxxx/PMC3983413.xml\n",
            "PMC003xxxxxx/PMC3983414.xml\n",
            "PMC003xxxxxx/PMC3983415.xml\n",
            "PMC003xxxxxx/PMC3983416.xml\n",
            "PMC003xxxxxx/PMC3983417.xml\n",
            "PMC003xxxxxx/PMC3983418.xml\n",
            "PMC003xxxxxx/PMC3983419.xml\n",
            "PMC003xxxxxx/PMC3983420.xml\n",
            "PMC003xxxxxx/PMC3983421.xml\n",
            "PMC003xxxxxx/PMC3983422.xml\n",
            "PMC003xxxxxx/PMC3983423.xml\n",
            "PMC003xxxxxx/PMC3983424.xml\n",
            "PMC003xxxxxx/PMC3983425.xml\n",
            "PMC003xxxxxx/PMC3983426.xml\n",
            "PMC003xxxxxx/PMC3983473.xml\n",
            "PMC003xxxxxx/PMC3983476.xml\n",
            "PMC003xxxxxx/PMC3983502.xml\n",
            "PMC003xxxxxx/PMC3983503.xml\n",
            "PMC003xxxxxx/PMC3983504.xml\n",
            "PMC003xxxxxx/PMC3983505.xml\n",
            "PMC003xxxxxx/PMC3983506.xml\n",
            "PMC003xxxxxx/PMC3983507.xml\n",
            "PMC003xxxxxx/PMC3983508.xml\n",
            "PMC003xxxxxx/PMC3983509.xml\n",
            "PMC003xxxxxx/PMC3983510.xml\n",
            "PMC003xxxxxx/PMC3983511.xml\n",
            "PMC003xxxxxx/PMC3983529.xml\n",
            "PMC003xxxxxx/PMC3983530.xml\n",
            "PMC003xxxxxx/PMC3983531.xml\n",
            "PMC003xxxxxx/PMC3983532.xml\n",
            "PMC003xxxxxx/PMC3983533.xml\n",
            "PMC003xxxxxx/PMC3983534.xml\n",
            "PMC003xxxxxx/PMC3983535.xml\n",
            "PMC003xxxxxx/PMC3983536.xml\n",
            "PMC003xxxxxx/PMC3983537.xml\n",
            "PMC003xxxxxx/PMC3983538.xml\n",
            "PMC003xxxxxx/PMC3983539.xml\n",
            "PMC003xxxxxx/PMC3983540.xml\n",
            "PMC003xxxxxx/PMC3983541.xml\n",
            "PMC003xxxxxx/PMC3983542.xml\n",
            "PMC003xxxxxx/PMC3983543.xml\n",
            "PMC003xxxxxx/PMC3983544.xml\n",
            "PMC003xxxxxx/PMC3983545.xml\n",
            "PMC003xxxxxx/PMC3983546.xml\n",
            "PMC003xxxxxx/PMC3983547.xml\n",
            "PMC003xxxxxx/PMC3983548.xml\n",
            "PMC003xxxxxx/PMC3983549.xml\n",
            "PMC003xxxxxx/PMC3983550.xml\n",
            "PMC003xxxxxx/PMC3983551.xml\n",
            "PMC003xxxxxx/PMC3983552.xml\n",
            "PMC003xxxxxx/PMC3983553.xml\n",
            "PMC003xxxxxx/PMC3983554.xml\n",
            "PMC003xxxxxx/PMC3983555.xml\n",
            "PMC003xxxxxx/PMC3983556.xml\n",
            "PMC003xxxxxx/PMC3983568.xml\n",
            "PMC003xxxxxx/PMC3983573.xml\n",
            "PMC003xxxxxx/PMC3983574.xml\n",
            "PMC003xxxxxx/PMC3983594.xml\n",
            "PMC003xxxxxx/PMC3983596.xml\n",
            "PMC003xxxxxx/PMC3983597.xml\n",
            "PMC003xxxxxx/PMC3983599.xml\n",
            "PMC003xxxxxx/PMC3983600.xml\n",
            "PMC003xxxxxx/PMC3983601.xml\n",
            "PMC003xxxxxx/PMC3983602.xml\n",
            "PMC003xxxxxx/PMC3983603.xml\n",
            "PMC003xxxxxx/PMC3983604.xml\n",
            "PMC003xxxxxx/PMC3983605.xml\n",
            "PMC003xxxxxx/PMC3983618.xml\n",
            "PMC003xxxxxx/PMC3983657.xml\n",
            "PMC003xxxxxx/PMC3983660.xml\n",
            "PMC003xxxxxx/PMC3983675.xml\n",
            "PMC003xxxxxx/PMC3983676.xml\n",
            "PMC003xxxxxx/PMC3983677.xml\n",
            "PMC003xxxxxx/PMC3983679.xml\n",
            "PMC003xxxxxx/PMC3983680.xml\n",
            "PMC003xxxxxx/PMC3983683.xml\n",
            "PMC003xxxxxx/PMC3983739.xml\n",
            "PMC003xxxxxx/PMC3983740.xml\n",
            "PMC003xxxxxx/PMC3983741.xml\n",
            "PMC003xxxxxx/PMC3983742.xml\n",
            "PMC003xxxxxx/PMC3983743.xml\n",
            "PMC003xxxxxx/PMC3983744.xml\n",
            "PMC003xxxxxx/PMC3983745.xml\n",
            "PMC003xxxxxx/PMC3983746.xml\n",
            "PMC003xxxxxx/PMC3983747.xml\n",
            "PMC003xxxxxx/PMC3983748.xml\n",
            "PMC003xxxxxx/PMC3983749.xml\n",
            "PMC003xxxxxx/PMC3983750.xml\n",
            "PMC003xxxxxx/PMC3983751.xml\n",
            "PMC003xxxxxx/PMC3983752.xml\n",
            "PMC003xxxxxx/PMC3983753.xml\n",
            "PMC003xxxxxx/PMC3983759.xml\n",
            "PMC003xxxxxx/PMC3983760.xml\n",
            "PMC003xxxxxx/PMC3983761.xml\n",
            "PMC003xxxxxx/PMC3983762.xml\n",
            "PMC003xxxxxx/PMC3983763.xml\n",
            "PMC003xxxxxx/PMC3983764.xml\n",
            "PMC003xxxxxx/PMC3983765.xml\n",
            "PMC003xxxxxx/PMC3983766.xml\n",
            "PMC003xxxxxx/PMC3983767.xml\n",
            "PMC003xxxxxx/PMC3983768.xml\n",
            "PMC003xxxxxx/PMC3983769.xml\n",
            "PMC003xxxxxx/PMC3983770.xml\n",
            "PMC003xxxxxx/PMC3983771.xml\n",
            "PMC003xxxxxx/PMC3983772.xml\n",
            "PMC003xxxxxx/PMC3983773.xml\n",
            "PMC003xxxxxx/PMC3983774.xml\n",
            "PMC003xxxxxx/PMC3983775.xml\n",
            "PMC003xxxxxx/PMC3983941.xml\n",
            "PMC003xxxxxx/PMC3983959.xml\n",
            "PMC003xxxxxx/PMC3984012.xml\n",
            "PMC003xxxxxx/PMC3984013.xml\n",
            "PMC003xxxxxx/PMC3984023.xml\n",
            "PMC003xxxxxx/PMC3984039.xml\n",
            "PMC003xxxxxx/PMC3984040.xml\n",
            "PMC003xxxxxx/PMC3984041.xml\n",
            "PMC003xxxxxx/PMC3984042.xml\n",
            "PMC003xxxxxx/PMC3984047.xml\n",
            "PMC003xxxxxx/PMC3984055.xml\n",
            "PMC003xxxxxx/PMC3984056.xml\n",
            "PMC003xxxxxx/PMC3984057.xml\n",
            "PMC003xxxxxx/PMC3984058.xml\n",
            "PMC003xxxxxx/PMC3984059.xml\n",
            "PMC003xxxxxx/PMC3984060.xml\n",
            "PMC003xxxxxx/PMC3984061.xml\n",
            "PMC003xxxxxx/PMC3984062.xml\n",
            "PMC003xxxxxx/PMC3984063.xml\n",
            "PMC003xxxxxx/PMC3984064.xml\n",
            "PMC003xxxxxx/PMC3984066.xml\n",
            "PMC003xxxxxx/PMC3984067.xml\n",
            "PMC003xxxxxx/PMC3984068.xml\n",
            "PMC003xxxxxx/PMC3984136.xml\n",
            "PMC003xxxxxx/PMC3984187.xml\n",
            "PMC003xxxxxx/PMC3984188.xml\n",
            "PMC003xxxxxx/PMC3984189.xml\n",
            "PMC003xxxxxx/PMC3984190.xml\n",
            "PMC003xxxxxx/PMC3984191.xml\n",
            "PMC003xxxxxx/PMC3984192.xml\n",
            "PMC003xxxxxx/PMC3984193.xml\n",
            "PMC003xxxxxx/PMC3984194.xml\n",
            "PMC003xxxxxx/PMC3984195.xml\n",
            "PMC003xxxxxx/PMC3984196.xml\n",
            "PMC003xxxxxx/PMC3984197.xml\n",
            "PMC003xxxxxx/PMC3984198.xml\n",
            "PMC003xxxxxx/PMC3984199.xml\n",
            "PMC003xxxxxx/PMC3984200.xml\n",
            "PMC003xxxxxx/PMC3984201.xml\n",
            "PMC003xxxxxx/PMC3984202.xml\n",
            "PMC003xxxxxx/PMC3984203.xml\n",
            "PMC003xxxxxx/PMC3984204.xml\n",
            "PMC003xxxxxx/PMC3984267.xml\n",
            "PMC003xxxxxx/PMC3984268.xml\n",
            "PMC003xxxxxx/PMC3984269.xml\n",
            "PMC003xxxxxx/PMC3984288.xml\n",
            "PMC003xxxxxx/PMC3984289.xml\n",
            "PMC003xxxxxx/PMC3984290.xml\n",
            "PMC003xxxxxx/PMC3984291.xml\n",
            "PMC003xxxxxx/PMC3984292.xml\n",
            "PMC003xxxxxx/PMC3984293.xml\n",
            "PMC003xxxxxx/PMC3984294.xml\n",
            "PMC003xxxxxx/PMC3984295.xml\n",
            "PMC003xxxxxx/PMC3984303.xml\n",
            "PMC003xxxxxx/PMC3984304.xml\n",
            "PMC003xxxxxx/PMC3984305.xml\n",
            "PMC003xxxxxx/PMC3984306.xml\n",
            "PMC003xxxxxx/PMC3984307.xml\n",
            "PMC003xxxxxx/PMC3984308.xml\n",
            "PMC003xxxxxx/PMC3984309.xml\n",
            "PMC003xxxxxx/PMC3984310.xml\n",
            "PMC003xxxxxx/PMC3984312.xml\n",
            "PMC003xxxxxx/PMC3984323.xml\n",
            "PMC003xxxxxx/PMC3984324.xml\n",
            "PMC003xxxxxx/PMC3984333.xml\n",
            "PMC003xxxxxx/PMC3984341.xml\n",
            "PMC003xxxxxx/PMC3984342.xml\n",
            "PMC003xxxxxx/PMC3984345.xml\n",
            "PMC003xxxxxx/PMC3984406.xml\n",
            "PMC003xxxxxx/PMC3984411.xml\n",
            "PMC003xxxxxx/PMC3984415.xml\n",
            "PMC003xxxxxx/PMC3984417.xml\n",
            "PMC003xxxxxx/PMC3984443.xml\n",
            "PMC003xxxxxx/PMC3984444.xml\n",
            "PMC003xxxxxx/PMC3984445.xml\n",
            "PMC003xxxxxx/PMC3984446.xml\n",
            "PMC003xxxxxx/PMC3984447.xml\n",
            "PMC003xxxxxx/PMC3984448.xml\n",
            "PMC003xxxxxx/PMC3984498.xml\n",
            "PMC003xxxxxx/PMC3984518.xml\n",
            "PMC003xxxxxx/PMC3984546.xml\n",
            "PMC003xxxxxx/PMC3984657.xml\n",
            "PMC003xxxxxx/PMC3984761.xml\n",
            "PMC003xxxxxx/PMC3984841.xml\n",
            "PMC003xxxxxx/PMC3984886.xml\n",
            "PMC003xxxxxx/PMC3984887.xml\n",
            "PMC003xxxxxx/PMC3984888.xml\n",
            "PMC003xxxxxx/PMC3984889.xml\n",
            "PMC003xxxxxx/PMC3984890.xml\n",
            "PMC003xxxxxx/PMC3984891.xml\n",
            "PMC003xxxxxx/PMC3984892.xml\n",
            "PMC003xxxxxx/PMC3984893.xml\n",
            "PMC003xxxxxx/PMC3984894.xml\n",
            "PMC003xxxxxx/PMC3984895.xml\n",
            "PMC003xxxxxx/PMC3984952.xml\n",
            "PMC003xxxxxx/PMC3984953.xml\n",
            "PMC003xxxxxx/PMC3984954.xml\n",
            "PMC003xxxxxx/PMC3984955.xml\n",
            "PMC003xxxxxx/PMC3984956.xml\n",
            "PMC003xxxxxx/PMC3984957.xml\n",
            "PMC003xxxxxx/PMC3984958.xml\n",
            "PMC003xxxxxx/PMC3984959.xml\n",
            "PMC003xxxxxx/PMC3984960.xml\n",
            "PMC003xxxxxx/PMC3984961.xml\n",
            "PMC003xxxxxx/PMC3985061.xml\n",
            "PMC003xxxxxx/PMC3985073.xml\n",
            "PMC003xxxxxx/PMC3985074.xml\n",
            "PMC003xxxxxx/PMC3985075.xml\n",
            "PMC003xxxxxx/PMC3985076.xml\n",
            "PMC003xxxxxx/PMC3985077.xml\n",
            "PMC003xxxxxx/PMC3985078.xml\n",
            "PMC003xxxxxx/PMC3985079.xml\n",
            "PMC003xxxxxx/PMC3985080.xml\n",
            "PMC003xxxxxx/PMC3985081.xml\n",
            "PMC003xxxxxx/PMC3985082.xml\n",
            "PMC003xxxxxx/PMC3985083.xml\n",
            "PMC003xxxxxx/PMC3985185.xml\n",
            "PMC003xxxxxx/PMC3985347.xml\n",
            "PMC003xxxxxx/PMC3985348.xml\n",
            "PMC003xxxxxx/PMC3985349.xml\n",
            "PMC003xxxxxx/PMC3985350.xml\n",
            "PMC003xxxxxx/PMC3985351.xml\n",
            "PMC003xxxxxx/PMC3985352.xml\n",
            "PMC003xxxxxx/PMC3985353.xml\n",
            "PMC003xxxxxx/PMC3985354.xml\n",
            "PMC003xxxxxx/PMC3985355.xml\n",
            "PMC003xxxxxx/PMC3985356.xml\n",
            "PMC003xxxxxx/PMC3985357.xml\n",
            "PMC003xxxxxx/PMC3985358.xml\n",
            "PMC003xxxxxx/PMC3985359.xml\n",
            "PMC003xxxxxx/PMC3985360.xml\n",
            "PMC003xxxxxx/PMC3985361.xml\n",
            "PMC003xxxxxx/PMC3985362.xml\n",
            "PMC003xxxxxx/PMC3985363.xml\n",
            "PMC003xxxxxx/PMC3985364.xml\n",
            "PMC003xxxxxx/PMC3985365.xml\n",
            "PMC003xxxxxx/PMC3985366.xml\n",
            "PMC003xxxxxx/PMC3985367.xml\n",
            "PMC003xxxxxx/PMC3985368.xml\n",
            "PMC003xxxxxx/PMC3985369.xml\n",
            "PMC003xxxxxx/PMC3985370.xml\n",
            "PMC003xxxxxx/PMC3985371.xml\n",
            "PMC003xxxxxx/PMC3985372.xml\n",
            "PMC003xxxxxx/PMC3985373.xml\n",
            "PMC003xxxxxx/PMC3985374.xml\n",
            "PMC003xxxxxx/PMC3985375.xml\n",
            "PMC003xxxxxx/PMC3985376.xml\n",
            "PMC003xxxxxx/PMC3985377.xml\n",
            "PMC003xxxxxx/PMC3985378.xml\n",
            "PMC003xxxxxx/PMC3985379.xml\n",
            "PMC003xxxxxx/PMC3985380.xml\n",
            "PMC003xxxxxx/PMC3985423.xml\n",
            "PMC003xxxxxx/PMC3985504.xml\n",
            "PMC003xxxxxx/PMC3985618.xml\n",
            "PMC003xxxxxx/PMC3985619.xml\n",
            "PMC003xxxxxx/PMC3985622.xml\n",
            "PMC003xxxxxx/PMC3985623.xml\n",
            "PMC003xxxxxx/PMC3985624.xml\n",
            "PMC003xxxxxx/PMC3985629.xml\n",
            "PMC003xxxxxx/PMC3985633.xml\n",
            "PMC003xxxxxx/PMC3985634.xml\n",
            "PMC003xxxxxx/PMC3985637.xml\n",
            "PMC003xxxxxx/PMC3985646.xml\n",
            "PMC003xxxxxx/PMC3985647.xml\n",
            "PMC003xxxxxx/PMC3985659.xml\n",
            "PMC003xxxxxx/PMC3985666.xml\n",
            "PMC003xxxxxx/PMC3985667.xml\n",
            "PMC003xxxxxx/PMC3985669.xml\n",
            "PMC003xxxxxx/PMC3985670.xml\n",
            "PMC003xxxxxx/PMC3985678.xml\n",
            "PMC003xxxxxx/PMC3985780.xml\n",
            "PMC003xxxxxx/PMC3985782.xml\n",
            "PMC003xxxxxx/PMC3985784.xml\n",
            "PMC003xxxxxx/PMC3985785.xml\n",
            "PMC003xxxxxx/PMC3985789.xml\n",
            "PMC003xxxxxx/PMC3985792.xml\n",
            "PMC003xxxxxx/PMC3985793.xml\n",
            "PMC003xxxxxx/PMC3985797.xml\n",
            "PMC003xxxxxx/PMC3985800.xml\n",
            "PMC003xxxxxx/PMC3985802.xml\n",
            "PMC003xxxxxx/PMC3985805.xml\n",
            "PMC003xxxxxx/PMC3985806.xml\n",
            "PMC003xxxxxx/PMC3986008.xml\n",
            "PMC003xxxxxx/PMC3986029.xml\n",
            "PMC003xxxxxx/PMC3986030.xml\n",
            "PMC003xxxxxx/PMC3986031.xml\n",
            "PMC003xxxxxx/PMC3986034.xml\n",
            "PMC003xxxxxx/PMC3986109.xml\n",
            "PMC003xxxxxx/PMC3986110.xml\n",
            "PMC003xxxxxx/PMC3986111.xml\n",
            "PMC003xxxxxx/PMC3986112.xml\n",
            "PMC003xxxxxx/PMC3986113.xml\n",
            "PMC003xxxxxx/PMC3986274.xml\n",
            "PMC003xxxxxx/PMC3986275.xml\n",
            "PMC003xxxxxx/PMC3986276.xml\n",
            "PMC003xxxxxx/PMC3986277.xml\n",
            "PMC003xxxxxx/PMC3986278.xml\n",
            "PMC003xxxxxx/PMC3986279.xml\n",
            "PMC003xxxxxx/PMC3986280.xml\n",
            "PMC003xxxxxx/PMC3986281.xml\n",
            "PMC003xxxxxx/PMC3986282.xml\n",
            "PMC003xxxxxx/PMC3986283.xml\n",
            "PMC003xxxxxx/PMC3986284.xml\n",
            "PMC003xxxxxx/PMC3986285.xml\n",
            "PMC003xxxxxx/PMC3986286.xml\n",
            "PMC003xxxxxx/PMC3986287.xml\n",
            "PMC003xxxxxx/PMC3986288.xml\n",
            "PMC003xxxxxx/PMC3986289.xml\n",
            "PMC003xxxxxx/PMC3986290.xml\n",
            "PMC003xxxxxx/PMC3986291.xml\n",
            "PMC003xxxxxx/PMC3986292.xml\n",
            "PMC003xxxxxx/PMC3986293.xml\n",
            "PMC003xxxxxx/PMC3986294.xml\n",
            "PMC003xxxxxx/PMC3986295.xml\n",
            "PMC003xxxxxx/PMC3986296.xml\n",
            "PMC003xxxxxx/PMC3986297.xml\n",
            "PMC003xxxxxx/PMC3986298.xml\n",
            "PMC003xxxxxx/PMC3986299.xml\n",
            "PMC003xxxxxx/PMC3986300.xml\n",
            "PMC003xxxxxx/PMC3986301.xml\n",
            "PMC003xxxxxx/PMC3986302.xml\n",
            "PMC003xxxxxx/PMC3986331.xml\n",
            "PMC003xxxxxx/PMC3986332.xml\n",
            "PMC003xxxxxx/PMC3986333.xml\n",
            "PMC003xxxxxx/PMC3986334.xml\n",
            "PMC003xxxxxx/PMC3986335.xml\n",
            "PMC003xxxxxx/PMC3986336.xml\n",
            "PMC003xxxxxx/PMC3986337.xml\n",
            "PMC003xxxxxx/PMC3986415.xml\n",
            "PMC003xxxxxx/PMC3986416.xml\n",
            "PMC003xxxxxx/PMC3986417.xml\n",
            "PMC003xxxxxx/PMC3986422.xml\n",
            "PMC003xxxxxx/PMC3986490.xml\n",
            "PMC003xxxxxx/PMC3986491.xml\n",
            "PMC003xxxxxx/PMC3986492.xml\n",
            "PMC003xxxxxx/PMC3986494.xml\n",
            "PMC003xxxxxx/PMC3986495.xml\n",
            "PMC003xxxxxx/PMC3986497.xml\n",
            "PMC003xxxxxx/PMC3986498.xml\n",
            "PMC003xxxxxx/PMC3986499.xml\n",
            "PMC003xxxxxx/PMC3986500.xml\n",
            "PMC003xxxxxx/PMC3986504.xml\n",
            "PMC003xxxxxx/PMC3986507.xml\n",
            "PMC003xxxxxx/PMC3986508.xml\n",
            "PMC003xxxxxx/PMC3986510.xml\n",
            "PMC003xxxxxx/PMC3986534.xml\n",
            "PMC003xxxxxx/PMC3986535.xml\n",
            "PMC003xxxxxx/PMC3986536.xml\n",
            "PMC003xxxxxx/PMC3986537.xml\n",
            "PMC003xxxxxx/PMC3986538.xml\n",
            "PMC003xxxxxx/PMC3986540.xml\n",
            "PMC003xxxxxx/PMC3986548.xml\n",
            "PMC003xxxxxx/PMC3986549.xml\n",
            "PMC003xxxxxx/PMC3986579.xml\n",
            "PMC003xxxxxx/PMC3986580.xml\n",
            "PMC003xxxxxx/PMC3986581.xml\n",
            "PMC003xxxxxx/PMC3986582.xml\n",
            "PMC003xxxxxx/PMC3986583.xml\n",
            "PMC003xxxxxx/PMC3986584.xml\n",
            "PMC003xxxxxx/PMC3986587.xml\n",
            "PMC003xxxxxx/PMC3986588.xml\n",
            "PMC003xxxxxx/PMC3986617.xml\n",
            "PMC003xxxxxx/PMC3986619.xml\n",
            "PMC003xxxxxx/PMC3986624.xml\n",
            "PMC003xxxxxx/PMC3986626.xml\n",
            "PMC003xxxxxx/PMC3986627.xml\n",
            "PMC003xxxxxx/PMC3986629.xml\n",
            "PMC003xxxxxx/PMC3986630.xml\n",
            "PMC003xxxxxx/PMC3986631.xml\n",
            "PMC003xxxxxx/PMC3986633.xml\n",
            "PMC003xxxxxx/PMC3986634.xml\n",
            "PMC003xxxxxx/PMC3986637.xml\n",
            "PMC003xxxxxx/PMC3986657.xml\n",
            "PMC003xxxxxx/PMC3986660.xml\n",
            "PMC003xxxxxx/PMC3986661.xml\n",
            "PMC003xxxxxx/PMC3986663.xml\n",
            "PMC003xxxxxx/PMC3986664.xml\n",
            "PMC003xxxxxx/PMC3986684.xml\n",
            "PMC003xxxxxx/PMC3986698.xml\n",
            "PMC003xxxxxx/PMC3986699.xml\n",
            "PMC003xxxxxx/PMC3986700.xml\n",
            "PMC003xxxxxx/PMC3986701.xml\n",
            "PMC003xxxxxx/PMC3986702.xml\n",
            "PMC003xxxxxx/PMC3986703.xml\n",
            "PMC003xxxxxx/PMC3986704.xml\n",
            "PMC003xxxxxx/PMC3986728.xml\n",
            "PMC003xxxxxx/PMC3986732.xml\n",
            "PMC003xxxxxx/PMC3986809.xml\n",
            "PMC003xxxxxx/PMC3986868.xml\n",
            "PMC003xxxxxx/PMC3986893.xml\n",
            "PMC003xxxxxx/PMC3986894.xml\n",
            "PMC003xxxxxx/PMC3987051.xml\n",
            "PMC003xxxxxx/PMC3987052.xml\n",
            "PMC003xxxxxx/PMC3987062.xml\n",
            "PMC003xxxxxx/PMC3987132.xml\n",
            "PMC003xxxxxx/PMC3987133.xml\n",
            "PMC003xxxxxx/PMC3987134.xml\n",
            "PMC003xxxxxx/PMC3987135.xml\n",
            "PMC003xxxxxx/PMC3987136.xml\n",
            "PMC003xxxxxx/PMC3987137.xml\n",
            "PMC003xxxxxx/PMC3987138.xml\n",
            "PMC003xxxxxx/PMC3987139.xml\n",
            "PMC003xxxxxx/PMC3987140.xml\n",
            "PMC003xxxxxx/PMC3987141.xml\n",
            "PMC003xxxxxx/PMC3987142.xml\n",
            "PMC003xxxxxx/PMC3987143.xml\n",
            "PMC003xxxxxx/PMC3987144.xml\n",
            "PMC003xxxxxx/PMC3987145.xml\n",
            "PMC003xxxxxx/PMC3987149.xml\n",
            "PMC003xxxxxx/PMC3987150.xml\n",
            "PMC003xxxxxx/PMC3987151.xml\n",
            "PMC003xxxxxx/PMC3987152.xml\n",
            "PMC003xxxxxx/PMC3987153.xml\n",
            "PMC003xxxxxx/PMC3987154.xml\n",
            "PMC003xxxxxx/PMC3987155.xml\n",
            "PMC003xxxxxx/PMC3987156.xml\n",
            "PMC003xxxxxx/PMC3987157.xml\n",
            "PMC003xxxxxx/PMC3987179.xml\n",
            "PMC003xxxxxx/PMC3987180.xml\n",
            "PMC003xxxxxx/PMC3987181.xml\n",
            "PMC003xxxxxx/PMC3987182.xml\n",
            "PMC003xxxxxx/PMC3987183.xml\n",
            "PMC003xxxxxx/PMC3987184.xml\n",
            "PMC003xxxxxx/PMC3987185.xml\n",
            "PMC003xxxxxx/PMC3987186.xml\n",
            "PMC003xxxxxx/PMC3987187.xml\n",
            "PMC003xxxxxx/PMC3987188.xml\n",
            "PMC003xxxxxx/PMC3987189.xml\n",
            "PMC003xxxxxx/PMC3987190.xml\n",
            "PMC003xxxxxx/PMC3987191.xml\n",
            "PMC003xxxxxx/PMC3987192.xml\n",
            "PMC003xxxxxx/PMC3987193.xml\n",
            "PMC003xxxxxx/PMC3987194.xml\n",
            "PMC003xxxxxx/PMC3987195.xml\n",
            "PMC003xxxxxx/PMC3987196.xml\n",
            "PMC003xxxxxx/PMC3987197.xml\n",
            "PMC003xxxxxx/PMC3987198.xml\n",
            "PMC003xxxxxx/PMC3987199.xml\n",
            "PMC003xxxxxx/PMC3987200.xml\n",
            "PMC003xxxxxx/PMC3987244.xml\n",
            "PMC003xxxxxx/PMC3987245.xml\n",
            "PMC003xxxxxx/PMC3987246.xml\n",
            "PMC003xxxxxx/PMC3987247.xml\n",
            "PMC003xxxxxx/PMC3987255.xml\n",
            "PMC003xxxxxx/PMC3987256.xml\n",
            "PMC003xxxxxx/PMC3987260.xml\n",
            "PMC003xxxxxx/PMC3987261.xml\n",
            "PMC003xxxxxx/PMC3987262.xml\n",
            "PMC003xxxxxx/PMC3987263.xml\n",
            "PMC003xxxxxx/PMC3987264.xml\n",
            "PMC003xxxxxx/PMC3987265.xml\n",
            "PMC003xxxxxx/PMC3987266.xml\n",
            "PMC003xxxxxx/PMC3987267.xml\n",
            "PMC003xxxxxx/PMC3987268.xml\n",
            "PMC003xxxxxx/PMC3987269.xml\n",
            "PMC003xxxxxx/PMC3987270.xml\n",
            "PMC003xxxxxx/PMC3987271.xml\n",
            "PMC003xxxxxx/PMC3987272.xml\n",
            "PMC003xxxxxx/PMC3987273.xml\n",
            "PMC003xxxxxx/PMC3987274.xml\n",
            "PMC003xxxxxx/PMC3987275.xml\n",
            "PMC003xxxxxx/PMC3987276.xml\n",
            "PMC003xxxxxx/PMC3987277.xml\n",
            "PMC003xxxxxx/PMC3987278.xml\n",
            "PMC003xxxxxx/PMC3987279.xml\n",
            "PMC003xxxxxx/PMC3987280.xml\n",
            "PMC003xxxxxx/PMC3987281.xml\n",
            "PMC003xxxxxx/PMC3987282.xml\n",
            "PMC003xxxxxx/PMC3987283.xml\n",
            "PMC003xxxxxx/PMC3987284.xml\n",
            "PMC003xxxxxx/PMC3987292.xml\n",
            "PMC003xxxxxx/PMC3987293.xml\n",
            "PMC003xxxxxx/PMC3987294.xml\n",
            "PMC003xxxxxx/PMC3987295.xml\n",
            "PMC003xxxxxx/PMC3987296.xml\n",
            "PMC003xxxxxx/PMC3987309.xml\n",
            "PMC003xxxxxx/PMC3987310.xml\n",
            "PMC003xxxxxx/PMC3987311.xml\n",
            "PMC003xxxxxx/PMC3987312.xml\n",
            "PMC003xxxxxx/PMC3987313.xml\n",
            "PMC003xxxxxx/PMC3987314.xml\n",
            "PMC003xxxxxx/PMC3987315.xml\n",
            "PMC003xxxxxx/PMC3987316.xml\n",
            "PMC003xxxxxx/PMC3987317.xml\n",
            "PMC003xxxxxx/PMC3987318.xml\n",
            "PMC003xxxxxx/PMC3987319.xml\n",
            "PMC003xxxxxx/PMC3987320.xml\n",
            "PMC003xxxxxx/PMC3987321.xml\n",
            "PMC003xxxxxx/PMC3987322.xml\n",
            "PMC003xxxxxx/PMC3987323.xml\n",
            "PMC003xxxxxx/PMC3987324.xml\n",
            "PMC003xxxxxx/PMC3987325.xml\n",
            "PMC003xxxxxx/PMC3987326.xml\n",
            "PMC003xxxxxx/PMC3987327.xml\n",
            "PMC003xxxxxx/PMC3987328.xml\n",
            "PMC003xxxxxx/PMC3987329.xml\n",
            "PMC003xxxxxx/PMC3987330.xml\n",
            "PMC003xxxxxx/PMC3987331.xml\n",
            "PMC003xxxxxx/PMC3987332.xml\n",
            "PMC003xxxxxx/PMC3987333.xml\n",
            "PMC003xxxxxx/PMC3987334.xml\n",
            "PMC003xxxxxx/PMC3987335.xml\n",
            "PMC003xxxxxx/PMC3987336.xml\n",
            "PMC003xxxxxx/PMC3987337.xml\n",
            "PMC003xxxxxx/PMC3987338.xml\n",
            "PMC003xxxxxx/PMC3987339.xml\n",
            "PMC003xxxxxx/PMC3987340.xml\n",
            "PMC003xxxxxx/PMC3987341.xml\n",
            "PMC003xxxxxx/PMC3987342.xml\n",
            "PMC003xxxxxx/PMC3987343.xml\n",
            "PMC003xxxxxx/PMC3987344.xml\n",
            "PMC003xxxxxx/PMC3987345.xml\n",
            "PMC003xxxxxx/PMC3987346.xml\n",
            "PMC003xxxxxx/PMC3987347.xml\n",
            "PMC003xxxxxx/PMC3987348.xml\n",
            "PMC003xxxxxx/PMC3987349.xml\n",
            "PMC003xxxxxx/PMC3987350.xml\n",
            "PMC003xxxxxx/PMC3987351.xml\n",
            "PMC003xxxxxx/PMC3987352.xml\n",
            "PMC003xxxxxx/PMC3987353.xml\n",
            "PMC003xxxxxx/PMC3987354.xml\n",
            "PMC003xxxxxx/PMC3987355.xml\n",
            "PMC003xxxxxx/PMC3987356.xml\n",
            "PMC003xxxxxx/PMC3987357.xml\n",
            "PMC003xxxxxx/PMC3987358.xml\n",
            "PMC003xxxxxx/PMC3987359.xml\n",
            "PMC003xxxxxx/PMC3987360.xml\n",
            "PMC003xxxxxx/PMC3987361.xml\n",
            "PMC003xxxxxx/PMC3987362.xml\n",
            "PMC003xxxxxx/PMC3987363.xml\n",
            "PMC003xxxxxx/PMC3987364.xml\n",
            "PMC003xxxxxx/PMC3987365.xml\n",
            "PMC003xxxxxx/PMC3987366.xml\n",
            "PMC003xxxxxx/PMC3987367.xml\n",
            "PMC003xxxxxx/PMC3987368.xml\n",
            "PMC003xxxxxx/PMC3987369.xml\n",
            "PMC003xxxxxx/PMC3987370.xml\n",
            "PMC003xxxxxx/PMC3987371.xml\n",
            "PMC003xxxxxx/PMC3987372.xml\n",
            "PMC003xxxxxx/PMC3987373.xml\n",
            "PMC003xxxxxx/PMC3987374.xml\n",
            "PMC003xxxxxx/PMC3987375.xml\n",
            "PMC003xxxxxx/PMC3987376.xml\n",
            "PMC003xxxxxx/PMC3987377.xml\n",
            "PMC003xxxxxx/PMC3987378.xml\n",
            "PMC003xxxxxx/PMC3987379.xml\n",
            "PMC003xxxxxx/PMC3987380.xml\n",
            "PMC003xxxxxx/PMC3987381.xml\n",
            "PMC003xxxxxx/PMC3987382.xml\n",
            "PMC003xxxxxx/PMC3987383.xml\n",
            "PMC003xxxxxx/PMC3987384.xml\n",
            "PMC003xxxxxx/PMC3987385.xml\n",
            "PMC003xxxxxx/PMC3987386.xml\n",
            "PMC003xxxxxx/PMC3987387.xml\n",
            "PMC003xxxxxx/PMC3987388.xml\n",
            "PMC003xxxxxx/PMC3987389.xml\n",
            "PMC003xxxxxx/PMC3987390.xml\n",
            "PMC003xxxxxx/PMC3987391.xml\n",
            "PMC003xxxxxx/PMC3987392.xml\n",
            "PMC003xxxxxx/PMC3987393.xml\n",
            "PMC003xxxxxx/PMC3987394.xml\n",
            "PMC003xxxxxx/PMC3987395.xml\n",
            "PMC003xxxxxx/PMC3987396.xml\n",
            "PMC003xxxxxx/PMC3987397.xml\n",
            "PMC003xxxxxx/PMC3987398.xml\n",
            "PMC003xxxxxx/PMC3987399.xml\n",
            "PMC003xxxxxx/PMC3987400.xml\n",
            "PMC003xxxxxx/PMC3987401.xml\n",
            "PMC003xxxxxx/PMC3987402.xml\n",
            "PMC003xxxxxx/PMC3987403.xml\n",
            "PMC003xxxxxx/PMC3987462.xml\n",
            "PMC003xxxxxx/PMC3987463.xml\n",
            "PMC003xxxxxx/PMC3987464.xml\n",
            "PMC003xxxxxx/PMC3987465.xml\n",
            "PMC003xxxxxx/PMC3987466.xml\n",
            "PMC003xxxxxx/PMC3987467.xml\n",
            "PMC003xxxxxx/PMC3987468.xml\n",
            "PMC003xxxxxx/PMC3987469.xml\n",
            "PMC003xxxxxx/PMC3987470.xml\n",
            "PMC003xxxxxx/PMC3987471.xml\n",
            "PMC003xxxxxx/PMC3987472.xml\n",
            "PMC003xxxxxx/PMC3987473.xml\n",
            "PMC003xxxxxx/PMC3987474.xml\n",
            "PMC003xxxxxx/PMC3987475.xml\n",
            "PMC003xxxxxx/PMC3987476.xml\n",
            "PMC003xxxxxx/PMC3987477.xml\n",
            "PMC003xxxxxx/PMC3987478.xml\n",
            "PMC003xxxxxx/PMC3987479.xml\n",
            "PMC003xxxxxx/PMC3987480.xml\n",
            "PMC003xxxxxx/PMC3987481.xml\n",
            "PMC003xxxxxx/PMC3987482.xml\n",
            "PMC003xxxxxx/PMC3987483.xml\n",
            "PMC003xxxxxx/PMC3987484.xml\n",
            "PMC003xxxxxx/PMC3987485.xml\n",
            "PMC003xxxxxx/PMC3987486.xml\n",
            "PMC003xxxxxx/PMC3987487.xml\n",
            "PMC003xxxxxx/PMC3987488.xml\n",
            "PMC003xxxxxx/PMC3987489.xml\n",
            "PMC003xxxxxx/PMC3987490.xml\n",
            "PMC003xxxxxx/PMC3987491.xml\n",
            "PMC003xxxxxx/PMC3987492.xml\n",
            "PMC003xxxxxx/PMC3987493.xml\n",
            "PMC003xxxxxx/PMC3987494.xml\n",
            "PMC003xxxxxx/PMC3987495.xml\n",
            "PMC003xxxxxx/PMC3987496.xml\n",
            "PMC003xxxxxx/PMC3987497.xml\n",
            "PMC003xxxxxx/PMC3987498.xml\n",
            "PMC003xxxxxx/PMC3987499.xml\n",
            "PMC003xxxxxx/PMC3987501.xml\n",
            "PMC003xxxxxx/PMC3987502.xml\n",
            "PMC003xxxxxx/PMC3987503.xml\n",
            "PMC003xxxxxx/PMC3987504.xml\n",
            "PMC003xxxxxx/PMC3987708.xml\n",
            "PMC003xxxxxx/PMC3987709.xml\n",
            "PMC003xxxxxx/PMC3987710.xml\n",
            "PMC003xxxxxx/PMC3987711.xml\n",
            "PMC003xxxxxx/PMC3987712.xml\n",
            "PMC003xxxxxx/PMC3987713.xml\n",
            "PMC003xxxxxx/PMC3987714.xml\n",
            "PMC003xxxxxx/PMC3987715.xml\n",
            "PMC003xxxxxx/PMC3987716.xml\n",
            "PMC003xxxxxx/PMC3987717.xml\n",
            "PMC003xxxxxx/PMC3987718.xml\n",
            "PMC003xxxxxx/PMC3987719.xml\n",
            "PMC003xxxxxx/PMC3987721.xml\n",
            "PMC003xxxxxx/PMC3987722.xml\n",
            "PMC003xxxxxx/PMC3987723.xml\n",
            "PMC003xxxxxx/PMC3987725.xml\n",
            "PMC003xxxxxx/PMC3987727.xml\n",
            "PMC003xxxxxx/PMC3987728.xml\n",
            "PMC003xxxxxx/PMC3987729.xml\n",
            "PMC003xxxxxx/PMC3987730.xml\n",
            "PMC003xxxxxx/PMC3987732.xml\n",
            "PMC003xxxxxx/PMC3987733.xml\n",
            "PMC003xxxxxx/PMC3987735.xml\n",
            "PMC003xxxxxx/PMC3987736.xml\n",
            "PMC003xxxxxx/PMC3987737.xml\n",
            "PMC003xxxxxx/PMC3987738.xml\n",
            "PMC003xxxxxx/PMC3987739.xml\n",
            "PMC003xxxxxx/PMC3987742.xml\n",
            "PMC003xxxxxx/PMC3988120.xml\n",
            "PMC003xxxxxx/PMC3988281.xml\n",
            "PMC003xxxxxx/PMC3988282.xml\n",
            "PMC003xxxxxx/PMC3988283.xml\n",
            "PMC003xxxxxx/PMC3988284.xml\n",
            "PMC003xxxxxx/PMC3988285.xml\n",
            "PMC003xxxxxx/PMC3988286.xml\n",
            "PMC003xxxxxx/PMC3988287.xml\n",
            "PMC003xxxxxx/PMC3988318.xml\n",
            "PMC003xxxxxx/PMC3988319.xml\n",
            "PMC003xxxxxx/PMC3988337.xml\n",
            "PMC003xxxxxx/PMC3988338.xml\n",
            "PMC003xxxxxx/PMC3988339.xml\n",
            "PMC003xxxxxx/PMC3988340.xml\n",
            "PMC003xxxxxx/PMC3988341.xml\n",
            "PMC003xxxxxx/PMC3988342.xml\n",
            "PMC003xxxxxx/PMC3988343.xml\n",
            "PMC003xxxxxx/PMC3988344.xml\n",
            "PMC003xxxxxx/PMC3988345.xml\n",
            "PMC003xxxxxx/PMC3988346.xml\n",
            "PMC003xxxxxx/PMC3988347.xml\n",
            "PMC003xxxxxx/PMC3988348.xml\n",
            "PMC003xxxxxx/PMC3988349.xml\n",
            "PMC003xxxxxx/PMC3988350.xml\n",
            "PMC003xxxxxx/PMC3988431.xml\n",
            "PMC003xxxxxx/PMC3988432.xml\n",
            "PMC003xxxxxx/PMC3988433.xml\n",
            "PMC003xxxxxx/PMC3988434.xml\n",
            "PMC003xxxxxx/PMC3988435.xml\n",
            "PMC003xxxxxx/PMC3988436.xml\n",
            "PMC003xxxxxx/PMC3988437.xml\n",
            "PMC003xxxxxx/PMC3988438.xml\n",
            "PMC003xxxxxx/PMC3988439.xml\n",
            "PMC003xxxxxx/PMC3988440.xml\n",
            "PMC003xxxxxx/PMC3988441.xml\n",
            "PMC003xxxxxx/PMC3988442.xml\n",
            "PMC003xxxxxx/PMC3988443.xml\n",
            "PMC003xxxxxx/PMC3988477.xml\n",
            "PMC003xxxxxx/PMC3988479.xml\n",
            "PMC003xxxxxx/PMC3988480.xml\n",
            "PMC003xxxxxx/PMC3988481.xml\n",
            "PMC003xxxxxx/PMC3988482.xml\n",
            "PMC003xxxxxx/PMC3988483.xml\n",
            "PMC003xxxxxx/PMC3988484.xml\n",
            "PMC003xxxxxx/PMC3988485.xml\n",
            "PMC003xxxxxx/PMC3988487.xml\n",
            "PMC003xxxxxx/PMC3988493.xml\n",
            "PMC003xxxxxx/PMC3988494.xml\n",
            "PMC003xxxxxx/PMC3988499.xml\n",
            "PMC003xxxxxx/PMC3988500.xml\n",
            "PMC003xxxxxx/PMC3988501.xml\n",
            "PMC003xxxxxx/PMC3988502.xml\n",
            "PMC003xxxxxx/PMC3988503.xml\n",
            "PMC003xxxxxx/PMC3988504.xml\n",
            "PMC003xxxxxx/PMC3988505.xml\n",
            "PMC003xxxxxx/PMC3988506.xml\n",
            "PMC003xxxxxx/PMC3988507.xml\n",
            "PMC003xxxxxx/PMC3988508.xml\n",
            "PMC003xxxxxx/PMC3988509.xml\n",
            "PMC003xxxxxx/PMC3988510.xml\n",
            "PMC003xxxxxx/PMC3988511.xml\n",
            "PMC003xxxxxx/PMC3988512.xml\n",
            "PMC003xxxxxx/PMC3988513.xml\n",
            "PMC003xxxxxx/PMC3988515.xml\n",
            "PMC003xxxxxx/PMC3988516.xml\n",
            "PMC003xxxxxx/PMC3988517.xml\n",
            "PMC003xxxxxx/PMC3988520.xml\n",
            "PMC003xxxxxx/PMC3988539.xml\n",
            "PMC003xxxxxx/PMC3988561.xml\n",
            "PMC003xxxxxx/PMC3988562.xml\n",
            "PMC003xxxxxx/PMC3988564.xml\n",
            "PMC003xxxxxx/PMC3988565.xml\n",
            "PMC003xxxxxx/PMC3988566.xml\n",
            "PMC003xxxxxx/PMC3988567.xml\n",
            "PMC003xxxxxx/PMC3988568.xml\n",
            "PMC003xxxxxx/PMC3988569.xml\n",
            "PMC003xxxxxx/PMC3988570.xml\n",
            "PMC003xxxxxx/PMC3988571.xml\n",
            "PMC003xxxxxx/PMC3988572.xml\n",
            "PMC003xxxxxx/PMC3988574.xml\n",
            "PMC003xxxxxx/PMC3988586.xml\n",
            "PMC003xxxxxx/PMC3988587.xml\n",
            "PMC003xxxxxx/PMC3988588.xml\n",
            "PMC003xxxxxx/PMC3988589.xml\n",
            "PMC003xxxxxx/PMC3988590.xml\n",
            "PMC003xxxxxx/PMC3988591.xml\n",
            "PMC003xxxxxx/PMC3988592.xml\n",
            "PMC003xxxxxx/PMC3988593.xml\n",
            "PMC003xxxxxx/PMC3988594.xml\n",
            "PMC003xxxxxx/PMC3988595.xml\n",
            "PMC003xxxxxx/PMC3988596.xml\n",
            "PMC003xxxxxx/PMC3988597.xml\n",
            "PMC003xxxxxx/PMC3988598.xml\n",
            "PMC003xxxxxx/PMC3988599.xml\n",
            "PMC003xxxxxx/PMC3988600.xml\n",
            "PMC003xxxxxx/PMC3988601.xml\n",
            "PMC003xxxxxx/PMC3988602.xml\n",
            "PMC003xxxxxx/PMC3988603.xml\n",
            "PMC003xxxxxx/PMC3988604.xml\n",
            "PMC003xxxxxx/PMC3988605.xml\n",
            "PMC003xxxxxx/PMC3988606.xml\n",
            "PMC003xxxxxx/PMC3988607.xml\n",
            "PMC003xxxxxx/PMC3988608.xml\n",
            "PMC003xxxxxx/PMC3988609.xml\n",
            "PMC003xxxxxx/PMC3988610.xml\n",
            "PMC003xxxxxx/PMC3988630.xml\n",
            "PMC003xxxxxx/PMC3988631.xml\n",
            "PMC003xxxxxx/PMC3988632.xml\n",
            "PMC003xxxxxx/PMC3988633.xml\n",
            "PMC003xxxxxx/PMC3988634.xml\n",
            "PMC003xxxxxx/PMC3988635.xml\n",
            "PMC003xxxxxx/PMC3988636.xml\n",
            "PMC003xxxxxx/PMC3988637.xml\n",
            "PMC003xxxxxx/PMC3988638.xml\n",
            "PMC003xxxxxx/PMC3988639.xml\n",
            "PMC003xxxxxx/PMC3988640.xml\n",
            "PMC003xxxxxx/PMC3988641.xml\n",
            "PMC003xxxxxx/PMC3988642.xml\n",
            "PMC003xxxxxx/PMC3988643.xml\n",
            "PMC003xxxxxx/PMC3988644.xml\n",
            "PMC003xxxxxx/PMC3988645.xml\n",
            "PMC003xxxxxx/PMC3988646.xml\n",
            "PMC003xxxxxx/PMC3988647.xml\n",
            "PMC003xxxxxx/PMC3988648.xml\n",
            "PMC003xxxxxx/PMC3988649.xml\n",
            "PMC003xxxxxx/PMC3988650.xml\n",
            "PMC003xxxxxx/PMC3988651.xml\n",
            "PMC003xxxxxx/PMC3988652.xml\n",
            "PMC003xxxxxx/PMC3988653.xml\n",
            "PMC003xxxxxx/PMC3988654.xml\n",
            "PMC003xxxxxx/PMC3988655.xml\n",
            "PMC003xxxxxx/PMC3988656.xml\n",
            "PMC003xxxxxx/PMC3988657.xml\n",
            "PMC003xxxxxx/PMC3988801.xml\n",
            "PMC003xxxxxx/PMC3988804.xml\n",
            "PMC003xxxxxx/PMC3988807.xml\n",
            "PMC003xxxxxx/PMC3988808.xml\n",
            "PMC003xxxxxx/PMC3988809.xml\n",
            "PMC003xxxxxx/PMC3988811.xml\n",
            "PMC003xxxxxx/PMC3988812.xml\n",
            "PMC003xxxxxx/PMC3988815.xml\n",
            "PMC003xxxxxx/PMC3988816.xml\n",
            "PMC003xxxxxx/PMC3988820.xml\n",
            "PMC003xxxxxx/PMC3988821.xml\n",
            "PMC003xxxxxx/PMC3988822.xml\n",
            "PMC003xxxxxx/PMC3988824.xml\n",
            "PMC003xxxxxx/PMC3988826.xml\n",
            "PMC003xxxxxx/PMC3988837.xml\n",
            "PMC003xxxxxx/PMC3988840.xml\n",
            "PMC003xxxxxx/PMC3988842.xml\n",
            "PMC003xxxxxx/PMC3988847.xml\n",
            "PMC003xxxxxx/PMC3988849.xml\n",
            "PMC003xxxxxx/PMC3988850.xml\n",
            "PMC003xxxxxx/PMC3988851.xml\n",
            "PMC003xxxxxx/PMC3988869.xml\n",
            "PMC003xxxxxx/PMC3988870.xml\n",
            "PMC003xxxxxx/PMC3988871.xml\n",
            "PMC003xxxxxx/PMC3988872.xml\n",
            "PMC003xxxxxx/PMC3988883.xml\n",
            "PMC003xxxxxx/PMC3988891.xml\n",
            "PMC003xxxxxx/PMC3988922.xml\n",
            "PMC003xxxxxx/PMC3988923.xml\n",
            "PMC003xxxxxx/PMC3988925.xml\n",
            "PMC003xxxxxx/PMC3988926.xml\n",
            "PMC003xxxxxx/PMC3988927.xml\n",
            "PMC003xxxxxx/PMC3988933.xml\n",
            "PMC003xxxxxx/PMC3988949.xml\n",
            "PMC003xxxxxx/PMC3988952.xml\n",
            "PMC003xxxxxx/PMC3988953.xml\n",
            "PMC003xxxxxx/PMC3988954.xml\n",
            "PMC003xxxxxx/PMC3988956.xml\n",
            "PMC003xxxxxx/PMC3988961.xml\n",
            "PMC003xxxxxx/PMC3988965.xml\n",
            "PMC003xxxxxx/PMC3988984.xml\n",
            "PMC003xxxxxx/PMC3988985.xml\n",
            "PMC003xxxxxx/PMC3988991.xml\n",
            "PMC003xxxxxx/PMC3988992.xml\n",
            "PMC003xxxxxx/PMC3988997.xml\n",
            "PMC003xxxxxx/PMC3988999.xml\n",
            "PMC003xxxxxx/PMC3989022.xml\n",
            "PMC003xxxxxx/PMC3989024.xml\n",
            "PMC003xxxxxx/PMC3989025.xml\n",
            "PMC003xxxxxx/PMC3989026.xml\n",
            "PMC003xxxxxx/PMC3989027.xml\n",
            "PMC003xxxxxx/PMC3989029.xml\n",
            "PMC003xxxxxx/PMC3989031.xml\n",
            "PMC003xxxxxx/PMC3989038.xml\n",
            "PMC003xxxxxx/PMC3989040.xml\n",
            "PMC003xxxxxx/PMC3989049.xml\n",
            "PMC003xxxxxx/PMC3989050.xml\n",
            "PMC003xxxxxx/PMC3989054.xml\n",
            "PMC003xxxxxx/PMC3989055.xml\n",
            "PMC003xxxxxx/PMC3989059.xml\n",
            "PMC003xxxxxx/PMC3989064.xml\n",
            "PMC003xxxxxx/PMC3989067.xml\n",
            "PMC003xxxxxx/PMC3989115.xml\n",
            "PMC003xxxxxx/PMC3989117.xml\n",
            "PMC003xxxxxx/PMC3989119.xml\n",
            "PMC003xxxxxx/PMC3989131.xml\n",
            "PMC003xxxxxx/PMC3989296.xml\n",
            "PMC003xxxxxx/PMC3989350.xml\n",
            "PMC003xxxxxx/PMC3989353.xml\n",
            "PMC003xxxxxx/PMC3989361.xml\n",
            "PMC003xxxxxx/PMC3989481.xml\n",
            "PMC003xxxxxx/PMC3989486.xml\n",
            "PMC003xxxxxx/PMC3989488.xml\n",
            "PMC003xxxxxx/PMC3989522.xml\n",
            "PMC003xxxxxx/PMC3989525.xml\n",
            "PMC003xxxxxx/PMC3989526.xml\n",
            "PMC003xxxxxx/PMC3989553.xml\n",
            "PMC003xxxxxx/PMC3989554.xml\n",
            "PMC003xxxxxx/PMC3989555.xml\n",
            "PMC003xxxxxx/PMC3989558.xml\n",
            "PMC003xxxxxx/PMC3989560.xml\n",
            "PMC003xxxxxx/PMC3989654.xml\n",
            "PMC003xxxxxx/PMC3989669.xml\n",
            "PMC003xxxxxx/PMC3989695.xml\n",
            "PMC003xxxxxx/PMC3989777.xml\n",
            "PMC003xxxxxx/PMC3989810.xml\n",
            "PMC003xxxxxx/PMC3989811.xml\n",
            "PMC003xxxxxx/PMC3989812.xml\n",
            "PMC003xxxxxx/PMC3989813.xml\n",
            "PMC003xxxxxx/PMC3989814.xml\n",
            "PMC003xxxxxx/PMC3989815.xml\n",
            "PMC003xxxxxx/PMC3989816.xml\n",
            "PMC003xxxxxx/PMC3989817.xml\n",
            "PMC003xxxxxx/PMC3989818.xml\n",
            "PMC003xxxxxx/PMC3989819.xml\n",
            "PMC003xxxxxx/PMC3989820.xml\n",
            "PMC003xxxxxx/PMC3989821.xml\n",
            "PMC003xxxxxx/PMC3989822.xml\n",
            "PMC003xxxxxx/PMC3989823.xml\n",
            "PMC003xxxxxx/PMC3989824.xml\n",
            "PMC003xxxxxx/PMC3989825.xml\n",
            "PMC003xxxxxx/PMC3989826.xml\n",
            "PMC003xxxxxx/PMC3989827.xml\n",
            "PMC003xxxxxx/PMC3989828.xml\n",
            "PMC003xxxxxx/PMC3989829.xml\n",
            "PMC003xxxxxx/PMC3989830.xml\n",
            "PMC003xxxxxx/PMC3989831.xml\n",
            "PMC003xxxxxx/PMC3989832.xml\n",
            "PMC003xxxxxx/PMC3989833.xml\n",
            "PMC003xxxxxx/PMC3989834.xml\n",
            "PMC003xxxxxx/PMC3989851.xml\n",
            "PMC003xxxxxx/PMC3989863.xml\n",
            "PMC003xxxxxx/PMC3989905.xml\n",
            "PMC003xxxxxx/PMC3989906.xml\n",
            "PMC003xxxxxx/PMC3989907.xml\n",
            "PMC003xxxxxx/PMC3989908.xml\n",
            "PMC003xxxxxx/PMC3989909.xml\n",
            "PMC003xxxxxx/PMC3989910.xml\n",
            "PMC003xxxxxx/PMC3989911.xml\n",
            "PMC003xxxxxx/PMC3989912.xml\n",
            "PMC003xxxxxx/PMC3989913.xml\n",
            "PMC003xxxxxx/PMC3989914.xml\n",
            "PMC003xxxxxx/PMC3989915.xml\n",
            "PMC003xxxxxx/PMC3989916.xml\n",
            "PMC003xxxxxx/PMC3989917.xml\n",
            "PMC003xxxxxx/PMC3989918.xml\n",
            "PMC003xxxxxx/PMC3989919.xml\n",
            "PMC003xxxxxx/PMC3989920.xml\n",
            "PMC003xxxxxx/PMC3989921.xml\n",
            "PMC003xxxxxx/PMC3989922.xml\n",
            "PMC003xxxxxx/PMC3989923.xml\n",
            "PMC003xxxxxx/PMC3989937.xml\n",
            "PMC003xxxxxx/PMC3989988.xml\n",
            "PMC003xxxxxx/PMC3989994.xml\n",
            "PMC003xxxxxx/PMC3990060.xml\n",
            "PMC003xxxxxx/PMC3990061.xml\n",
            "PMC003xxxxxx/PMC3990062.xml\n",
            "PMC003xxxxxx/PMC3990063.xml\n",
            "PMC003xxxxxx/PMC3990064.xml\n",
            "PMC003xxxxxx/PMC3990065.xml\n",
            "PMC003xxxxxx/PMC3990066.xml\n",
            "PMC003xxxxxx/PMC3990067.xml\n",
            "PMC003xxxxxx/PMC3990068.xml\n",
            "PMC003xxxxxx/PMC3990069.xml\n",
            "PMC003xxxxxx/PMC3990070.xml\n",
            "PMC003xxxxxx/PMC3990071.xml\n",
            "PMC003xxxxxx/PMC3990072.xml\n",
            "PMC003xxxxxx/PMC3990073.xml\n",
            "PMC003xxxxxx/PMC3990074.xml\n",
            "PMC003xxxxxx/PMC3990075.xml\n",
            "PMC003xxxxxx/PMC3990076.xml\n",
            "PMC003xxxxxx/PMC3990077.xml\n",
            "PMC003xxxxxx/PMC3990078.xml\n",
            "PMC003xxxxxx/PMC3990079.xml\n",
            "PMC003xxxxxx/PMC3990080.xml\n",
            "PMC003xxxxxx/PMC3990081.xml\n",
            "PMC003xxxxxx/PMC3990082.xml\n",
            "PMC003xxxxxx/PMC3990083.xml\n",
            "PMC003xxxxxx/PMC3990084.xml\n",
            "PMC003xxxxxx/PMC3990085.xml\n",
            "PMC003xxxxxx/PMC3990086.xml\n",
            "PMC003xxxxxx/PMC3990087.xml\n",
            "PMC003xxxxxx/PMC3990088.xml\n",
            "PMC003xxxxxx/PMC3990089.xml\n",
            "PMC003xxxxxx/PMC3990090.xml\n",
            "PMC003xxxxxx/PMC3990091.xml\n",
            "PMC003xxxxxx/PMC3990092.xml\n",
            "PMC003xxxxxx/PMC3990093.xml\n",
            "PMC003xxxxxx/PMC3990094.xml\n",
            "PMC003xxxxxx/PMC3990095.xml\n",
            "PMC003xxxxxx/PMC3990096.xml\n",
            "PMC003xxxxxx/PMC3990097.xml\n",
            "PMC003xxxxxx/PMC3990098.xml\n",
            "PMC003xxxxxx/PMC3990114.xml\n",
            "PMC003xxxxxx/PMC3990115.xml\n",
            "PMC003xxxxxx/PMC3990116.xml\n",
            "PMC003xxxxxx/PMC3990117.xml\n",
            "PMC003xxxxxx/PMC3990118.xml\n",
            "PMC003xxxxxx/PMC3990119.xml\n",
            "PMC003xxxxxx/PMC3990120.xml\n",
            "PMC003xxxxxx/PMC3990121.xml\n",
            "PMC003xxxxxx/PMC3990122.xml\n",
            "PMC003xxxxxx/PMC3990123.xml\n",
            "PMC003xxxxxx/PMC3990124.xml\n",
            "PMC003xxxxxx/PMC3990125.xml\n",
            "PMC003xxxxxx/PMC3990126.xml\n",
            "PMC003xxxxxx/PMC3990127.xml\n",
            "PMC003xxxxxx/PMC3990128.xml\n",
            "PMC003xxxxxx/PMC3990129.xml\n",
            "PMC003xxxxxx/PMC3990130.xml\n",
            "PMC003xxxxxx/PMC3990131.xml\n",
            "PMC003xxxxxx/PMC3990132.xml\n",
            "PMC003xxxxxx/PMC3990133.xml\n",
            "PMC003xxxxxx/PMC3990134.xml\n",
            "PMC003xxxxxx/PMC3990135.xml\n",
            "PMC003xxxxxx/PMC3990136.xml\n",
            "PMC003xxxxxx/PMC3990137.xml\n",
            "PMC003xxxxxx/PMC3990138.xml\n",
            "PMC003xxxxxx/PMC3990139.xml\n",
            "PMC003xxxxxx/PMC3990140.xml\n",
            "PMC003xxxxxx/PMC3990141.xml\n",
            "PMC003xxxxxx/PMC3990142.xml\n",
            "PMC003xxxxxx/PMC3990143.xml\n",
            "PMC003xxxxxx/PMC3990144.xml\n",
            "PMC003xxxxxx/PMC3990145.xml\n",
            "PMC003xxxxxx/PMC3990146.xml\n",
            "PMC003xxxxxx/PMC3990147.xml\n",
            "PMC003xxxxxx/PMC3990148.xml\n",
            "PMC003xxxxxx/PMC3990149.xml\n",
            "PMC003xxxxxx/PMC3990150.xml\n",
            "PMC003xxxxxx/PMC3990151.xml\n",
            "PMC003xxxxxx/PMC3990152.xml\n",
            "PMC003xxxxxx/PMC3990153.xml\n",
            "PMC003xxxxxx/PMC3990154.xml\n",
            "PMC003xxxxxx/PMC3990175.xml\n",
            "PMC003xxxxxx/PMC3990176.xml\n",
            "PMC003xxxxxx/PMC3990177.xml\n",
            "PMC003xxxxxx/PMC3990178.xml\n",
            "PMC003xxxxxx/PMC3990179.xml\n",
            "PMC003xxxxxx/PMC3990180.xml\n",
            "PMC003xxxxxx/PMC3990181.xml\n",
            "PMC003xxxxxx/PMC3990182.xml\n",
            "PMC003xxxxxx/PMC3990183.xml\n",
            "PMC003xxxxxx/PMC3990186.xml\n",
            "PMC003xxxxxx/PMC3990238.xml\n",
            "PMC003xxxxxx/PMC3990239.xml\n",
            "PMC003xxxxxx/PMC3990287.xml\n",
            "PMC003xxxxxx/PMC3990288.xml\n",
            "PMC003xxxxxx/PMC3990289.xml\n",
            "PMC003xxxxxx/PMC3990290.xml\n",
            "PMC003xxxxxx/PMC3990362.xml\n",
            "PMC003xxxxxx/PMC3990363.xml\n",
            "PMC003xxxxxx/PMC3990364.xml\n",
            "PMC003xxxxxx/PMC3990365.xml\n",
            "PMC003xxxxxx/PMC3990366.xml\n",
            "PMC003xxxxxx/PMC3990367.xml\n",
            "PMC003xxxxxx/PMC3990369.xml\n",
            "PMC003xxxxxx/PMC3990370.xml\n",
            "PMC003xxxxxx/PMC3990371.xml\n",
            "PMC003xxxxxx/PMC3990372.xml\n",
            "PMC003xxxxxx/PMC3990373.xml\n",
            "PMC003xxxxxx/PMC3990374.xml\n",
            "PMC003xxxxxx/PMC3990375.xml\n",
            "PMC003xxxxxx/PMC3990376.xml\n",
            "PMC003xxxxxx/PMC3990377.xml\n",
            "PMC003xxxxxx/PMC3990378.xml\n",
            "PMC003xxxxxx/PMC3990379.xml\n",
            "PMC003xxxxxx/PMC3990380.xml\n",
            "PMC003xxxxxx/PMC3990381.xml\n",
            "PMC003xxxxxx/PMC3990382.xml\n",
            "PMC003xxxxxx/PMC3990383.xml\n",
            "PMC003xxxxxx/PMC3990384.xml\n",
            "PMC003xxxxxx/PMC3990387.xml\n",
            "PMC003xxxxxx/PMC3990388.xml\n",
            "PMC003xxxxxx/PMC3990389.xml\n",
            "PMC003xxxxxx/PMC3990390.xml\n",
            "PMC003xxxxxx/PMC3990391.xml\n",
            "PMC003xxxxxx/PMC3990392.xml\n",
            "PMC003xxxxxx/PMC3990393.xml\n",
            "PMC003xxxxxx/PMC3990394.xml\n",
            "PMC003xxxxxx/PMC3990395.xml\n",
            "PMC003xxxxxx/PMC3990396.xml\n",
            "PMC003xxxxxx/PMC3990397.xml\n",
            "PMC003xxxxxx/PMC3990398.xml\n",
            "PMC003xxxxxx/PMC3990399.xml\n",
            "PMC003xxxxxx/PMC3990400.xml\n",
            "PMC003xxxxxx/PMC3990401.xml\n",
            "PMC003xxxxxx/PMC3990402.xml\n",
            "PMC003xxxxxx/PMC3990416.xml\n",
            "PMC003xxxxxx/PMC3990422.xml\n",
            "PMC003xxxxxx/PMC3990424.xml\n",
            "PMC003xxxxxx/PMC3990425.xml\n",
            "PMC003xxxxxx/PMC3990427.xml\n",
            "PMC003xxxxxx/PMC3990443.xml\n",
            "PMC003xxxxxx/PMC3990449.xml\n",
            "PMC003xxxxxx/PMC3990450.xml\n",
            "PMC003xxxxxx/PMC3990454.xml\n",
            "PMC003xxxxxx/PMC3990463.xml\n",
            "PMC003xxxxxx/PMC3990464.xml\n",
            "PMC003xxxxxx/PMC3990465.xml\n",
            "PMC003xxxxxx/PMC3990504.xml\n",
            "PMC003xxxxxx/PMC3990505.xml\n",
            "PMC003xxxxxx/PMC3990506.xml\n",
            "PMC003xxxxxx/PMC3990594.xml\n",
            "PMC003xxxxxx/PMC3990595.xml\n",
            "PMC003xxxxxx/PMC3990760.xml\n",
            "PMC003xxxxxx/PMC3990761.xml\n",
            "PMC003xxxxxx/PMC3990762.xml\n",
            "PMC003xxxxxx/PMC3990763.xml\n",
            "PMC003xxxxxx/PMC3990764.xml\n",
            "PMC003xxxxxx/PMC3990765.xml\n",
            "PMC003xxxxxx/PMC3990777.xml\n",
            "PMC003xxxxxx/PMC3990778.xml\n",
            "PMC003xxxxxx/PMC3990779.xml\n",
            "PMC003xxxxxx/PMC3990780.xml\n",
            "PMC003xxxxxx/PMC3990781.xml\n",
            "PMC003xxxxxx/PMC3990782.xml\n",
            "PMC003xxxxxx/PMC3990783.xml\n",
            "PMC003xxxxxx/PMC3990784.xml\n",
            "PMC003xxxxxx/PMC3990785.xml\n",
            "PMC003xxxxxx/PMC3990786.xml\n",
            "PMC003xxxxxx/PMC3990816.xml\n",
            "PMC003xxxxxx/PMC3990817.xml\n",
            "PMC003xxxxxx/PMC3990818.xml\n",
            "PMC003xxxxxx/PMC3990819.xml\n",
            "PMC003xxxxxx/PMC3990820.xml\n",
            "PMC003xxxxxx/PMC3990821.xml\n",
            "PMC003xxxxxx/PMC3990822.xml\n",
            "PMC003xxxxxx/PMC3990823.xml\n",
            "PMC003xxxxxx/PMC3990824.xml\n",
            "PMC003xxxxxx/PMC3990825.xml\n",
            "PMC003xxxxxx/PMC3990826.xml\n",
            "PMC003xxxxxx/PMC3990827.xml\n",
            "PMC003xxxxxx/PMC3990828.xml\n",
            "PMC003xxxxxx/PMC3990829.xml\n",
            "PMC003xxxxxx/PMC3990832.xml\n",
            "PMC003xxxxxx/PMC3990835.xml\n",
            "PMC003xxxxxx/PMC3990836.xml\n",
            "PMC003xxxxxx/PMC3990855.xml\n",
            "PMC003xxxxxx/PMC3990856.xml\n",
            "PMC003xxxxxx/PMC3990916.xml\n",
            "PMC003xxxxxx/PMC3990917.xml\n",
            "PMC003xxxxxx/PMC3990918.xml\n",
            "PMC003xxxxxx/PMC3990919.xml\n",
            "PMC003xxxxxx/PMC3990920.xml\n",
            "PMC003xxxxxx/PMC3990921.xml\n",
            "PMC003xxxxxx/PMC3990922.xml\n",
            "PMC003xxxxxx/PMC3990923.xml\n",
            "PMC003xxxxxx/PMC3990924.xml\n",
            "PMC003xxxxxx/PMC3990925.xml\n",
            "PMC003xxxxxx/PMC3990926.xml\n",
            "PMC003xxxxxx/PMC3990927.xml\n",
            "PMC003xxxxxx/PMC3990940.xml\n",
            "PMC003xxxxxx/PMC3991139.xml\n",
            "PMC003xxxxxx/PMC3991140.xml\n",
            "PMC003xxxxxx/PMC3991141.xml\n",
            "PMC003xxxxxx/PMC3991145.xml\n",
            "PMC003xxxxxx/PMC3991146.xml\n",
            "PMC003xxxxxx/PMC3991303.xml\n",
            "PMC003xxxxxx/PMC3991320.xml\n",
            "PMC003xxxxxx/PMC3991322.xml\n",
            "PMC003xxxxxx/PMC3991327.xml\n",
            "PMC003xxxxxx/PMC3991336.xml\n",
            "PMC003xxxxxx/PMC3991337.xml\n",
            "PMC003xxxxxx/PMC3991351.xml\n",
            "PMC003xxxxxx/PMC3991373.xml\n",
            "PMC003xxxxxx/PMC3991391.xml\n",
            "PMC003xxxxxx/PMC3991397.xml\n",
            "PMC003xxxxxx/PMC3991398.xml\n",
            "PMC003xxxxxx/PMC3991399.xml\n",
            "PMC003xxxxxx/PMC3991400.xml\n",
            "PMC003xxxxxx/PMC3991401.xml\n",
            "PMC003xxxxxx/PMC3991402.xml\n",
            "PMC003xxxxxx/PMC3991403.xml\n",
            "PMC003xxxxxx/PMC3991404.xml\n",
            "PMC003xxxxxx/PMC3991405.xml\n",
            "PMC003xxxxxx/PMC3991406.xml\n",
            "PMC003xxxxxx/PMC3991407.xml\n",
            "PMC003xxxxxx/PMC3991408.xml\n",
            "PMC003xxxxxx/PMC3991409.xml\n",
            "PMC003xxxxxx/PMC3991410.xml\n",
            "PMC003xxxxxx/PMC3991411.xml\n",
            "PMC003xxxxxx/PMC3991412.xml\n",
            "PMC003xxxxxx/PMC3991413.xml\n",
            "PMC003xxxxxx/PMC3991414.xml\n",
            "PMC003xxxxxx/PMC3991415.xml\n",
            "PMC003xxxxxx/PMC3991419.xml\n",
            "PMC003xxxxxx/PMC3991437.xml\n",
            "PMC003xxxxxx/PMC3991438.xml\n",
            "PMC003xxxxxx/PMC3991439.xml\n",
            "PMC003xxxxxx/PMC3991440.xml\n",
            "PMC003xxxxxx/PMC3991441.xml\n",
            "PMC003xxxxxx/PMC3991442.xml\n",
            "PMC003xxxxxx/PMC3991443.xml\n",
            "PMC003xxxxxx/PMC3991444.xml\n",
            "PMC003xxxxxx/PMC3991445.xml\n",
            "PMC003xxxxxx/PMC3991446.xml\n",
            "PMC003xxxxxx/PMC3991447.xml\n",
            "PMC003xxxxxx/PMC3991448.xml\n",
            "PMC003xxxxxx/PMC3991449.xml\n",
            "PMC003xxxxxx/PMC3991450.xml\n",
            "PMC003xxxxxx/PMC3991451.xml\n",
            "PMC003xxxxxx/PMC3991452.xml\n",
            "PMC003xxxxxx/PMC3991453.xml\n",
            "PMC003xxxxxx/PMC3991454.xml\n",
            "PMC003xxxxxx/PMC3991455.xml\n",
            "PMC003xxxxxx/PMC3991456.xml\n",
            "PMC003xxxxxx/PMC3991457.xml\n",
            "PMC003xxxxxx/PMC3991458.xml\n",
            "PMC003xxxxxx/PMC3991459.xml\n",
            "PMC003xxxxxx/PMC3991468.xml\n",
            "PMC003xxxxxx/PMC3991473.xml\n",
            "PMC003xxxxxx/PMC3991552.xml\n",
            "PMC003xxxxxx/PMC3991787.xml\n",
            "PMC003xxxxxx/PMC3991788.xml\n",
            "PMC003xxxxxx/PMC3991789.xml\n",
            "PMC003xxxxxx/PMC3991790.xml\n",
            "PMC003xxxxxx/PMC3991791.xml\n",
            "PMC003xxxxxx/PMC3991792.xml\n",
            "PMC003xxxxxx/PMC3991793.xml\n",
            "PMC003xxxxxx/PMC3991794.xml\n",
            "PMC003xxxxxx/PMC3991795.xml\n",
            "PMC003xxxxxx/PMC3991796.xml\n",
            "PMC003xxxxxx/PMC3991797.xml\n",
            "PMC003xxxxxx/PMC3991798.xml\n",
            "PMC003xxxxxx/PMC3991799.xml\n",
            "PMC003xxxxxx/PMC3991800.xml\n",
            "PMC003xxxxxx/PMC3991801.xml\n",
            "PMC003xxxxxx/PMC3991802.xml\n",
            "PMC003xxxxxx/PMC3991803.xml\n",
            "PMC003xxxxxx/PMC3991804.xml\n",
            "PMC003xxxxxx/PMC3991805.xml\n",
            "PMC003xxxxxx/PMC3991806.xml\n",
            "PMC003xxxxxx/PMC3991807.xml\n",
            "PMC003xxxxxx/PMC3991808.xml\n",
            "PMC003xxxxxx/PMC3991809.xml\n",
            "PMC003xxxxxx/PMC3991810.xml\n",
            "PMC003xxxxxx/PMC3991811.xml\n",
            "PMC003xxxxxx/PMC3991812.xml\n",
            "PMC003xxxxxx/PMC3991813.xml\n",
            "PMC003xxxxxx/PMC3991828.xml\n",
            "PMC003xxxxxx/PMC3991829.xml\n",
            "PMC003xxxxxx/PMC3991834.xml\n",
            "PMC003xxxxxx/PMC3991838.xml\n",
            "PMC003xxxxxx/PMC3991839.xml\n",
            "PMC003xxxxxx/PMC3991843.xml\n",
            "PMC003xxxxxx/PMC3991845.xml\n",
            "PMC003xxxxxx/PMC3991849.xml\n",
            "PMC003xxxxxx/PMC3991850.xml\n",
            "PMC003xxxxxx/PMC3991853.xml\n",
            "PMC003xxxxxx/PMC3991854.xml\n",
            "PMC003xxxxxx/PMC3991856.xml\n",
            "PMC003xxxxxx/PMC3991932.xml\n",
            "PMC003xxxxxx/PMC3991933.xml\n",
            "PMC003xxxxxx/PMC3991934.xml\n",
            "PMC003xxxxxx/PMC3991935.xml\n",
            "PMC003xxxxxx/PMC3991936.xml\n",
            "PMC003xxxxxx/PMC3991937.xml\n",
            "PMC003xxxxxx/PMC3991938.xml\n",
            "PMC003xxxxxx/PMC3991939.xml\n",
            "PMC003xxxxxx/PMC3991940.xml\n",
            "PMC003xxxxxx/PMC3991941.xml\n",
            "PMC003xxxxxx/PMC3991942.xml\n",
            "PMC003xxxxxx/PMC3991943.xml\n",
            "PMC003xxxxxx/PMC3991944.xml\n",
            "PMC003xxxxxx/PMC3991945.xml\n",
            "PMC003xxxxxx/PMC3991946.xml\n",
            "PMC003xxxxxx/PMC3991947.xml\n",
            "PMC003xxxxxx/PMC3991948.xml\n",
            "PMC003xxxxxx/PMC3991949.xml\n",
            "PMC003xxxxxx/PMC3991950.xml\n",
            "PMC003xxxxxx/PMC3991951.xml\n",
            "PMC003xxxxxx/PMC3991952.xml\n",
            "PMC003xxxxxx/PMC3991953.xml\n",
            "PMC003xxxxxx/PMC3991954.xml\n",
            "PMC003xxxxxx/PMC3991955.xml\n",
            "PMC003xxxxxx/PMC3991956.xml\n",
            "PMC003xxxxxx/PMC3991957.xml\n",
            "PMC003xxxxxx/PMC3991958.xml\n",
            "PMC003xxxxxx/PMC3991959.xml\n",
            "PMC003xxxxxx/PMC3992011.xml\n",
            "PMC003xxxxxx/PMC3992113.xml\n",
            "PMC003xxxxxx/PMC3992324.xml\n",
            "PMC003xxxxxx/PMC3992325.xml\n",
            "PMC003xxxxxx/PMC3992326.xml\n",
            "PMC003xxxxxx/PMC3992327.xml\n",
            "PMC003xxxxxx/PMC3992328.xml\n",
            "PMC003xxxxxx/PMC3992329.xml\n",
            "PMC003xxxxxx/PMC3992330.xml\n",
            "PMC003xxxxxx/PMC3992331.xml\n",
            "PMC003xxxxxx/PMC3992332.xml\n",
            "PMC003xxxxxx/PMC3992333.xml\n",
            "PMC003xxxxxx/PMC3992334.xml\n",
            "PMC003xxxxxx/PMC3992335.xml\n",
            "PMC003xxxxxx/PMC3992336.xml\n",
            "PMC003xxxxxx/PMC3992342.xml\n",
            "PMC003xxxxxx/PMC3992343.xml\n",
            "PMC003xxxxxx/PMC3992344.xml\n",
            "PMC003xxxxxx/PMC3992345.xml\n",
            "PMC003xxxxxx/PMC3992346.xml\n",
            "PMC003xxxxxx/PMC3992347.xml\n",
            "PMC003xxxxxx/PMC3992348.xml\n",
            "PMC003xxxxxx/PMC3992349.xml\n",
            "PMC003xxxxxx/PMC3992350.xml\n",
            "PMC003xxxxxx/PMC3992351.xml\n",
            "PMC003xxxxxx/PMC3992352.xml\n",
            "PMC003xxxxxx/PMC3992353.xml\n",
            "PMC003xxxxxx/PMC3992354.xml\n",
            "PMC003xxxxxx/PMC3992355.xml\n",
            "PMC003xxxxxx/PMC3992356.xml\n",
            "PMC003xxxxxx/PMC3992357.xml\n",
            "PMC003xxxxxx/PMC3992358.xml\n",
            "PMC003xxxxxx/PMC3992359.xml\n",
            "PMC003xxxxxx/PMC3992360.xml\n",
            "PMC003xxxxxx/PMC3992361.xml\n",
            "PMC003xxxxxx/PMC3992362.xml\n",
            "PMC003xxxxxx/PMC3992363.xml\n",
            "PMC003xxxxxx/PMC3992364.xml\n",
            "PMC003xxxxxx/PMC3992427.xml\n",
            "PMC003xxxxxx/PMC3992428.xml\n",
            "PMC003xxxxxx/PMC3992439.xml\n",
            "PMC003xxxxxx/PMC3992440.xml\n",
            "PMC003xxxxxx/PMC3992441.xml\n",
            "PMC003xxxxxx/PMC3992442.xml\n",
            "PMC003xxxxxx/PMC3992443.xml\n",
            "PMC003xxxxxx/PMC3992444.xml\n",
            "PMC003xxxxxx/PMC3992445.xml\n",
            "PMC003xxxxxx/PMC3992446.xml\n",
            "PMC003xxxxxx/PMC3992447.xml\n",
            "PMC003xxxxxx/PMC3992448.xml\n",
            "PMC003xxxxxx/PMC3992449.xml\n",
            "PMC003xxxxxx/PMC3992450.xml\n",
            "PMC003xxxxxx/PMC3992451.xml\n",
            "PMC003xxxxxx/PMC3992452.xml\n",
            "PMC003xxxxxx/PMC3992453.xml\n",
            "PMC003xxxxxx/PMC3992454.xml\n",
            "PMC003xxxxxx/PMC3992455.xml\n",
            "PMC003xxxxxx/PMC3992456.xml\n",
            "PMC003xxxxxx/PMC3992457.xml\n",
            "PMC003xxxxxx/PMC3992458.xml\n",
            "PMC003xxxxxx/PMC3992459.xml\n",
            "PMC003xxxxxx/PMC3992460.xml\n",
            "PMC003xxxxxx/PMC3992461.xml\n",
            "PMC003xxxxxx/PMC3992462.xml\n",
            "PMC003xxxxxx/PMC3992463.xml\n",
            "PMC003xxxxxx/PMC3992464.xml\n",
            "PMC003xxxxxx/PMC3992465.xml\n",
            "PMC003xxxxxx/PMC3992466.xml\n",
            "PMC003xxxxxx/PMC3992467.xml\n",
            "PMC003xxxxxx/PMC3992486.xml\n",
            "PMC003xxxxxx/PMC3992487.xml\n",
            "PMC003xxxxxx/PMC3992488.xml\n",
            "PMC003xxxxxx/PMC3992489.xml\n",
            "PMC003xxxxxx/PMC3992490.xml\n",
            "PMC003xxxxxx/PMC3992491.xml\n",
            "PMC003xxxxxx/PMC3992492.xml\n",
            "PMC003xxxxxx/PMC3992493.xml\n",
            "PMC003xxxxxx/PMC3992494.xml\n",
            "PMC003xxxxxx/PMC3992495.xml\n",
            "PMC003xxxxxx/PMC3992496.xml\n",
            "PMC003xxxxxx/PMC3992497.xml\n",
            "PMC003xxxxxx/PMC3992498.xml\n",
            "PMC003xxxxxx/PMC3992499.xml\n",
            "PMC003xxxxxx/PMC3992500.xml\n",
            "PMC003xxxxxx/PMC3992501.xml\n",
            "PMC003xxxxxx/PMC3992502.xml\n",
            "PMC003xxxxxx/PMC3992503.xml\n",
            "PMC003xxxxxx/PMC3992504.xml\n",
            "PMC003xxxxxx/PMC3992505.xml\n",
            "PMC003xxxxxx/PMC3992506.xml\n",
            "PMC003xxxxxx/PMC3992507.xml\n",
            "PMC003xxxxxx/PMC3992508.xml\n",
            "PMC003xxxxxx/PMC3992509.xml\n",
            "PMC003xxxxxx/PMC3992510.xml\n",
            "PMC003xxxxxx/PMC3992511.xml\n",
            "PMC003xxxxxx/PMC3992512.xml\n",
            "PMC003xxxxxx/PMC3992513.xml\n",
            "PMC003xxxxxx/PMC3992514.xml\n",
            "PMC003xxxxxx/PMC3992515.xml\n",
            "PMC003xxxxxx/PMC3992516.xml\n",
            "PMC003xxxxxx/PMC3992524.xml\n",
            "PMC003xxxxxx/PMC3992571.xml\n",
            "PMC003xxxxxx/PMC3992580.xml\n",
            "PMC003xxxxxx/PMC3992742.xml\n",
            "PMC003xxxxxx/PMC3992743.xml\n",
            "PMC003xxxxxx/PMC3992744.xml\n",
            "PMC003xxxxxx/PMC3992745.xml\n",
            "PMC003xxxxxx/PMC3992746.xml\n",
            "PMC003xxxxxx/PMC3992747.xml\n",
            "PMC003xxxxxx/PMC3992748.xml\n",
            "PMC003xxxxxx/PMC3992749.xml\n",
            "PMC003xxxxxx/PMC3992750.xml\n",
            "PMC003xxxxxx/PMC3992751.xml\n",
            "PMC003xxxxxx/PMC3992752.xml\n",
            "PMC003xxxxxx/PMC3992753.xml\n",
            "PMC003xxxxxx/PMC3992754.xml\n",
            "PMC003xxxxxx/PMC3992755.xml\n",
            "PMC003xxxxxx/PMC3992756.xml\n",
            "PMC003xxxxxx/PMC3992757.xml\n",
            "PMC003xxxxxx/PMC3992758.xml\n",
            "PMC003xxxxxx/PMC3992759.xml\n",
            "PMC003xxxxxx/PMC3992760.xml\n",
            "PMC003xxxxxx/PMC3992761.xml\n",
            "PMC003xxxxxx/PMC3992762.xml\n",
            "PMC003xxxxxx/PMC3992763.xml\n",
            "PMC003xxxxxx/PMC3992764.xml\n",
            "PMC003xxxxxx/PMC3992765.xml\n",
            "PMC003xxxxxx/PMC3992766.xml\n",
            "PMC003xxxxxx/PMC3992767.xml\n",
            "PMC003xxxxxx/PMC3992768.xml\n",
            "PMC003xxxxxx/PMC3992769.xml\n",
            "PMC003xxxxxx/PMC3992770.xml\n",
            "PMC003xxxxxx/PMC3992771.xml\n",
            "PMC003xxxxxx/PMC3992772.xml\n",
            "PMC003xxxxxx/PMC3992773.xml\n",
            "PMC003xxxxxx/PMC3992774.xml\n",
            "PMC003xxxxxx/PMC3992775.xml\n",
            "PMC003xxxxxx/PMC3992776.xml\n",
            "PMC003xxxxxx/PMC3992777.xml\n",
            "PMC003xxxxxx/PMC3992778.xml\n",
            "PMC003xxxxxx/PMC3992779.xml\n",
            "PMC003xxxxxx/PMC3992780.xml\n",
            "PMC003xxxxxx/PMC3992781.xml\n",
            "PMC003xxxxxx/PMC3992782.xml\n",
            "PMC003xxxxxx/PMC3992794.xml\n",
            "PMC003xxxxxx/PMC3992795.xml\n",
            "PMC003xxxxxx/PMC3992796.xml\n",
            "PMC003xxxxxx/PMC3992797.xml\n",
            "PMC003xxxxxx/PMC3992798.xml\n",
            "PMC003xxxxxx/PMC3992799.xml\n",
            "PMC003xxxxxx/PMC3992800.xml\n",
            "PMC003xxxxxx/PMC3992801.xml\n",
            "PMC003xxxxxx/PMC3992802.xml\n",
            "PMC003xxxxxx/PMC3992803.xml\n",
            "PMC003xxxxxx/PMC3992804.xml\n",
            "PMC003xxxxxx/PMC3992805.xml\n",
            "PMC003xxxxxx/PMC3992806.xml\n",
            "PMC003xxxxxx/PMC3992807.xml\n",
            "PMC003xxxxxx/PMC3992808.xml\n",
            "PMC003xxxxxx/PMC3992809.xml\n",
            "PMC003xxxxxx/PMC3992810.xml\n",
            "PMC003xxxxxx/PMC3992811.xml\n",
            "PMC003xxxxxx/PMC3992812.xml\n",
            "PMC003xxxxxx/PMC3992813.xml\n",
            "PMC003xxxxxx/PMC3992850.xml\n",
            "PMC003xxxxxx/PMC3992871.xml\n",
            "PMC003xxxxxx/PMC3992883.xml\n",
            "PMC003xxxxxx/PMC3992902.xml\n",
            "PMC003xxxxxx/PMC3992914.xml\n",
            "PMC003xxxxxx/PMC3992951.xml\n",
            "PMC003xxxxxx/PMC3993850.xml\n",
            "PMC003xxxxxx/PMC3993851.xml\n",
            "PMC003xxxxxx/PMC3993852.xml\n",
            "PMC003xxxxxx/PMC3993853.xml\n",
            "PMC003xxxxxx/PMC3993854.xml\n",
            "PMC003xxxxxx/PMC3993855.xml\n",
            "PMC003xxxxxx/PMC3993856.xml\n",
            "PMC003xxxxxx/PMC3993857.xml\n",
            "PMC003xxxxxx/PMC3993858.xml\n",
            "PMC003xxxxxx/PMC3993859.xml\n",
            "PMC003xxxxxx/PMC3993860.xml\n",
            "PMC003xxxxxx/PMC3993861.xml\n",
            "PMC003xxxxxx/PMC3993862.xml\n",
            "PMC003xxxxxx/PMC3994179.xml\n",
            "PMC003xxxxxx/PMC3994251.xml\n",
            "PMC003xxxxxx/PMC3994252.xml\n",
            "PMC003xxxxxx/PMC3994253.xml\n",
            "PMC003xxxxxx/PMC3994254.xml\n",
            "PMC003xxxxxx/PMC3994255.xml\n",
            "PMC003xxxxxx/PMC3994256.xml\n",
            "PMC003xxxxxx/PMC3994257.xml\n",
            "PMC003xxxxxx/PMC3994258.xml\n",
            "PMC003xxxxxx/PMC3994259.xml\n",
            "PMC003xxxxxx/PMC3994260.xml\n",
            "PMC003xxxxxx/PMC3994261.xml\n",
            "PMC003xxxxxx/PMC3994262.xml\n",
            "PMC003xxxxxx/PMC3994263.xml\n",
            "PMC003xxxxxx/PMC3994264.xml\n",
            "PMC003xxxxxx/PMC3994265.xml\n",
            "PMC003xxxxxx/PMC3994266.xml\n",
            "PMC003xxxxxx/PMC3994296.xml\n",
            "PMC003xxxxxx/PMC3994297.xml\n",
            "PMC003xxxxxx/PMC3994298.xml\n",
            "PMC003xxxxxx/PMC3994299.xml\n",
            "PMC003xxxxxx/PMC3994300.xml\n",
            "PMC003xxxxxx/PMC3994301.xml\n",
            "PMC003xxxxxx/PMC3994302.xml\n",
            "PMC003xxxxxx/PMC3994303.xml\n",
            "PMC003xxxxxx/PMC3994304.xml\n",
            "PMC003xxxxxx/PMC3994305.xml\n",
            "PMC003xxxxxx/PMC3994306.xml\n",
            "PMC003xxxxxx/PMC3994307.xml\n",
            "PMC003xxxxxx/PMC3994308.xml\n",
            "PMC003xxxxxx/PMC3994309.xml\n",
            "PMC003xxxxxx/PMC3994435.xml\n",
            "PMC003xxxxxx/PMC3994437.xml\n",
            "PMC003xxxxxx/PMC3994438.xml\n",
            "PMC003xxxxxx/PMC3994439.xml\n",
            "PMC003xxxxxx/PMC3994440.xml\n",
            "PMC003xxxxxx/PMC3994441.xml\n",
            "PMC003xxxxxx/PMC3994443.xml\n",
            "PMC003xxxxxx/PMC3994444.xml\n",
            "PMC003xxxxxx/PMC3994446.xml\n",
            "PMC003xxxxxx/PMC3994447.xml\n",
            "PMC003xxxxxx/PMC3994497.xml\n",
            "PMC003xxxxxx/PMC3994498.xml\n",
            "PMC003xxxxxx/PMC3994499.xml\n",
            "PMC003xxxxxx/PMC3994500.xml\n",
            "PMC003xxxxxx/PMC3994501.xml\n",
            "PMC003xxxxxx/PMC3994502.xml\n",
            "PMC003xxxxxx/PMC3994503.xml\n",
            "PMC003xxxxxx/PMC3994507.xml\n",
            "PMC003xxxxxx/PMC3994508.xml\n",
            "PMC003xxxxxx/PMC3994509.xml\n",
            "PMC003xxxxxx/PMC3994511.xml\n",
            "PMC003xxxxxx/PMC3994512.xml\n",
            "PMC003xxxxxx/PMC3994513.xml\n",
            "PMC003xxxxxx/PMC3994514.xml\n",
            "PMC003xxxxxx/PMC3994515.xml\n",
            "PMC003xxxxxx/PMC3994517.xml\n",
            "PMC003xxxxxx/PMC3994578.xml\n",
            "PMC003xxxxxx/PMC3994579.xml\n",
            "PMC003xxxxxx/PMC3994580.xml\n",
            "PMC003xxxxxx/PMC3994581.xml\n",
            "PMC003xxxxxx/PMC3994582.xml\n",
            "PMC003xxxxxx/PMC3994583.xml\n",
            "PMC003xxxxxx/PMC3994584.xml\n",
            "PMC003xxxxxx/PMC3994585.xml\n",
            "PMC003xxxxxx/PMC3994586.xml\n",
            "PMC003xxxxxx/PMC3994587.xml\n",
            "PMC003xxxxxx/PMC3994588.xml\n",
            "PMC003xxxxxx/PMC3994589.xml\n",
            "PMC003xxxxxx/PMC3994596.xml\n",
            "PMC003xxxxxx/PMC3994597.xml\n",
            "PMC003xxxxxx/PMC3994598.xml\n",
            "PMC003xxxxxx/PMC3994599.xml\n",
            "PMC003xxxxxx/PMC3994600.xml\n",
            "PMC003xxxxxx/PMC3994601.xml\n",
            "PMC003xxxxxx/PMC3994602.xml\n",
            "PMC003xxxxxx/PMC3994603.xml\n",
            "PMC003xxxxxx/PMC3994604.xml\n",
            "PMC003xxxxxx/PMC3994605.xml\n",
            "PMC003xxxxxx/PMC3994606.xml\n",
            "PMC003xxxxxx/PMC3994607.xml\n",
            "PMC003xxxxxx/PMC3994608.xml\n",
            "PMC003xxxxxx/PMC3994609.xml\n",
            "PMC003xxxxxx/PMC3994610.xml\n",
            "PMC003xxxxxx/PMC3994611.xml\n",
            "PMC003xxxxxx/PMC3994612.xml\n",
            "PMC003xxxxxx/PMC3994613.xml\n",
            "PMC003xxxxxx/PMC3994618.xml\n",
            "PMC003xxxxxx/PMC3994619.xml\n",
            "PMC003xxxxxx/PMC3994620.xml\n",
            "PMC003xxxxxx/PMC3994621.xml\n",
            "PMC003xxxxxx/PMC3994622.xml\n",
            "PMC003xxxxxx/PMC3994623.xml\n",
            "PMC003xxxxxx/PMC3994624.xml\n",
            "PMC003xxxxxx/PMC3994625.xml\n",
            "PMC003xxxxxx/PMC3994626.xml\n",
            "PMC003xxxxxx/PMC3994635.xml\n",
            "PMC003xxxxxx/PMC3994681.xml\n",
            "PMC003xxxxxx/PMC3994682.xml\n",
            "PMC003xxxxxx/PMC3994683.xml\n",
            "PMC003xxxxxx/PMC3994684.xml\n",
            "PMC003xxxxxx/PMC3994685.xml\n",
            "PMC003xxxxxx/PMC3994686.xml\n",
            "PMC003xxxxxx/PMC3994687.xml\n",
            "PMC003xxxxxx/PMC3994688.xml\n",
            "PMC003xxxxxx/PMC3994689.xml\n",
            "PMC003xxxxxx/PMC3994690.xml\n",
            "PMC003xxxxxx/PMC3994691.xml\n",
            "PMC003xxxxxx/PMC3994692.xml\n",
            "PMC003xxxxxx/PMC3994693.xml\n",
            "PMC003xxxxxx/PMC3994694.xml\n",
            "PMC003xxxxxx/PMC3994695.xml\n",
            "PMC003xxxxxx/PMC3994696.xml\n",
            "PMC003xxxxxx/PMC3994698.xml\n",
            "PMC003xxxxxx/PMC3994699.xml\n",
            "PMC003xxxxxx/PMC3994705.xml\n",
            "PMC003xxxxxx/PMC3994706.xml\n",
            "PMC003xxxxxx/PMC3994707.xml\n",
            "PMC003xxxxxx/PMC3994708.xml\n",
            "PMC003xxxxxx/PMC3994709.xml\n",
            "PMC003xxxxxx/PMC3994710.xml\n",
            "PMC003xxxxxx/PMC3994713.xml\n",
            "PMC003xxxxxx/PMC3994714.xml\n",
            "PMC003xxxxxx/PMC3994715.xml\n",
            "PMC003xxxxxx/PMC3994716.xml\n",
            "PMC003xxxxxx/PMC3994717.xml\n",
            "PMC003xxxxxx/PMC3994718.xml\n",
            "PMC003xxxxxx/PMC3994719.xml\n",
            "PMC003xxxxxx/PMC3994720.xml\n",
            "PMC003xxxxxx/PMC3994721.xml\n",
            "PMC003xxxxxx/PMC3994724.xml\n",
            "PMC003xxxxxx/PMC3994725.xml\n",
            "PMC003xxxxxx/PMC3994726.xml\n",
            "PMC003xxxxxx/PMC3994727.xml\n",
            "PMC003xxxxxx/PMC3994728.xml\n",
            "PMC003xxxxxx/PMC3994729.xml\n",
            "PMC003xxxxxx/PMC3994730.xml\n",
            "PMC003xxxxxx/PMC3994731.xml\n",
            "PMC003xxxxxx/PMC3994732.xml\n",
            "PMC003xxxxxx/PMC3994733.xml\n",
            "PMC003xxxxxx/PMC3994734.xml\n",
            "PMC003xxxxxx/PMC3994735.xml\n",
            "PMC003xxxxxx/PMC3994736.xml\n",
            "PMC003xxxxxx/PMC3994737.xml\n",
            "PMC003xxxxxx/PMC3994738.xml\n",
            "PMC003xxxxxx/PMC3994739.xml\n",
            "PMC003xxxxxx/PMC3994740.xml\n",
            "PMC003xxxxxx/PMC3994741.xml\n",
            "PMC003xxxxxx/PMC3994742.xml\n",
            "PMC003xxxxxx/PMC3994743.xml\n",
            "PMC003xxxxxx/PMC3994744.xml\n",
            "PMC003xxxxxx/PMC3994745.xml\n",
            "PMC003xxxxxx/PMC3994746.xml\n",
            "PMC003xxxxxx/PMC3994747.xml\n",
            "PMC003xxxxxx/PMC3994845.xml\n",
            "PMC003xxxxxx/PMC3994846.xml\n",
            "PMC003xxxxxx/PMC3994847.xml\n",
            "PMC003xxxxxx/PMC3994848.xml\n",
            "PMC003xxxxxx/PMC3994850.xml\n",
            "PMC003xxxxxx/PMC3994851.xml\n",
            "PMC003xxxxxx/PMC3994852.xml\n",
            "PMC003xxxxxx/PMC3994853.xml\n",
            "PMC003xxxxxx/PMC3994855.xml\n",
            "PMC003xxxxxx/PMC3994857.xml\n",
            "PMC003xxxxxx/PMC3994866.xml\n",
            "PMC003xxxxxx/PMC3994916.xml\n",
            "PMC003xxxxxx/PMC3994917.xml\n",
            "PMC003xxxxxx/PMC3994918.xml\n",
            "PMC003xxxxxx/PMC3994919.xml\n",
            "PMC003xxxxxx/PMC3994924.xml\n",
            "PMC003xxxxxx/PMC3994928.xml\n",
            "PMC003xxxxxx/PMC3994929.xml\n",
            "PMC003xxxxxx/PMC3994930.xml\n",
            "PMC003xxxxxx/PMC3994931.xml\n",
            "PMC003xxxxxx/PMC3994932.xml\n",
            "PMC003xxxxxx/PMC3994933.xml\n",
            "PMC003xxxxxx/PMC3994934.xml\n",
            "PMC003xxxxxx/PMC3994935.xml\n",
            "PMC003xxxxxx/PMC3994936.xml\n",
            "PMC003xxxxxx/PMC3994937.xml\n",
            "PMC003xxxxxx/PMC3994938.xml\n",
            "PMC003xxxxxx/PMC3994939.xml\n",
            "PMC003xxxxxx/PMC3994940.xml\n",
            "PMC003xxxxxx/PMC3994941.xml\n",
            "PMC003xxxxxx/PMC3994947.xml\n",
            "PMC003xxxxxx/PMC3994948.xml\n",
            "PMC003xxxxxx/PMC3994949.xml\n",
            "PMC003xxxxxx/PMC3994950.xml\n",
            "PMC003xxxxxx/PMC3994951.xml\n",
            "PMC003xxxxxx/PMC3994952.xml\n",
            "PMC003xxxxxx/PMC3994953.xml\n",
            "PMC003xxxxxx/PMC3994954.xml\n",
            "PMC003xxxxxx/PMC3994955.xml\n",
            "PMC003xxxxxx/PMC3994956.xml\n",
            "PMC003xxxxxx/PMC3994957.xml\n",
            "PMC003xxxxxx/PMC3994958.xml\n",
            "PMC003xxxxxx/PMC3994959.xml\n",
            "PMC003xxxxxx/PMC3994960.xml\n",
            "PMC003xxxxxx/PMC3994961.xml\n",
            "PMC003xxxxxx/PMC3994962.xml\n",
            "PMC003xxxxxx/PMC3994963.xml\n",
            "PMC003xxxxxx/PMC3995006.xml\n",
            "PMC003xxxxxx/PMC3995009.xml\n",
            "PMC003xxxxxx/PMC3995015.xml\n",
            "PMC003xxxxxx/PMC3995018.xml\n",
            "PMC003xxxxxx/PMC3995088.xml\n",
            "PMC003xxxxxx/PMC3995091.xml\n",
            "PMC003xxxxxx/PMC3995154.xml\n",
            "PMC003xxxxxx/PMC3995213.xml\n",
            "PMC003xxxxxx/PMC3995216.xml\n",
            "PMC003xxxxxx/PMC3995217.xml\n",
            "PMC003xxxxxx/PMC3995218.xml\n",
            "PMC003xxxxxx/PMC3995219.xml\n",
            "PMC003xxxxxx/PMC3995220.xml\n",
            "PMC003xxxxxx/PMC3995221.xml\n",
            "PMC003xxxxxx/PMC3995222.xml\n",
            "PMC003xxxxxx/PMC3995223.xml\n",
            "PMC003xxxxxx/PMC3995225.xml\n",
            "PMC003xxxxxx/PMC3995226.xml\n",
            "PMC003xxxxxx/PMC3995227.xml\n",
            "PMC003xxxxxx/PMC3995228.xml\n",
            "PMC003xxxxxx/PMC3995229.xml\n",
            "PMC003xxxxxx/PMC3995233.xml\n",
            "PMC003xxxxxx/PMC3995235.xml\n",
            "PMC003xxxxxx/PMC3995236.xml\n",
            "PMC003xxxxxx/PMC3995237.xml\n",
            "PMC003xxxxxx/PMC3995239.xml\n",
            "PMC003xxxxxx/PMC3995240.xml\n",
            "PMC003xxxxxx/PMC3995242.xml\n",
            "PMC003xxxxxx/PMC3995243.xml\n",
            "PMC003xxxxxx/PMC3995244.xml\n",
            "PMC003xxxxxx/PMC3995245.xml\n",
            "PMC003xxxxxx/PMC3995246.xml\n",
            "PMC003xxxxxx/PMC3995247.xml\n",
            "PMC003xxxxxx/PMC3995248.xml\n",
            "PMC003xxxxxx/PMC3995249.xml\n",
            "PMC003xxxxxx/PMC3995250.xml\n",
            "PMC003xxxxxx/PMC3995251.xml\n",
            "PMC003xxxxxx/PMC3995252.xml\n",
            "PMC003xxxxxx/PMC3995253.xml\n",
            "PMC003xxxxxx/PMC3995254.xml\n",
            "PMC003xxxxxx/PMC3995255.xml\n",
            "PMC003xxxxxx/PMC3995256.xml\n",
            "PMC003xxxxxx/PMC3995257.xml\n",
            "PMC003xxxxxx/PMC3995259.xml\n",
            "PMC003xxxxxx/PMC3995260.xml\n",
            "PMC003xxxxxx/PMC3995261.xml\n",
            "PMC003xxxxxx/PMC3995262.xml\n",
            "PMC003xxxxxx/PMC3995263.xml\n",
            "PMC003xxxxxx/PMC3995264.xml\n",
            "PMC003xxxxxx/PMC3995265.xml\n",
            "PMC003xxxxxx/PMC3995266.xml\n",
            "PMC003xxxxxx/PMC3995267.xml\n",
            "PMC003xxxxxx/PMC3995269.xml\n",
            "PMC003xxxxxx/PMC3995270.xml\n",
            "PMC003xxxxxx/PMC3995271.xml\n",
            "PMC003xxxxxx/PMC3995272.xml\n",
            "PMC003xxxxxx/PMC3995273.xml\n",
            "PMC003xxxxxx/PMC3995274.xml\n",
            "PMC003xxxxxx/PMC3995275.xml\n",
            "PMC003xxxxxx/PMC3995276.xml\n",
            "PMC003xxxxxx/PMC3995277.xml\n",
            "PMC003xxxxxx/PMC3995278.xml\n",
            "PMC003xxxxxx/PMC3995279.xml\n",
            "PMC003xxxxxx/PMC3995280.xml\n",
            "PMC003xxxxxx/PMC3995282.xml\n",
            "PMC003xxxxxx/PMC3995283.xml\n",
            "PMC003xxxxxx/PMC3995285.xml\n",
            "PMC003xxxxxx/PMC3995286.xml\n",
            "PMC003xxxxxx/PMC3995287.xml\n",
            "PMC003xxxxxx/PMC3995288.xml\n",
            "PMC003xxxxxx/PMC3995289.xml\n",
            "PMC003xxxxxx/PMC3995290.xml\n",
            "PMC003xxxxxx/PMC3995291.xml\n",
            "PMC003xxxxxx/PMC3995292.xml\n",
            "PMC003xxxxxx/PMC3995329.xml\n",
            "PMC003xxxxxx/PMC3995332.xml\n",
            "PMC003xxxxxx/PMC3995337.xml\n",
            "PMC003xxxxxx/PMC3995338.xml\n",
            "PMC003xxxxxx/PMC3995340.xml\n",
            "PMC003xxxxxx/PMC3995342.xml\n",
            "PMC003xxxxxx/PMC3995370.xml\n",
            "PMC003xxxxxx/PMC3995373.xml\n",
            "PMC003xxxxxx/PMC3995378.xml\n",
            "PMC003xxxxxx/PMC3995379.xml\n",
            "PMC003xxxxxx/PMC3995384.xml\n",
            "PMC003xxxxxx/PMC3995386.xml\n",
            "PMC003xxxxxx/PMC3995387.xml\n",
            "PMC003xxxxxx/PMC3995388.xml\n",
            "PMC003xxxxxx/PMC3995389.xml\n",
            "PMC003xxxxxx/PMC3995392.xml\n",
            "PMC003xxxxxx/PMC3995393.xml\n",
            "PMC003xxxxxx/PMC3995394.xml\n",
            "PMC003xxxxxx/PMC3995395.xml\n",
            "PMC003xxxxxx/PMC3995396.xml\n",
            "PMC003xxxxxx/PMC3995397.xml\n",
            "PMC003xxxxxx/PMC3995400.xml\n",
            "PMC003xxxxxx/PMC3995401.xml\n",
            "PMC003xxxxxx/PMC3995726.xml\n",
            "PMC003xxxxxx/PMC3995727.xml\n",
            "PMC003xxxxxx/PMC3995728.xml\n",
            "PMC003xxxxxx/PMC3995729.xml\n",
            "PMC003xxxxxx/PMC3995730.xml\n",
            "PMC003xxxxxx/PMC3995731.xml\n",
            "PMC003xxxxxx/PMC3995732.xml\n",
            "PMC003xxxxxx/PMC3995733.xml\n",
            "PMC003xxxxxx/PMC3995734.xml\n",
            "PMC003xxxxxx/PMC3995735.xml\n",
            "PMC003xxxxxx/PMC3995737.xml\n",
            "PMC003xxxxxx/PMC3995738.xml\n",
            "PMC003xxxxxx/PMC3996244.xml\n",
            "PMC003xxxxxx/PMC3996245.xml\n",
            "PMC003xxxxxx/PMC3996246.xml\n",
            "PMC003xxxxxx/PMC3996247.xml\n",
            "PMC003xxxxxx/PMC3996248.xml\n",
            "PMC003xxxxxx/PMC3996249.xml\n",
            "PMC003xxxxxx/PMC3996250.xml\n",
            "PMC003xxxxxx/PMC3996251.xml\n",
            "PMC003xxxxxx/PMC3996252.xml\n",
            "PMC003xxxxxx/PMC3996253.xml\n",
            "PMC003xxxxxx/PMC3996259.xml\n",
            "PMC003xxxxxx/PMC3996260.xml\n",
            "PMC003xxxxxx/PMC3996261.xml\n",
            "PMC003xxxxxx/PMC3996262.xml\n",
            "PMC003xxxxxx/PMC3996263.xml\n",
            "PMC003xxxxxx/PMC3996264.xml\n",
            "PMC003xxxxxx/PMC3996265.xml\n",
            "PMC003xxxxxx/PMC3996266.xml\n",
            "PMC003xxxxxx/PMC3996267.xml\n",
            "PMC003xxxxxx/PMC3996268.xml\n",
            "PMC003xxxxxx/PMC3996269.xml\n",
            "PMC003xxxxxx/PMC3996270.xml\n",
            "PMC003xxxxxx/PMC3996271.xml\n",
            "PMC003xxxxxx/PMC3996285.xml\n",
            "PMC003xxxxxx/PMC3996333.xml\n",
            "PMC003xxxxxx/PMC3996334.xml\n",
            "PMC003xxxxxx/PMC3996335.xml\n",
            "PMC003xxxxxx/PMC3996336.xml\n",
            "PMC003xxxxxx/PMC3996337.xml\n",
            "PMC003xxxxxx/PMC3996338.xml\n",
            "PMC003xxxxxx/PMC3996339.xml\n",
            "PMC003xxxxxx/PMC3996340.xml\n",
            "PMC003xxxxxx/PMC3996341.xml\n",
            "PMC003xxxxxx/PMC3996342.xml\n",
            "PMC003xxxxxx/PMC3996343.xml\n",
            "PMC003xxxxxx/PMC3996344.xml\n",
            "PMC003xxxxxx/PMC3996345.xml\n",
            "PMC003xxxxxx/PMC3996346.xml\n",
            "PMC003xxxxxx/PMC3996347.xml\n",
            "PMC003xxxxxx/PMC3996348.xml\n",
            "PMC003xxxxxx/PMC3996349.xml\n",
            "PMC003xxxxxx/PMC3996350.xml\n",
            "PMC003xxxxxx/PMC3996360.xml\n",
            "PMC003xxxxxx/PMC3996401.xml\n",
            "PMC003xxxxxx/PMC3996402.xml\n",
            "PMC003xxxxxx/PMC3996403.xml\n",
            "PMC003xxxxxx/PMC3996404.xml\n",
            "PMC003xxxxxx/PMC3996405.xml\n",
            "PMC003xxxxxx/PMC3996406.xml\n",
            "PMC003xxxxxx/PMC3996407.xml\n",
            "PMC003xxxxxx/PMC3996408.xml\n",
            "PMC003xxxxxx/PMC3996409.xml\n",
            "PMC003xxxxxx/PMC3996410.xml\n",
            "PMC003xxxxxx/PMC3996411.xml\n",
            "PMC003xxxxxx/PMC3996412.xml\n",
            "PMC003xxxxxx/PMC3996413.xml\n",
            "PMC003xxxxxx/PMC3996414.xml\n",
            "PMC003xxxxxx/PMC3996415.xml\n",
            "PMC003xxxxxx/PMC3996416.xml\n",
            "PMC003xxxxxx/PMC3996417.xml\n",
            "PMC003xxxxxx/PMC3996418.xml\n",
            "PMC003xxxxxx/PMC3996419.xml\n",
            "PMC003xxxxxx/PMC3996420.xml\n",
            "PMC003xxxxxx/PMC3996421.xml\n",
            "PMC003xxxxxx/PMC3996422.xml\n",
            "PMC003xxxxxx/PMC3996423.xml\n",
            "PMC003xxxxxx/PMC3996424.xml\n",
            "PMC003xxxxxx/PMC3996425.xml\n",
            "PMC003xxxxxx/PMC3996426.xml\n",
            "PMC003xxxxxx/PMC3996427.xml\n",
            "PMC003xxxxxx/PMC3996428.xml\n",
            "PMC003xxxxxx/PMC3996429.xml\n",
            "PMC003xxxxxx/PMC3996430.xml\n",
            "PMC003xxxxxx/PMC3996431.xml\n",
            "PMC003xxxxxx/PMC3996432.xml\n",
            "PMC003xxxxxx/PMC3996433.xml\n",
            "PMC003xxxxxx/PMC3996434.xml\n",
            "PMC003xxxxxx/PMC3996435.xml\n",
            "PMC003xxxxxx/PMC3996436.xml\n",
            "PMC003xxxxxx/PMC3996460.xml\n",
            "PMC003xxxxxx/PMC3996461.xml\n",
            "PMC003xxxxxx/PMC3996462.xml\n",
            "PMC003xxxxxx/PMC3996464.xml\n",
            "PMC003xxxxxx/PMC3996465.xml\n",
            "PMC003xxxxxx/PMC3996466.xml\n",
            "PMC003xxxxxx/PMC3996467.xml\n",
            "PMC003xxxxxx/PMC3996478.xml\n",
            "PMC003xxxxxx/PMC3996479.xml\n",
            "PMC003xxxxxx/PMC3996480.xml\n",
            "PMC003xxxxxx/PMC3996481.xml\n",
            "PMC003xxxxxx/PMC3996482.xml\n",
            "PMC003xxxxxx/PMC3996483.xml\n",
            "PMC003xxxxxx/PMC3996486.xml\n",
            "PMC003xxxxxx/PMC3996532.xml\n",
            "PMC003xxxxxx/PMC3996535.xml\n",
            "PMC003xxxxxx/PMC3996537.xml\n",
            "PMC003xxxxxx/PMC3996541.xml\n",
            "PMC003xxxxxx/PMC3996544.xml\n",
            "PMC003xxxxxx/PMC3996546.xml\n",
            "PMC003xxxxxx/PMC3996650.xml\n",
            "PMC003xxxxxx/PMC3996688.xml\n",
            "PMC003xxxxxx/PMC3996715.xml\n",
            "PMC003xxxxxx/PMC3996716.xml\n",
            "PMC003xxxxxx/PMC3996717.xml\n",
            "PMC003xxxxxx/PMC3996718.xml\n",
            "PMC003xxxxxx/PMC3996719.xml\n",
            "PMC003xxxxxx/PMC3996720.xml\n",
            "PMC003xxxxxx/PMC3996721.xml\n",
            "PMC003xxxxxx/PMC3996722.xml\n",
            "PMC003xxxxxx/PMC3996723.xml\n",
            "PMC003xxxxxx/PMC3996725.xml\n",
            "PMC003xxxxxx/PMC3996726.xml\n",
            "PMC003xxxxxx/PMC3996731.xml\n",
            "PMC003xxxxxx/PMC3996732.xml\n",
            "PMC003xxxxxx/PMC3996733.xml\n",
            "PMC003xxxxxx/PMC3996734.xml\n",
            "PMC003xxxxxx/PMC3996735.xml\n",
            "PMC003xxxxxx/PMC3996736.xml\n",
            "PMC003xxxxxx/PMC3996737.xml\n",
            "PMC003xxxxxx/PMC3996738.xml\n",
            "PMC003xxxxxx/PMC3996739.xml\n",
            "PMC003xxxxxx/PMC3996740.xml\n",
            "PMC003xxxxxx/PMC3996741.xml\n",
            "PMC003xxxxxx/PMC3996742.xml\n",
            "PMC003xxxxxx/PMC3996743.xml\n",
            "PMC003xxxxxx/PMC3996744.xml\n",
            "PMC003xxxxxx/PMC3996745.xml\n",
            "PMC003xxxxxx/PMC3996746.xml\n",
            "PMC003xxxxxx/PMC3996747.xml\n",
            "PMC003xxxxxx/PMC3996748.xml\n",
            "PMC003xxxxxx/PMC3996749.xml\n",
            "PMC003xxxxxx/PMC3996750.xml\n",
            "PMC003xxxxxx/PMC3996751.xml\n",
            "PMC003xxxxxx/PMC3996752.xml\n",
            "PMC003xxxxxx/PMC3996753.xml\n",
            "PMC003xxxxxx/PMC3996754.xml\n",
            "PMC003xxxxxx/PMC3996755.xml\n",
            "PMC003xxxxxx/PMC3996756.xml\n",
            "PMC003xxxxxx/PMC3996757.xml\n",
            "PMC003xxxxxx/PMC3996758.xml\n",
            "PMC003xxxxxx/PMC3996762.xml\n",
            "PMC003xxxxxx/PMC3996763.xml\n",
            "PMC003xxxxxx/PMC3996764.xml\n",
            "PMC003xxxxxx/PMC3996765.xml\n",
            "PMC003xxxxxx/PMC3996766.xml\n",
            "PMC003xxxxxx/PMC3996767.xml\n",
            "PMC003xxxxxx/PMC3996768.xml\n",
            "PMC003xxxxxx/PMC3996769.xml\n",
            "PMC003xxxxxx/PMC3996770.xml\n",
            "PMC003xxxxxx/PMC3996771.xml\n",
            "PMC003xxxxxx/PMC3996772.xml\n",
            "PMC003xxxxxx/PMC3996773.xml\n",
            "PMC003xxxxxx/PMC3996774.xml\n",
            "PMC003xxxxxx/PMC3996775.xml\n",
            "PMC003xxxxxx/PMC3996776.xml\n",
            "PMC003xxxxxx/PMC3996777.xml\n",
            "PMC003xxxxxx/PMC3996778.xml\n",
            "PMC003xxxxxx/PMC3996779.xml\n",
            "PMC003xxxxxx/PMC3996780.xml\n",
            "PMC003xxxxxx/PMC3996781.xml\n",
            "PMC003xxxxxx/PMC3996782.xml\n",
            "PMC003xxxxxx/PMC3996783.xml\n",
            "PMC003xxxxxx/PMC3996784.xml\n",
            "PMC003xxxxxx/PMC3996785.xml\n",
            "PMC003xxxxxx/PMC3996786.xml\n",
            "PMC003xxxxxx/PMC3996787.xml\n",
            "PMC003xxxxxx/PMC3996788.xml\n",
            "PMC003xxxxxx/PMC3996789.xml\n",
            "PMC003xxxxxx/PMC3996790.xml\n",
            "PMC003xxxxxx/PMC3996791.xml\n",
            "PMC003xxxxxx/PMC3996792.xml\n",
            "PMC003xxxxxx/PMC3996793.xml\n",
            "PMC003xxxxxx/PMC3996794.xml\n",
            "PMC003xxxxxx/PMC3996795.xml\n",
            "PMC003xxxxxx/PMC3996796.xml\n",
            "PMC003xxxxxx/PMC3996797.xml\n",
            "PMC003xxxxxx/PMC3996798.xml\n",
            "PMC003xxxxxx/PMC3996799.xml\n",
            "PMC003xxxxxx/PMC3996800.xml\n",
            "PMC003xxxxxx/PMC3996801.xml\n",
            "PMC003xxxxxx/PMC3996802.xml\n",
            "PMC003xxxxxx/PMC3996809.xml\n",
            "PMC003xxxxxx/PMC3996810.xml\n",
            "PMC003xxxxxx/PMC3996811.xml\n",
            "PMC003xxxxxx/PMC3996812.xml\n",
            "PMC003xxxxxx/PMC3996813.xml\n",
            "PMC003xxxxxx/PMC3996814.xml\n",
            "PMC003xxxxxx/PMC3996816.xml\n",
            "PMC003xxxxxx/PMC3996817.xml\n",
            "PMC003xxxxxx/PMC3996818.xml\n",
            "PMC003xxxxxx/PMC3996820.xml\n",
            "PMC003xxxxxx/PMC3996821.xml\n",
            "PMC003xxxxxx/PMC3996822.xml\n",
            "PMC003xxxxxx/PMC3996823.xml\n",
            "PMC003xxxxxx/PMC3996825.xml\n",
            "PMC003xxxxxx/PMC3997238.xml\n",
            "PMC003xxxxxx/PMC3997278.xml\n",
            "PMC003xxxxxx/PMC3997346.xml\n",
            "PMC003xxxxxx/PMC3997378.xml\n",
            "PMC003xxxxxx/PMC3997517.xml\n",
            "PMC003xxxxxx/PMC3997800.xml\n",
            "PMC003xxxxxx/PMC3997804.xml\n",
            "PMC003xxxxxx/PMC3997811.xml\n",
            "PMC003xxxxxx/PMC3997827.xml\n",
            "PMC003xxxxxx/PMC3997830.xml\n",
            "PMC003xxxxxx/PMC3997831.xml\n",
            "PMC003xxxxxx/PMC3997832.xml\n",
            "PMC003xxxxxx/PMC3997833.xml\n",
            "PMC003xxxxxx/PMC3997922.xml\n",
            "PMC003xxxxxx/PMC3997923.xml\n",
            "PMC003xxxxxx/PMC3997924.xml\n",
            "PMC003xxxxxx/PMC3997925.xml\n",
            "PMC003xxxxxx/PMC3997926.xml\n",
            "PMC003xxxxxx/PMC3997927.xml\n",
            "PMC003xxxxxx/PMC3998012.xml\n",
            "PMC003xxxxxx/PMC3998013.xml\n",
            "PMC003xxxxxx/PMC3998015.xml\n",
            "PMC003xxxxxx/PMC3998016.xml\n",
            "PMC003xxxxxx/PMC3998018.xml\n",
            "PMC003xxxxxx/PMC3998019.xml\n",
            "PMC003xxxxxx/PMC3998020.xml\n",
            "PMC003xxxxxx/PMC3998027.xml\n",
            "PMC003xxxxxx/PMC3998028.xml\n",
            "PMC003xxxxxx/PMC3998029.xml\n",
            "PMC003xxxxxx/PMC3998030.xml\n",
            "PMC003xxxxxx/PMC3998031.xml\n",
            "PMC003xxxxxx/PMC3998032.xml\n",
            "PMC003xxxxxx/PMC3998034.xml\n",
            "PMC003xxxxxx/PMC3998072.xml\n",
            "PMC003xxxxxx/PMC3998074.xml\n",
            "PMC003xxxxxx/PMC3998088.xml\n",
            "PMC003xxxxxx/PMC3998127.xml\n",
            "PMC003xxxxxx/PMC3998138.xml\n",
            "PMC003xxxxxx/PMC3998140.xml\n",
            "PMC003xxxxxx/PMC3998144.xml\n",
            "PMC003xxxxxx/PMC3998145.xml\n",
            "PMC003xxxxxx/PMC3998146.xml\n",
            "PMC003xxxxxx/PMC3998147.xml\n",
            "PMC003xxxxxx/PMC3998148.xml\n",
            "PMC003xxxxxx/PMC3998149.xml\n",
            "PMC003xxxxxx/PMC3998150.xml\n",
            "PMC003xxxxxx/PMC3998151.xml\n",
            "PMC003xxxxxx/PMC3998152.xml\n",
            "PMC003xxxxxx/PMC3998153.xml\n",
            "PMC003xxxxxx/PMC3998154.xml\n",
            "PMC003xxxxxx/PMC3998155.xml\n",
            "PMC003xxxxxx/PMC3998156.xml\n",
            "PMC003xxxxxx/PMC3998157.xml\n",
            "PMC003xxxxxx/PMC3998158.xml\n",
            "PMC003xxxxxx/PMC3998159.xml\n",
            "PMC003xxxxxx/PMC3998160.xml\n",
            "PMC003xxxxxx/PMC3998161.xml\n",
            "PMC003xxxxxx/PMC3998162.xml\n",
            "PMC003xxxxxx/PMC3998163.xml\n",
            "PMC003xxxxxx/PMC3998164.xml\n",
            "PMC003xxxxxx/PMC3998165.xml\n",
            "PMC003xxxxxx/PMC3998166.xml\n",
            "PMC003xxxxxx/PMC3998167.xml\n",
            "PMC003xxxxxx/PMC3998168.xml\n",
            "PMC003xxxxxx/PMC3998169.xml\n",
            "PMC003xxxxxx/PMC3998170.xml\n",
            "PMC003xxxxxx/PMC3998171.xml\n",
            "PMC003xxxxxx/PMC3998172.xml\n",
            "PMC003xxxxxx/PMC3998173.xml\n",
            "PMC003xxxxxx/PMC3998174.xml\n",
            "PMC003xxxxxx/PMC3998175.xml\n",
            "PMC003xxxxxx/PMC3998176.xml\n",
            "PMC003xxxxxx/PMC3998177.xml\n",
            "PMC003xxxxxx/PMC3998178.xml\n",
            "PMC003xxxxxx/PMC3998179.xml\n",
            "PMC003xxxxxx/PMC3998180.xml\n",
            "PMC003xxxxxx/PMC3998181.xml\n",
            "PMC003xxxxxx/PMC3998182.xml\n",
            "PMC003xxxxxx/PMC3998183.xml\n",
            "PMC003xxxxxx/PMC3998184.xml\n",
            "PMC003xxxxxx/PMC3998199.xml\n",
            "PMC003xxxxxx/PMC3998211.xml\n",
            "PMC003xxxxxx/PMC3998212.xml\n",
            "PMC003xxxxxx/PMC3998213.xml\n",
            "PMC003xxxxxx/PMC3998214.xml\n",
            "PMC003xxxxxx/PMC3998522.xml\n",
            "PMC003xxxxxx/PMC3998635.xml\n",
            "PMC003xxxxxx/PMC3998636.xml\n",
            "PMC003xxxxxx/PMC3998637.xml\n",
            "PMC003xxxxxx/PMC3998638.xml\n",
            "PMC003xxxxxx/PMC3998724.xml\n",
            "PMC003xxxxxx/PMC3998726.xml\n",
            "PMC003xxxxxx/PMC3998787.xml\n",
            "PMC003xxxxxx/PMC3998791.xml\n",
            "PMC003xxxxxx/PMC3998792.xml\n",
            "PMC003xxxxxx/PMC3998793.xml\n",
            "PMC003xxxxxx/PMC3998794.xml\n",
            "PMC003xxxxxx/PMC3998795.xml\n",
            "PMC003xxxxxx/PMC3998796.xml\n",
            "PMC003xxxxxx/PMC3998797.xml\n",
            "PMC003xxxxxx/PMC3998798.xml\n",
            "PMC003xxxxxx/PMC3998799.xml\n",
            "PMC003xxxxxx/PMC3998800.xml\n",
            "PMC003xxxxxx/PMC3998801.xml\n",
            "PMC003xxxxxx/PMC3998802.xml\n",
            "PMC003xxxxxx/PMC3998803.xml\n",
            "PMC003xxxxxx/PMC3998804.xml\n",
            "PMC003xxxxxx/PMC3998805.xml\n",
            "PMC003xxxxxx/PMC3998806.xml\n",
            "PMC003xxxxxx/PMC3998807.xml\n",
            "PMC003xxxxxx/PMC3998808.xml\n",
            "PMC003xxxxxx/PMC3998809.xml\n",
            "PMC003xxxxxx/PMC3998810.xml\n",
            "PMC003xxxxxx/PMC3998811.xml\n",
            "PMC003xxxxxx/PMC3998812.xml\n",
            "PMC003xxxxxx/PMC3998846.xml\n",
            "PMC003xxxxxx/PMC3998847.xml\n",
            "PMC003xxxxxx/PMC3998848.xml\n",
            "PMC003xxxxxx/PMC3998849.xml\n",
            "PMC003xxxxxx/PMC3998850.xml\n",
            "PMC003xxxxxx/PMC3998851.xml\n",
            "PMC003xxxxxx/PMC3998852.xml\n",
            "PMC003xxxxxx/PMC3998853.xml\n",
            "PMC003xxxxxx/PMC3998854.xml\n",
            "PMC003xxxxxx/PMC3998855.xml\n",
            "PMC003xxxxxx/PMC3998856.xml\n",
            "PMC003xxxxxx/PMC3998857.xml\n",
            "PMC003xxxxxx/PMC3998858.xml\n",
            "PMC003xxxxxx/PMC3998859.xml\n",
            "PMC003xxxxxx/PMC3998860.xml\n",
            "PMC003xxxxxx/PMC3998861.xml\n",
            "PMC003xxxxxx/PMC3998862.xml\n",
            "PMC003xxxxxx/PMC3998863.xml\n",
            "PMC003xxxxxx/PMC3998864.xml\n",
            "PMC003xxxxxx/PMC3998865.xml\n",
            "PMC003xxxxxx/PMC3998866.xml\n",
            "PMC003xxxxxx/PMC3998867.xml\n",
            "PMC003xxxxxx/PMC3998868.xml\n",
            "PMC003xxxxxx/PMC3998869.xml\n",
            "PMC003xxxxxx/PMC3999056.xml\n",
            "PMC003xxxxxx/PMC3999062.xml\n",
            "PMC003xxxxxx/PMC3999072.xml\n",
            "PMC003xxxxxx/PMC3999075.xml\n",
            "PMC003xxxxxx/PMC3999083.xml\n",
            "PMC003xxxxxx/PMC3999136.xml\n",
            "PMC003xxxxxx/PMC3999272.xml\n",
            "PMC003xxxxxx/PMC3999273.xml\n",
            "PMC003xxxxxx/PMC3999274.xml\n",
            "PMC003xxxxxx/PMC3999275.xml\n",
            "PMC003xxxxxx/PMC3999276.xml\n",
            "PMC003xxxxxx/PMC3999277.xml\n",
            "PMC003xxxxxx/PMC3999315.xml\n",
            "PMC003xxxxxx/PMC3999316.xml\n",
            "PMC003xxxxxx/PMC3999317.xml\n",
            "PMC003xxxxxx/PMC3999318.xml\n",
            "PMC003xxxxxx/PMC3999319.xml\n",
            "PMC003xxxxxx/PMC3999320.xml\n",
            "PMC003xxxxxx/PMC3999321.xml\n",
            "PMC003xxxxxx/PMC3999322.xml\n",
            "PMC003xxxxxx/PMC3999323.xml\n",
            "PMC003xxxxxx/PMC3999324.xml\n",
            "PMC003xxxxxx/PMC3999325.xml\n",
            "PMC003xxxxxx/PMC3999326.xml\n",
            "PMC003xxxxxx/PMC3999327.xml\n",
            "PMC003xxxxxx/PMC3999328.xml\n",
            "PMC003xxxxxx/PMC3999329.xml\n",
            "PMC003xxxxxx/PMC3999351.xml\n",
            "PMC003xxxxxx/PMC3999352.xml\n",
            "PMC003xxxxxx/PMC3999353.xml\n",
            "PMC003xxxxxx/PMC3999354.xml\n",
            "PMC003xxxxxx/PMC3999355.xml\n",
            "PMC003xxxxxx/PMC3999356.xml\n",
            "PMC003xxxxxx/PMC3999357.xml\n",
            "PMC003xxxxxx/PMC3999358.xml\n",
            "PMC003xxxxxx/PMC3999388.xml\n",
            "PMC003xxxxxx/PMC3999389.xml\n",
            "PMC003xxxxxx/PMC3999390.xml\n",
            "PMC003xxxxxx/PMC3999391.xml\n",
            "PMC003xxxxxx/PMC3999392.xml\n",
            "PMC003xxxxxx/PMC3999393.xml\n",
            "PMC003xxxxxx/PMC3999394.xml\n",
            "PMC003xxxxxx/PMC3999395.xml\n",
            "PMC003xxxxxx/PMC3999396.xml\n",
            "PMC003xxxxxx/PMC3999397.xml\n",
            "PMC003xxxxxx/PMC3999398.xml\n",
            "PMC003xxxxxx/PMC3999399.xml\n",
            "PMC003xxxxxx/PMC3999441.xml\n",
            "PMC003xxxxxx/PMC3999442.xml\n",
            "PMC003xxxxxx/PMC3999443.xml\n",
            "PMC003xxxxxx/PMC3999444.xml\n",
            "PMC003xxxxxx/PMC3999446.xml\n",
            "PMC003xxxxxx/PMC3999447.xml\n",
            "PMC003xxxxxx/PMC3999448.xml\n",
            "PMC003xxxxxx/PMC3999449.xml\n",
            "PMC003xxxxxx/PMC3999451.xml\n",
            "PMC003xxxxxx/PMC3999452.xml\n",
            "PMC003xxxxxx/PMC3999454.xml\n",
            "PMC003xxxxxx/PMC3999455.xml\n",
            "PMC003xxxxxx/PMC3999456.xml\n",
            "PMC003xxxxxx/PMC3999485.xml\n",
            "PMC003xxxxxx/PMC3999511.xml\n",
            "PMC003xxxxxx/PMC3999516.xml\n",
            "PMC003xxxxxx/PMC3999573.xml\n",
            "PMC003xxxxxx/PMC3999574.xml\n",
            "PMC003xxxxxx/PMC3999575.xml\n",
            "PMC003xxxxxx/PMC3999576.xml\n",
            "PMC003xxxxxx/PMC3999577.xml\n",
            "PMC003xxxxxx/PMC3999578.xml\n",
            "PMC003xxxxxx/PMC3999579.xml\n",
            "PMC003xxxxxx/PMC3999580.xml\n",
            "PMC003xxxxxx/PMC3999594.xml\n",
            "PMC003xxxxxx/PMC3999595.xml\n",
            "PMC003xxxxxx/PMC3999596.xml\n",
            "PMC003xxxxxx/PMC3999597.xml\n",
            "PMC003xxxxxx/PMC3999598.xml\n",
            "PMC003xxxxxx/PMC3999599.xml\n",
            "PMC003xxxxxx/PMC3999600.xml\n",
            "PMC003xxxxxx/PMC3999601.xml\n",
            "PMC003xxxxxx/PMC3999602.xml\n",
            "PMC003xxxxxx/PMC3999603.xml\n",
            "PMC003xxxxxx/PMC3999604.xml\n",
            "PMC003xxxxxx/PMC3999605.xml\n",
            "PMC003xxxxxx/PMC3999606.xml\n",
            "PMC003xxxxxx/PMC3999607.xml\n",
            "PMC003xxxxxx/PMC3999608.xml\n",
            "PMC003xxxxxx/PMC3999609.xml\n",
            "PMC003xxxxxx/PMC3999610.xml\n",
            "PMC003xxxxxx/PMC3999611.xml\n",
            "PMC003xxxxxx/PMC3999612.xml\n",
            "PMC003xxxxxx/PMC3999613.xml\n",
            "PMC003xxxxxx/PMC3999614.xml\n",
            "PMC003xxxxxx/PMC3999644.xml\n",
            "PMC003xxxxxx/PMC3999645.xml\n",
            "PMC003xxxxxx/PMC3999646.xml\n",
            "PMC003xxxxxx/PMC3999647.xml\n",
            "PMC003xxxxxx/PMC3999648.xml\n",
            "PMC003xxxxxx/PMC3999649.xml\n",
            "PMC003xxxxxx/PMC3999650.xml\n",
            "PMC003xxxxxx/PMC3999651.xml\n",
            "PMC003xxxxxx/PMC3999652.xml\n",
            "PMC003xxxxxx/PMC3999653.xml\n",
            "PMC003xxxxxx/PMC3999654.xml\n",
            "PMC003xxxxxx/PMC3999655.xml\n",
            "PMC003xxxxxx/PMC3999656.xml\n",
            "PMC003xxxxxx/PMC3999657.xml\n",
            "PMC003xxxxxx/PMC3999658.xml\n",
            "PMC003xxxxxx/PMC3999659.xml\n",
            "PMC003xxxxxx/PMC3999660.xml\n",
            "PMC003xxxxxx/PMC3999661.xml\n",
            "PMC003xxxxxx/PMC3999662.xml\n",
            "PMC003xxxxxx/PMC3999666.xml\n",
            "PMC003xxxxxx/PMC3999667.xml\n",
            "PMC003xxxxxx/PMC3999668.xml\n",
            "PMC003xxxxxx/PMC3999669.xml\n",
            "PMC003xxxxxx/PMC3999670.xml\n",
            "PMC003xxxxxx/PMC3999671.xml\n",
            "PMC003xxxxxx/PMC3999672.xml\n",
            "PMC003xxxxxx/PMC3999673.xml\n",
            "PMC003xxxxxx/PMC3999674.xml\n",
            "PMC003xxxxxx/PMC3999675.xml\n",
            "PMC003xxxxxx/PMC3999676.xml\n",
            "PMC003xxxxxx/PMC3999677.xml\n",
            "PMC003xxxxxx/PMC3999678.xml\n",
            "PMC003xxxxxx/PMC3999679.xml\n",
            "PMC003xxxxxx/PMC3999680.xml\n",
            "PMC003xxxxxx/PMC3999681.xml\n",
            "PMC003xxxxxx/PMC3999682.xml\n",
            "PMC003xxxxxx/PMC3999683.xml\n",
            "PMC003xxxxxx/PMC3999684.xml\n",
            "PMC003xxxxxx/PMC3999685.xml\n",
            "PMC003xxxxxx/PMC3999686.xml\n",
            "PMC003xxxxxx/PMC3999687.xml\n",
            "PMC003xxxxxx/PMC3999688.xml\n",
            "PMC003xxxxxx/PMC3999689.xml\n",
            "PMC003xxxxxx/PMC3999690.xml\n",
            "PMC003xxxxxx/PMC3999691.xml\n",
            "PMC003xxxxxx/PMC3999692.xml\n",
            "PMC003xxxxxx/PMC3999693.xml\n",
            "PMC003xxxxxx/PMC3999694.xml\n",
            "PMC003xxxxxx/PMC3999695.xml\n",
            "PMC003xxxxxx/PMC3999696.xml\n",
            "PMC003xxxxxx/PMC3999697.xml\n",
            "PMC003xxxxxx/PMC3999703.xml\n",
            "PMC003xxxxxx/PMC3999704.xml\n",
            "PMC003xxxxxx/PMC3999709.xml\n",
            "PMC003xxxxxx/PMC3999718.xml\n",
            "PMC003xxxxxx/PMC3999720.xml\n",
            "PMC003xxxxxx/PMC3999726.xml\n",
            "PMC003xxxxxx/PMC3999800.xml\n",
            "PMC003xxxxxx/PMC3999802.xml\n",
            "PMC003xxxxxx/PMC3999803.xml\n",
            "PMC003xxxxxx/PMC3999804.xml\n",
            "PMC003xxxxxx/PMC3999805.xml\n",
            "PMC003xxxxxx/PMC3999806.xml\n",
            "PMC003xxxxxx/PMC3999807.xml\n",
            "PMC003xxxxxx/PMC3999808.xml\n",
            "PMC003xxxxxx/PMC3999809.xml\n",
            "PMC003xxxxxx/PMC3999949.xml\n",
            "PMC003xxxxxx/PMC3999951.xml\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!sudo apt-get install -y libxml2-utils\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "g0zFYEjAqRa_",
        "outputId": "24c1641e-a4a7-4324-c118-d2196652d170"
      },
      "execution_count": 15,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "The following NEW packages will be installed:\n",
            "  libxml2-utils\n",
            "0 upgraded, 1 newly installed, 0 to remove and 49 not upgraded.\n",
            "Need to get 40.2 kB of archives.\n",
            "After this operation, 207 kB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libxml2-utils amd64 2.9.13+dfsg-1ubuntu0.4 [40.2 kB]\n",
            "Fetched 40.2 kB in 0s (90.2 kB/s)\n",
            "debconf: unable to initialize frontend: Dialog\n",
            "debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 78, <> line 1.)\n",
            "debconf: falling back to frontend: Readline\n",
            "debconf: unable to initialize frontend: Readline\n",
            "debconf: (This frontend requires a controlling tty.)\n",
            "debconf: falling back to frontend: Teletype\n",
            "dpkg-preconfigure: unable to re-open stdin: \n",
            "Selecting previously unselected package libxml2-utils.\n",
            "(Reading database ... 123633 files and directories currently installed.)\n",
            "Preparing to unpack .../libxml2-utils_2.9.13+dfsg-1ubuntu0.4_amd64.deb ...\n",
            "Unpacking libxml2-utils (2.9.13+dfsg-1ubuntu0.4) ...\n",
            "Setting up libxml2-utils (2.9.13+dfsg-1ubuntu0.4) ...\n",
            "Processing triggers for man-db (2.10.2-1) ...\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!xmllint --format /content/data-for-clinical-note-summarizer/extracted_xml_files/PMC001xxxxxx/PMC1193645.xml"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "s-2RwdaNpkAg",
        "outputId": "06476c9b-37f6-4344-87cb-27e08f1b7e3c"
      },
      "execution_count": 16,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "<?xml version=\"1.0\"?>\n",
            "<!DOCTYPE article PUBLIC \"-//NLM//DTD JATS (Z39.96) Journal Archiving and Interchange DTD v1.0 20120330//EN\" \"JATS-archivearticle1.dtd\">\n",
            "<article xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" article-type=\"research-article\">\n",
            "  <?properties open_access?>\n",
            "  <front>\n",
            "    <journal-meta>\n",
            "      <journal-id journal-id-type=\"nlm-ta\">J Exp Med</journal-id>\n",
            "      <journal-id journal-id-type=\"iso-abbrev\">J. Exp. Med</journal-id>\n",
            "      <journal-title-group>\n",
            "        <journal-title>The Journal of Experimental Medicine</journal-title>\n",
            "      </journal-title-group>\n",
            "      <issn pub-type=\"ppub\">0022-1007</issn>\n",
            "      <issn pub-type=\"epub\">1540-9538</issn>\n",
            "      <publisher>\n",
            "        <publisher-name>The Rockefeller University Press</publisher-name>\n",
            "      </publisher>\n",
            "    </journal-meta>\n",
            "    <article-meta>\n",
            "      <article-id pub-id-type=\"pmid\">14699080</article-id>\n",
            "      <article-id pub-id-type=\"pmc\">PMC1193645</article-id>\n",
            "      <article-id pub-id-type=\"publisher-id\">20020509</article-id>\n",
            "      <article-id pub-id-type=\"doi\">10.1084/jem.20020509</article-id>\n",
            "      <article-categories>\n",
            "        <subj-group subj-group-type=\"heading\">\n",
            "          <subject>Article</subject>\n",
            "        </subj-group>\n",
            "      </article-categories>\n",
            "      <title-group>\n",
            "        <article-title>Impaired Development of CD4<sup>+</sup> CD25<sup>+</sup> Regulatory T Cells in the Absence of STAT1</article-title>\n",
            "        <subtitle>Increased Susceptibility to Autoimmune Disease</subtitle>\n",
            "      </title-group>\n",
            "      <contrib-group>\n",
            "        <contrib contrib-type=\"author\">\n",
            "          <name>\n",
            "            <surname>Nishibori</surname>\n",
            "            <given-names>Takeaki</given-names>\n",
            "          </name>\n",
            "        </contrib>\n",
            "        <contrib contrib-type=\"author\">\n",
            "          <name>\n",
            "            <surname>Tanabe</surname>\n",
            "            <given-names>Yoshinari</given-names>\n",
            "          </name>\n",
            "        </contrib>\n",
            "        <contrib contrib-type=\"author\">\n",
            "          <name>\n",
            "            <surname>Su</surname>\n",
            "            <given-names>Leon</given-names>\n",
            "          </name>\n",
            "        </contrib>\n",
            "        <contrib contrib-type=\"author\">\n",
            "          <name>\n",
            "            <surname>David</surname>\n",
            "            <given-names>Michael</given-names>\n",
            "          </name>\n",
            "        </contrib>\n",
            "      </contrib-group>\n",
            "      <aff id=\"N0x1c60f60N0x1df1d98\">Division of Biological Sciences and University of California San Diego Cancer Center, University of California San Diego, La Jolla, CA 92093</aff>\n",
            "      <author-notes>\n",
            "        <fn>\n",
            "          <p>Address correspondence to Michael David, Department of Biology, University of California San Diego, Bonner Hall 3138, 9500 Gilman Drive, La Jolla, CA 92093. Phone: (858) 822-1108; Fax: (858) 822-1106; email: <email>midavid@ucsd.edu</email>\n",
            "</p>\n",
            "        </fn>\n",
            "      </author-notes>\n",
            "      <pub-date pub-type=\"ppub\">\n",
            "        <day>5</day>\n",
            "        <month>1</month>\n",
            "        <year>2004</year>\n",
            "      </pub-date>\n",
            "      <volume>199</volume>\n",
            "      <issue>1</issue>\n",
            "      <fpage>25</fpage>\n",
            "      <lpage>34</lpage>\n",
            "      <history>\n",
            "        <date date-type=\"received\">\n",
            "          <day>1</day>\n",
            "          <month>4</month>\n",
            "          <year>2002</year>\n",
            "        </date>\n",
            "        <date date-type=\"accepted\">\n",
            "          <day>5</day>\n",
            "          <month>11</month>\n",
            "          <year>2003</year>\n",
            "        </date>\n",
            "      </history>\n",
            "      <permissions>\n",
            "        <copyright-statement>Copyright &#xA9; 2004, The Rockefeller University Press</copyright-statement>\n",
            "        <license license-type=\"openaccess\">\n",
            "          <license-p>This article is distributed under the terms of an Attribution&#x2013;Noncommercial&#x2013;Share Alike&#x2013;No Mirror Sites license for the first six months after the publication date (see <ext-link ext-link-type=\"uri\" xlink:href=\"http://www.rupress.org/terms\">http://www.rupress.org/terms</ext-link>). After six months it is available under a Creative Commons License (Attribution&#x2013;Noncommercial&#x2013;Share Alike 4.0 Unported license, as described at <ext-link ext-link-type=\"uri\" xlink:href=\"http://creativecommons.org/licenses/by-nc-sa/4.0/\">http://creativecommons.org/licenses/by-nc-sa/4.0/</ext-link>).</license-p>\n",
            "        </license>\n",
            "      </permissions>\n",
            "      <abstract>\n",
            "        <p>Type I and II interferons (IFNs) exert opposing effects on the progression of multiple sclerosis, even though both IFNs use the signal transducer and activator of transcription 1 (STAT1) as a signaling mediator. Here we report that STAT1-deficient mice expressing a transgenic T cell receptor against myelin basic protein spontaneously develop experimental autoimmune encephalomyelitis with dramatically increased frequency. The heightened susceptibility to this autoimmune disease appears to be triggered by a reduced number as well as a functional impairment of the CD4<sup>+</sup> CD25<sup>+</sup> regulatory T cells in STAT1-deficient animals. Adoptive transfer of wild-type regulatory T cells into STAT1-deficient hosts is sufficient to prevent the development of autoimmune disease. These results demonstrate an essential role of STAT1 in the maintenance of immunological self-tolerance.</p>\n",
            "      </abstract>\n",
            "      <kwd-group>\n",
            "        <kwd>STAT1</kwd>\n",
            "        <kwd>EAE</kwd>\n",
            "        <kwd>autoimmune disease</kwd>\n",
            "        <kwd>regulatory T cells</kwd>\n",
            "      </kwd-group>\n",
            "    </article-meta>\n",
            "    <notes>\n",
            "      <fn-group>\n",
            "        <fn>\n",
            "          <p><italic>Abbreviations used in this paper:</italic> CFSE, carboxyfluorescein diacetate succinimidyl ester; CNS, central nervous system; EAE, experimental autoimmune encephalomyelitis; MBP, myelin basic protein; MS, multiple sclerosis; STAT1, signal transducer and activator of transcription 1; TDS, T cell&#x2013;depleted spleen cells.</p>\n",
            "        </fn>\n",
            "        <fn>\n",
            "          <p>T. Nishibori and Y. Tanabe contributed equally to this work.</p>\n",
            "        </fn>\n",
            "      </fn-group>\n",
            "    </notes>\n",
            "  </front>\n",
            "  <body>\n",
            "    <sec>\n",
            "      <title>Introduction</title>\n",
            "      <p>Multiple sclerosis (MS) is a T cell&#x2013;mediated, demyelinating autoimmune disease of the central nervous system characterized by T cells reactive to the major components of the myelin sheaths, such as myelin basic protein (MBP), myelin oligodendrocyte glycoprotein, and proteolipid protein (<xref rid=\"bib1\" ref-type=\"bibr\">1</xref>). In animals, MS-like symptoms can be induced by immunization with MBP or with peptides representing specific epitopes of proteins associated with the myelin sheats, leading to the development of experimental autoimmune encephalomyelitis (EAE; <xref rid=\"bib2\" ref-type=\"bibr\">2</xref>).</p>\n",
            "      <p>Similarly, MS-like disease symptoms emerge in mice that express a transgenic TCR against MBP (<xref rid=\"bib3\" ref-type=\"bibr\">3</xref>) after immunization of such animals with MBP. In this case, the TCR &#x3B1; and &#x3B2; chains were derived from a CD4<sup>+</sup> encephalomyelitogenic T cell clone that recognizes MBP in an H-2 I-A<sup>u</sup>&#x2013;restricted context. Due to allelic exclusion, &lt;5% of CD4<sup>+</sup> T cells in these animals express antigen receptors of different antigen specificity. In transgenic mice of H-2<sup>u</sup> haplotype, 14% of the animals develop the disease within 12 mo, whereas no symptoms arise in an H-2<sup>b</sup> background. One of the most striking discoveries resulted from the cross of MBP TCR transgenic mice with RAG-1&#x2013;deficient animals. The RAG-1 protein is required for the successful rearrangement of antibody and TCR genes. Thus, in RAG-1&#x2013;deficient mice, lymphocytes can only mature if they express functional antigen receptors as transgenes. RAG-1&#x2013;deficient mice transgenic for the MBP-directed TCR succumb to EAE at a rate of 100% by 12 mo (<xref rid=\"bib4\" ref-type=\"bibr\">4</xref>). These findings unequivocally demonstrate that nontransgenic lymphocytes, which are absent in the RAG-1&#x2013;deficient animals, exert a protective effect against the development of the autoimmune disorder.</p>\n",
            "      <p>One cell population that can mediate a protective effect has been identified as a subset of CD4<sup>+</sup> T cells that coexpress CD25, but are distinct from activated T cells (<xref rid=\"bib5\" ref-type=\"bibr\">5</xref>). Based on their function, this T cell subpopulation has been termed &#x201C;suppressor&#x201D; or &#x201C;regulatory&#x201D; T cells (<xref rid=\"bib6\" ref-type=\"bibr\">6</xref>&#x2013;<xref rid=\"bib8\" ref-type=\"bibr\">8</xref>). Importantly, adoptive transfer of wild-type CD4<sup>+</sup> cells into RAG-1&#x2013;deficient, MBP TCR transgenic mice was found to protect the animals from developing EAE (<xref rid=\"bib9\" ref-type=\"bibr\">9</xref>). Once activated, CD4<sup>+</sup> CD25<sup>+</sup> cells act in an antigen-independent manner (<xref rid=\"bib10\" ref-type=\"bibr\">10</xref>), and appear to represent a unique lineage of CD4<sup>+</sup> T cells, albeit of elusive origin.</p>\n",
            "      <p>Among several immunomodulatory therapies for MS treatment, IFN-&#x3B2; administration has become the treatment of choice. IFNs as well as other cytokines and growth factors mediate their antiviral and antiproliferative effects through the induction of immediate early response genes (<xref rid=\"bib11\" ref-type=\"bibr\">11</xref>). The signal transducer and activator of transcription 1 (STAT1) transcription factor has been identified as an important part of the IFN-&#x3B1;/&#x3B2; and IFN-&#x3B3;&#x2013;induced signaling cascades (<xref rid=\"bib12\" ref-type=\"bibr\">12</xref>).</p>\n",
            "      <p>Several years ago, two labs independently generated STAT1-deficient mice to elucidate the role of STAT1 in vivo (<xref rid=\"bib13\" ref-type=\"bibr\">13</xref>, <xref rid=\"bib14\" ref-type=\"bibr\">14</xref>). As expected, the most prominent phenotype was a significantly increased sensitivity of these animals to viral infections due to their inability to respond to the antiviral effects of IFNs. The lack of IFN-mediated up-regulation of MHC expression also results in an inability to mount an effective immune response to bacterial infections. Interestingly, it appeared that lymphocyte development was not substantially altered in the absence of STAT1 (<xref rid=\"bib13\" ref-type=\"bibr\">13</xref>, <xref rid=\"bib14\" ref-type=\"bibr\">14</xref>). As such, STAT1<sup>&#x2212;/&#x2212;</sup> mice display a normal CD4/CD8 profile in the thymus, and no obvious defects were found in the B cell compartment. A later report described that splenocytes derived from STAT1<sup>&#x2212;/&#x2212;</sup> animals displayed a reduced rate of apoptosis due to a lack of caspase 1 and 11 expression (<xref rid=\"bib15\" ref-type=\"bibr\">15</xref>).</p>\n",
            "      <p>IFNs were first used in MS therapy because of their antiviral properties, as it has been observed that acute viral infection triggers the onset and clinical relapses of the disease. Initially, IFN-&#x3B3; was administered, however, the IFN itself provoked significant exacerbation of the disease (<xref rid=\"bib16\" ref-type=\"bibr\">16</xref>). This observation consequently led to the therapeutic application of IFN-&#x3B1;/&#x3B2; because type I IFNs were known to counteract several actions of IFN-&#x3B3;.</p>\n",
            "      <p>The opposing effects of IFN-&#x3B1;/&#x3B2; and IFN-&#x3B3; in the progression of MS raise the intriguing question as to what role STAT1 plays in the pathogenic process because both IFNs use STAT1 as the primary mediator of their biological actions. To address this issue we used EAE as a model for MS and examined the effects of STAT1 expression on the induction and progression of this autoimmune disease.</p>\n",
            "    </sec>\n",
            "    <sec sec-type=\"materials|methods\">\n",
            "      <title>Materials and Methods</title>\n",
            "      <sec>\n",
            "        <title>Animals.</title>\n",
            "        <p>Wild-type and STAT1<sup>&#x2212;/&#x2212;</sup> 129/SvEv (H-2<sup>b/b</sup>) mice were purchased from Taconic Inc. and mice expressing the transgenic TCR &#x3B1; and &#x3B2; chains (TCR<sup>+</sup>) specific for the MBP (MBP1-9) were provided by S. Tonegawa (Massachusetts Institute of Technology, Cambridge, MA). These animals were crossed to generate STAT1<sup>+/&#x2212;</sup> TCR<sup>+</sup> H-2<sup>u/u</sup> or H-2<sup>b/u</sup> and STAT1<sup>&#x2212;/&#x2212;</sup> TCR<sup>+</sup> H-2<sup>u/u</sup> or H-2<sup>b/u</sup> mice. All mice used in these experiments were maintained in a pathogen-free environment and were bred and cared for in accordance with University of California San Diego Animal Care Facility regulations.</p>\n",
            "      </sec>\n",
            "      <sec>\n",
            "        <title>EAE Scoring and Immunizations.</title>\n",
            "        <p>Clinical symptoms of EAE were graded as follows (<xref rid=\"bib17\" ref-type=\"bibr\">17</xref>): level 0, no symptoms; level 1, limp tail; level 2, partial paralysis of hind legs; level 3, complete paralysis of hind legs; level 4, paralysis of fore and hind legs; level 5, moribund. For some experiments, 6&#x2013;10-wk-old STAT1<sup>+/&#x2212;</sup> TCR &#x3B1;&#x3B2;<sup>1</sup> H-2<sup>b/u</sup> mice were immunized with 200 &#x3BC;g MBP per mouse emulsified in CFA.</p>\n",
            "      </sec>\n",
            "      <sec>\n",
            "        <title>Flow Cytometry and Intracellular Cytokine Staining.</title>\n",
            "        <p>FITC-labeled anti-CD4 mAb, PE-labeled anti-CD8 mAb, FITC-labeled anti-V&#x3B2;8 mAb, PE-labeled anti-CD69, biotin-conjugated anti-CD25 mAb, and biotin-conjugated anti&#x2013;&#x3B3;/&#x3B4; TCR mAb were obtained from BD Biosciences. Biotin-conjugated anti-B220, anti-CD4 mAbs, and APC-conjugated streptavidin were purchased from eBioscience. Anti-CD3&#x25B; mAb (clone 2C11) and IL-2 supernatant were provided by S. Hedrick (University of California San Diego, San Diego, CA).</p>\n",
            "        <p>For immunostaining, single cell suspensions were prepared from thymus, spleen, and peripheral lymph node cells. After removing red blood cells, one million cells were resuspended in the staining buffer (PBS, 1% FCS, 0.02% NaN<sub>3</sub>) and incubated with the antibodies for 30 min on ice. For intracellular cytokine staining, cells were stimulated with 50 ng/ml PMA and 500 ng/ml ionomycin for 4 h. 10 &#x3BC;g/ml brefeldin A was added during the last 2 h of the incubation. Stimulated cells were stained with FITC-labeled anti-CD4 Ab before fixing with 4% paraformaldehyde. Cells were resuspended in permeabilization buffer (PBS, 1% saponin, 1% FCS) and stained with PE-labeled anti&#x2013;IFN-&#x3B3; mAb (BD Biosciences) for 30 min on ice.</p>\n",
            "      </sec>\n",
            "      <sec>\n",
            "        <title>T Cell and APC Purification.</title>\n",
            "        <p>CD4<sup>+</sup> T cells from spleen and thymus were isolated by magnetic sorting using magnetic microbeads (Miltenyi Biotec). In brief, single cell suspensions were incubated with biotinylated anti-CD8, anti-B220, anti-MAC1, anti-Gr1, and anti&#x2013;&#x3B3;/&#x3B4; TCR antibodies (BD Biosciences) for 30 min on ice, washed, and further incubated with anti-biotin beads (Miltenyi Biotec) for 15 min. Cells were then applied to a SuperMacs magnet column and CD4<sup>+</sup> T cells that did not bind to the column were collected. This fraction was then stained with PE-labeled anti-CD25 (PC61) and FITC-labeled anti-CD4 and FACS<sup>&#xAE;</sup> sorted to obtain CD4<sup>+</sup> CD25<sup>+</sup> and CD4<sup>+</sup> CD25<sup>&#x2212;</sup> T cells. For depletion experiments, CD4<sup>+</sup> T cells were additionally labeled with biotinylated anti-CD25 antibody at the same time and CD4<sup>+</sup> CD25<sup>&#x2212;</sup> cells were collected. T cell&#x2013;depleted spleen cells (TDS) as APCs were obtained by removal of T cells with biotinylated anti-CD4, anti-CD8, and anti&#x2013;&#x3B3;/&#x3B4; TCR antibodies as outlined above. APCs were treated with 50 &#x3BC;g/ml mitomycin C for 30 min.</p>\n",
            "      </sec>\n",
            "      <sec>\n",
            "        <title>Carboxyfluorescein Diacetate Succinimidyl Ester (CFSE) Assays.</title>\n",
            "        <p>CFSE was obtained from Molecular Probes, Inc. Purified T cell populations of interest were washed and resuspended in PBS containing 5 &#x3BC;M CFSE. After incubation for 8 min, media containing serum was added and labeled cells were washed and subjected to proliferation assays. T cells (10<sup>6</sup> cells/well) were cultured with mitomycin-treated TDS (3 &#xD7; 10<sup>6</sup> cells/well) as APCs in a 24-well flat-bottom plate in the absence or presence of 1.0 &#x3BC;g/ml Con A. After 3 d of incubation, stained cells were analyzed with a FACScan&#x2122; (Becton Dickinson). To analyze the suppressive properties of regulatory T cells, CD4<sup>+</sup> CD25<sup>&#x2212;</sup> T cells were labeled with CFSE and cultured in 96-well plates (10<sup>5</sup> cells/well) with TDS as APCs (3 &#xD7; 10<sup>5</sup> cells/well), 25 ng/ml anti-CD3 antibody, and the indicated numbers (1:0, 1:0.1, 1:0.2, 1:0.4, and 1:0.8) of unlabeled wild-type or STAT1-deficient CD4<sup>+</sup> CD25<sup>+</sup> cells. After 3 d of incubation, stained cells were analyzed with FACScan&#x2122; (Becton Dickinson).</p>\n",
            "      </sec>\n",
            "      <sec>\n",
            "        <title>Proliferation Assays by [<sup>3</sup>H]thymidine Incorporation.</title>\n",
            "        <p>Purified T cells (2 &#xD7; 10<sup>4</sup> cells/well) were cultured with mitomycin-treated TDS (6 &#xD7; 10<sup>4</sup> cells/well) as APCs in a 96-well round-bottom plate. For stimulation either 1.0 &#x3BC;g/ml Con A or 50 ng/ml anti-CD3 antibody in combination with 50 U/ml human rIL-2 were added. Stimulated cells were cultured in complete medium for 2 d, pulsed with 1 &#x3BC;Ci [<sup>3</sup>H]thymidine per well, and harvested 16 h later. All data shown represent an average of at least three independent experiments.</p>\n",
            "      </sec>\n",
            "      <sec>\n",
            "        <title>Adoptive T Cell Transfer.</title>\n",
            "        <p>CD4<sup>+</sup> T cells or CD4<sup>+</sup> CD25<sup>+</sup> T cells derived from STAT1<sup>+/&#x2212;</sup> TCR<sup>+</sup> H-2<sup>b/u</sup> mice were purified as outlined above, and injected intravenously in 200 &#x3BC;l PBS into 37&#x2013;41-d-old STAT1<sup>&#x2212;/&#x2212;</sup> TCR<sup>+</sup> H-2<sup>b/u</sup> littermates. Recipient mice were monitored every other day for EAE development and progression.</p>\n",
            "      </sec>\n",
            "    </sec>\n",
            "    <sec>\n",
            "      <title>Results</title>\n",
            "      <sec>\n",
            "        <title>STAT1-deficient Mice Are Highly Susceptible to Autoimmune Disease.</title>\n",
            "        <p>Mice transgenic for an MBP-reactive T cell receptor and of the required H-2<sup>u/u</sup> haplotype (<xref rid=\"bib4\" ref-type=\"bibr\">4</xref>) succumb to EAE after immunization with MBP. These transgenes were originally injected into C57BL/6 eggs, and the required H-2<sup>u/u</sup> haplotype was introduced into the transgenic background by crosses with PL/J mice (<xref rid=\"bib4\" ref-type=\"bibr\">4</xref>). Several reports illustrated the need for a homozygous H-2<sup>u/u</sup> background, as mice of H-2<sup>b/u</sup> or H-2<sup>b/b</sup> genotype failed to develop EAE.</p>\n",
            "        <p>To address the role of STAT1 in autoimmune disease development, we crossed these transgenic mice onto STAT1<sup>&#x2212;/&#x2212;</sup> animals with the goal to obtain STAT1-deficient mice bearing the transgenic TCR on an H-2<sup>u/u</sup> background. Surprisingly, we observed already in the F2 generation that several STAT1-deficient animals carrying the transgenic TCR (TCR<sup>+</sup> STAT1<sup>&#x2212;/&#x2212;</sup> H-2<sup>b/u</sup>) spontaneously developed progressive paralysis, even though these mice were of H-2<sup>b/u</sup> genotype.</p>\n",
            "        <p>By the age of 10 wk, 50% of these animals had developed level 5 paralysis, and by 24 wk 90% of the mice had succumbed to EAE (<xref rid=\"fig1\" ref-type=\"fig\">Fig. 1</xref> A). In contrast, none of their TCR<sup>+</sup> STAT1<sup>+/&#x2212;</sup> H-2<sup>b/u</sup> littermates displayed any symptoms. Strikingly, even repeated immunization of TCR<sup>+</sup> ST<sup>+/&#x2212;</sup> H-2<sup>b/u</sup> mice with MBP was unable to provoke EAE development, confirming previously published results on the importance of the H-2<sup>u/u</sup> background (<xref rid=\"bib4\" ref-type=\"bibr\">4</xref>).</p>\n",
            "        <fig id=\"fig1\" position=\"float\">\n",
            "          <label>Figure 1.</label>\n",
            "          <caption>\n",
            "            <p>Spontaneous EAE development in wild-type and STAT1-deficient mice. (A) TCR<sup>+</sup> STAT1<sup>&#x2212;/&#x2212;</sup> H-2<sup>b/u</sup> mice and their TCR<sup>+</sup> STAT1<sup>+/&#x2212;</sup> H-2<sup>b/u</sup> littermates (30 animals/group) were monitored for 6 mo and the spontaneous development of EAE (level 5) was recorded. (B) Same as in A, but TCR<sup>+</sup> STAT1<sup>&#x2212;/&#x2212;</sup> H-2<sup>u/u</sup> mice and their TCR<sup>+</sup> STAT1<sup>+/&#x2212;</sup> H-2<sup>u/u</sup> littermates (20 animals/group) were used.</p>\n",
            "          </caption>\n",
            "          <graphic xlink:href=\"20020509f1\"/>\n",
            "        </fig>\n",
            "        <p>Identical to mice of H-2<sup>b/u</sup> genotype, TCR<sup>+</sup> STAT1<sup>&#x2212;/&#x2212;</sup> H-2<sup>u/u</sup> mice succumb to spontaneous EAE at a drastically increased rate when compared with TCR<sup>+</sup> STAT1<sup>+/&#x2212;</sup> H-2<sup>u/u</sup> animals, whereas TCR<sup>+</sup> STAT1<sup>+/&#x2212;</sup> H-2<sup>u/u</sup> mice also spontaneously developed EAE at the previously published frequency (<xref rid=\"bib4\" ref-type=\"bibr\">4</xref>) of &#x223C;15% (<xref rid=\"fig1\" ref-type=\"fig\">Fig. 1</xref> B).</p>\n",
            "      </sec>\n",
            "      <sec>\n",
            "        <title>Increased IFN-&#x3B3; Production and CD69 Up-regulation in STAT1-deficient Paralyzed Mice.</title>\n",
            "        <p>T cells expressing the transgenic TCR can be identified by immunostaining with V&#x3B2;8-specific antibodies (<xref rid=\"bib4\" ref-type=\"bibr\">4</xref>). To verify that the absence of STAT1 does not affect the expression of the transgenic TCR, peripheral lymph node T cells derived from either 10&#x2013;12-wk-old asymptomatic or level 4 paralyzed TCR<sup>+</sup> STAT1<sup>&#x2212;/&#x2212;</sup> H-2<sup>b/u</sup> mice, or from their heterozygous littermates, were analyzed for V&#x3B2;8 expression. As shown in <xref rid=\"fig2\" ref-type=\"fig\">Fig. 2</xref> A, the fraction of CD4<sup>+</sup> V&#x3B2;8<sup>+</sup> cells (&gt;90%) is virtually identical among the three animal types. Notably, only V&#x3B2;8<sup>+</sup> cells displayed increased CD69 expression in the paralyzed STAT1<sup>&#x2212;/&#x2212;</sup> mice, whereas no such up-regulation is observed in CD4<sup>+</sup> V&#x3B2;8<sup>&#x2212;</sup> cells. In accordance with the manifestation that MS/EAE is a Th1 cell&#x2013;mediated autoimmune disorder, CD4<sup>+</sup> T cells from level 4 paralyzed TCR<sup>+</sup> STAT1<sup>&#x2212;/&#x2212;</sup> H-2<sup>b/u</sup> mice displayed a significant up-regulation in intracellular IFN-&#x3B3; production (<xref rid=\"fig2\" ref-type=\"fig\">Fig. 2</xref> B).</p>\n",
            "        <fig id=\"fig2\" position=\"float\">\n",
            "          <label>Figure 2.</label>\n",
            "          <caption>\n",
            "            <p>Flow cytometric analysis of wild-type, asymptomatic, or paralyzed STAT1-deficient mice. (A) Cells derived from lymph nodes of wild-type, asymptomatic, or paralyzed STAT1-deficient animals were stained for CD4, CD69, and V&#x3B2;8. Horizontal axis represents V&#x3B2;8 staining, vertical axis corresponds to CD4 (top) or CD69 (bottom) staining of CD8<sup>&#x2212;</sup> cells. Cells displayed in lower panels are CD4<sup>+</sup>. (B) Splenocytes derived from wild-type, asymptomatic, or paralyzed STAT1-deficient animals were stimulated with 50 ng/ml PMA/500 ng/ml ionomycin for 4 h (10 &#x3BC;g/ml brefeldin A was added during the last 2 h) and stained for CD4, CD8, and intracellular IFN-&#x3B3;. CD4 staining (horizontal axis) and IFN-&#x3B3; production (vertical axis) of CD8<sup>&#x2212;</sup> cells is shown.</p>\n",
            "          </caption>\n",
            "          <graphic xlink:href=\"20020509f2\"/>\n",
            "        </fig>\n",
            "      </sec>\n",
            "      <sec>\n",
            "        <title>Increased Lymphocyte/Macrophage Infiltration in the Central Nervous System (CNS) of STAT1-deficient Paralyzed Mice.</title>\n",
            "        <p>To ensure that the observed paralysis was indeed the consequence of an inflammatory response of the CNS, brains and spines were removed from asymptomatic or paralyzed STAT1-deficient TCR<sup>+</sup> H-2<sup>b/u</sup> mice and subjected to histopathological analysis after hematoxylin/eosin staining. As shown in <xref rid=\"fig3\" ref-type=\"fig\">Fig. 3</xref> , there is a multifocal to diffuse encephalitis composed of an admixture of neutrophils and mononuclear cells visible in the brain stem and hippocampus of the paralyzed animals. In addition, the spinal cord sections of EAE<sup>+</sup> mice display a focal cavitating necropurulent area at the distal aspect of the section. As such, the histological analysis confirms the presence of massive inflammatory processes and typical EAE lesions in the CNS of TCR<sup>+</sup> STAT1<sup>&#x2212;/&#x2212;</sup> H-2<sup>b/u</sup> mice that had progressed to level 5 paralysis.</p>\n",
            "        <fig id=\"fig3\" position=\"float\">\n",
            "          <label>Figure 3.</label>\n",
            "          <caption>\n",
            "            <p>Immunohistological analysis of CNS tissue from asymptomatic and paralyzed STAT1-deficient mice. Brains and spines were removed from asymptomatic or paralyzed (level 5) STAT1-deficient mice and fixed in formaldehyde. Paraffin-embedded sections were stained with hematoxylin and eosin.</p>\n",
            "          </caption>\n",
            "          <graphic xlink:href=\"20020509f3\"/>\n",
            "        </fig>\n",
            "      </sec>\n",
            "      <sec>\n",
            "        <title>Hyperproliferation of STAT1-deficient CD4<sup>+</sup> T Cells In Vitro.</title>\n",
            "        <p>Total splenocytes isolated from STAT1-deficient animals have been shown to hyperproliferate in vitro (<xref rid=\"bib15\" ref-type=\"bibr\">15</xref>). To explore specifically the mitogenic behavior of CD4<sup>+</sup> cells, these cells were purified from wild-type and STAT1<sup>&#x2212;/&#x2212;</sup> 129/SvEv mice as outlined in Materials and Methods, and subjected to in vitro proliferation assays in the presence of mitomycin-treated, wild-type APCs and 1 &#x3BC;g/ml Con A. Under these conditions STAT1-deficient cells displayed an approximately fourfold increased proliferation rate when compared with their wild-type counterparts (<xref rid=\"fig4\" ref-type=\"fig\">Fig. 4</xref> A).</p>\n",
            "        <fig id=\"fig4\" position=\"float\">\n",
            "          <label>Figure 4.</label>\n",
            "          <caption>\n",
            "            <p>Proliferation of wild-type and STAT1-deficient CD4<sup>+</sup> cells. (A) CD4<sup>+</sup> cells were purified from the spleens of wild-type or STAT1-deficient mice as described in Materials and Methods, and proliferation was assessed by thymidine incorporation in response to stimulation with 1.0 &#x3BC;g/ml Con A in the presence of mitomycin C&#x2013;treated APCs. (B) CD4<sup>+</sup> cells were purified from the spleens of TCR<sup>+</sup> STAT1<sup>&#x2212;/&#x2212;</sup> H-2<sup>b/u</sup> mice and their TCR<sup>+</sup> STAT1<sup>+/&#x2212;</sup> H-2<sup>b/u</sup> littermates, and proliferation was measured by thymidine incorporation. Mitomycin C&#x2013;treated APCs were generated from STAT1<sup>+/&#x2212;</sup> H-2<sup>b/u</sup> animals and preincubated with 100 &#x3BC;g/ml MBP as indicated.</p>\n",
            "          </caption>\n",
            "          <graphic xlink:href=\"20020509f4\"/>\n",
            "        </fig>\n",
            "        <p>To confirm this difference in cell growth in T cells expressing the transgenic TCR and in response to the dominant autoantigen of EAE, CD4<sup>+</sup> cells were also isolated from TCR<sup>+</sup> STAT1<sup>&#x2212;/&#x2212;</sup> H-2<sup>b/u</sup> mice and their TCR<sup>+</sup> STAT1<sup>+/&#x2212;</sup> H-2<sup>b/u</sup> littermates. Mitomycin C&#x2013;treated APCs were generated from STAT1<sup>+/&#x2212;</sup> H-2<sup>b/u</sup> animals and preincubated with 100 &#x3BC;g/ml MBP as indicated. Similar to nonspecific stimulation by Con A, CD4<sup>+</sup> T cells derived from TCR<sup>+</sup> STAT1<sup>&#x2212;/&#x2212;</sup> H-2<sup>b/u</sup> mice responded with substantially increased proliferation in the presence of APCs presenting MBP antigen, whereas no mitogenesis was observed without APCs or with APCs in the absence of MBP (<xref rid=\"fig4\" ref-type=\"fig\">Fig. 4</xref> B).</p>\n",
            "      </sec>\n",
            "      <sec>\n",
            "        <title>Reduced Number of CD4<sup>+</sup> CD25<sup>+</sup> Regulatory T Cells in the Absence of STAT1.</title>\n",
            "        <p>The dramatically increased frequency of spontaneous EAE development in TCR<sup>MBP</sup>-transgenic STAT1-deficient animals is reminiscent of the phenotype observed in mice expressing the TCR<sup>MBP</sup> in the absence of RAG-1 (<xref rid=\"bib4\" ref-type=\"bibr\">4</xref>). In this model, a deficiency of CD4<sup>+</sup> CD25<sup>+</sup> regulatory T cells was found to be ultimately responsible for the lack of immunological self-tolerance (<xref rid=\"bib9\" ref-type=\"bibr\">9</xref>). Therefore, we decided to analyze thymic CD4<sup>+</sup> cells for the presence of the CD25<sup>+</sup> subpopulation in wild-type and STAT1<sup>&#x2212;/&#x2212;</sup> 129/SvEv mice. In wild-type animals, 9&#x2013;10% of the CD4<sup>+</sup> cells were CD25<sup>+</sup>, conforming to previously published findings. In contrast, only &#x223C;5% of CD4<sup>+</sup> cells accounted for CD25<sup>+</sup> regulatory T cells in STAT1-deficient animals (<xref rid=\"fig5\" ref-type=\"fig\">Fig. 5</xref>) . Similar, albeit smaller differences were observed in the peripheral lymphoid organs (not depicted). We further analyzed the CD4<sup>+</sup> CD25<sup>+</sup> T cells from wild-type and STAT1<sup>&#x2212;/&#x2212;</sup> mice for the expression of additional cell surface markers such as CTLA-4, CD45RB, and CD62L, which are typical for regulatory T cells, but did not observe any significant differences between these animals (not depicted).</p>\n",
            "        <fig id=\"fig5\" position=\"float\">\n",
            "          <label>Figure 5.</label>\n",
            "          <caption>\n",
            "            <p>(A and B) Flow cytometric analysis of thymocytes in wild-type and STAT1-deficient mice. Cells derived from thymi of wild-type or STAT1-deficient animals were stained for CD4, CD8, and CD25. The ratio of CD4<sup>+</sup> CD8<sup>&#x2212;</sup> CD25<sup>+</sup> cells as the percentage of total CD4<sup>+</sup> CD8<sup>&#x2212;</sup> cells is shown.</p>\n",
            "          </caption>\n",
            "          <graphic xlink:href=\"20020509f5\"/>\n",
            "        </fig>\n",
            "      </sec>\n",
            "      <sec>\n",
            "        <title>Wild-type CD4<sup>+</sup> Cells Suppress the Hyperproliferation of STAT1-deficient T Cells.</title>\n",
            "        <p>The finding of a reduced CD4<sup>+</sup> CD25<sup>+</sup> T subpopulation in STAT1<sup>&#x2212;/&#x2212;</sup> mice raised the possibility that the hyperproliferation of STAT1-deficient CD4<sup>+</sup> cells is due to a loss of the intrinsic suppressor effect of regulatory T cells. Therefore, we tested whether wild-type CD4<sup>+</sup> cells had the ability to restrain the proliferation of their STAT1-deficient counterparts. CD4<sup>+</sup> cells were purified from wild-type and STAT1<sup>&#x2212;/&#x2212;</sup> 129/SvEv mice and subjected to thymidine incorporation assays. As found previously, STAT1<sup>&#x2212;/&#x2212;</sup> cells displayed an approximately fourfold increase in the Con A&#x2013;stimulated mitogenic response when compared with wild-type cells. However, mixing of wild-type and STAT1-deficient CD4<sup>+</sup> cells in a 50:50 ratio resulted in a proliferation rate significantly lower than the expected average of the two individual populations, suggesting that wild-type CD4<sup>+</sup> cells exert a suppressive effect on their STAT1<sup>&#x2212;/&#x2212;</sup> equivalents (<xref rid=\"fig6\" ref-type=\"fig\">Fig. 6</xref> A). To further investigate whether the wild-type CD4<sup>+</sup> cells mediate their suppressive function truly through the activities of CD4<sup>+</sup> CD25<sup>+</sup> regulatory T cells, we performed analogous mixing experiments, this time with wild-type CD4<sup>+</sup> cells depleted of their CD25<sup>+</sup> subpopulation. Indeed, removal of CD25<sup>+</sup> cells from the wild-type CD4<sup>+</sup> cells resulted in significant reduction in their ability to exert antiproliferative effects upon STAT1-deficient cells (<xref rid=\"fig6\" ref-type=\"fig\">Fig. 6</xref> A). This finding is not restricted to Con A stimulation, as a similar pattern of trans-suppression emerged when anti-CD3 antibodies were used for TCR stimulation in the presence of exogenous IL-2 (<xref rid=\"fig6\" ref-type=\"fig\">Fig. 6</xref> B). The trans-suppressive effect required the coculture of the CD4<sup>+</sup> cells from both animals, as the transfer of conditioned media from wild-type cells into the cultures of their STAT1-deficient counterparts was unable to reproduce the antiproliferative effect (not depicted).</p>\n",
            "        <fig id=\"fig6\" position=\"float\">\n",
            "          <label>Figure 6.</label>\n",
            "          <caption>\n",
            "            <p>Effect of wild-type CD4<sup>+</sup> CD25<sup>+</sup> cells on the proliferation of STAT1<sup>&#x2212;/&#x2212;</sup> CD4<sup>+</sup> cells. (A) CD4<sup>+</sup> cells were purified from the spleens of wild-type or STAT1-deficient mice as described in Materials and Methods. Wild-type or STAT1-deficient CD4<sup>+</sup> cells were either incubated separately, mixed in a 1:1 ratio, or wild-type cells were CD25 depleted before mixing 1:1 with STAT1-deficient cells as indicated. Proliferation was then assessed by thymidine incorporation in response to stimulation with 1.0 &#x3BC;g/ml Con A in the presence of mitomycin C&#x2013;treated APCs. (B) Same as in A, but cells were stimulated with 50 ng/ml anti-CD3 antibody in combination with 50 U/ml human rIL-2. (C) CD4<sup>+</sup> cells were purified from the spleens of wild-type or STAT1-deficient mice as described in Materials and Methods. STAT1-deficient cells were labeled with CFSE and their proliferation in response to 1.0 &#x3BC;g/ml Con A in the presence of either unlabeled wild-type CD4<sup>+</sup> cells or unlabeled CD25-depleted wild-type CD4<sup>+</sup> cells (1:1 ratio) was determined by flow cytometry. (D) CD4<sup>+</sup> cells were purified from the spleens of wild-type mice, depleted of CD25<sup>+</sup> cells, and labeled with CFSE. Proliferation in response to 1.0 &#x3BC;g/ml Con A in the presence of either unlabeled STAT1<sup>&#x2212;/&#x2212;</sup> CD4<sup>+</sup> cells or unlabeled CD25-depleted STAT1<sup>&#x2212;/&#x2212;</sup> CD4<sup>+</sup> cells (1:1 ratio) was determined by flow cytometry.</p>\n",
            "          </caption>\n",
            "          <graphic xlink:href=\"20020509f6ab\"/>\n",
            "          <graphic xlink:href=\"20020509f6c\"/>\n",
            "          <graphic xlink:href=\"20020509f6d\"/>\n",
            "        </fig>\n",
            "      </sec>\n",
            "      <sec>\n",
            "        <title>Wild-type, but Not STAT1-deficient CD4<sup>+</sup> CD25<sup>+</sup> Cells Suppress T Cell Proliferation.</title>\n",
            "        <p>Next, we wanted to ensure that the reduced rate of [<sup>3</sup>H]thymidine incorporation in the mixed T cell populations was indeed due to a suppressive effect of the wild-type CD4<sup>+</sup> CD25<sup>+</sup> regulatory T cells on the proliferation of STAT1-deficient cells. To specifically measure the mitogenic response of only one of the T cell populations in the mixture, we used CFSE staining to analyze cell proliferation. CD4<sup>+</sup> cells were derived from wild-type and STAT1<sup>&#x2212;/&#x2212;</sup> mice, however, only STAT1-deficient cells were labeled with CFSE such that only proliferation of these cells, but not of added wild-type cells, would be monitored. Stimulation with 1 &#x3BC;g/ml Con A resulted in a massive proliferative response of STAT1<sup>&#x2212;/&#x2212;</sup> CD4<sup>+</sup> cells (<xref rid=\"fig6\" ref-type=\"fig\">Fig. 6</xref> C, panels 1 and 2), which was considerably reduced when unlabeled, wild-type CD4<sup>+</sup> T cells were present during the incubation (<xref rid=\"fig6\" ref-type=\"fig\">Fig. 6</xref> C, panel 3). Removal of CD25<sup>+</sup> cells from the wild-type cells before their addition to labeled, STAT1-deficient cells resulted in a significant, albeit not complete, abrogation of the antiproliferative effect (<xref rid=\"fig6\" ref-type=\"fig\">Fig. 6</xref> C, panel 4).</p>\n",
            "        <p>CD4<sup>+</sup> CD25<sup>+</sup> T cells can still be found in STAT1<sup>&#x2212;/&#x2212;</sup> mice, even though their population is &#x223C;50% reduced compared with wild-type animals. Because it seemed unlikely that the lower ratio of CD4<sup>+</sup> CD25<sup>+</sup> cells alone would account for the substantial differences in cell proliferation, we decided to analyze the suppressive ability of the remaining STAT1-deficient CD4<sup>+</sup> CD25<sup>+</sup> cells. Resembling the approach outlined above, wild-type CD4<sup>+</sup> cells were labeled with CFSE after they had been depleted of their own CD25<sup>+</sup> subpopulation. Subsequently, proliferation of these cells was monitored in the absence or presence of unlabeled STAT1-deficient CD4<sup>+</sup> cells. Interestingly, the addition of STAT1<sup>&#x2212;/&#x2212;</sup> CD4<sup>+</sup> cells was unable to trigger any change in the proliferation rate of their wild-type counterparts (<xref rid=\"fig6\" ref-type=\"fig\">Fig. 6</xref> D, panels 2 and 3). Accordingly, the removal of CD4<sup>+</sup> CD25<sup>+</sup> cells from the STAT1-deficient population was also without any effect on the mitogenic response of the wild-type CD4<sup>+</sup> cells (<xref rid=\"fig6\" ref-type=\"fig\">Fig. 6</xref> D, panel 4).</p>\n",
            "        <p>To further corroborate these results, we also added increasing numbers of purified wild-type or STAT1-deficient CD4<sup>+</sup> CD25<sup>+</sup> cells to either wild-type or STAT1-deficient CD4<sup>+</sup> CD25<sup>&#x2212;</sup> cells that had been labeled with CFSE to analyze their proliferation. As shown in <xref rid=\"fig7\" ref-type=\"fig\">Fig. 7</xref> , wild-type CD4<sup>+</sup> CD25<sup>+</sup> cells were able to suppress the proliferation of their wild-type or STAT1-deficient CD25<sup>+</sup> counterparts in a dose-dependent manner (striped bars). In contrast, STAT1<sup>&#x2212;/&#x2212;</sup> CD4<sup>+</sup> CD25<sup>+</sup> cells failed to exert any growth inhibitory effect on either wild-type or STAT1<sup>&#x2212;/&#x2212;</sup> CD4<sup>+</sup> CD25<sup>&#x2212;</sup> cells (dotted bars). These results not only demonstrate that STAT1<sup>&#x2212;/&#x2212;</sup> CD4<sup>+</sup> CD25<sup>+</sup> cells fail to act as functional regulatory T cells, but also illustrate that STAT1<sup>&#x2212;/&#x2212;</sup> CD4<sup>+</sup> CD25<sup>&#x2212;</sup> cells retain their ability to respond to the inhibitory effects of wild-type CD4<sup>+</sup> CD25<sup>+</sup> cells.</p>\n",
            "        <fig id=\"fig7\" position=\"float\">\n",
            "          <label>Figure 7.</label>\n",
            "          <caption>\n",
            "            <p>STAT1-deficient CD4<sup>+</sup> CD25<sup>+</sup> cells fail to suppress T cell proliferation. CD4<sup>+</sup> CD25<sup>+</sup> and CD4<sup>+</sup> CD25<sup>&#x2212;</sup> cells were purified from spleens as described in Materials and Methods. Wild-type or STAT1-deficient CD4<sup>+</sup> CD25<sup>&#x2212;</sup> cells were labeled with CFSE and their proliferation in response to 1.0 &#x3BC;g/ml Con A in the absence or presence of either unlabeled wild-type or STAT1-deficient CD4<sup>+</sup> CD25<sup>+</sup> cells was determined by flow cytometry (ratios of CD4<sup>+</sup> CD25<sup>&#x2212;</sup>/CD4<sup>+</sup> CD25<sup>+</sup> cells were 1:0.1, 1:0.2, 1:0.4, and 1:0.8 in lanes 3&#x2013;6 and 10&#x2013;13, respectively). A representative of three independent experiments is shown.</p>\n",
            "          </caption>\n",
            "          <graphic xlink:href=\"20020509f7\"/>\n",
            "        </fig>\n",
            "      </sec>\n",
            "      <sec>\n",
            "        <title>Adoptive Transfer of Wild-type CD4<sup>+</sup> CD25<sup>+</sup> Cells Prevents EAE Development in STAT1<sup>&#x2212;/&#x2212;</sup> TCR<sup>+</sup> Mice.</title>\n",
            "        <p>To determine whether the observed deficiencies in CD4<sup>+</sup> CD25<sup>+</sup> regulatory T cell function are indeed responsible for the increased susceptibility of STAT1-deficient mice toward the development of EAE, we transferred either total CD4<sup>+</sup> cells or CD4<sup>+</sup> CD25<sup>+</sup> cells derived from TCR<sup>+</sup> STAT1<sup>+/&#x2212;</sup> H-2<sup>b/u</sup> into their TCR<sup>+</sup> STAT1<sup>&#x2212;/&#x2212;</sup> H-2<sup>b/u</sup> littermates. As summarized in <xref ref-type=\"table\" rid=\"tbl1\">Table I</xref>, 80% of TCR<sup>+</sup> STAT1<sup>+/&#x2212;</sup> H-2<sup>b/u</sup> mice developed level 5 paralysis at the age of 17 wk. In contrast, mice that had received wild-type CD4<sup>+</sup> cells succumbed to the disease at a substantially reduced (32%) rate. This protective effect was further increased when purified CD4<sup>+ </sup>CD25<sup>+</sup> cells were transferred into the TCR<sup>+</sup> STAT1<sup>+/&#x2212;</sup> H-2<sup>b/u</sup> mice, illustrating the physiological importance of this cell population in the protection against EAE.</p>\n",
            "        <table-wrap id=\"tbl1\" position=\"float\">\n",
            "          <label>Table I.</label>\n",
            "          <caption>\n",
            "            <p>Protection of STAT1<sup>&#x2212;/&#x2212;</sup> TCR against EAE after Adoptive Transfer of CD4<sup>+</sup> T Cells</p>\n",
            "          </caption>\n",
            "          <table frame=\"hsides\" rules=\"groups\">\n",
            "            <thead>\n",
            "              <tr>\n",
            "                <th colspan=\"1\" rowspan=\"1\" align=\"left\">Transferred WT cells</th>\n",
            "                <th colspan=\"1\" rowspan=\"1\" align=\"center\">Lethality after 17 wk (%)</th>\n",
            "              </tr>\n",
            "            </thead>\n",
            "            <tbody>\n",
            "              <tr>\n",
            "                <td colspan=\"1\" rowspan=\"1\" align=\"left\">No transfer</td>\n",
            "                <td colspan=\"1\" rowspan=\"1\" align=\"center\">24/30 (80)</td>\n",
            "              </tr>\n",
            "              <tr>\n",
            "                <td colspan=\"1\" rowspan=\"1\" align=\"left\">CD4<sup>+</sup> (10<sup>6</sup> cells)</td>\n",
            "                <td colspan=\"1\" rowspan=\"1\" align=\"center\">2/6 (32)</td>\n",
            "              </tr>\n",
            "              <tr>\n",
            "                <td colspan=\"1\" rowspan=\"1\" align=\"left\">CD4<sup>+</sup> CD25<sup>+</sup> (5 &#xD7; 10<sup>4</sup> cells)</td>\n",
            "                <td colspan=\"1\" rowspan=\"1\" align=\"center\">1/6 (16)</td>\n",
            "              </tr>\n",
            "            </tbody>\n",
            "          </table>\n",
            "        </table-wrap>\n",
            "      </sec>\n",
            "    </sec>\n",
            "    <sec>\n",
            "      <title>Discussion</title>\n",
            "      <p>The immunomodulatory functions of IFNs have long provoked interest in the role of these cytokines in the development or treatment of numerous autoimmune disorders, particularly of MS (<xref rid=\"bib18\" ref-type=\"bibr\">18</xref>). The opposing effects that types I and II IFNs elicit in the progression of the disease prompted intense investigation into the underlying mechanism (<xref rid=\"bib16\" ref-type=\"bibr\">16</xref>). It is generally accepted that MS, as well as the widely used murine model EAE, represent Th1-mediated autoimmune diseases in which CD4<sup>+</sup> cells specific for MBP trigger demyelination of the CNS with devastating consequences (<xref rid=\"bib1\" ref-type=\"bibr\">1</xref>). The immune system in STAT1-deficient mice is skewed toward Th2-type responses (<xref rid=\"bib19\" ref-type=\"bibr\">19</xref>). Therefore, contrary to our observations, a reduced sensitivity toward Th1-mediated autoimmune diseases would be expected. Intriguingly, 35% of the CD4<sup>+</sup> cells in paralyzed TCR<sup>+</sup> STAT1<sup>&#x2212;/&#x2212;</sup> H-2<sup>b/u</sup> mice stained positive for intracellular IFN-&#x3B3;, demonstrating that these animals were able to mount a strong Th1 response despite their inability to respond to IFN-&#x3B3;.</p>\n",
            "      <p>Previous studies have shown that STAT1-deficient cells lack the expression of several caspase family members (<xref rid=\"bib15\" ref-type=\"bibr\">15</xref>, <xref rid=\"bib20\" ref-type=\"bibr\">20</xref>), thus raising the possibility of an increased susceptibility toward autoimmune disease due to a defect in the negative selection of autoreactive T cells in nontransgenic mice. However, virtually identical numbers of TCR<sup>MBP</sup>-transgenic T cells were found in the peripheral lymphoid organs of TCR<sup>+</sup> wild-type and STAT1-deficient animals, making the possibility of reduced negative selection as the reason for the increased EAE rate in STAT1<sup>&#x2212;/&#x2212;</sup> mice rather unlikely.</p>\n",
            "      <p>In mice carrying the transgene for the MBP-directed TCR, &#x223C;95% of the CD4<sup>+</sup> cells actually express the TCR<sup>MBP</sup>, whereas the remaining cells harbor TCRs of undetermined specificity. This minor population of phenotypically nontransgenic CD4<sup>+</sup> T cells is absolutely crucial for maintaining immunological self-tolerance, as evidenced by crosses of the TCR<sup>MBP</sup>-transgenic mice onto Rag1<sup>&#x2212;/&#x2212;</sup> animals (<xref rid=\"bib4\" ref-type=\"bibr\">4</xref>). There, 100% of the animals succumb spontaneously to EAE due to the complete absence of regulatory T cells. Further support for the existence of regulatory T cells stems from the observation that reconstitution of immune-deficient mice with CD4<sup>+</sup> cells depleted of CD4<sup>+</sup> CD25<sup>+</sup> cells induces autoimmune disease, whereas complementation with CD4<sup>+</sup> CD25<sup>+</sup> cells prevents autoimmune syndromes elicited by transfer of CD4<sup>+</sup> CD25<sup>&#x2212;</sup> cells (<xref rid=\"bib9\" ref-type=\"bibr\">9</xref>, <xref rid=\"bib21\" ref-type=\"bibr\">21</xref>). Although the existence of such a &#x201C;suppressor&#x201D; T cell population was originally somewhat controversial, the presence of protective regulatory CD4<sup>+</sup> cells is now well accepted (<xref rid=\"bib7\" ref-type=\"bibr\">7</xref>, <xref rid=\"bib22\" ref-type=\"bibr\">22</xref>&#x2013;<xref rid=\"bib26\" ref-type=\"bibr\">26</xref>).</p>\n",
            "      <p>Crosses of TCR<sup>MBP</sup> mice onto STAT1-null mice results in essentially the same pathological phenotype as the respective cross to RAG-1&#x2013;deficient animals and is accompanied by a significant reduction in the number of CD4<sup>+ </sup>CD25<sup>+</sup> cells. Further evidence for a disturbed balance of CD4<sup>+</sup> CD25<sup>+</sup> cells in the absence of STAT1 comes from the finding that wild-type CD4<sup>+</sup> CD25<sup>+</sup> cells are able to suppress the proliferation of STAT1<sup>&#x2212;/&#x2212;</sup> T cells, whereas conversely, STAT1-deficient CD4<sup>+</sup> CD25<sup>+</sup> cells fail to exert any effect on CD4<sup>+</sup> CD25<sup>&#x2212;</sup> cells derived from either wild-type or STAT1<sup>&#x2212;/&#x2212;</sup> mice. In accordance with previously published observations that the function of regulatory T cells requires cell to cell contact, conditioned media from wild-type T cell cultures was unable to inhibit the excessive proliferative response of STAT1-deficient CD4<sup>+</sup> cells. The final proof that diminished CD4<sup>+</sup> CD25<sup>+</sup> cell function accounts for the increased incidence of EAE in the absence of STAT1 comes from the fact that adoptive transfer of this cell population from wild-type mice into their STAT1<sup>&#x2212;/&#x2212;</sup> littermates protected the host animals against the development of the disease.</p>\n",
            "      <p>Many key aspects of the origin of regulatory T cells remain unsolved. B7 and CD28 are required elements in the generation of CD4<sup>+</sup> CD25<sup>+</sup> cells (<xref rid=\"bib27\" ref-type=\"bibr\">27</xref>) as are CD62L (<xref rid=\"bib28\" ref-type=\"bibr\">28</xref>) and IL-2 (<xref rid=\"bib29\" ref-type=\"bibr\">29</xref>). IL-12 is also a crucial factor in the development of EAE, as IL-12<sup>&#x2212;/&#x2212;</sup> mice are completely resistant to the disease (<xref rid=\"bib30\" ref-type=\"bibr\">30</xref>). The efficacy of type I IFNs in the treatment of MS is likely due to the suppressive effects of IFN-&#x3B1;/&#x3B2; on the IL-12 production by APCs (<xref rid=\"bib16\" ref-type=\"bibr\">16</xref>). However, initial theories that IL-12 promotes EAE development by inducing IFN-&#x3B3; are contradicted by the finding that IFN-&#x3B3;&#x2013; or IFN-&#x3B3;R&#x2013;deficient mice still develop EAE, surprisingly with even higher incidence (<xref rid=\"bib31\" ref-type=\"bibr\">31</xref>, <xref rid=\"bib32\" ref-type=\"bibr\">32</xref>). The latter finding prompted the hypothesis that IFN-&#x3B3; plays an important role in the development and activation of regulatory T cells (<xref rid=\"bib33\" ref-type=\"bibr\">33</xref>). The presence and functional integrity of regulatory T cells in IFN-&#x3B3;&#x2013; or IFN-&#x3B3;R&#x2013;deficient mice remains to be determined. Intriguingly, a recent study reported that STAT1 and Jak2 expression are significantly increased after TCR stimulation in CD4<sup>+</sup> CD25<sup>+</sup> regulatory cells, but is not affected in CD4<sup>+</sup> CD25<sup>&#x2212;</sup> T cells (<xref rid=\"bib34\" ref-type=\"bibr\">34</xref>). The results presented in this work support the notion that STAT1 signaling might be required for the development and function of functional regulatory T cells.</p>\n",
            "    </sec>\n",
            "  </body>\n",
            "  <back>\n",
            "    <ack>\n",
            "      <p>TCR<sup>MBP</sup>-transgenic mice were kindly provided by Dr. S. Tonegawa. Anti-CD3 antibody and rhuIL-2 were generously supplied by Dr. S. Hedrick. We would also like to thank Drs. R. Rickert, C. Walsh, and S. Hedrick for many helpful discussions, and Drs. D. Otero and C. Meschter for their technical assistance.</p>\n",
            "      <p>This work was supported by National Institutes of Health grant CA80105 (to M. David) and a grant from the National Multiple Sclerosis Society (to M. David).</p>\n",
            "    </ack>\n",
            "    <ref-list>\n",
            "      <ref id=\"bib1\">\n",
            "        <label>1</label>\n",
            "        <mixed-citation publication-type=\"journal\">Steinman, L. <year>1996</year>. Multiple sclerosis: a coordinated immunological attack against myelin in the central nervous system. <source>Cell.</source>\n",
            "<volume>85</volume>:<fpage>299</fpage>&#x2013;302.\n",
            "<pub-id pub-id-type=\"pmid\">8616884</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib2\">\n",
            "        <label>2</label>\n",
            "        <mixed-citation publication-type=\"journal\">Steinman, L. <year>1999</year>. Assessment of animal models for MS and demyelinating disease in the design of rational therapy. <source>Neuron.</source>\n",
            "<volume>24</volume>:<fpage>511</fpage>&#x2013;514.\n",
            "<pub-id pub-id-type=\"pmid\">10595504</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib3\">\n",
            "        <label>3</label>\n",
            "        <mixed-citation publication-type=\"journal\">Goverman, J., A. Woods, L. Larson, L.P. Weiner, L. Hood, and D.M. Zaller. <year>1993</year>. Transgenic mice that express a myelin basic protein-specific T cell receptor develop spontaneous autoimmunity. <source>Cell.</source>\n",
            "<volume>72</volume>:<fpage>551</fpage>&#x2013;560.\n",
            "<pub-id pub-id-type=\"pmid\">7679952</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib4\">\n",
            "        <label>4</label>\n",
            "        <mixed-citation publication-type=\"journal\">Lafaille, J.J., K. Nagashima, M. Katsuki, and S. Tonegawa. <year>1994</year>. High incidence of spontaneous autoimmune encephalomyelitis in immunodeficient anti-myelin basic protein T cell receptor transgenic mice. <source>Cell.</source>\n",
            "<volume>78</volume>:<fpage>399</fpage>&#x2013;408.\n",
            "<pub-id pub-id-type=\"pmid\">7520367</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib5\">\n",
            "        <label>5</label>\n",
            "        <mixed-citation publication-type=\"journal\">Sakaguchi, S., N. Sakaguchi, M. Asano, M. Itoh, and M. Toda. <year>1995</year>. Immunologic self-tolerance maintained by activated T cells expressing IL-2 receptor alpha-chains (CD25). Breakdown of a single mechanism of self-tolerance causes various autoimmune diseases. <source>J. Immunol.</source>\n",
            "<volume>155</volume>:<fpage>1151</fpage>&#x2013;1164.\n",
            "<pub-id pub-id-type=\"pmid\">7636184</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib6\">\n",
            "        <label>6</label>\n",
            "        <mixed-citation publication-type=\"journal\">Suri-Payer, E., A.Z. Amar, A.M. Thornton, and E.M. Shevach. <year>1998</year>. CD4+CD25+ T cells inhibit both the induction and effector function of autoreactive T cells and represent a unique lineage of immunoregulatory cells. <source>J. Immunol.</source>\n",
            "<volume>160</volume>:<fpage>1212</fpage>&#x2013;1218.\n",
            "<pub-id pub-id-type=\"pmid\">9570536</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib7\">\n",
            "        <label>7</label>\n",
            "        <mixed-citation publication-type=\"journal\">Shevach, E.M. <year>2001</year>. Certified professionals: CD4<sup>+</sup> CD25<sup>+</sup> suppressor T cells. <source>J. Exp. Med.</source>\n",
            "<volume>193</volume>:<fpage>F41</fpage>&#x2013;F46.\n",
            "<pub-id pub-id-type=\"pmid\">11390442</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib8\">\n",
            "        <label>8</label>\n",
            "        <mixed-citation publication-type=\"journal\">Thornton, A.M., and E.M. Shevach. <year>1998</year>. CD4<sup>+</sup> CD25<sup>+</sup> immunoregulatory T cells suppress polyclonal T cell activation in vitro by inhibiting interleukin 2 production. <source>J. Exp. Med.</source>\n",
            "<volume>188</volume>:<fpage>287</fpage>&#x2013;296.\n",
            "<pub-id pub-id-type=\"pmid\">9670041</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib9\">\n",
            "        <label>9</label>\n",
            "        <mixed-citation publication-type=\"journal\">Van de Keere, F., and S. Tonegawa. <year>1998</year>. CD4<sup>+</sup> T cells prevent spontaneous experimental autoimmune encephalomyelitis in anti-myelin basic protein T cell receptor transgenic mice. <source>J. Exp. Med.</source>\n",
            "<volume>188</volume>:<fpage>1875</fpage>&#x2013;1882.\n",
            "<pub-id pub-id-type=\"pmid\">9815265</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib10\">\n",
            "        <label>10</label>\n",
            "        <mixed-citation publication-type=\"journal\">Thornton, A.M., and E.M. Shevach. <year>2000</year>. Suppressor effector function of CD4+CD25+ immunoregulatory T cells is antigen nonspecific. <source>J. Immunol.</source>\n",
            "<volume>164</volume>:<fpage>183</fpage>&#x2013;190.\n",
            "<pub-id pub-id-type=\"pmid\">10605010</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib11\">\n",
            "        <label>11</label>\n",
            "        <mixed-citation publication-type=\"journal\">Schindler, C., and J.E. Darnell, Jr. <year>1995</year>. Transcriptional responses to polypeptide ligands: the JAK-STAT pathway. <source>Annu. Rev. Biochem.</source>\n",
            "<volume>64</volume>:<fpage>621</fpage>&#x2013;651.\n",
            "<pub-id pub-id-type=\"pmid\">7574495</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib12\">\n",
            "        <label>12</label>\n",
            "        <mixed-citation publication-type=\"journal\">Muller, M., C. Laxton, J. Briscoe, C. Schindler, T. Improta, J.E. Darnell, Jr., G.R. Stark, and I.M. Kerr. <year>1993</year>. Complementation of a mutant cell line: central role of the 91 kDa polypeptide of ISGF3 in the interferon-&#x3B1; and -&#x3B3; signal transduction pathways. <source>EMBO J.</source>\n",
            "<volume>12</volume>:<fpage>4221</fpage>&#x2013;4228.\n",
            "<pub-id pub-id-type=\"pmid\">7693454</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib13\">\n",
            "        <label>13</label>\n",
            "        <mixed-citation publication-type=\"journal\">Meraz, M.A., J.M. White, K.C.F. Sheehan, E.A. Bach, S.J. Rodig, A.S. Dighe, D.H. Kaplan, J.K. Riley, A.C. Greenlund, D. Campbell, et al. <year>1996</year>. Targeted disruption of the <italic>Stat1</italic> gene in mice reveals unexpected physiologic specificity in the JAK-STAT signaling pathway. <source>Cell.</source>\n",
            "<volume>84</volume>:<fpage>431</fpage>&#x2013;442.\n",
            "<pub-id pub-id-type=\"pmid\">8608597</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib14\">\n",
            "        <label>14</label>\n",
            "        <mixed-citation publication-type=\"journal\">Durbin, J.E., R. Hackenmiller, M.C. Simon, and D.E. Levy. <year>1996</year>. Targeted disruption of the mouse <italic>Stat1</italic> gene results in compromised innate immunity to viral disease. <source>Cell.</source>\n",
            "<volume>84</volume>:<fpage>443</fpage>&#x2013;450.\n",
            "<pub-id pub-id-type=\"pmid\">8608598</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib15\">\n",
            "        <label>15</label>\n",
            "        <mixed-citation publication-type=\"journal\">Lee, C.K., E. Smith, R. Gimeno, R. Gertner, and D.E. Levy. <year>2000</year>. STAT1 affects lymphocyte survival and proliferation partially independent of its role downstream of IFN-gamma. <source>J. Immunol.</source>\n",
            "<volume>164</volume>:<fpage>1286</fpage>&#x2013;1292.\n",
            "<pub-id pub-id-type=\"pmid\">10640742</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib16\">\n",
            "        <label>16</label>\n",
            "        <mixed-citation publication-type=\"journal\">Karp, C.L., C.A. Biron, and D.N. Irani. <year>2000</year>. Interferon beta in multiple sclerosis: is IL-12 suppression the key? <source>Immunol. Today.</source>\n",
            "<volume>21</volume>:<fpage>24</fpage>&#x2013;28.\n",
            "<pub-id pub-id-type=\"pmid\">10637555</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib17\">\n",
            "        <label>17</label>\n",
            "        <mixed-citation publication-type=\"journal\">Baron, J.L., J.A. Madri, N.H. Ruddle, G. Hashim, and C.A. Janeway, Jr. <year>1993</year>. Surface expression of alpha 4 integrin by CD4 T cells is required for their entry into brain parenchyma. <source>J. Exp. Med.</source>\n",
            "<volume>177</volume>:<fpage>57</fpage>&#x2013;68.\n",
            "<pub-id pub-id-type=\"pmid\">7678116</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib18\">\n",
            "        <label>18</label>\n",
            "        <mixed-citation publication-type=\"journal\">Biron, C.A. <year>2001</year>. Interferons alpha and beta as immune regulators&#x2013;a new look. <source>Immunity.</source>\n",
            "<volume>14</volume>:<fpage>661</fpage>&#x2013;664.\n",
            "<pub-id pub-id-type=\"pmid\">11420036</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib19\">\n",
            "        <label>19</label>\n",
            "        <mixed-citation publication-type=\"journal\">Durbin, J.E., A. Fernandez-Sesma, C.K. Lee, T.D. Rao, A.B. Frey, T.M. Moran, S. Vukmanovic, A. Garcia-Sastre, and D.E. Levy. <year>2000</year>. Type I IFN modulates innate and specific antiviral immunity. <source>J. Immunol.</source>\n",
            "<volume>164</volume>:<fpage>4220</fpage>&#x2013;4228.\n",
            "<pub-id pub-id-type=\"pmid\">10754318</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib20\">\n",
            "        <label>20</label>\n",
            "        <mixed-citation publication-type=\"journal\">Kumar, A., M. Commane, T.W. Flickinger, C.M. Horvath, and G.R. Stark. <year>1997</year>. Defective TNF-alpha-induced apoptosis in STAT1-null cells due to low constitutive levels of caspases. <source>Science.</source>\n",
            "<volume>278</volume>:<fpage>1630</fpage>&#x2013;1632.\n",
            "<pub-id pub-id-type=\"pmid\">9374464</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib21\">\n",
            "        <label>21</label>\n",
            "        <mixed-citation publication-type=\"journal\">Kuniyasu, Y., T. Takahashi, M. Itoh, J. Shimizu, G. Toda, and S. Sakaguchi. <year>2000</year>. Naturally anergic and suppressive CD25(+)CD4(+) T cells as a functionally and phenotypically distinct immunoregulatory T cell subpopulation. <source>Int. Immunol.</source>\n",
            "<volume>12</volume>:<fpage>1145</fpage>&#x2013;1155.\n",
            "<pub-id pub-id-type=\"pmid\">10917889</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib22\">\n",
            "        <label>22</label>\n",
            "        <mixed-citation publication-type=\"journal\">Maloy, K.J., and F. Powrie. <year>2001</year>. Regulatory T cells in the control of immune pathology. <source>Nat. Immunol.</source>\n",
            "<volume>2</volume>:<fpage>816</fpage>&#x2013;822.\n",
            "<pub-id pub-id-type=\"pmid\">11526392</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib23\">\n",
            "        <label>23</label>\n",
            "        <mixed-citation publication-type=\"journal\">Shevach, E.M., R.S. McHugh, A.M. Thornton, C. Piccirillo, K. Natarajan, and D.H. Margulies. <year>2001</year>. Control of autoimmunity by regulatory T cells. <source>Adv. Exp. Med. Biol.</source>\n",
            "<volume>490</volume>:<fpage>21</fpage>&#x2013;32.\n",
            "<pub-id pub-id-type=\"pmid\">11505971</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib24\">\n",
            "        <label>24</label>\n",
            "        <mixed-citation publication-type=\"journal\">O'Garra, A., L. Steinman, and K. Gijbels. <year>1997</year>. CD4+ T-cell subsets in autoimmunity. <source>Curr. Opin. Immunol.</source>\n",
            "<volume>9</volume>:<fpage>872</fpage>&#x2013;883.\n",
            "<pub-id pub-id-type=\"pmid\">9492992</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib25\">\n",
            "        <label>25</label>\n",
            "        <mixed-citation publication-type=\"journal\">Shevach, E.M. <year>2000</year>. Regulatory T cells in autoimmunity. <source>Annu. Rev. Immunol.</source>\n",
            "<volume>18</volume>:<fpage>423</fpage>&#x2013;449.\n",
            "<pub-id pub-id-type=\"pmid\">10837065</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib26\">\n",
            "        <label>26</label>\n",
            "        <mixed-citation publication-type=\"journal\">Sakaguchi, S. <year>2000</year>. Regulatory T cells: key controllers of immunologic self-tolerance. <source>Cell.</source>\n",
            "<volume>101</volume>:<fpage>455</fpage>&#x2013;458.\n",
            "<pub-id pub-id-type=\"pmid\">10850488</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib27\">\n",
            "        <label>27</label>\n",
            "        <mixed-citation publication-type=\"journal\">Salomon, B., D.J. Lenschow, L. Rhee, N. Ashourian, B. Singh, A. Sharpe, and J.A. Bluestone. <year>2000</year>. B7/CD28 costimulation is essential for the homeostasis of the CD4+CD25+ immunoregulatory T cells that control autoimmune diabetes. <source>Immunity.</source>\n",
            "<volume>12</volume>:<fpage>431</fpage>&#x2013;440.\n",
            "<pub-id pub-id-type=\"pmid\">10795741</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib28\">\n",
            "        <label>28</label>\n",
            "        <mixed-citation publication-type=\"journal\">Grewal, I.S., H.G. Foellmer, K.D. Grewal, H. Wang, W.P. Lee, D. Tumas, C.A. Janeway, Jr., and R.A. Flavell. <year>2001</year>. CD62L is required on effector cells for local interactions in the CNS to cause myelin damage in experimental allergic encephalomyelitis. <source>Immunity.</source>\n",
            "<volume>14</volume>:<fpage>291</fpage>&#x2013;302.\n",
            "<pub-id pub-id-type=\"pmid\">11290338</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib29\">\n",
            "        <label>29</label>\n",
            "        <mixed-citation publication-type=\"journal\">Papiernik, M., M.L. de Moraes, C. Pontoux, F. Vasseur, and C. Penit. <year>1998</year>. Regulatory CD4 T cells: expression of IL-2R alpha chain, resistance to clonal deletion and IL-2 dependency. <source>Int. Immunol.</source>\n",
            "<volume>10</volume>:<fpage>371</fpage>&#x2013;378.\n",
            "<pub-id pub-id-type=\"pmid\">9620592</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib30\">\n",
            "        <label>30</label>\n",
            "        <mixed-citation publication-type=\"journal\">Segal, B.M., B.K. Dwyer, and E.M. Shevach. <year>1998</year>. An interleukin (IL)-10/IL-12 immunoregulatory circuit controls susceptibility to autoimmune disease. <source>J. Exp. Med.</source>\n",
            "<volume>187</volume>:<fpage>537</fpage>&#x2013;546.\n",
            "<pub-id pub-id-type=\"pmid\">9463404</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib31\">\n",
            "        <label>31</label>\n",
            "        <mixed-citation publication-type=\"journal\">Willenborg, D.O., S. Fordham, C.C. Bernard, W.B. Cowden, and I.A. Ramshaw. <year>1996</year>. IFN-gamma plays a critical down-regulatory role in the induction and effector phase of myelin oligodendrocyte glycoprotein-induced autoimmune encephalomyelitis. <source>J. Immunol.</source>\n",
            "<volume>157</volume>:<fpage>3223</fpage>&#x2013;3227.\n",
            "<pub-id pub-id-type=\"pmid\">8871615</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib32\">\n",
            "        <label>32</label>\n",
            "        <mixed-citation publication-type=\"journal\">Chu, C.Q., S. Wittmer, and D.K. Dalton. <year>2000</year>. Failure to suppress the expansion of the activated CD4 T cell population in interferon &#x3B3;&#x2013;deficient mice leads to exacerbation of experimental autoimmune encephalomyelitis. <source>J. Exp. Med.</source>\n",
            "<volume>192</volume>:<fpage>123</fpage>&#x2013;128.\n",
            "<pub-id pub-id-type=\"pmid\">10880533</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib33\">\n",
            "        <label>33</label>\n",
            "        <mixed-citation publication-type=\"journal\">Kumar, V., and E. Sercarz. <year>2001</year>. An integrative model of regulation centered on recognition of TCR peptide/MHC complexes. <source>Immunol. Rev.</source>\n",
            "<volume>182</volume>:<fpage>113</fpage>&#x2013;121.\n",
            "<pub-id pub-id-type=\"pmid\">11722628</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "      <ref id=\"bib34\">\n",
            "        <label>34</label>\n",
            "        <mixed-citation publication-type=\"journal\">McHugh, R.S., M.J. Whitters, C.A. Piccirillo, D.A. Young, E.M. Shevach, M. Collins, and M.C. Byrne. <year>2002</year>. CD4+CD25+ immunoregulatory T cells: gene expression analysis reveals a functional role for the glucocorticoid-induced TNF receptor. <source>Immunity.</source>\n",
            "<volume>16</volume>:<fpage>311</fpage>&#x2013;323.\n",
            "<pub-id pub-id-type=\"pmid\">11869690</pub-id></mixed-citation>\n",
            "      </ref>\n",
            "    </ref-list>\n",
            "  </back>\n",
            "</article>\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import xml.etree.ElementTree as ET\n",
        "\n",
        "def parse_xml(file_path):\n",
        "    tree = ET.parse(file_path)\n",
        "    root = tree.getroot()\n",
        "    return root\n",
        "\n",
        "root = parse_xml('/content/data-for-clinical-note-summarizer/extracted_xml_files/PMC001xxxxxx/PMC1193645.xml')\n"
      ],
      "metadata": {
        "id": "Bsk6RhWQuLU5"
      },
      "execution_count": 28,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "def collect_tags(element, tags_set):\n",
        "    tags_set.add(element.tag)\n",
        "    for child in element:\n",
        "        collect_tags(child, tags_set)\n",
        "\n",
        "tags_set = set()\n",
        "collect_tags(root, tags_set)\n",
        "\n",
        "unique_tags = list(tags_set)\n"
      ],
      "metadata": {
        "id": "rXflbnhMsLph"
      },
      "execution_count": 29,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "print(\"Unique Tags in the XML File:\")\n",
        "for tag in unique_tags:\n",
        "    print(tag)\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "vWV_XjIdsSuI",
        "outputId": "ac795f29-f854-488a-f33e-b4b07ee1e161"
      },
      "execution_count": 30,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Unique Tags in the XML File:\n",
            "abstract\n",
            "month\n",
            "source\n",
            "contrib\n",
            "journal-title-group\n",
            "contrib-group\n",
            "issue\n",
            "caption\n",
            "thead\n",
            "surname\n",
            "fn-group\n",
            "ref\n",
            "history\n",
            "copyright-statement\n",
            "license-p\n",
            "fig\n",
            "table\n",
            "volume\n",
            "article-meta\n",
            "issn\n",
            "body\n",
            "pub-date\n",
            "article-categories\n",
            "fpage\n",
            "article-title\n",
            "given-names\n",
            "year\n",
            "back\n",
            "front\n",
            "email\n",
            "permissions\n",
            "journal-meta\n",
            "journal-title\n",
            "name\n",
            "article\n",
            "mixed-citation\n",
            "sup\n",
            "td\n",
            "subj-group\n",
            "ext-link\n",
            "sec\n",
            "fn\n",
            "author-notes\n",
            "notes\n",
            "day\n",
            "kwd\n",
            "xref\n",
            "title\n",
            "subject\n",
            "sub\n",
            "tbody\n",
            "title-group\n",
            "th\n",
            "publisher-name\n",
            "table-wrap\n",
            "pub-id\n",
            "article-id\n",
            "graphic\n",
            "lpage\n",
            "tr\n",
            "date\n",
            "journal-id\n",
            "kwd-group\n",
            "label\n",
            "publisher\n",
            "italic\n",
            "license\n",
            "aff\n",
            "p\n",
            "ref-list\n",
            "ack\n",
            "subtitle\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import xml.etree.ElementTree as ET\n",
        "import json\n",
        "\n",
        "def parse_xml(file_path):\n",
        "    \"\"\"\n",
        "    Parse the XML file and return its root element.\n",
        "    \"\"\"\n",
        "    tree = ET.parse(file_path)\n",
        "    return tree.getroot()\n",
        "\n",
        "def extract_metadata(root):\n",
        "    \"\"\"\n",
        "    Extract metadata such as journal information, article title, subtitle, authors, correspondence,\n",
        "    publication date, volume, issue, pages, DOI, keywords, and abbreviations.\n",
        "    \"\"\"\n",
        "    metadata = {\n",
        "        \"journal_title\": root.findtext(\".//journal-title\", default=\"\").strip(),\n",
        "        \"issns\": {\n",
        "            \"print\": root.findtext(\".//issn[@pub-type='ppub']\", default=\"\").strip(),\n",
        "            \"online\": root.findtext(\".//issn[@pub-type='epub']\", default=\"\").strip()\n",
        "        },\n",
        "        \"publisher\": root.findtext(\".//publisher-name\", default=\"\").strip(),\n",
        "        \"article_title\": root.findtext(\".//article-title\", default=\"\").strip(),\n",
        "        \"subtitle\": root.findtext(\".//subtitle\", default=\"\").strip(),\n",
        "        \"authors\": [],\n",
        "        \"correspondence\": [],\n",
        "        \"publication_date\": root.findtext(\".//pub-date/year\", default=\"\").strip(),\n",
        "        \"volume\": root.findtext(\".//volume\", default=\"\").strip(),\n",
        "        \"issue\": root.findtext(\".//issue\", default=\"\").strip(),\n",
        "        \"pages\": root.findtext(\".//fpage\", default=\"\").strip() + \"-\" + root.findtext(\".//lpage\", default=\"\").strip(),\n",
        "        \"doi\": root.findtext(\".//article-id[@pub-id-type='doi']\", default=\"\").strip(),\n",
        "        \"abstract\": [],\n",
        "        \"keywords\": [],\n",
        "        \"abbreviations\": []\n",
        "    }\n",
        "\n",
        "    # Extract authors\n",
        "    for contrib in root.findall(\".//contrib[@contrib-type='author']\"):\n",
        "        surname = contrib.findtext(\".//surname\", default=\"\").strip()\n",
        "        given_names = contrib.findtext(\".//given-names\", default=\"\").strip()\n",
        "        if surname or given_names:\n",
        "            metadata[\"authors\"].append(f\"{given_names} {surname}\".strip())\n",
        "\n",
        "    # Extract correspondence\n",
        "    for corr in root.findall(\".//address/email\"):\n",
        "        metadata[\"correspondence\"].append(corr.text.strip())\n",
        "\n",
        "    # Extract abstract\n",
        "    for abstract in root.findall(\".//abstract\"):\n",
        "        metadata[\"abstract\"].append(\" \".join(abstract.itertext()).strip())\n",
        "\n",
        "    # Extract keywords\n",
        "    for kwd in root.findall(\".//kwd\"):\n",
        "        metadata[\"keywords\"].append(kwd.text.strip())\n",
        "\n",
        "    # Extract abbreviations\n",
        "    for abbrev in root.findall(\".//abbrev\"):\n",
        "        metadata[\"abbreviations\"].append(abbrev.text.strip())\n",
        "\n",
        "    return metadata\n",
        "\n",
        "def extract_sections(root):\n",
        "    \"\"\"\n",
        "    Recursively extract sections, including titles, paragraphs, figures, tables, lists, and subsections.\n",
        "    \"\"\"\n",
        "    def parse_section(section):\n",
        "        section_data = {\n",
        "            \"title\": section.findtext(\"title\", default=\"\").strip(),\n",
        "            \"paragraphs\": [\" \".join(p.itertext()).strip() for p in section.findall(\"p\")],\n",
        "            \"figures\": [],\n",
        "            \"tables\": [],\n",
        "            \"lists\": [],\n",
        "            \"subsections\": []\n",
        "        }\n",
        "\n",
        "        # Extract figures\n",
        "        for fig in section.findall(\"fig\"):\n",
        "            caption = \" \".join(fig.find(\"caption\").itertext()).strip() if fig.find(\"caption\") else \"\"\n",
        "            section_data[\"figures\"].append({\"caption\": caption})\n",
        "\n",
        "        # Extract tables\n",
        "        for table in section.findall(\"table-wrap\"):\n",
        "            caption = \" \".join(table.find(\"caption\").itertext()).strip() if table.find(\"caption\") else \"\"\n",
        "            section_data[\"tables\"].append({\"caption\": caption})\n",
        "\n",
        "        # Extract lists\n",
        "        for list_element in section.findall(\"list\"):\n",
        "            list_items = [item.text.strip() for item in list_element.findall(\"list-item\") if item.text]\n",
        "            section_data[\"lists\"].append(list_items)\n",
        "\n",
        "        # Recursively extract subsections\n",
        "        for subsection in section.findall(\"sec\"):\n",
        "            section_data[\"subsections\"].append(parse_section(subsection))\n",
        "\n",
        "        return section_data\n",
        "\n",
        "    sections = []\n",
        "    for top_section in root.findall(\".//sec\"):\n",
        "        sections.append(parse_section(top_section))\n",
        "    return sections\n",
        "\n",
        "def extract_references(root):\n",
        "    \"\"\"\n",
        "    Extract references from the article, including titles and other metadata.\n",
        "    \"\"\"\n",
        "    references = []\n",
        "    for ref in root.findall(\".//ref\"):\n",
        "        ref_data = {\n",
        "            \"title\": ref.findtext(\".//article-title\", default=\"\").strip(),\n",
        "            \"authors\": [],\n",
        "            \"journal\": ref.findtext(\".//source\", default=\"\").strip(),\n",
        "            \"year\": ref.findtext(\".//year\", default=\"\").strip()\n",
        "        }\n",
        "\n",
        "        # Extract authors in reference\n",
        "        for person in ref.findall(\".//name\"):\n",
        "            surname = person.findtext(\"surname\", default=\"\").strip()\n",
        "            given_names = person.findtext(\"given-names\", default=\"\").strip()\n",
        "            if surname or given_names:\n",
        "                ref_data[\"authors\"].append(f\"{given_names} {surname}\".strip())\n",
        "\n",
        "        references.append(ref_data)\n",
        "\n",
        "    return references\n",
        "\n",
        "def extract_full_article(file_path):\n",
        "    \"\"\"\n",
        "    Extract the complete article data, including metadata, sections, and references.\n",
        "    \"\"\"\n",
        "    root = parse_xml(file_path)\n",
        "    data = extract_metadata(root)\n",
        "    data[\"sections\"] = extract_sections(root)\n",
        "    data[\"references\"] = extract_references(root)\n",
        "    return data\n",
        "\n",
        "def save_to_json(data, output_file):\n",
        "    \"\"\"\n",
        "    Save extracted data to a JSON file.\n",
        "    \"\"\"\n",
        "    with open(output_file, \"w\", encoding=\"utf-8\") as f:\n",
        "        json.dump(data, f, ensure_ascii=False, indent=4)\n",
        "\n",
        "if __name__ == \"__main__\":\n",
        "    # Input XML file path\n",
        "    xml_file = \"/content/data-for-clinical-note-summarizer/extracted_xml_files/PMC001xxxxxx/PMC1474074.xml\"  # Replace with your XML file path\n",
        "    output_file = \"output.json\"        # Output JSON file\n",
        "\n",
        "    # Extract data and save to JSON\n",
        "    article_data = extract_full_article(xml_file)\n",
        "    save_to_json(article_data, output_file)\n",
        "\n",
        "    print(f\"Extraction complete. Data saved to {output_file}.\")\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Fb0lAE550UTy",
        "outputId": "bb257ea4-38f0-42b7-9a01-2d6ece47c54f"
      },
      "execution_count": 43,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Extraction complete. Data saved to output.json.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import os\n",
        "import csv\n",
        "import xml.etree.ElementTree as ET\n",
        "\n",
        "def parse_xml(file_path):\n",
        "    \"\"\"\n",
        "    Parse the XML file and return its root element.\n",
        "    \"\"\"\n",
        "    tree = ET.parse(file_path)\n",
        "    return tree.getroot()\n",
        "\n",
        "def extract_metadata(root):\n",
        "    \"\"\"\n",
        "    Extract metadata such as journal information, article title, abstract, keywords, and publication details.\n",
        "    \"\"\"\n",
        "    metadata = {\n",
        "        \"journal_title\": root.findtext(\".//journal-title\", default=\"\").strip(),\n",
        "        \"article_title\": root.findtext(\".//article-title\", default=\"\").strip(),\n",
        "        \"authors\": \"; \".join(\n",
        "            f\"{author.findtext('.//given-names', default='').strip()} {author.findtext('.//surname', default='').strip()}\"\n",
        "            for author in root.findall(\".//contrib[@contrib-type='author']\")\n",
        "        ),\n",
        "        \"publication_date\": root.findtext(\".//pub-date/year\", default=\"\").strip(),\n",
        "        \"doi\": root.findtext(\".//article-id[@pub-id-type='doi']\", default=\"\").strip(),\n",
        "    }\n",
        "\n",
        "    # Extract abstract - fix for joining all text\n",
        "    abstract = root.find(\".//abstract\")\n",
        "    if abstract is not None:\n",
        "        metadata[\"abstract\"] = \" \".join(abstract.itertext()).strip()\n",
        "    else:\n",
        "        metadata[\"abstract\"] = \"\"\n",
        "\n",
        "    # Extract keywords\n",
        "    metadata[\"keywords\"] = \"; \".join([kwd.text.strip() for kwd in root.findall(\".//kwd\")])\n",
        "\n",
        "    return metadata\n",
        "\n",
        "def extract_sections(root):\n",
        "    \"\"\"\n",
        "    Extract content (paragraphs, figures, tables) for each section and return as strings.\n",
        "    \"\"\"\n",
        "    sections_content = {\n",
        "        \"paragraphs\": \"\",\n",
        "        \"figures\": \"\",\n",
        "        \"tables\": \"\",\n",
        "        \"lists\": \"\",\n",
        "    }\n",
        "\n",
        "    def parse_section(section, parent_title=\"\"):\n",
        "        section_title = section.findtext(\"title\", default=\"\").strip()\n",
        "        full_title = f\"{parent_title} > {section_title}\" if parent_title else section_title\n",
        "\n",
        "        # Append paragraphs\n",
        "        for paragraph in section.findall(\"p\"):\n",
        "            sections_content[\"paragraphs\"] += \" \".join(paragraph.itertext()).strip() + \" \"\n",
        "\n",
        "        # Append figures\n",
        "        for figure in section.findall(\"fig\"):\n",
        "            caption = figure.find(\"caption\")\n",
        "            if caption is not None:\n",
        "                sections_content[\"figures\"] += \" \".join(caption.itertext()).strip() + \" \"\n",
        "\n",
        "        # Append tables\n",
        "        for table in section.findall(\"table-wrap\"):\n",
        "            caption = table.find(\"caption\")\n",
        "            if caption is not None:\n",
        "                sections_content[\"tables\"] += \" \".join(caption.itertext()).strip() + \" \"\n",
        "\n",
        "        # Recursively process subsections\n",
        "        for subsection in section.findall(\"sec\"):\n",
        "            parse_section(subsection, parent_title=full_title)\n",
        "\n",
        "    # Start processing top-level sections\n",
        "    for top_section in root.findall(\".//sec\"):\n",
        "        parse_section(top_section)\n",
        "\n",
        "    return sections_content\n",
        "\n",
        "def process_folder(input_folder, output_csv):\n",
        "    \"\"\"\n",
        "    Process all XML files in a folder and save the extracted data to a CSV file.\n",
        "    \"\"\"\n",
        "    # CSV header: Metadata fields + content fields for sections\n",
        "    csv_header = [\n",
        "        \"journal_title\", \"article_title\", \"authors\", \"publication_date\", \"doi\",\n",
        "        \"abstract\", \"keywords\", \"paragraphs\", \"figures\", \"tables\", \"lists\"\n",
        "    ]\n",
        "\n",
        "    rows = []\n",
        "\n",
        "    # Process each XML file in the folder\n",
        "    for file_name in os.listdir(input_folder):\n",
        "        if file_name.endswith(\".xml\"):\n",
        "            file_path = os.path.join(input_folder, file_name)\n",
        "            try:\n",
        "                root = parse_xml(file_path)\n",
        "                article_metadata = extract_metadata(root)\n",
        "                sections_content = extract_sections(root)\n",
        "\n",
        "                # Combine metadata with sections content\n",
        "                row = {**article_metadata, **sections_content}\n",
        "                rows.append(row)\n",
        "\n",
        "            except Exception as e:\n",
        "                print(f\"Error processing {file_name}: {e}\")\n",
        "\n",
        "    # Write all data to CSV\n",
        "    with open(output_csv, mode=\"w\", encoding=\"utf-8\", newline=\"\") as csvfile:\n",
        "        writer = csv.DictWriter(csvfile, fieldnames=csv_header)\n",
        "        writer.writeheader()\n",
        "        writer.writerows(rows)\n",
        "\n",
        "if __name__ == \"__main__\":\n",
        "    # Folder containing XML files\n",
        "    input_folder = \"/content/extracted_xml_files/PMC002xxxxxx\"  # Replace with your folder path\n",
        "    output_csv = \"final_output.csv\"  # Output CSV file\n",
        "\n",
        "    # Process all files and save to CSV\n",
        "    process_folder(input_folder, output_csv)\n",
        "    print(f\"Extraction complete. Data saved to {output_csv}.\")\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "FvWJNMA1F7JD",
        "outputId": "f67ca49d-9cdb-40be-b2cc-f702f44c3888"
      },
      "execution_count": 81,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Error processing PMC2249741.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2782211.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2671205.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2610305.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2575888.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217465.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2829409.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2980633.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193847.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2788184.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2975056.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174350.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2900597.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2529237.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2846443.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2576647.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2817400.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2789544.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2789941.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132879.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2211911.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193001.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195571.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2999852.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2561159.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194170.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195806.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2671619.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2686619.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779904.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2842876.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2656928.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2807266.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2526493.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2791119.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2887653.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2802727.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2755035.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2533591.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2767514.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193041.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174831.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2852658.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174295.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194099.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816971.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2173630.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890890.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193147.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148166.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2671203.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2914296.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2983006.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2694388.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132889.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2791127.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148136.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872623.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133144.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695126.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2925526.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2801134.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148152.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872691.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2882083.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2802491.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2770101.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2788186.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193743.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2267737.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229549.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217494.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192380.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2810371.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2694629.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2682507.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2682495.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2615335.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2714412.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195800.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2952102.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2935671.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2575889.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2952767.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2802914.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2651465.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826645.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229426.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2451336.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826732.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104745.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2991177.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2799627.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2690828.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2762532.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2899255.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2199622.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2440086.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193661.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695776.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2694264.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2782306.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759214.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955747.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2877814.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2908433.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2832745.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2860680.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2529239.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2991233.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195761.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2243254.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837180.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2279743.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2652064.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2230544.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104734.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2827799.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2413272.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2755781.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2878174.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2818188.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2694378.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2873812.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2852659.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133139.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132825.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2682504.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2993542.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2997560.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2211939.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2905201.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2164970.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212012.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104729.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955755.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2975445.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2720514.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194132.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2791123.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2900819.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2860568.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2822270.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2758717.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212490.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2441495.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695131.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2233755.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2694382.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104725.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193296.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2561145.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2995223.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2855018.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150670.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2825539.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2668829.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2967007.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174219.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2905515.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2945463.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174563.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2954306.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2988865.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2213497.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175229.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2811846.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2882259.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2987532.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2757579.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2993540.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2413283.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2413282.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2795858.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2900051.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2850540.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2920756.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193593.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826090.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2766280.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2233682.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2976066.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2724695.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148193.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2692015.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2852671.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2852660.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2879483.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174050.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2249740.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195847.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2744769.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148226.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2561152.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759202.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132980.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174571.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2584333.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217244.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2206240.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2639266.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2789547.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2600653.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2755030.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2718217.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133081.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2952766.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2628142.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826658.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2981729.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2995224.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2773370.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2486466.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2933843.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2967008.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2778772.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826660.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2941057.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2811250.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2853384.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2848776.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2156123.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174797.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2387230.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2646357.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2657534.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2866305.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2583063.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2729390.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2211819.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2556091.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174543.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217459.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2760466.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2811425.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2833432.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148195.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2907713.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2652052.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2846122.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2615304.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175158.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2290942.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2907017.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2900820.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2627247.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2651458.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2778765.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133149.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193616.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2865352.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2876270.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2921200.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2525850.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2975446.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2898109.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2867229.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193615.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2775894.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2905040.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2810364.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2275197.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779343.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872710.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779890.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2998775.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2504349.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195801.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104717.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2831567.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195996.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2856857.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2556238.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132800.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2877898.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2999732.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2504348.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2943750.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2423652.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955749.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2995975.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2718222.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2831610.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2900594.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193429.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2656929.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2976079.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212549.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2189801.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2953268.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2946664.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2895876.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2815839.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133178.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2834352.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2862945.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2657536.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2824527.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2974165.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2974927.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2935869.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759208.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695129.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2610298.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2993919.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2825308.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2459236.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2877907.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2486473.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2232397.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2515545.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2553423.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175199.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148222.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2836210.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2892354.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2488377.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2860561.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2824522.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2652058.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2185590.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148119.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2429961.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2703834.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2700692.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2515876.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195916.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2916109.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2561157.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955235.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2443231.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2853382.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2877902.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2789535.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759205.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816970.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2769548.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2775109.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175259.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2892343.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2817404.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2691177.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2920754.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2895895.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2834777.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2899760.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2822221.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2801102.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2997556.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2805926.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2755032.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2788202.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212289.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2652061.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174796.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2519116.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2789540.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2846134.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2627236.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174577.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2486463.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695014.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217112.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2736885.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2999729.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2939358.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2886142.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2989172.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2396481.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192968.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2233902.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2649283.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195715.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2687582.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2820231.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193502.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872712.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2921503.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194079.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2839095.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2814112.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104746.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2860696.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2636165.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193436.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2802733.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2918868.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2905194.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2722427.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2629953.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2935873.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2993946.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2492730.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2988858.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2933847.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132977.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2973230.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2801118.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2644267.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2854345.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955236.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2941156.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2803206.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2803207.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2894203.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2736893.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2782320.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174810.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193885.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872711.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2874033.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2854365.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2486476.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193758.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2994130.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2802725.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132834.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2724690.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217227.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816388.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217225.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150732.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2576645.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2646356.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2671617.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195809.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826655.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2705846.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2615262.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193252.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104735.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2736898.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2740637.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2892140.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2948653.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2924425.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2529238.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2877315.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2526494.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2732789.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2789533.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193325.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2797622.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195841.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2990201.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2815834.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2865353.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2823043.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2628133.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2950081.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212510.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2844956.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2486459.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826739.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2839348.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2802729.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2803205.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2223373.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2211857.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2714901.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2839090.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2796120.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193887.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2275199.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816968.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104722.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229637.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2988896.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2840663.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2413286.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2995969.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2785513.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2900052.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890160.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2736887.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192389.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2650625.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816964.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2718224.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2686959.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2213443.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2997555.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2172762.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2852657.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2156185.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2802723.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2880694.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193007.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2582982.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816376.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2900821.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2671627.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2814116.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2882561.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890325.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193550.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2525752.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2169474.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2575883.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2493533.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816952.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2956061.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2518948.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2997546.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826661.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2424297.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2733956.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2719186.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2882255.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2565736.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2995968.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2722160.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2610306.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193536.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779907.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2957574.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217153.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2993911.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2981784.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2651451.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104732.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2702828.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816390.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150771.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2516362.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2692010.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2671615.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2275201.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2949088.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779087.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2718225.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2841269.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2964123.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2876265.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2877225.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104726.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2249733.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2576629.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2956262.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175260.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150789.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2902736.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2628025.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2946701.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2486475.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872078.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2233689.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229361.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2811302.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2807929.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193703.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133106.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150685.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2649871.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194070.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2948172.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2714554.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2690067.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133058.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2802732.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872024.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2987182.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2877894.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2387232.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2172900.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2172233.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2628156.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2504342.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695015.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194265.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104716.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133068.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2924483.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2657551.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759213.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2825735.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2839907.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2882263.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890899.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2990202.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2932892.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2576641.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229360.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2151187.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2895863.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2702832.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2213298.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2211989.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2211827.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2831187.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2525751.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2993904.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2649872.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2249738.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2213312.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2709885.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2975008.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2842563.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2639114.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2173414.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2817410.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2173083.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826653.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2211908.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2467505.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2500205.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217076.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2701181.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837260.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2771125.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150581.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2921502.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2668941.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217197.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229577.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2772964.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2275198.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2173507.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837155.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2970807.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2173378.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2918872.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2845516.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2677216.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2952408.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2913398.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2844513.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229501.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2974930.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2882262.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2627169.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2724688.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2692007.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2820228.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2486480.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2564731.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2886099.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2757581.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2651454.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2865348.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2034494.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2561153.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174540.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2868130.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759203.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779292.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2698201.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132871.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2944959.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2803221.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2886118.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2812417.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2993905.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2718215.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2655743.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2801119.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2864151.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2887652.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2196008.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837830.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195619.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2803216.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2290933.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2929429.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229607.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2814114.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175163.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148135.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193734.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955911.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2801124.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2803213.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2647179.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195591.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2275200.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217047.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2702910.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2529246.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2817432.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2989448.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2815831.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779905.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2718214.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2789542.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2949563.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2844966.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104733.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2169444.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2673709.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2871094.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2874492.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2687682.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955744.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2805925.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2852663.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2199606.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2504351.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2789545.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2882271.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2817425.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193428.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2413277.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2832885.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2819973.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150673.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2725789.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2776166.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759201.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2529236.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2805927.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2995211.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779322.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2636063.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192387.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2803203.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2635971.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2935972.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2668587.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2921500.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2151188.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2692016.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2211861.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2782307.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779371.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2997541.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2877893.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2935868.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2868155.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2791121.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2213346.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2199724.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2628080.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2914253.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2576634.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890164.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2803755.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2719744.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2536710.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695028.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193251.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837241.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2213308.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212290.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695024.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2841267.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2849020.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759210.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217214.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826726.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2721949.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872695.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816831.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2892773.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2918870.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2211821.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2529235.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872020.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175131.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2752466.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2888404.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2652045.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2733990.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955711.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174858.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2576638.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2935877.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779908.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2935865.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2995857.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2773364.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2628065.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816389.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2687649.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193203.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193866.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193677.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2615280.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2173991.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2213412.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2639328.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816973.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2652054.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2469278.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212417.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2846127.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2793310.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2628116.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2946560.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2807642.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779288.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229644.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192648.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2817398.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2954731.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2797847.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2396877.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2862943.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2892053.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2580732.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2920072.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2386084.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2615305.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759211.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2908750.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2721991.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133120.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132905.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2727395.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195737.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104713.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2887934.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104718.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2892350.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2639447.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2413288.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2998739.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2423654.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2920175.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2777209.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2561154.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175237.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955243.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194252.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2568038.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2993921.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2610300.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2814031.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2565773.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2822627.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2486460.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104723.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779896.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2846130.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2198807.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2909122.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2529383.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195639.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872698.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2864426.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2652063.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2185517.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2486464.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2908850.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2892771.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2913391.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148185.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2480603.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2671631.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2413275.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890161.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2423660.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2515875.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2694635.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193510.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2172833.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837254.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2989830.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2885296.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2766497.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2671629.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2741629.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2781778.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2951530.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2899024.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2924485.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133088.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2806527.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2811833.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2791111.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2504343.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195839.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193660.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2839092.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2803209.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212378.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2839466.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2149350.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2895783.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2639027.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193467.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2846136.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2267734.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2895206.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2671788.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2904664.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2902141.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2586314.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2989829.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2693820.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2924491.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2939349.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2928441.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2211786.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2941043.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2820232.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104728.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2781782.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837258.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2919686.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212287.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229642.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194259.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2610312.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2693613.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2998776.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837264.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2156147.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837831.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2922620.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2978740.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2839091.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2924489.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2722211.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2698627.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195581.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2702072.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2791116.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2814108.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193922.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2516911.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2883927.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2886097.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2657539.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132896.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193330.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2882274.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2834769.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2189814.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2935974.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872022.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2573398.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2775849.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2845889.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150571.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2817429.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2265772.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2817405.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2882270.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2650639.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2275193.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2921202.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890322.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2835637.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2726571.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2896735.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2804881.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872713.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837263.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2636040.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2809315.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132975.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2688021.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2707303.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2799959.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2610310.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175205.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2721989.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2835564.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2998743.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2692014.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2900048.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2811841.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193084.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2876954.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2724696.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212018.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2831181.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2775870.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2764078.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2782796.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133069.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2856000.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2817434.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174553.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193667.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837266.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2724705.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2185080.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2486468.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2898108.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2817422.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2661522.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2493603.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217505.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2820667.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2650636.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955918.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2692011.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2947714.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2789534.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2408938.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2921262.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2525545.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2671612.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2586320.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193512.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2811248.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2846132.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2156145.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2852654.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2918796.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2798988.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212745.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955746.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194188.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2921201.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132811.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2423651.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2517101.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2846277.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2486467.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890163.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2636017.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2763177.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2983015.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2924991.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192941.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2667134.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826651.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194239.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2698425.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695022.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2933446.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2718220.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2845782.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2880268.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2636104.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2914289.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2526449.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212295.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2954293.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837256.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2169485.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2975003.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2570083.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174805.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2853383.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2788183.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2199735.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2149354.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2777221.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2887323.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2687828.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2797627.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872714.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816972.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2063582.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2517153.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2724699.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2938456.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2692020.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2945380.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2852667.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193797.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2924422.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2839094.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2841259.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2812422.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2803747.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2686953.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2974207.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175160.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2998746.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2515874.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2474556.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2998744.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2882266.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133084.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194199.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955742.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2486462.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2542407.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2904666.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194175.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2556095.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2724689.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2994123.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2185533.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2852662.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194074.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2290936.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132972.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2877896.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2652049.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890329.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2908927.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195807.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890156.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2947428.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148168.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133073.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2752777.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2801136.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2825322.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2652051.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2491421.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816966.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2877889.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2211905.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2947195.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2529380.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217452.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193354.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2586321.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2651452.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2529244.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229384.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193220.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779313.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2846736.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2736896.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2387229.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2775857.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2789329.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872716.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150677.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2657543.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2952113.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2526484.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2918794.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2791120.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2173673.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2820688.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2652062.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2799969.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2947807.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2213292.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192974.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2993907.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2586309.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816900.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2233659.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132965.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192995.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2706732.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195731.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779327.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2845873.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2610309.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2151195.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2504350.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2839115.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2810578.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2846123.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2518949.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2993917.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759198.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2905195.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2755034.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2722197.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2923801.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2855960.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192032.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2983002.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2486481.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2718218.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104743.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2687521.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2935872.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2233905.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2820398.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2962788.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2935866.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2864897.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2189807.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194077.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2910860.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2802728.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193727.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2849048.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2628119.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133192.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2662490.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2699387.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826654.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2552378.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2413280.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2997539.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2486472.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175223.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695774.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2700687.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2913393.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695021.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2806245.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2652066.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2808596.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2807733.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132907.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2911544.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890985.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2792339.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2877887.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2845890.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175258.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194083.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2803223.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192030.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2169445.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2172280.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2956055.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2875467.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2800005.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2810594.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2935874.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2814102.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2998745.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2568036.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2995421.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2956051.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133132.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2196021.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217355.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2491418.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826662.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2156159.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2817435.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2636076.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2981516.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193704.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194237.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2819658.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2860565.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2995220.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2748272.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2825318.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2773040.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2576636.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2898060.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2636027.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2924484.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193414.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2213529.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2866370.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2814109.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2935544.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2529378.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2797434.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826657.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837259.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2639208.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104740.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955748.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2504341.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2922622.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2755796.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2811840.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132917.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2673734.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104744.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2480516.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2156175.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2615326.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2923707.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2833430.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2639335.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2834246.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193435.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148167.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2801105.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2905205.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872801.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2573028.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2938637.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2811788.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132875.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2172388.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955739.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2788165.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2860807.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2211931.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2854889.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2649860.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2990660.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2814006.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2849034.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2921209.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229645.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175293.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2892143.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2810593.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2824876.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2740636.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2970806.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2582980.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2576643.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2871096.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175197.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2755025.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2693841.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2825733.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695036.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2649422.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2694679.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2722214.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104714.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2721990.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2657546.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2935973.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816252.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2908448.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150686.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2847828.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2820237.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872717.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2990200.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2908440.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2945468.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2839103.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174229.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2887326.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195995.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2396471.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2650637.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890876.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2736892.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759199.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2575882.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2650627.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193299.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2839100.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2491414.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2789537.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2576642.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2998748.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192370.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2802730.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2610302.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2156142.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2639016.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217064.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2996532.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2649284.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194157.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2804795.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2854609.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2631610.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2811246.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2839279.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2692008.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2211851.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2731109.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2898099.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2724692.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2529384.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2156126.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2610299.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2899017.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2692009.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2687664.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2921060.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2693823.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2824126.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2998023.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2868132.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2682508.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2794310.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2882267.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193679.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2935864.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2927047.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2839074.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2770829.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2934729.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229421.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2686969.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217221.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175100.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104712.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2504352.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2238713.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2803212.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2882269.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2834331.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2718212.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890878.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2522301.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2671623.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2883083.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2504346.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2921207.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2629956.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2797858.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2892156.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2719184.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2724704.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2995201.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2791118.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2737235.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955740.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2852673.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695023.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212016.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2722008.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2185084.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2172793.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193543.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174222.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837240.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2800239.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2169461.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133071.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2199227.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174235.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2847205.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2858443.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2896650.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2693848.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2526481.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2765655.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2983020.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104715.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2718223.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2990664.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2698622.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2688167.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2705080.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695585.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2867230.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217217.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955242.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2908884.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2814117.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2651453.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2892945.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2828619.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174857.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2850514.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759337.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2877813.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193244.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2266784.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2552391.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2610297.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150666.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2423655.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2707282.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2846133.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2886098.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2948654.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2575893.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2396466.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2858275.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2807174.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2529231.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2169469.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695580.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2910950.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2992129.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2814751.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2814104.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2249742.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2789541.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2950082.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2522291.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217150.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133138.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2691653.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2998749.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2908803.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133023.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2651438.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2885220.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174538.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132829.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2575890.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826644.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2998343.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779326.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2832883.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2811777.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2173394.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2822231.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2682500.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192926.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2698194.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2803215.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2949565.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826641.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2424280.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2459234.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192644.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2908792.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2173206.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175092.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2265110.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2990639.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194110.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2964117.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2845781.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2486461.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2486479.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2651464.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194179.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2677215.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2480597.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104738.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2811295.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2198836.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2668626.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826254.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2900823.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2886126.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2198819.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2844965.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2233702.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2185597.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2807264.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2169431.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212520.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2213401.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2693615.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2976077.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2814115.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195777.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2814107.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2648905.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2525750.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2881426.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2992402.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2839491.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2639446.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2525842.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2947007.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2872699.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2948173.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195848.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2989792.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150560.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193460.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2213355.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175104.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2151193.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2687700.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217115.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148174.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2515544.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2784600.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2532783.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2517152.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2990636.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2710499.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2185524.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192019.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2628085.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2211925.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195603.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2874020.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2718210.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2651460.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2780611.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2736901.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2651459.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150715.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2529242.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2529390.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229411.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2791124.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2997536.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2196004.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2610308.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2962554.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2852668.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229597.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2797410.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826646.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133169.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2568037.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174565.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2857794.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150573.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2275196.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2882583.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2797435.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2702731.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2895300.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229581.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2232402.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2995856.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2185564.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2849021.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779895.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2198876.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2892608.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2652065.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217211.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2904663.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2839101.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2744828.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2829128.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2223370.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2198806.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217434.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174330.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104742.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2504344.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2808595.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175213.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2852672.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229449.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2172418.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2934706.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2800027.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194066.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2797408.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816969.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2839462.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837235.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192390.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132887.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2921059.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2536722.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2892296.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2887324.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2682498.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2997969.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195683.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2964122.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2797415.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2945543.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2864668.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779900.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2974933.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2185078.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2900050.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816965.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2671613.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148216.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217149.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2921204.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2692502.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2815822.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2504345.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2995234.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2886914.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132932.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2682494.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2782317.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2814118.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890327.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2946597.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195703.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2233785.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890155.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2978622.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2990009.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2650623.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174038.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2788098.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2939334.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2764081.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2823024.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2922838.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133118.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2808587.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2913729.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132929.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104727.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2935867.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2877888.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195656.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2687727.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2423663.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2238716.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2736649.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174860.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2862174.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2865355.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217447.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2639024.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2692017.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212719.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2651456.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133047.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2575887.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2151185.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2693570.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2525848.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2657537.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2168094.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2935879.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890165.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150558.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2518083.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2881322.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2911331.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2995237.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2583083.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2151178.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2164975.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2213429.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2196428.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2859322.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2657548.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2926902.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2865354.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2440947.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2989109.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2868139.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2925449.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2880261.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2233662.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2993906.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104736.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2833427.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2172186.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2895867.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2234082.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104720.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2868149.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2905197.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193262.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174828.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2698188.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2892294.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2833429.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2748471.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2172161.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695026.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2908432.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192981.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2698204.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2928679.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2736886.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837551.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2797620.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2885302.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2846126.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2185522.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2866304.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2898543.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2657547.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2516365.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2941811.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2924480.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212291.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2833435.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2892146.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2212427.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2692019.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2887325.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2952768.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2693819.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150578.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195810.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2830628.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229366.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837093.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2903691.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2905191.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2874021.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2561151.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174570.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2842512.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2773366.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2737232.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2652053.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2925528.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2789927.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2176149.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2222902.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2993914.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2886115.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2807175.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2877900.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2698427.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2966996.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2852669.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2213350.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2692013.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2826659.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2848724.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174223.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955278.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2837255.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2802731.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2213294.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759204.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2172305.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194012.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2945470.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2846128.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217061.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2995241.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2865351.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2983021.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2194026.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2802722.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148139.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2993913.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2988991.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2785514.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2758499.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2978885.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2615309.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2948182.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193237.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2920004.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2671624.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175164.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2575884.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2195588.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2233763.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2767516.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2847171.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2196872.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2671628.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2808715.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2773371.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2877197.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2997562.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695031.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2880267.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2718211.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2233999.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2692018.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2993916.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217195.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2885275.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2172714.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2525541.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2885270.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2169472.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2782145.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2895864.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2805920.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2199198.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2771142.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2174855.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104719.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2819475.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2211781.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2556093.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2944026.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193811.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2610307.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759206.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2990659.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2628021.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229513.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2639029.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2816239.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104739.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2730030.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2898501.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2854344.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2206118.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2396478.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2694397.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2169343.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2710213.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2192934.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2811648.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193624.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2169456.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2232889.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2903697.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2801109.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2504356.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2234460.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2840664.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193117.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2636096.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2529388.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2171981.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217258.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2844595.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695775.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2639449.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2811844.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2749534.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2890157.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2233906.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2533593.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229365.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2873814.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2413274.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217039.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2735678.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2721984.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2933133.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2568035.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2903700.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2830412.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2900824.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2175188.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2173373.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148188.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2206244.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759197.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2947009.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2800007.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2898105.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2150714.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2610314.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2808599.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2758193.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2693542.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2575886.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2104730.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2841273.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2938442.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2386081.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2780626.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2467463.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217067.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779082.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2671202.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2628143.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2695586.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2820692.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759207.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2759209.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2877895.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2868127.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2830605.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2690845.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2978894.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2504354.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2844528.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2615293.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2148126.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2674291.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2779274.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2886088.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955745.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2776144.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193875.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2650880.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2672125.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2172828.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2846129.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2640162.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2681174.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2173089.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193225.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2724697.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2133080.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2757427.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2132948.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2955752.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2217199.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2229371.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2947637.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2789538.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2149348.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2722006.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2874742.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2193362.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2810813.xml: 'NoneType' object has no attribute 'strip'\n",
            "Error processing PMC2608844.xml: 'NoneType' object has no attribute 'strip'\n",
            "Extraction complete. Data saved to final_output.csv.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [],
      "metadata": {
        "id": "e7TV1BjyF7Az"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import pandas as pd\n",
        "csv2 = pd.read_csv(\"/content/data-for-clinical-note-summarizer/detailed_output.csv\")"
      ],
      "metadata": {
        "id": "KFfobZoiGoZg"
      },
      "execution_count": 75,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "csv2.head(10)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 364
        },
        "id": "uwc-6KmuGtgc",
        "outputId": "1e072416-63b2-44d7-9830-c9f87ce5a939"
      },
      "execution_count": 76,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "            journal_title                                      article_title  \\\n",
              "0  Nucleic Acids Research  C1D and hMtr4p associate with the human exosom...   \n",
              "1  Nucleic Acids Research  C1D and hMtr4p associate with the human exosom...   \n",
              "2  Nucleic Acids Research  C1D and hMtr4p associate with the human exosom...   \n",
              "3  Nucleic Acids Research  C1D and hMtr4p associate with the human exosom...   \n",
              "4  Nucleic Acids Research  C1D and hMtr4p associate with the human exosom...   \n",
              "5  Nucleic Acids Research  C1D and hMtr4p associate with the human exosom...   \n",
              "6  Nucleic Acids Research  C1D and hMtr4p associate with the human exosom...   \n",
              "7  Nucleic Acids Research  C1D and hMtr4p associate with the human exosom...   \n",
              "8  Nucleic Acids Research  C1D and hMtr4p associate with the human exosom...   \n",
              "9  Nucleic Acids Research  C1D and hMtr4p associate with the human exosom...   \n",
              "\n",
              "                                            authors  publication_date  \\\n",
              "0  Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn              2007   \n",
              "1  Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn              2007   \n",
              "2  Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn              2007   \n",
              "3  Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn              2007   \n",
              "4  Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn              2007   \n",
              "5  Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn              2007   \n",
              "6  Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn              2007   \n",
              "7  Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn              2007   \n",
              "8  Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn              2007   \n",
              "9  Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn              2007   \n",
              "\n",
              "                  doi                                      section_title  \\\n",
              "0  10.1093/nar/gkm082                                       INTRODUCTION   \n",
              "1  10.1093/nar/gkm082                                       INTRODUCTION   \n",
              "2  10.1093/nar/gkm082                                       INTRODUCTION   \n",
              "3  10.1093/nar/gkm082                                       INTRODUCTION   \n",
              "4  10.1093/nar/gkm082               MATERIALS AND METHODS > cDNA cloning   \n",
              "5  10.1093/nar/gkm082        MATERIALS AND METHODS > Immunoprecipitation   \n",
              "6  10.1093/nar/gkm082      MATERIALS AND METHODS > Western blot analysis   \n",
              "7  10.1093/nar/gkm082  MATERIALS AND METHODS > Expression and purific...   \n",
              "8  10.1093/nar/gkm082                           MATERIALS AND METHODS >    \n",
              "9  10.1093/nar/gkm082  MATERIALS AND METHODS > Transient transfection...   \n",
              "\n",
              "  content_type                                            content  \n",
              "0    paragraph  The human exosome is a complex with 3′–5′ exor...  \n",
              "1    paragraph  A variety of nuclear functions have been descr...  \n",
              "2    paragraph  Besides the core exosome components, several c...  \n",
              "3    paragraph  In this study, we investigated the exosome ass...  \n",
              "4    paragraph  The cDNA of human C1D was obtained by a PCR-ba...  \n",
              "5    paragraph  Polyclonal rabbit anti-EGFP antibodies were co...  \n",
              "6    paragraph  For western blot analysis, proteins were separ...  \n",
              "7    paragraph  For prokaryotic expression, the C1D and MPP6 c...  \n",
              "8    paragraph  The open reading frames of PM/Scl-100, MPP6 an...  \n",
              "9    paragraph  The cDNAs were cloned into suitable pEGFP vect...  "
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-97df37d1-c3c5-4ce9-a7e8-5ddeb448b10d\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>journal_title</th>\n",
              "      <th>article_title</th>\n",
              "      <th>authors</th>\n",
              "      <th>publication_date</th>\n",
              "      <th>doi</th>\n",
              "      <th>section_title</th>\n",
              "      <th>content_type</th>\n",
              "      <th>content</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>Nucleic Acids Research</td>\n",
              "      <td>C1D and hMtr4p associate with the human exosom...</td>\n",
              "      <td>Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn</td>\n",
              "      <td>2007</td>\n",
              "      <td>10.1093/nar/gkm082</td>\n",
              "      <td>INTRODUCTION</td>\n",
              "      <td>paragraph</td>\n",
              "      <td>The human exosome is a complex with 3′–5′ exor...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>Nucleic Acids Research</td>\n",
              "      <td>C1D and hMtr4p associate with the human exosom...</td>\n",
              "      <td>Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn</td>\n",
              "      <td>2007</td>\n",
              "      <td>10.1093/nar/gkm082</td>\n",
              "      <td>INTRODUCTION</td>\n",
              "      <td>paragraph</td>\n",
              "      <td>A variety of nuclear functions have been descr...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>Nucleic Acids Research</td>\n",
              "      <td>C1D and hMtr4p associate with the human exosom...</td>\n",
              "      <td>Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn</td>\n",
              "      <td>2007</td>\n",
              "      <td>10.1093/nar/gkm082</td>\n",
              "      <td>INTRODUCTION</td>\n",
              "      <td>paragraph</td>\n",
              "      <td>Besides the core exosome components, several c...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>Nucleic Acids Research</td>\n",
              "      <td>C1D and hMtr4p associate with the human exosom...</td>\n",
              "      <td>Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn</td>\n",
              "      <td>2007</td>\n",
              "      <td>10.1093/nar/gkm082</td>\n",
              "      <td>INTRODUCTION</td>\n",
              "      <td>paragraph</td>\n",
              "      <td>In this study, we investigated the exosome ass...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>Nucleic Acids Research</td>\n",
              "      <td>C1D and hMtr4p associate with the human exosom...</td>\n",
              "      <td>Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn</td>\n",
              "      <td>2007</td>\n",
              "      <td>10.1093/nar/gkm082</td>\n",
              "      <td>MATERIALS AND METHODS &gt; cDNA cloning</td>\n",
              "      <td>paragraph</td>\n",
              "      <td>The cDNA of human C1D was obtained by a PCR-ba...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>Nucleic Acids Research</td>\n",
              "      <td>C1D and hMtr4p associate with the human exosom...</td>\n",
              "      <td>Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn</td>\n",
              "      <td>2007</td>\n",
              "      <td>10.1093/nar/gkm082</td>\n",
              "      <td>MATERIALS AND METHODS &gt; Immunoprecipitation</td>\n",
              "      <td>paragraph</td>\n",
              "      <td>Polyclonal rabbit anti-EGFP antibodies were co...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>Nucleic Acids Research</td>\n",
              "      <td>C1D and hMtr4p associate with the human exosom...</td>\n",
              "      <td>Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn</td>\n",
              "      <td>2007</td>\n",
              "      <td>10.1093/nar/gkm082</td>\n",
              "      <td>MATERIALS AND METHODS &gt; Western blot analysis</td>\n",
              "      <td>paragraph</td>\n",
              "      <td>For western blot analysis, proteins were separ...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>Nucleic Acids Research</td>\n",
              "      <td>C1D and hMtr4p associate with the human exosom...</td>\n",
              "      <td>Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn</td>\n",
              "      <td>2007</td>\n",
              "      <td>10.1093/nar/gkm082</td>\n",
              "      <td>MATERIALS AND METHODS &gt; Expression and purific...</td>\n",
              "      <td>paragraph</td>\n",
              "      <td>For prokaryotic expression, the C1D and MPP6 c...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>Nucleic Acids Research</td>\n",
              "      <td>C1D and hMtr4p associate with the human exosom...</td>\n",
              "      <td>Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn</td>\n",
              "      <td>2007</td>\n",
              "      <td>10.1093/nar/gkm082</td>\n",
              "      <td>MATERIALS AND METHODS &gt;</td>\n",
              "      <td>paragraph</td>\n",
              "      <td>The open reading frames of PM/Scl-100, MPP6 an...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td>Nucleic Acids Research</td>\n",
              "      <td>C1D and hMtr4p associate with the human exosom...</td>\n",
              "      <td>Geurt Schilders; Erwin van Dijk; Ger J.M. Pruijn</td>\n",
              "      <td>2007</td>\n",
              "      <td>10.1093/nar/gkm082</td>\n",
              "      <td>MATERIALS AND METHODS &gt; Transient transfection...</td>\n",
              "      <td>paragraph</td>\n",
              "      <td>The cDNAs were cloned into suitable pEGFP vect...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-97df37d1-c3c5-4ce9-a7e8-5ddeb448b10d')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-97df37d1-c3c5-4ce9-a7e8-5ddeb448b10d button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-97df37d1-c3c5-4ce9-a7e8-5ddeb448b10d');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-8ccdde26-25db-4cfb-8f5d-f29b9b9bf2c3\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-8ccdde26-25db-4cfb-8f5d-f29b9b9bf2c3')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-8ccdde26-25db-4cfb-8f5d-f29b9b9bf2c3 button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "csv2",
              "summary": "{\n  \"name\": \"csv2\",\n  \"rows\": 89599,\n  \"fields\": [\n    {\n      \"column\": \"journal_title\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 25,\n        \"samples\": [\n          \"Evidence-based Complementary and Alternative Medicine : eCAM\",\n          \"Biophysical Journal\",\n          \"Nucleic Acids Research\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"article_title\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 1569,\n        \"samples\": [\n          \"Selecton 2007: advanced models for detecting positive and purifying selection using a Bayesian inference approach\",\n          \"Epitope mapping using combinatorial phage-display libraries: a graph-based algorithm\",\n          \"High-throughput discovery of rare human nucleotide polymorphisms by Ecotilling\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"authors\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 1605,\n        \"samples\": [\n          \"Tanya Barrett; Dennis B. Troup; Stephen E. Wilhite; Pierre Ledoux; Dmitry Rudnev; Carlos Evangelista; Irene F. Kim; Alexandra Soboleva; Maxim Tomashevsky; Ron Edgar\",\n          \"Nathan J. Felix; W. June Brickey; Robert Griffiths; Jinghua Zhang; Luc Van Kaer; Thomas Coffman; Jenny P.-Y. Ting\",\n          \"Joachim L. Schultze; Sabine Michalak; Joel Lowne; Adam Wong; Maria H. Gilleece; John G. Gribben; Lee M. Nadler\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"publication_date\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 1,\n        \"min\": 1997,\n        \"max\": 2007,\n        \"num_unique_values\": 11,\n        \"samples\": [\n          1997,\n          2007,\n          2003\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"doi\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 1532,\n        \"samples\": [\n          \"10.1093/nar/gkm507\",\n          \"10.1093/nar/gkl934\",\n          \"10.1093/nar/gkl675\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"section_title\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 31158,\n        \"samples\": [\n          \"Analysis of Cytokine Production in SJL/J Mice with Established TNBS Colitis Treated with pCMV-TGF-\\u03b21 with or without Concomitant Administration of Intraperitoneal Anti\\u2013IL-10.\",\n          \"MATERIALS AND METHODS > Construction of expression vector and TR2-inducible 3T3-L1 clones\",\n          \"MATERIALS AND METHODS > Northern blots and 5\\u2032 end mapping\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"content_type\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 3,\n        \"samples\": [\n          \"paragraph\",\n          \"figure\",\n          \"table\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"content\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 53310,\n        \"samples\": [\n          \"PDI 12  subjected to Fe/NTA affinity cleavage showed strand scission 5\\u2032-distal to the PNA target ( Figure 2E , lane 3). Because the NTA moiety is conjugated via the N-terminus of the PNA (see Materials and Methods), these data are compatible with a parallel orientation of the PNA oligomer and formally establish the binding of the PNA strand to the entire target (residues A 1 \\u2013G 12 ) ( Figure 2F , PDI 12 ).\",\n          \"While the computational cost of the present analysis is negligible, we believe that DNA deformation (free) energies in protein\\u2013DNA co-crystals substantially extend the insights that can be gained from structural data. To encourage the application to other systems, we have included the required Mathematica ( 33 ) scripts as Supplementary Data.\",\n          \"Percentage of 2-5A oligomers produced with different RNAs\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 76
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import pandas as pd\n",
        "csv = pd.read_csv(\"/content/data-for-clinical-note-summarizer/final_output.csv\")"
      ],
      "metadata": {
        "id": "EjU67OutEA-m"
      },
      "execution_count": 84,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "csv.head(10)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 364
        },
        "id": "IspGW5kbIiql",
        "outputId": "fb3be94b-7e79-4ba2-9e60-b617b142cccf"
      },
      "execution_count": 86,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "                          journal_title  \\\n",
              "0           The Journal of Cell Biology   \n",
              "1           The Journal of Cell Biology   \n",
              "2                              Diabetes   \n",
              "3                                  Lung   \n",
              "4  The Journal of Experimental Medicine   \n",
              "5            Nanoscale Research Letters   \n",
              "6     Journal of Korean Medical Science   \n",
              "7                         Diabetes Care   \n",
              "8           The Journal of Cell Biology   \n",
              "9  The Journal of Experimental Medicine   \n",
              "\n",
              "                                       article_title  \\\n",
              "0                                         Correction   \n",
              "1  Cells in the Sun: The American Society for Cel...   \n",
              "2  Higher Plasma Soluble Receptor for Advanced Gl...   \n",
              "3  Use of Mannitol Inhalation Challenge in Assess...   \n",
              "4  THE INTRAVENOUS INJECTION OF MAGNESIUM SULPHAT...   \n",
              "5  Atomic Layer Deposition of ZnO on Multi-walled...   \n",
              "6  The Breakdown of Preformed Peritoneal Advanced...   \n",
              "7  Risk Factors for Childhood Overweight in Offsp...   \n",
              "8                      A cytokine that packs a punch   \n",
              "9  Phagocytic chimeric receptors require both tra...   \n",
              "\n",
              "                                             authors  publication_date  \\\n",
              "0                                                NaN              1991   \n",
              "1                William A. Wells; Nicole LeBrasseur              2007   \n",
              "2  Johanna W.M. Nin; Anders Jorsal; Isabel Ferrei...              2010   \n",
              "3                                    Sheldon Spector              2009   \n",
              "4                             J. Auer; S. J. Meltzer              1916   \n",
              "5     XL Li; C Li; Y Zhang; DP Chu; WI Milne; HJ Fan              2010   \n",
              "6  Yong-Kook Lee; Joon-Yeop Lee; Jun-Seup Kim; Ki...              2009   \n",
              "7  Maarten Rijpert; Inge M. Evers; Monique A.M.J....              2009   \n",
              "8                                  Nicole LeBrasseur              2003   \n",
              "9                                                NaN              1992   \n",
              "\n",
              "                            doi  \\\n",
              "0                           NaN   \n",
              "1            10.1083/jcb.1765mr   \n",
              "2             10.2337/db09-1509   \n",
              "3     10.1007/s00408-009-9174-2   \n",
              "4                           NaN   \n",
              "5     10.1007/s11671-010-9721-z   \n",
              "6  10.3346/jkms.2009.24.S1.S189   \n",
              "7             10.2337/dc09-0652   \n",
              "8            10.1083/jcb1615rr4   \n",
              "9                           NaN   \n",
              "\n",
              "                                            abstract  \\\n",
              "0                                                NaN   \n",
              "1                                                NaN   \n",
              "2  OBJECTIVE To investigate the associations of p...   \n",
              "3  Bronchial provocation testing uses a variety o...   \n",
              "4  These experiments justify the following genera...   \n",
              "5  In this article, direct coating of ZnO on PECV...   \n",
              "6  It has been demonstrated that inhibitors of ad...   \n",
              "7  OBJECTIVE Pregnancy in type 1 diabetic women r...   \n",
              "8                                                NaN   \n",
              "9  Phagocytosis has traditionally been viewed as ...   \n",
              "\n",
              "                                            keywords  \\\n",
              "0                                                NaN   \n",
              "1                                                NaN   \n",
              "2                                                NaN   \n",
              "3  Mannitol; Bronchoprovocation; Inhalation chall...   \n",
              "4                                                NaN   \n",
              "5  ZnO; Atomic layer deposition; Carbon nanotubes...   \n",
              "6  Advanced Glycation End Products; Aminoguanidin...   \n",
              "7                                                NaN   \n",
              "8                                                NaN   \n",
              "9                                                NaN   \n",
              "\n",
              "                                          paragraphs  \\\n",
              "0                                                NaN   \n",
              "1  RNAi can unite the precision view from microsc...   \n",
              "2  To investigate the associations of plasma leve...   \n",
              "3  Bronchial provocation testing may be employed ...   \n",
              "4                                                NaN   \n",
              "5  Carbon nanotube (CNT) and inorganic composite ...   \n",
              "6  Advanced glycation end products (AGE) have bee...   \n",
              "7  Pregnancy in type 1 diabetic women remains a h...   \n",
              "8                                                NaN   \n",
              "9                                                NaN   \n",
              "\n",
              "                                             figures  \\\n",
              "0                                                NaN   \n",
              "1  The power of RNAi and microscopy are united in...   \n",
              "2  Cumulative hazard for CVD morbidity and mortal...   \n",
              "3                                                NaN   \n",
              "4                                                NaN   \n",
              "5  a  TEM image of PECVD-grown CNTs.  b  TEM imag...   \n",
              "6                                                NaN   \n",
              "7  Prevalence of childhood overweight according t...   \n",
              "8                                                NaN   \n",
              "9                                                NaN   \n",
              "\n",
              "                                              tables  lists  \n",
              "0                                                NaN    NaN  \n",
              "1                                                NaN    NaN  \n",
              "2  Baseline characteristics according to the occu...    NaN  \n",
              "3                                                NaN    NaN  \n",
              "4                                                NaN    NaN  \n",
              "5                                                NaN    NaN  \n",
              "6                                                NaN    NaN  \n",
              "7  Possible risk factors for development of child...    NaN  \n",
              "8                                                NaN    NaN  \n",
              "9                                                NaN    NaN  "
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-5d2592f5-f933-4f5e-a764-ea3e7e87a13c\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>journal_title</th>\n",
              "      <th>article_title</th>\n",
              "      <th>authors</th>\n",
              "      <th>publication_date</th>\n",
              "      <th>doi</th>\n",
              "      <th>abstract</th>\n",
              "      <th>keywords</th>\n",
              "      <th>paragraphs</th>\n",
              "      <th>figures</th>\n",
              "      <th>tables</th>\n",
              "      <th>lists</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>The Journal of Cell Biology</td>\n",
              "      <td>Correction</td>\n",
              "      <td>NaN</td>\n",
              "      <td>1991</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>The Journal of Cell Biology</td>\n",
              "      <td>Cells in the Sun: The American Society for Cel...</td>\n",
              "      <td>William A. Wells; Nicole LeBrasseur</td>\n",
              "      <td>2007</td>\n",
              "      <td>10.1083/jcb.1765mr</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>RNAi can unite the precision view from microsc...</td>\n",
              "      <td>The power of RNAi and microscopy are united in...</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>Diabetes</td>\n",
              "      <td>Higher Plasma Soluble Receptor for Advanced Gl...</td>\n",
              "      <td>Johanna W.M. Nin; Anders Jorsal; Isabel Ferrei...</td>\n",
              "      <td>2010</td>\n",
              "      <td>10.2337/db09-1509</td>\n",
              "      <td>OBJECTIVE To investigate the associations of p...</td>\n",
              "      <td>NaN</td>\n",
              "      <td>To investigate the associations of plasma leve...</td>\n",
              "      <td>Cumulative hazard for CVD morbidity and mortal...</td>\n",
              "      <td>Baseline characteristics according to the occu...</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>Lung</td>\n",
              "      <td>Use of Mannitol Inhalation Challenge in Assess...</td>\n",
              "      <td>Sheldon Spector</td>\n",
              "      <td>2009</td>\n",
              "      <td>10.1007/s00408-009-9174-2</td>\n",
              "      <td>Bronchial provocation testing uses a variety o...</td>\n",
              "      <td>Mannitol; Bronchoprovocation; Inhalation chall...</td>\n",
              "      <td>Bronchial provocation testing may be employed ...</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>The Journal of Experimental Medicine</td>\n",
              "      <td>THE INTRAVENOUS INJECTION OF MAGNESIUM SULPHAT...</td>\n",
              "      <td>J. Auer; S. J. Meltzer</td>\n",
              "      <td>1916</td>\n",
              "      <td>NaN</td>\n",
              "      <td>These experiments justify the following genera...</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>Nanoscale Research Letters</td>\n",
              "      <td>Atomic Layer Deposition of ZnO on Multi-walled...</td>\n",
              "      <td>XL Li; C Li; Y Zhang; DP Chu; WI Milne; HJ Fan</td>\n",
              "      <td>2010</td>\n",
              "      <td>10.1007/s11671-010-9721-z</td>\n",
              "      <td>In this article, direct coating of ZnO on PECV...</td>\n",
              "      <td>ZnO; Atomic layer deposition; Carbon nanotubes...</td>\n",
              "      <td>Carbon nanotube (CNT) and inorganic composite ...</td>\n",
              "      <td>a  TEM image of PECVD-grown CNTs.  b  TEM imag...</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>Journal of Korean Medical Science</td>\n",
              "      <td>The Breakdown of Preformed Peritoneal Advanced...</td>\n",
              "      <td>Yong-Kook Lee; Joon-Yeop Lee; Jun-Seup Kim; Ki...</td>\n",
              "      <td>2009</td>\n",
              "      <td>10.3346/jkms.2009.24.S1.S189</td>\n",
              "      <td>It has been demonstrated that inhibitors of ad...</td>\n",
              "      <td>Advanced Glycation End Products; Aminoguanidin...</td>\n",
              "      <td>Advanced glycation end products (AGE) have bee...</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>Diabetes Care</td>\n",
              "      <td>Risk Factors for Childhood Overweight in Offsp...</td>\n",
              "      <td>Maarten Rijpert; Inge M. Evers; Monique A.M.J....</td>\n",
              "      <td>2009</td>\n",
              "      <td>10.2337/dc09-0652</td>\n",
              "      <td>OBJECTIVE Pregnancy in type 1 diabetic women r...</td>\n",
              "      <td>NaN</td>\n",
              "      <td>Pregnancy in type 1 diabetic women remains a h...</td>\n",
              "      <td>Prevalence of childhood overweight according t...</td>\n",
              "      <td>Possible risk factors for development of child...</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>The Journal of Cell Biology</td>\n",
              "      <td>A cytokine that packs a punch</td>\n",
              "      <td>Nicole LeBrasseur</td>\n",
              "      <td>2003</td>\n",
              "      <td>10.1083/jcb1615rr4</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td>The Journal of Experimental Medicine</td>\n",
              "      <td>Phagocytic chimeric receptors require both tra...</td>\n",
              "      <td>NaN</td>\n",
              "      <td>1992</td>\n",
              "      <td>NaN</td>\n",
              "      <td>Phagocytosis has traditionally been viewed as ...</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-5d2592f5-f933-4f5e-a764-ea3e7e87a13c')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-5d2592f5-f933-4f5e-a764-ea3e7e87a13c button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-5d2592f5-f933-4f5e-a764-ea3e7e87a13c');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-bd5fe75f-1c56-4db4-918e-3a375fb61020\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-bd5fe75f-1c56-4db4-918e-3a375fb61020')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-bd5fe75f-1c56-4db4-918e-3a375fb61020 button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "csv",
              "summary": "{\n  \"name\": \"csv\",\n  \"rows\": 76244,\n  \"fields\": [\n    {\n      \"column\": \"journal_title\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 819,\n        \"samples\": [\n          \"Nature Communications\",\n          \"Journal of Cardiovascular Translational Research\",\n          \"Journal of Gambling Studies\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"article_title\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 72088,\n        \"samples\": [\n          \"Identification of histone H2b as a heat-shock protein in\\nDrosophila\",\n          \"Anaphylaxis with angioedema by rocuronium during induction of general anesthesia -A case report-\",\n          \"Molecular composition of the node of Ranvier: identification of ankyrin- binding cell adhesion molecules neurofascin (mucin+/third FNIII domain- ) and NrCAM at nodal axon segments\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"authors\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 48525,\n        \"samples\": [\n          \"K. Cheng-Minoda; J. Davidowitz; A. Liebowitz; G. M. Breinin\",\n          \"Sang Yul Choi; Mi-Sook Kim; SungYul Yoo; ChulKoo Cho; YoungHoon Ji; KumBae Kim; YoungSeok Seo; Kyung Duk Park; JunAh Lee; Tai-Won Lee\",\n          \"Kael Duprey; Michelle Lin\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"publication_date\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 24,\n        \"min\": 1896,\n        \"max\": 2011,\n        \"num_unique_values\": 114,\n        \"samples\": [\n          1908,\n          1916,\n          1974\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"doi\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 33292,\n        \"samples\": [\n          \"10.1093/jxb/ern057\",\n          \"10.4111/kju.2010.51.10.671\",\n          \"10.1136/bmj.c4616\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 68310,\n        \"samples\": [\n          \"In this paper we describe a 66-kD protein that co-purifies with intermediate filaments from rat optic nerve and spinal cord but can be separated further by ion-exchange chromatography. This protein is distinct from the 68-kD neurofilament subunit protein as judged by isoelectric focusing, immunoblotting, peptide mapping, and tests of polymerization competence. This protein is avidly recognized by the monoclonal anti-intermediate filament antigen antibody, previously demonstrated to recognize a common antigenic determinant in all five known classes of intermediate filaments. Also, when isolated this protein binds to various intermediate filament subunit proteins, which suggests an in vivo interaction with the intermediate filament cytoskeleton, and it appears to be axonally transported in the rat optic nerve. Because of this ability to bind to intermediate filaments in situ and in vitro we have named this protein alpha-internexin. A possible functional role for the protein in organizing filament assembly and distribution is discussed.\",\n          \"Discrete fluctuations in conductance of lipid bilayer membranes may be observed during the initial stages of membrane interaction with EIM (\\\"excitability inducing material\\\"), during destruction of the EIM conductance by proteolysis, and during the potential-dependent transitions between low and high conductance states in the \\\"excitable\\\" membranes. The discrete conductance steps observed during the initial reaction of EIM with the lipid membranes are remarkably uniform, even in membranes of widely varying lipid composition. They range only from 2 to 6 x 10 -10  ohm -1  and average 4 x 10 -10  ohm -1 . Steps found during destruction of the EIM conductance by proteolysis are somewhat smaller. The transition between high conductance and low conductance states may involve steps as small as 0.5 x 10 -10  ohm -1 . These phenomena are consistent with the formation of a stable protein bridge across the lipid membrane to provide a polar channel for the transport of cations. T6he uniform conductance fluctuations observed during the formation of these macromolecular channels may indicate that the ions in a conductive channel, in its open state, are largely protected from the influence of the polar groups of the membrane lipids. Potential-dependent changes in conductance may be due to configurational or positional changes in the protein channel. Differences in lipid-lipid and lipid-macromolecule interactions may account for the variations in switching kinetics in various membrane systems.\",\n          \"Uncoupling proteins 2 and 3 (UCP2/3) are essential for mitochondrial Ca 2+  uptake but both proteins exhibit distinct activities in regard to the source and mode of Ca 2+  mobilization. In the present work, structural determinants of their contribution to mitochondrial Ca 2+  uptake were explored. Previous findings indicate the importance of the intermembrane loop 2 (IML2) for the contribution of UCP2/3. Thus, the IML2 of UCP2/3 was substituted by that of UCP1. These chimeras had no activity in mitochondrial uptake of intracellularly released Ca 2+ , while they mimicked the wild-type proteins by potentiating mitochondrial sequestration of entering Ca 2+ . Alignment of the IML2 sequences revealed that UCP1, UCP2 and UCP3 share a basic amino acid in positions 163, 164 and 167, while only UCP2 and UCP3 contain a second basic residue in positions 168 and 171, respectively. Accordingly, mutants of UCP3 in positions 167 and 171/172 were made. In permeabilized cells, these mutants exhibited distinct Ca 2+  sensitivities in regard to mitochondrial Ca 2+  sequestration. In intact cells, these mutants established different activities in mitochondrial uptake of either intracellularly released (UCP3 R171,E172 ) or entering (UCP3 R167 ) Ca 2+ . Our data demonstrate that distinct sites in the IML2 of UCP3 effect mitochondrial uptake of high and low Ca 2+  signals.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"keywords\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 19584,\n        \"samples\": [\n          \"cardiovascular disease; dietary; risk factors; sodium\",\n          \"Bone regeneration; Collagen; Synthetic peptide\",\n          \"melanoma; melanocyte; tyrosinase-related protein; T cell; perforin\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"paragraphs\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 34792,\n        \"samples\": [\n          \"C57BL/6 (H-2 b ) male mice and 129/SEV and IFN-\\u03b3 receptor knockout (IFN-\\u03b3R KO; reference  17 ) mice of both sexes were either purchased from The  Jackson Laboratory  (Bar Harbor, ME) or else bred in our facility and used at 2\\u201312 mo of age. LCMV, strain Armstrong, an RNA virus in the Old World arenavirus family, was propagated in BHK21 baby hamster kidney cells ( 18 ). The WR strain of VV, a DNA virus in the poxvirus family ( 11 ), and the AN3739 strain of PV, an RNA virus in the New World arenavirus family only distantly related to LCMV, were propagated in L929 or BHK21 cells ( 11 ,  18 ,  19 ). MCMV, strain Smith, a DNA virus in the herpesvirus family, was obtained from salivary glands of infected BALB/c mice ( 20 ). For acute virus infections, mice were injected intraperitoneally with 4 \\u00d7 10 4  PFU of LCMV, 2 \\u00d7 10 4 , 4 \\u00d7 10 5 , or 10 6  PFU of VV, 10 6  PFU of PV, or 10 4  or 10 5  PFU of MCMV. KO (H-2 b ), an SV40-transformed kidney cell line derived from a C57BL/6 mouse ( 21 ) and provided to us by Dr. Satvir Tevethia (Pennsylvania State Medical Center, Hershey, PA), was propagated in DMEM ( GIBCO BRL , Gaithersburg, MD) supplemented with 5 \\u00d7 10 \\u22125  M 2-ME. L929 (H-2 k ), a continuous liver cell line derived from C3H mice, MC57G (H-2 b ), a methylcholanthrene-induced fibroblast cell line from C57BL/6 mice, early passage mouse embryo fibroblast (MEF) cells from C57BL/6 mice, and ATCC Vero cells were propagated in Eagle's MEM ( GIBCO BRL ). KO and MC57G cells were infected with LCMV and PV at a multiplicity of infection (MOI) of 0.1\\u20130.2 PFU/cell and incubated for 2 d at 37\\u00b0C. All cell lines were supplemented with 100 U/ml penicillin G, 100 \\u03bcg/ml streptomycin sulfate, 2 mM  l -glutamine, 10 mM Hepes, and 10% heat-inactivated (56\\u00b0C, 30 min) fetal bovine serum (FBS;  Sigma Chemical Co. , St. Louis, MO). Mice were immunized with a sublethal dose of one virus. After the rise and fall of the acute T cell response and when the immune system had returned to homeostasis (usually 6 wk or longer), the mouse was challenged with the second virus. At 3\\u20135 d after the second virus infection, the spleen, fat pads, and/or liver were harvested, homogenized, and titrated for virus plaques, as previously described ( 22 ). In mice infected with LCMV only the spleens were titrated, as this strain of LCMV does not replicate in the liver or fat pads. These viruses were used either in crude tissue culture supernatants or were purified over sucrose gradients and diluted in PBS. Control naive mice were either left uninjected or injected with tissue culture media or with culture media sedimented like virus over a sucrose gradient, and there was no significant difference in virus titer when any one of these methods was compared with another. The control mice were always age matched to the experimental group and housed under exactly the same pathogen-free conditions as the experimental group for the identical time period. All mice used were healthy with no evidence of any underlying disease. Peritoneal cells from individual mice were stained for intracellular IFN-\\u03b3 based on a method previously described ( 23 ). Peritoneal cells at 2 \\u00d7 10 6  per tube were stimulated with PMA ( Sigma Chemical Co. ) at 50 ng/ml and ionomycin ( Sigma Chemical Co. ) at 500 ng/ml for 2 h at 37\\u00b0C. Then brefeldin A ( Sigma Chemical Co. ) was added at 10 \\u03bcg/ml for 2 h at 37\\u00b0C enabling intracellular proteins to accumulate. After the cells were washed with FACS \\u00ae  buffer (PBS with 2% FBS and 0.02% sodium azide) they were incubated with anti-Fc \\u03b3II receptor antibody ( PharMingen , San Diego, CA) as a blocking antibody, for 10 min at 4\\u00b0C. Then tricolor-conjugated anti\\u2013 mouse CD4 (clone CT-CD4; Caltag Laboratories, San Francisco, CA) or CD8 (clone CT-CD8\\u03b1; Caltag Laboratories) was added for 30 min at 4\\u00b0C. After two washes with FACS \\u00ae  buffer the cells were fixed with 200 \\u03bcl of 2% paraformaldehyde for 20 min at room temperature. They were then washed with FACS \\u00ae  buffer containing 0.5% saponin ( Sigma Chemical Co. ), a permeabilizing agent, and then incubated in this buffer for another 10 min at room temperature before adding PE-conjugated rat anti\\u2013mouse anti-IFN-\\u03b3 mAb ( PharMingen ) or control PE-conjugated rat IgG 1  isotype for 30 min at room temperature. They were then washed two more times with FACS \\u00ae  buffer with saponin and then one time with FACS \\u00ae  buffer before analysis on either a FACS \\u00ae  440 ( Becton Dickinson  & Co., Sparks, MD) or FACStar \\u00ae  Plus ( Becton Dickinson  & Co.). The total number of CD8 +  and CD4 +  or the total number of CD8 +  and CD4 +  cells secreting IFN-\\u03b3 was determined by multiplying the percentage times the total cell yield from each mouse. The total CD4 +  percentages may be an underestimate as these cells tend to downregulate the CD4 +  receptor on stimulation with PMA and ionomycin. That was not a problem with the CD8 +  cells. Adult 129/SEV and IFN-\\u03b3 receptor knockout mice (IFN-\\u03b3R KO) mice were inoculated intraperitoneally with LCMV, and at the indicated times after infection spleens were harvested. The LCMV-specific precursor frequency (p/f) per CD8 +  cell was quantified by LDAs of unsorted cells ( 12 ,  24 ). The percentage of CD8 +  T cells was determined by fluorescent antibody staining and FACS \\u00ae  analysis, as previously described ( 12 ,  24 ). The limiting dilution assays used a previously described method ( 12 ,  24 ). In brief, splenic lymphocytes from infected mice were harvested and titrated in U-bottomed, 96-well plates with 24 replicates at each dilution. They were stimulated with virus-infected PECs (3\\u20134 \\u00d7 10 4 /well) and supplemented with irradiated splenic feeders (1\\u20132 \\u00d7 10 5 /well) and growth factors provided by using a 16% culture supernatant from IL-2\\u2013secreting, gibbon lymphoma tumor cell line MLA.144 (American Type Culture Collection, Rockville, MD; reference  25 ). After 4 d, the cultures were fed with 10 4  irradiated, virus-infected PEC. On days 7\\u20138 of culture, individual wells were split twofold and assayed for cytolytic function on LCMV-infected or uninfected syngeneic target cells (KO) using a modified  51 Cr-release assay.  51 Cr-labeled targets (5 \\u00d7 10 3 ) were added to all wells, and after an 8\\u201310 h incubation at 37\\u00b0C the supernatant was harvested. Positive wells were defined as those wells whose  51 Cr-release exceeded the mean spontaneous release by >3 standard deviations. All wells that lysed uninfected syngeneic targets were eliminated from the analysis. Frequencies were calculated using \\u03c7 2  analysis according to Taswell ( 26 ) on a computer program kindly provided by Dr. Richard Miller (University of Michigan, Ann Arbor, MI). Cell-mediated cytotoxicity was determined using a standard microcytotoxicity (CTL) assay ( 19 ). Varying numbers of effector leukocytes were plated in triplicate to achieve the desired E/T ratio.  51 Cr-labeled MC57G target cells (5 \\u00d7 10 3 ), either uninfected or infected with virus, were added to all wells, and after a 6 h incubation at 37\\u00b0C the supernatant was harvested and counted. Data are expressed as percent specific  51 Cr-release = 100 \\u00d7 ([experimental cpm \\u2212 spontaneous cpm]/[maximum release cpm \\u2212 spontaneous release cpm]). Lytic units were calculated using exponential fit method ( 27 ) provided by software from Proteins International (Rochester Hills, MI). One lytic unit was defined as the number of effector cells required to lyse 5% of a population of 5 \\u00d7 10 3  targets in a 6 h CTL assay. A low percentage lysis was used for these calculations because some of the samples examined had relatively low levels of killing, and we wished to make quantitative comparisons within a wide range of very low and high CTL activity. The spontaneous release for each target used in these assays was <20%. C57BL/6 mice were injected intravenously with one spleen equivalent (\\u223c5\\u201310 \\u00d7 10 7  splenocytes) of immunologically naive (nonimmune) leukocytes or LCMV- immune (>6 wk after infection) leukocytes and challenged I.P. with PV (10 6  PFU) or VV (10 6  PFU). In some experiments LCMV-immune leukocytes were treated with anti-CD8 (rat anti\\u2013mouse CD8 mAb, clone Lyt 2.43; reference  28 ) and complement (rabbit serum) to remove CD8 +  T cells ( 29 ) or with anti-CD4 (rat anti\\u2013mouse CD4 mAb, clone GK1.5) and complement (guinea pig serum) to remove CD4 +  T cells before transfer. Spleens were harvested, homogenized, and plaque-assayed for virus titer at 3\\u20134 d after infection. The effectiveness of the CD4 +  or CD8 +  T cell depletion was monitored by staining with anti\\u2013 mouse CD4 (clone L3T4 RM4-4;  PharMingen ) or anti\\u2013mouse CD8 (clone CD8a Ly-2(53-6.7);  PharMingen ; reference  3 ). FACS \\u00ae  analyses showed that the CD4 +  or CD8 +  T cell contamination was <2%. The infiltrating leukocytes of VV-challenged LCMV-immune mice were isolated from the fat pads by mincing and digesting with collagenase B (200 mg/ml) in MEM media plus 4% BSA for 1 h at 37\\u00b0C, and then the lymphocytes were separated over Lympholite M (Cedarlane Labs., Hornby, Ontario, Canada) and analyzed. The number of PFU was determined by plaque assay using a 10% homogenate of tissue taken from individual mice and 10-fold dilutions of this homogenate on the appropriate cell line, ATCC vero cells for LCMV, PV, VV, and MEF cells for MCMV. Results were expressed as the geometric mean titers, i.e., the arithmetic averages of the logs for four or five separate animals titrated for virus individually plus or minus the SEM. Titers reported are log 10  PFU per whole spleen, liver, and both abdominal fat pads. The level of acute fatty necrosis was scored visually based on the severity of disease from level 0 to 7. Levels 1 and 2 represent very mild to mild disease with a few white necrotic spots on one or both lower abdominal fat pads; levels 3 and 4 represent mildly moderate and moderate with larger patches of necrosis of the lower abdominal fat pads and extension into the upper left quadrant fat pad around the spleen; and levels 5 and 6 represent moderately severe to severe with very extensive large patches of necrosis on the lower abdominal fat pads and spotty fatty necrosis throughout omental fat pads as well as the splenic fat pad; level 7 represents very severe disease with such severe fatty necrosis that the organs are adherent to each other. C57BL/6 (H-2 b ) male mice and 129/SEV and IFN-\\u03b3 receptor knockout (IFN-\\u03b3R KO; reference  17 ) mice of both sexes were either purchased from The  Jackson Laboratory  (Bar Harbor, ME) or else bred in our facility and used at 2\\u201312 mo of age. LCMV, strain Armstrong, an RNA virus in the Old World arenavirus family, was propagated in BHK21 baby hamster kidney cells ( 18 ). The WR strain of VV, a DNA virus in the poxvirus family ( 11 ), and the AN3739 strain of PV, an RNA virus in the New World arenavirus family only distantly related to LCMV, were propagated in L929 or BHK21 cells ( 11 ,  18 ,  19 ). MCMV, strain Smith, a DNA virus in the herpesvirus family, was obtained from salivary glands of infected BALB/c mice ( 20 ). For acute virus infections, mice were injected intraperitoneally with 4 \\u00d7 10 4  PFU of LCMV, 2 \\u00d7 10 4 , 4 \\u00d7 10 5 , or 10 6  PFU of VV, 10 6  PFU of PV, or 10 4  or 10 5  PFU of MCMV. KO (H-2 b ), an SV40-transformed kidney cell line derived from a C57BL/6 mouse ( 21 ) and provided to us by Dr. Satvir Tevethia (Pennsylvania State Medical Center, Hershey, PA), was propagated in DMEM ( GIBCO BRL , Gaithersburg, MD) supplemented with 5 \\u00d7 10 \\u22125  M 2-ME. L929 (H-2 k ), a continuous liver cell line derived from C3H mice, MC57G (H-2 b ), a methylcholanthrene-induced fibroblast cell line from C57BL/6 mice, early passage mouse embryo fibroblast (MEF) cells from C57BL/6 mice, and ATCC Vero cells were propagated in Eagle's MEM ( GIBCO BRL ). KO and MC57G cells were infected with LCMV and PV at a multiplicity of infection (MOI) of 0.1\\u20130.2 PFU/cell and incubated for 2 d at 37\\u00b0C. All cell lines were supplemented with 100 U/ml penicillin G, 100 \\u03bcg/ml streptomycin sulfate, 2 mM  l -glutamine, 10 mM Hepes, and 10% heat-inactivated (56\\u00b0C, 30 min) fetal bovine serum (FBS;  Sigma Chemical Co. , St. Louis, MO). Mice were immunized with a sublethal dose of one virus. After the rise and fall of the acute T cell response and when the immune system had returned to homeostasis (usually 6 wk or longer), the mouse was challenged with the second virus. At 3\\u20135 d after the second virus infection, the spleen, fat pads, and/or liver were harvested, homogenized, and titrated for virus plaques, as previously described ( 22 ). In mice infected with LCMV only the spleens were titrated, as this strain of LCMV does not replicate in the liver or fat pads. These viruses were used either in crude tissue culture supernatants or were purified over sucrose gradients and diluted in PBS. Control naive mice were either left uninjected or injected with tissue culture media or with culture media sedimented like virus over a sucrose gradient, and there was no significant difference in virus titer when any one of these methods was compared with another. The control mice were always age matched to the experimental group and housed under exactly the same pathogen-free conditions as the experimental group for the identical time period. All mice used were healthy with no evidence of any underlying disease. Peritoneal cells from individual mice were stained for intracellular IFN-\\u03b3 based on a method previously described ( 23 ). Peritoneal cells at 2 \\u00d7 10 6  per tube were stimulated with PMA ( Sigma Chemical Co. ) at 50 ng/ml and ionomycin ( Sigma Chemical Co. ) at 500 ng/ml for 2 h at 37\\u00b0C. Then brefeldin A ( Sigma Chemical Co. ) was added at 10 \\u03bcg/ml for 2 h at 37\\u00b0C enabling intracellular proteins to accumulate. After the cells were washed with FACS \\u00ae  buffer (PBS with 2% FBS and 0.02% sodium azide) they were incubated with anti-Fc \\u03b3II receptor antibody ( PharMingen , San Diego, CA) as a blocking antibody, for 10 min at 4\\u00b0C. Then tricolor-conjugated anti\\u2013 mouse CD4 (clone CT-CD4; Caltag Laboratories, San Francisco, CA) or CD8 (clone CT-CD8\\u03b1; Caltag Laboratories) was added for 30 min at 4\\u00b0C. After two washes with FACS \\u00ae  buffer the cells were fixed with 200 \\u03bcl of 2% paraformaldehyde for 20 min at room temperature. They were then washed with FACS \\u00ae  buffer containing 0.5% saponin ( Sigma Chemical Co. ), a permeabilizing agent, and then incubated in this buffer for another 10 min at room temperature before adding PE-conjugated rat anti\\u2013mouse anti-IFN-\\u03b3 mAb ( PharMingen ) or control PE-conjugated rat IgG 1  isotype for 30 min at room temperature. They were then washed two more times with FACS \\u00ae  buffer with saponin and then one time with FACS \\u00ae  buffer before analysis on either a FACS \\u00ae  440 ( Becton Dickinson  & Co., Sparks, MD) or FACStar \\u00ae  Plus ( Becton Dickinson  & Co.). The total number of CD8 +  and CD4 +  or the total number of CD8 +  and CD4 +  cells secreting IFN-\\u03b3 was determined by multiplying the percentage times the total cell yield from each mouse. The total CD4 +  percentages may be an underestimate as these cells tend to downregulate the CD4 +  receptor on stimulation with PMA and ionomycin. That was not a problem with the CD8 +  cells. Adult 129/SEV and IFN-\\u03b3 receptor knockout mice (IFN-\\u03b3R KO) mice were inoculated intraperitoneally with LCMV, and at the indicated times after infection spleens were harvested. The LCMV-specific precursor frequency (p/f) per CD8 +  cell was quantified by LDAs of unsorted cells ( 12 ,  24 ). The percentage of CD8 +  T cells was determined by fluorescent antibody staining and FACS \\u00ae  analysis, as previously described ( 12 ,  24 ). The limiting dilution assays used a previously described method ( 12 ,  24 ). In brief, splenic lymphocytes from infected mice were harvested and titrated in U-bottomed, 96-well plates with 24 replicates at each dilution. They were stimulated with virus-infected PECs (3\\u20134 \\u00d7 10 4 /well) and supplemented with irradiated splenic feeders (1\\u20132 \\u00d7 10 5 /well) and growth factors provided by using a 16% culture supernatant from IL-2\\u2013secreting, gibbon lymphoma tumor cell line MLA.144 (American Type Culture Collection, Rockville, MD; reference  25 ). After 4 d, the cultures were fed with 10 4  irradiated, virus-infected PEC. On days 7\\u20138 of culture, individual wells were split twofold and assayed for cytolytic function on LCMV-infected or uninfected syngeneic target cells (KO) using a modified  51 Cr-release assay.  51 Cr-labeled targets (5 \\u00d7 10 3 ) were added to all wells, and after an 8\\u201310 h incubation at 37\\u00b0C the supernatant was harvested. Positive wells were defined as those wells whose  51 Cr-release exceeded the mean spontaneous release by >3 standard deviations. All wells that lysed uninfected syngeneic targets were eliminated from the analysis. Frequencies were calculated using \\u03c7 2  analysis according to Taswell ( 26 ) on a computer program kindly provided by Dr. Richard Miller (University of Michigan, Ann Arbor, MI). Cell-mediated cytotoxicity was determined using a standard microcytotoxicity (CTL) assay ( 19 ). Varying numbers of effector leukocytes were plated in triplicate to achieve the desired E/T ratio.  51 Cr-labeled MC57G target cells (5 \\u00d7 10 3 ), either uninfected or infected with virus, were added to all wells, and after a 6 h incubation at 37\\u00b0C the supernatant was harvested and counted. Data are expressed as percent specific  51 Cr-release = 100 \\u00d7 ([experimental cpm \\u2212 spontaneous cpm]/[maximum release cpm \\u2212 spontaneous release cpm]). Lytic units were calculated using exponential fit method ( 27 ) provided by software from Proteins International (Rochester Hills, MI). One lytic unit was defined as the number of effector cells required to lyse 5% of a population of 5 \\u00d7 10 3  targets in a 6 h CTL assay. A low percentage lysis was used for these calculations because some of the samples examined had relatively low levels of killing, and we wished to make quantitative comparisons within a wide range of very low and high CTL activity. The spontaneous release for each target used in these assays was <20%. C57BL/6 mice were injected intravenously with one spleen equivalent (\\u223c5\\u201310 \\u00d7 10 7  splenocytes) of immunologically naive (nonimmune) leukocytes or LCMV- immune (>6 wk after infection) leukocytes and challenged I.P. with PV (10 6  PFU) or VV (10 6  PFU). In some experiments LCMV-immune leukocytes were treated with anti-CD8 (rat anti\\u2013mouse CD8 mAb, clone Lyt 2.43; reference  28 ) and complement (rabbit serum) to remove CD8 +  T cells ( 29 ) or with anti-CD4 (rat anti\\u2013mouse CD4 mAb, clone GK1.5) and complement (guinea pig serum) to remove CD4 +  T cells before transfer. Spleens were harvested, homogenized, and plaque-assayed for virus titer at 3\\u20134 d after infection. The effectiveness of the CD4 +  or CD8 +  T cell depletion was monitored by staining with anti\\u2013 mouse CD4 (clone L3T4 RM4-4;  PharMingen ) or anti\\u2013mouse CD8 (clone CD8a Ly-2(53-6.7);  PharMingen ; reference  3 ). FACS \\u00ae  analyses showed that the CD4 +  or CD8 +  T cell contamination was <2%. The infiltrating leukocytes of VV-challenged LCMV-immune mice were isolated from the fat pads by mincing and digesting with collagenase B (200 mg/ml) in MEM media plus 4% BSA for 1 h at 37\\u00b0C, and then the lymphocytes were separated over Lympholite M (Cedarlane Labs., Hornby, Ontario, Canada) and analyzed. The number of PFU was determined by plaque assay using a 10% homogenate of tissue taken from individual mice and 10-fold dilutions of this homogenate on the appropriate cell line, ATCC vero cells for LCMV, PV, VV, and MEF cells for MCMV. Results were expressed as the geometric mean titers, i.e., the arithmetic averages of the logs for four or five separate animals titrated for virus individually plus or minus the SEM. Titers reported are log 10  PFU per whole spleen, liver, and both abdominal fat pads. The level of acute fatty necrosis was scored visually based on the severity of disease from level 0 to 7. Levels 1 and 2 represent very mild to mild disease with a few white necrotic spots on one or both lower abdominal fat pads; levels 3 and 4 represent mildly moderate and moderate with larger patches of necrosis of the lower abdominal fat pads and extension into the upper left quadrant fat pad around the spleen; and levels 5 and 6 represent moderately severe to severe with very extensive large patches of necrosis on the lower abdominal fat pads and spotty fatty necrosis throughout omental fat pads as well as the splenic fat pad; level 7 represents very severe disease with such severe fatty necrosis that the organs are adherent to each other. Table  1  shows viral titration data from experiments in which mice immune to one virus were challenged with any of a panel of heterologous viruses. Prior immunity to LCMV, PV or MCMV resulted in significant 10\\u2013300-fold lower VV titers in three major organs (spleen, fat pads, and liver) at 3\\u20135 d after VV infection. Prior immunity to LCMV also resulted in a significant 5\\u201313-fold lower PV titers 3\\u20134 d after PV infection, while immunity to MCMV provided a less dramatic but still significant 2.5-fold decrease in PV titer of the fat pads. Immunity to MCMV resulted in a significant sixfold lower virus titer in the spleen of LCMV-infected mice; immunity to PV lowered the splenic LCMV titer only 2.5-fold (the Armstrong strain of LCMV does not replicate in the liver or fat pad). Immunity to LCMV and PV resulted in a four- and fivefold decrease, respectively, in liver titers in MCMV-infected mice. In contrast, immunity to VV did not significantly lower LCMV, PV, or MCMV titers. Prior immunity to LCMV or PV significantly lowered VV titers over a 2 log 10  range of challenge virus dose (Table  2 ). This protective effect of prior immunity lasted for long periods of time after the original virus infection, as mice either 8 or 29 wk after LCMV infection demonstrated equal efficiency at resisting PV challenge (Fig.  1 \\n A ) as did mice either 6 or 19 wk after MCMV infection challenged with VV (Fig.  1 \\n B ). These studies suggest that protective immunity between these four heterologous viruses is a common occurrence, but it is not universal to all virus combinations, as there is a selective protective effect dependent on the sequence of virus infections. These experiments were initially done with standard virus stocks derived from serum-containing tissue culture supernatants but then were performed with purified virus stocks to determine if cross-reactive responses to cellular contaminants or FBS were influencing the results. Impurities such as FBS and cellular debris were removed by purification of the viruses over sucrose gradients, followed by substantial 20\\u2013100-fold dilutions in PBS before inoculation of mice. Use of purified and diluted viruses eliminated any anamnestic response to FBS or cellular contaminants. With elispot assays we did detect FBS-specific responses of CD4 +  cells to IFN-\\u03b3 and IL-4 when inoculations were done with virus prepared in media containing 10% FBS (data not shown), but no such responses were detected when purified or diluted virus preparations were used. For the protective immunity experiments, however, the conventional virus stocks and purified viruses produced similar results, so all the results were pooled for the analyses in Table  1 . It should be further noted that the virus stocks were grown in different cell lines; LCMV was grown in BHK cells, PV was grown in BHK or L929 cells, and VV was grown in NCTC929 cells or L929 cells (11 and 12). MCMV was derived directly from mouse salivary glands ( 11 ). Also, serum from LCMV-, PV-, and VV-immune mice infected with the unpurified virus stocks did not demonstrate any antibody production against L929 or BHK cell surface molecules by FACS \\u00ae  analysis (data not shown). All this information makes it highly unlikely that a cellular contaminant influenced the protective immunity. Adoptive cell transfer of LCMV-immune splenocytes into naive mice resulted in five- to eightfold more rapid virus clearance in the spleen and fat pads, on VV or PV challenge than in mice adoptively reconstituted with naive splenocytes (Fig.  2 ). These results supported the concept that the protective heterologous immunity against acute VV and PV infections was dependent on immune lymphocytes. Depletion of either CD8 +  (Fig.  2 ,  A  and  C ) or CD4 +  (Fig.  2 ,  B  and  D ) T cells before the cell transfer resulted in a loss of this protective effect, suggesting that both memory T cell types were required. Studies were done to examine whether CD4 +  memory T cells were required for reactivation of the LCMV-specific memory CTL in mice challenged acutely with PV. Depletion of CD4 +  T cells in vivo by inoculation with anti-CD4 mAb did not have a significant effect on PV-specific CTL induction in PV-infected control or LCMV-immune mice. In LCMV-immune mice challenged with PV, the reactivated LCMV-specific bulk CTL activity ( 3 ) in the control group was 6.3 \\u00b1 1.4 lytic units (LU; calculated as 5% lysis of 5 \\u00d7 10 3  targets) per 10 6  splenocytes ( n  = 18) in bulk assays, but in the CD4 +  T cell\\u2013 depleted mice it was dramatically lower, at <1.2 \\u00b1 0.1 LU per 10 6  splenocytes ( n  = 6). All of these mice were depleted of NK cells with mAb to NK1.1 in order to reduce background cytotoxicity to better resolve the virus-specific killing ( 12 ). These results collectively suggest that CD4 +  T cells were required for the heterologous activation of CD8 +  CTL and that both CD4 +  and CD8 +  memory T cells were required for protective immunity. A property of memory T cells is their ability to produce much higher levels of cytokines than do naive cells after stimulation ( 14 ). Therefore, we hypothesized that a heterologous virus that stimulated memory T cells would elicit a potent IFN-\\u03b3 response, a cytokine to which VV is known to be extremely sensitive ( 17 ,  30 ). Analysis by ELISA assay of IFN-\\u03b3 concentrations (Fig.  3 \\n I ) in the peritoneal fluid of LCMV-immune mice at day 3 after VV infection (16,115 \\u00b1 4,262 pg/ml,  n  = 12) revealed >1,074-fold higher titers of IFN-\\u03b3 than in uninfected naive mice (<15 \\u00b1 0 pg/ml,  n  = 8) or LCMV-immune mice (<15 \\u00b1 0 pg/ml,  n  = 8) and 5.4-fold (2,979 \\u00b1 1,039 pg/ml;  n  = 12; Student's  t  test,  P  = 0.007) more than control mice challenged with VV. LCMV-immune mice challenged with PV did not show this enhanced production of IFN-\\u03b3 (Fig.  3 \\n II ) nor did VV-immune mice challenged with LCMV (data not shown). We then examined the peritoneal cavity more closely, as this is the site of initial VV replication where the most significant impact on early virus replication could occur. FACS \\u00ae  staining of PECs resolved a significant 8- and 2.7-fold enhanced recruitment of CD8 +  and CD4 +  T cells, respectively, in the LCMV-immune versus naive mice 3 d after challenged with VV (Fig.  4 ,  A  and  B ). Furthermore, the peritoneal T cells in the LCMV-immune mice 3 d after challenge with VV were predominantly of the memory phenotype, CD44 hi . Basing our cutoff value for CD44 hi  (FITC-conjugated anti-CD44;  PharMingen ) at a mean fluorescence of 12.6 units as determined on naive splenocytes, it was found that 92 \\u00b1 5% ( n  = 3) of the peritoneal CD8 +  cells and 75 \\u00b1 19% ( n  = 3) of the peritoneal CD4 +  cells were CD44 hi  memory phenotype. The CD8 +  cells had a mean fluorescence of 182 \\u00b1 40 and the CD4 +  cells had a mean fluorescence of 177 \\u00b1 58. These cells were examined for expression of intracellular IFN-\\u03b3 after treatment with PMA and ionomycin. There was a significant 10- and 2.7-fold increase in the number of IFN-\\u03b3\\u2013producing CD8 +  and CD4 +  T cells, respectively, in LCMV-immune mice versus naive mice challenged with VV (Fig.  4 ,  A  and  B ). To further examine whether IFN-\\u03b3 was playing a role in the protective effect against heterologous viruses, we initially treated LCMV-immune mice with anti-IFN-\\u03b3 antibodies, as previously described ( 31 ). In brief, we gave 0.2 ml of a 1/5 dilution of anti-IFN-\\u03b3 ascites intraperitoneally daily, in combination with a similar intravenous dose on day 0 and day 3 of VV infection. In the first experiment at 4 d after VV infection there was a 20-fold increase in the VV fat pad titer in the anti-IFN-\\u03b3\\u2013treated group (LCMV-immune + VV, 2.77 \\u00b1 0.23 PFU/ml vs. LCMV-immune + VV + anti-IFN-\\u03b3, 4.07 \\u00b1 0.45 PFU/ml;  n  = 5;  P  = 0.04, Student's  t  test). In the second experiment at day 6 after VV infection there was a 13-fold increase in the anti-IFN-\\u03b3\\u2013treated group (LCMV-immune + VV, 4.23 \\u00b1 0.61 PFU/ml vs. LCMV-immune + VV + anti\\u2013IFN-\\u03b3, 5.31 \\u00b1 0.22;  n  = 5;  P  = 0.006, Student's  t  test). To further confirm this observation, we extended these studies into IFN-\\u03b3 receptor knockout (IFN-\\u03b3R KO) mice. Since the IFN-\\u03b3R KO mice were on the 129/SEV background instead of the C57BL/6 background, normal 129/ SEV mice were used as controls (Fig.  5 ). The 129/SEV and 129 IFN-\\u03b3R KO mice developed equivalent high frequency LCMV-specific memory pCTL, as shown by limiting dilution assays (Fig.  5 ,  A  and  B ). 129/SEV mice immune to LCMV developed protective heterologous immunity to PV and VV, similar to that in the C57BL/6 mice. However, IFN-\\u03b3R KO mice immune to LCMV did not resist VV infection, confirming a role for IFN-\\u03b3 in the control of VV by LCMV-induced memory T cells (Fig.  6 ,  A  and  B ). Similar experiments demonstrated that IFN-\\u03b3 was not required for the protective effect against PV (Fig.  6 ,  C  and  D ), suggesting the possibility that other cytokines or even direct cytotoxicity by the cross-reactive CTL may be effective. A role for cytotoxicity could not be easily addressed in these systems, as cytotoxicity-deficient perforin KO mice develop persistent LCMV infections instead of memory T cell responses ( 32 ). Substantial evidence has been provided that T cells can not only be mediators of protective immunity, but also of immunopathology ( 10 ,  33 \\u2013 37 ). Our studies described below of heterologous immunity between viruses indicate that a previous virus infection may predispose a host to a much more severe disease upon infection with an unrelated heterologous virus. In general accordance with an earlier report ( 22 ), mice immunized with LCMV, unlike naive mice, developed severe immunopathology consisting of extensive acute fat necrosis (AFN) with a cellular infiltrate in visceral fat pads 5\\u20137 d after VV challenge (Fig.  7 ). In contrast, prior immunity to PV or MCMV, although protective against VV challenge, did not result in AFN on VV infection. The leukocyte infiltrate within the abdominal fat pads contained 19% \\u00b1 7 (SEM;  n  = 2) CD4 +  T cells and 23% \\u00b1 5 (SEM;  n  = 2) CD8 +  T cells and mediated CTL activity against both VV- (8 \\u00b1 4 lytic units per 10 6  cells;  n  = 5) and LCMV- (9.4 \\u00b1 6.9 lytic units per 10 6  cells;  n  = 5) infected targets. The CTL activity segregated to the CD8 +  T cell population on sorting (data not shown). Therefore, these fat pads contained infiltrates of activated CTL specific not only for VV but also for LCMV. A reactivation of the LCMV-specific memory splenic CTL by VV has previously been documented ( 11 ,  12 ). Adoptive transfer studies showed that splenocytes from LCMV-immune mice not only provided some protective immunity against VV synthesis, but also induced rather severe acute fatty necrosis (Fig.  8 \\n A ). This immunopathology did not occur in mice lacking IFN-\\u03b3 receptors (Fig.  8 \\n B ). Further, LCMV-immune mice acutely infected with VV and treated with anti-IFN-\\u03b3 daily had decreased evidence of AFN (level of AFN: LCMV-immune + VV, 4 \\u00b1 1.2; LCMV-immune + VV + anti-IFN-\\u03b3, 1 \\u00b1 0.5;  n  = 5). As shown above, LCMV-immune mice challenged with VV (Fig.  3 \\n I ) had higher levels of IFN-\\u03b3 in the peritoneal fluid than did naive mice challenged with VV. This enhanced IFN-\\u03b3 response in immune mice may thus play a role in protective immunity as well as in the induction of immunopathology. In this sequence of viruses the immunological contribution of the LCMV-specific memory cells may result in an overwhelming delayed-type hypersensitivity-like cell-mediated immune response, resulting in enhanced pathogenesis, even though the VV is cleared more rapidly. Table  1  shows viral titration data from experiments in which mice immune to one virus were challenged with any of a panel of heterologous viruses. Prior immunity to LCMV, PV or MCMV resulted in significant 10\\u2013300-fold lower VV titers in three major organs (spleen, fat pads, and liver) at 3\\u20135 d after VV infection. Prior immunity to LCMV also resulted in a significant 5\\u201313-fold lower PV titers 3\\u20134 d after PV infection, while immunity to MCMV provided a less dramatic but still significant 2.5-fold decrease in PV titer of the fat pads. Immunity to MCMV resulted in a significant sixfold lower virus titer in the spleen of LCMV-infected mice; immunity to PV lowered the splenic LCMV titer only 2.5-fold (the Armstrong strain of LCMV does not replicate in the liver or fat pad). Immunity to LCMV and PV resulted in a four- and fivefold decrease, respectively, in liver titers in MCMV-infected mice. In contrast, immunity to VV did not significantly lower LCMV, PV, or MCMV titers. Prior immunity to LCMV or PV significantly lowered VV titers over a 2 log 10  range of challenge virus dose (Table  2 ). This protective effect of prior immunity lasted for long periods of time after the original virus infection, as mice either 8 or 29 wk after LCMV infection demonstrated equal efficiency at resisting PV challenge (Fig.  1 \\n A ) as did mice either 6 or 19 wk after MCMV infection challenged with VV (Fig.  1 \\n B ). These studies suggest that protective immunity between these four heterologous viruses is a common occurrence, but it is not universal to all virus combinations, as there is a selective protective effect dependent on the sequence of virus infections. These experiments were initially done with standard virus stocks derived from serum-containing tissue culture supernatants but then were performed with purified virus stocks to determine if cross-reactive responses to cellular contaminants or FBS were influencing the results. Impurities such as FBS and cellular debris were removed by purification of the viruses over sucrose gradients, followed by substantial 20\\u2013100-fold dilutions in PBS before inoculation of mice. Use of purified and diluted viruses eliminated any anamnestic response to FBS or cellular contaminants. With elispot assays we did detect FBS-specific responses of CD4 +  cells to IFN-\\u03b3 and IL-4 when inoculations were done with virus prepared in media containing 10% FBS (data not shown), but no such responses were detected when purified or diluted virus preparations were used. For the protective immunity experiments, however, the conventional virus stocks and purified viruses produced similar results, so all the results were pooled for the analyses in Table  1 . It should be further noted that the virus stocks were grown in different cell lines; LCMV was grown in BHK cells, PV was grown in BHK or L929 cells, and VV was grown in NCTC929 cells or L929 cells (11 and 12). MCMV was derived directly from mouse salivary glands ( 11 ). Also, serum from LCMV-, PV-, and VV-immune mice infected with the unpurified virus stocks did not demonstrate any antibody production against L929 or BHK cell surface molecules by FACS \\u00ae  analysis (data not shown). All this information makes it highly unlikely that a cellular contaminant influenced the protective immunity. Adoptive cell transfer of LCMV-immune splenocytes into naive mice resulted in five- to eightfold more rapid virus clearance in the spleen and fat pads, on VV or PV challenge than in mice adoptively reconstituted with naive splenocytes (Fig.  2 ). These results supported the concept that the protective heterologous immunity against acute VV and PV infections was dependent on immune lymphocytes. Depletion of either CD8 +  (Fig.  2 ,  A  and  C ) or CD4 +  (Fig.  2 ,  B  and  D ) T cells before the cell transfer resulted in a loss of this protective effect, suggesting that both memory T cell types were required. Studies were done to examine whether CD4 +  memory T cells were required for reactivation of the LCMV-specific memory CTL in mice challenged acutely with PV. Depletion of CD4 +  T cells in vivo by inoculation with anti-CD4 mAb did not have a significant effect on PV-specific CTL induction in PV-infected control or LCMV-immune mice. In LCMV-immune mice challenged with PV, the reactivated LCMV-specific bulk CTL activity ( 3 ) in the control group was 6.3 \\u00b1 1.4 lytic units (LU; calculated as 5% lysis of 5 \\u00d7 10 3  targets) per 10 6  splenocytes ( n  = 18) in bulk assays, but in the CD4 +  T cell\\u2013 depleted mice it was dramatically lower, at <1.2 \\u00b1 0.1 LU per 10 6  splenocytes ( n  = 6). All of these mice were depleted of NK cells with mAb to NK1.1 in order to reduce background cytotoxicity to better resolve the virus-specific killing ( 12 ). These results collectively suggest that CD4 +  T cells were required for the heterologous activation of CD8 +  CTL and that both CD4 +  and CD8 +  memory T cells were required for protective immunity. A property of memory T cells is their ability to produce much higher levels of cytokines than do naive cells after stimulation ( 14 ). Therefore, we hypothesized that a heterologous virus that stimulated memory T cells would elicit a potent IFN-\\u03b3 response, a cytokine to which VV is known to be extremely sensitive ( 17 ,  30 ). Analysis by ELISA assay of IFN-\\u03b3 concentrations (Fig.  3 \\n I ) in the peritoneal fluid of LCMV-immune mice at day 3 after VV infection (16,115 \\u00b1 4,262 pg/ml,  n  = 12) revealed >1,074-fold higher titers of IFN-\\u03b3 than in uninfected naive mice (<15 \\u00b1 0 pg/ml,  n  = 8) or LCMV-immune mice (<15 \\u00b1 0 pg/ml,  n  = 8) and 5.4-fold (2,979 \\u00b1 1,039 pg/ml;  n  = 12; Student's  t  test,  P  = 0.007) more than control mice challenged with VV. LCMV-immune mice challenged with PV did not show this enhanced production of IFN-\\u03b3 (Fig.  3 \\n II ) nor did VV-immune mice challenged with LCMV (data not shown). We then examined the peritoneal cavity more closely, as this is the site of initial VV replication where the most significant impact on early virus replication could occur. FACS \\u00ae  staining of PECs resolved a significant 8- and 2.7-fold enhanced recruitment of CD8 +  and CD4 +  T cells, respectively, in the LCMV-immune versus naive mice 3 d after challenged with VV (Fig.  4 ,  A  and  B ). Furthermore, the peritoneal T cells in the LCMV-immune mice 3 d after challenge with VV were predominantly of the memory phenotype, CD44 hi . Basing our cutoff value for CD44 hi  (FITC-conjugated anti-CD44;  PharMingen ) at a mean fluorescence of 12.6 units as determined on naive splenocytes, it was found that 92 \\u00b1 5% ( n  = 3) of the peritoneal CD8 +  cells and 75 \\u00b1 19% ( n  = 3) of the peritoneal CD4 +  cells were CD44 hi  memory phenotype. The CD8 +  cells had a mean fluorescence of 182 \\u00b1 40 and the CD4 +  cells had a mean fluorescence of 177 \\u00b1 58. These cells were examined for expression of intracellular IFN-\\u03b3 after treatment with PMA and ionomycin. There was a significant 10- and 2.7-fold increase in the number of IFN-\\u03b3\\u2013producing CD8 +  and CD4 +  T cells, respectively, in LCMV-immune mice versus naive mice challenged with VV (Fig.  4 ,  A  and  B ). To further examine whether IFN-\\u03b3 was playing a role in the protective effect against heterologous viruses, we initially treated LCMV-immune mice with anti-IFN-\\u03b3 antibodies, as previously described ( 31 ). In brief, we gave 0.2 ml of a 1/5 dilution of anti-IFN-\\u03b3 ascites intraperitoneally daily, in combination with a similar intravenous dose on day 0 and day 3 of VV infection. In the first experiment at 4 d after VV infection there was a 20-fold increase in the VV fat pad titer in the anti-IFN-\\u03b3\\u2013treated group (LCMV-immune + VV, 2.77 \\u00b1 0.23 PFU/ml vs. LCMV-immune + VV + anti-IFN-\\u03b3, 4.07 \\u00b1 0.45 PFU/ml;  n  = 5;  P  = 0.04, Student's  t  test). In the second experiment at day 6 after VV infection there was a 13-fold increase in the anti-IFN-\\u03b3\\u2013treated group (LCMV-immune + VV, 4.23 \\u00b1 0.61 PFU/ml vs. LCMV-immune + VV + anti\\u2013IFN-\\u03b3, 5.31 \\u00b1 0.22;  n  = 5;  P  = 0.006, Student's  t  test). To further confirm this observation, we extended these studies into IFN-\\u03b3 receptor knockout (IFN-\\u03b3R KO) mice. Since the IFN-\\u03b3R KO mice were on the 129/SEV background instead of the C57BL/6 background, normal 129/ SEV mice were used as controls (Fig.  5 ). The 129/SEV and 129 IFN-\\u03b3R KO mice developed equivalent high frequency LCMV-specific memory pCTL, as shown by limiting dilution assays (Fig.  5 ,  A  and  B ). 129/SEV mice immune to LCMV developed protective heterologous immunity to PV and VV, similar to that in the C57BL/6 mice. However, IFN-\\u03b3R KO mice immune to LCMV did not resist VV infection, confirming a role for IFN-\\u03b3 in the control of VV by LCMV-induced memory T cells (Fig.  6 ,  A  and  B ). Similar experiments demonstrated that IFN-\\u03b3 was not required for the protective effect against PV (Fig.  6 ,  C  and  D ), suggesting the possibility that other cytokines or even direct cytotoxicity by the cross-reactive CTL may be effective. A role for cytotoxicity could not be easily addressed in these systems, as cytotoxicity-deficient perforin KO mice develop persistent LCMV infections instead of memory T cell responses ( 32 ). Substantial evidence has been provided that T cells can not only be mediators of protective immunity, but also of immunopathology ( 10 ,  33 \\u2013 37 ). Our studies described below of heterologous immunity between viruses indicate that a previous virus infection may predispose a host to a much more severe disease upon infection with an unrelated heterologous virus. In general accordance with an earlier report ( 22 ), mice immunized with LCMV, unlike naive mice, developed severe immunopathology consisting of extensive acute fat necrosis (AFN) with a cellular infiltrate in visceral fat pads 5\\u20137 d after VV challenge (Fig.  7 ). In contrast, prior immunity to PV or MCMV, although protective against VV challenge, did not result in AFN on VV infection. The leukocyte infiltrate within the abdominal fat pads contained 19% \\u00b1 7 (SEM;  n  = 2) CD4 +  T cells and 23% \\u00b1 5 (SEM;  n  = 2) CD8 +  T cells and mediated CTL activity against both VV- (8 \\u00b1 4 lytic units per 10 6  cells;  n  = 5) and LCMV- (9.4 \\u00b1 6.9 lytic units per 10 6  cells;  n  = 5) infected targets. The CTL activity segregated to the CD8 +  T cell population on sorting (data not shown). Therefore, these fat pads contained infiltrates of activated CTL specific not only for VV but also for LCMV. A reactivation of the LCMV-specific memory splenic CTL by VV has previously been documented ( 11 ,  12 ). Adoptive transfer studies showed that splenocytes from LCMV-immune mice not only provided some protective immunity against VV synthesis, but also induced rather severe acute fatty necrosis (Fig.  8 \\n A ). This immunopathology did not occur in mice lacking IFN-\\u03b3 receptors (Fig.  8 \\n B ). Further, LCMV-immune mice acutely infected with VV and treated with anti-IFN-\\u03b3 daily had decreased evidence of AFN (level of AFN: LCMV-immune + VV, 4 \\u00b1 1.2; LCMV-immune + VV + anti-IFN-\\u03b3, 1 \\u00b1 0.5;  n  = 5). As shown above, LCMV-immune mice challenged with VV (Fig.  3 \\n I ) had higher levels of IFN-\\u03b3 in the peritoneal fluid than did naive mice challenged with VV. This enhanced IFN-\\u03b3 response in immune mice may thus play a role in protective immunity as well as in the induction of immunopathology. In this sequence of viruses the immunological contribution of the LCMV-specific memory cells may result in an overwhelming delayed-type hypersensitivity-like cell-mediated immune response, resulting in enhanced pathogenesis, even though the VV is cleared more rapidly. Most models for the immune response to viral infections have used infected animals sheltered from other environmental pathogens, a rather unnatural circumstance. In this study, we show that these sheltered mice responded quite differently to infection than did mice previously exposed to other pathogens, as early protective immunity can be mediated by memory T cells generated by a previous heterologous viral infection. Our results would suggest that both memory CD4 +  and CD8 +  T cells can be reactivated, as both populations were required for the protective immunity to occur in the virus combinations tested. For instance, in LCMV-immune versus a naive host challenged with VV, both memory cell types were recruited early in infection into the peritoneal cavity, the site of virus challenge, and were producing significantly more IFN-\\u03b3, a cytokine to which VV is very sensitive ( 17 ,  30 ). We have not examined the role of memory B cells in this system but clearly memory T cells are required for this type of protective immunity. Although T cells are generally thought to be highly specific for peptide\\u2013MHC complexes, more extensive investigations have shown that a single T cell clone can recognize two unrelated peptide sequences on the same protein, two different proteins of the same virus, and two different proteins from two unrelated viruses ( 38 \\u2013 40 ). It is not surprising that peptides from a second viral infection could cross-react with and stimulate some of the many clones of T cells comprising the memory pool specific for another virus. We have reported that memory CD8 +  T cells can often cross-react between heterologous viruses ( 12 ), and such cross- reactive T cells may contribute to this more rapid early resistance to a second infection. The T cell repertoire to each new infection would be altered as cross-reactive memory cells join in the response, resembling in some ways the original antigenic sin network of antibody responses to different strains of influenza virus, where primary immunity is boosted not by a homologous but, by a cross-reacting vaccine of a different influenza virus serotype, and some of the newly formed antibodies react better with the primary virus than with the virus actually eliciting the response ( 41 ). Memory T cells are known to express high levels of adhesion molecules ( 8 \\u2013 10 ) and to be easier to stimulate ( 13 \\u2013 16 ) than naive cells, making them susceptible to low affinity cross-reactive TcR triggering. Some reports have suggested that memory T cells can be stimulated nonspecifically by cytokines ( 5 ,  6 ,  42 ), and may be cross-activated by an infection that does not require stimulation through their T cell receptor by cross-reactive viral peptides. However, data from our laboratory suggests that memory T cells do not nonspecifically proliferate in response to an antigen to which there are no cross-reactive TcR epitopes detectable ( 43 ), and our experiments on heterologous immunity between viruses argue on behalf of cross-reactive memory cells rather than nonspecific activation effects, because the specific sequence of virus infections determines if there is any protective immunity. Nevertheless, it is possible there is some as yet unknown selective mechanism for cross-activation of different memory T cell populations. The contribution of memory cells specific for one virus to the natural resistance against another virus is presently not predictable and is not necessarily reciprocal. If the immunodominant memory T cell response to an earlier virus could recognize a cross-reactive determinant on the challenge virus, the challenge virus would find a large pool of memory T cells capable of a rapid response. Should the challenge virus not be recognized by part of the immunodominant memory T cell repertoire, then less of a protective effect may ensue. Individuals vary considerably in their responses to viral infections, ranging from subclinical to severe. There are many factors that contribute to this variation in responsiveness, including the dose and route of infection, as well as the physiological state and genetic background of the host. In this study, we show that memory T cells specific to unrelated viruses may also contribute to the host's primary response to a second virus. The beneficial effect of these early protective memory T cells is to slow down the spread of infection, much like a natural immune-mediated response, allowing time for more suitable high affinity antigen-specific T and B cell responses to develop. It is reasonable to expect that this level of resistance may be the difference between clinical and subclinical infections or lethal and nonlethal infections. The detrimental immunopathological effect of memory T cells leads us to question whether they may play a role in human infections like with Epstein-Barr virus or varicella-zoster virus that lead to much more severe disease in young adults than in young children, who presumably have a more restricted memory T cell repertoire ( 44 ,  45 ). Reactivation of memory T cells may also play a role in exacerbation of autoimmune diseases, such as multiple sclerosis or diabetes. For example, in an experimental model for chronic virus-induced CNS autoimmunity, disease may be induced by infection with a virus sharing epitopes with a protein expressed in oligodendrocytes, and this T cell\\u2013dependent disease could be exacerbated by a second infection with an unrelated virus ( 46 ). Thus, alterations in acute responses caused by the memory T cell modulations induced by the host's history of prior infections may have the paradoxical effect of producing both beneficial protective heterologous immunity and/or immunopathology. \",\n          \"Reactive oxygen species (H 2 O 2 ,  \\u2022 O 2 \\n -,\\u2022 OH) are important mediators of endothelial cell (EC) *  injury and likely play a central role in cardiovascular, cerebrovascular, and pulmonary syndromes such as myocardial infarction, ischemic stroke, acute respiratory distress syndrome, and acute organ transplant rejection ( Lounsbury et al., 2000 ). H 2 O 2  appears to be particularly important in the context of neutrophil\\u2013endothelial interaction in inflammation ( Martin, 1984 ;  Lewis et al., 1988 ;  Gasic et al., 1991 ), during which local H 2 O 2  levels may exceed 100\\u2013400 \\u03bcM ( Test and Weiss, 1984 ). H 2 O 2  causes cell depolarization ( Matoba et al., 2000 ) and increases cytosolic-free Ca 2+  concentration ([Ca 2+ ] i ) in EC through a range of concentration-dependent [Ca 2+ ] i  responses, including Ca 2+  influx into EC ( Bowles et al., 2001 ) and Ca 2+  release from intracellular stores ( Schilling and Elliott, 1992 ;  Doan et al., 1994 ;  Siflinger-Birnboim et al., 1996 ;  Yolk et al., 1997 ;  Hu et al., 1998 ). This rise in [Ca 2+ ] i  triggers the release of platelet-activating factor ( Lewis et al., 1988 ) and PGI 2  ( Harlan and Callahan, 1984 ), endothelial retraction, adhesion molecule redistribution ( Bradley et al., 1995 ), EC permeability ( Siflinger-Birnboim et al., 1996 ;  Carbajal and Schaeffer, 1998 ), gene transcription ( Barchowsky et al., 1995 ), and apoptosis ( Martin, 1984 ;  Suzucki et al., 1997 ). H 2 O 2  can also activate tyrosine kinases and inhibit protein tyrosine phosphatases in ECs ( Barchowsky et al., 1995 ;  Carbajal and Schaeffer, 1998 ). Despite the recognized importance of these EC responses to oxidant exposure, the key regulatory proteins modulating EC calcium responses remain largely unknown. One potential participant in this regulation is platelet endothelial adhesion molecule (PECAM)-1/CD-31, a 130-kD type 1 membrane glycoprotein belonging to the Ig superfamily of CAMs that plays an important role in the modulation of EC\\u2013neutrophil interaction and transmigration in inflammation ( Vaporciyan et al., 1993 ;  Nakada et al., 2000 ) and oxidant injury ( Scalia and Lefer, 1998 ). PECAM-1, which functions as both an adhesion and signaling protein, is highly expressed on ECs where it localizes to cell\\u2013cell borders, as well as on leukocytes and platelets ( Newman, 1997 ). Recently, it has been shown that engagement of PECAM-1 elicits prolonged EC calcium transients through a calcium-permeant, plasmalemmel nonselective cation (NSC) channel. Activation of this channel is dependent on Src family kinase activity, but independent of inositol trisphosphate (IP 3 )-mediated store release or phosphoinositide turnover ( Gurubhagavatula et al., 1998 ;  O'Brien et al., 2001 ). In addition to a large extracellular domain containing six Ig-like loops, PECAM-1 has a 118\\u2013amino acid cytoplasmic domain containing a dual tyrosine SH2 binding motif (Y663/Y686). Under a range of conditions, including phosphatase inhibition ( Jackson et al., 1997 ;  Sagawa et al., 1997 ), Src and Csk overexpression ( Lu et al., 1997 ;  Cao et al., 1998 ), and anti\\u2013PECAM-1 antibody engagement ( Varon et al., 1998 ), this motif is selectively tyrosine phosphorylated. Once phosphorylated, PECAM-1 can associate with SH2 domain proteins such as SHP-2, SHP-1, PLC\\u03b3, and PI3-kinase, although the physiologic relevance of these interactions in EC are unknown ( Newman, 1999 ). In light of the evidence that PECAM-1 participates in ionic signaling and may serve as a substrate for phosphotyrosine modification, we hypothesized that PECAM-1 might participate in the regulation of oxidant-mediated calcium signals in ECs. In order to test our hypothesis that PECAM-1 may play a role in oxidant-mediated signaling, we performed voltage clamp experiments on human umbilical vein EC (HUVEC), REN cells, and REN cells stably transfected with PECAM-1 (RHP). We have previously utilized REN cells, a mesothelioma cell line that does not express endogenous PECAM-1, as an EC model, and have found that many EC signaling processes may be reconstituted after PECAM-1 expression ( Gurubhagavatula et al., 1998 ;  O'Brien et al., 2001 ). Puffer pipette application of 200 \\u03bcM H 2 O 2  (a dose based on preliminary studies described below) to HUVEC and RHP cells voltage clamped at a holding potential of \\u201360 mV, elicited a large inward current in 93% of HUVEC ( n  = 15) with a peak amplitude of 648.4 \\u00b1 36.6 pA. A similar current (peak amplitude 682.5 \\u00b1 41.55 pA) was detected in 89% of RHP ( n  = 18), whereas no currents were observed in untransfected REN cells exposed to 200 ( n  = 16) or 400 \\u03bcM ( n  = 11) H 2 O 2  ( Fig. 1 ). H 2 O 2  currents were typically activated 30\\u201390 s after H 2 O 2  exposure and were of prolonged time course, features similar to cation currents evoked in HUVEC and RHP cells after PECAM-1 engagement ( O'Brien et al., 2001 ). The response to H 2 O 2  was concentration dependent in that RHP cells treated with H 2 O 2  at concentrations of 100 \\u03bcM or lower elicited current less consistently (though those responding manifested similar current kinetics). Cells that responded to 400 or 500 \\u03bcM H 2 O 2  displayed evident cell toxicity over 24 h (unpublished data), consistent with previous reports ( Barchowsky et al., 1994 ). Puffer application of vehicle alone did not elicit a current, further eliminating a potential role for shear stress\\u2013gated NSC channels or transient loss of seal in the response (unpublished data). To further confirm the role of PECAM-1 in both HUVEC and RHP cells, H 2 O 2  current responses were recorded in cells dialyzed with polyclonal anti\\u2013PECAM-1 cytoplasmic domain (PCD) antibody or control polyclonal antibody directed against herpes simplex virus thymidine kinase ([anti-] HSVTK). 200 \\u03bcM H 2 O 2  elicited currents with identical kinetics to nondialyzed cells in 83% of HUVEC ( n  = 12) and 80% RHP cells ( n  = 5) dialyzed with anti-HSVTK antibody. By contrast, in cells dialyzed with PCD, currents were completely blocked in 81% of HUVEC ( n  = 16) and 75% of RHP ( n  = 4) cells, and markedly attenuated in those cells in which currents were observed ( Fig. 2 ). The ion selectivity and voltage dependence of the PECAM-1 current were examined in HUVEC and RHP. The reversal potential of the evoked current was determined by voltage ramp (\\u221260 up to +50 mV for 500 ms) and voltage step (\\u221260 to +40 mV in 20 mV increments) protocols in both RHP cells ( Fig. 3 ) and HUVEC (unpublished data). The current\\u2013voltage relationship of the PECAM-1 current was ohmic, with no evidence of voltage-dependent gating or rectification. Similarly, initial trials with instantaneous voltage ramps to +100 mV showed no rectification (unpublished data). The reversal potential in standard physiologic solutions was \\u223c0 mV, close to the theoretical monovalent cation equilibrium potential of \\u22121.2 mV. Consistent with a NSC current, replacement of extracellular Na +  ions with impermeant Tris resulted in a left shift of the current reversal potential and a decrease in the slope conductance of the inward current. Conversely, asymmetric impermeant anion and symmetric cation solutions (CsAcetate pipette and CsCl bath) did not shift the reversal potential nor alter the slope conductance (unpublished data). To assess the permeability of PECAM-1\\u2013activated NSC channels to Ca 2+ , we examined ionic signaling in extracellular solution containing 100 mM Ca 2+  as the only permeant cation. As shown in  Fig. 4 , exposure of cells to 200 \\u03bcM H 2 O 2  activated a prolonged Ca 2+  current in 5/6 (83%) HUVEC and 6/6 (100%) RHP cells. The kinetics of this current were similar to those observed in monovalent cation solutions, and the current reversal potential was shifted to positive potentials, as predicted for Ca 2+ -permeant channels. These results are identical to those previously obtained after homophilic PECAM-1 engagement and anti\\u2013PECAM-1 antibody ligation ( O'Brien et al., 2001 ), suggesting that the same channel mediates both PECAM-1\\u2013dependent ionic signaling events. To further examine the role of PECAM-1 in mediating H 2 O 2 -activated calcium signaling, we measured H 2 O 2 -induced Ca 2+  fluorescence with laser scanning confocal microscopy in individual Fluo-4\\u2013loaded REN and RHP cells exposed to 200 \\u03bcM H 2 O 2 . As shown in  Fig. 5 , H 2 O 2  evoked a prolonged rise in intracellular calcium lasting about 5 min ( Fig. 5, A and B ), a time course quite similar to that of the PECAM-1\\u2013dependent cation current (compare  Figs. 1  and  5 ). Conversely, in REN cells, no increase in intracellular calcium was observed, confirming the requirement for PECAM-1 in this calcium response ( Fig. 5  B, inset). Addition of thrombin yielded calcium signals in both cell lines that were of rapid onset, but of much shorter duration, whereas buffer did not produce a rise in [Ca 2+ ] i  in either cell type ( Fig. 5  B). These findings suggest that EC PECAM-1 regulates the gating of a calcium-permeant, voltage-independent, NSC channel that mediates prolonged calcium signals observed in ECs after physiologic H 2 O 2  exposure. We have previously reported that PECAM-1 engagement activates a calcium-permeant channel that is independent of intracellular Ca 2+  store release ( O'Brien et al., 2001 ). We next sought to determine the relationship of the PECAM-1\\u2013dependent H 2 O 2  cation current to phosphoinositide-mediated intracellular calcium store release and to subsequently activated endothelial plasmalemmal nonspecific cation channels termed depletion-activated channel or store-operated channels (SOCs) that mediate capacitative calcium entry ( Vaca and Kunze, 1995 ;  Parekh and Penner, 1997 ;  Kamouchi et al., 1999 ). After dialysis with 5 mg/ml heparin, a compound known to act as an IP 3  receptor antagonist as well as a potential inhibitor of G protein receptor interaction ( Taylor and Broad, 1998 ), H 2 O 2 - and thrombin-induced currents were measured in RHP cells ( Fig. 6 ) and HUVEC (unpublished data). Treatment with 200 \\u03bcM H 2 O 2  evoked typical NSC currents in 8/9 (89%) RHP cells and 3/3 (100%) HUVEC, suggesting that the PECAM-1\\u2013activated NSC is neither IP 3  gated nor dependent on IP 3 -mediated intracellular calcium store release mechanisms, similar to findings after antibody engagement in whole-cell fluorimetry and voltage clamp studies ( O'Brien et al., 2001 ). Conversely, thrombin (1.5 U/ml), a G protein\\u2013coupled receptor agonist known to activate intracellular calcium store release and subsequent NSC activation in HUVEC and REN cells ( Gurubhagavatula et al., 1998 ;  O'Brien et al., 2001 ), activated a large inward current in control recordings ( n  = 9), but this current was completely blocked in cells dialyzed with heparin ( Fig. 6 , inset). Phosphoinositol turnover assays were also performed in RHP cells to confirm these findings as previously described ( Krymskaya et al., 1999 ;  O'Brien et al., 2001 ). Treatment with 200 \\u03bcM H 2 O 2  failed to elicit significant stimulation of phosphoinositide formation, whereas addition of thrombin yielded a robust increase in phosphoinositide turnover (unpublished data). These findings further support a phosphoinositide-independent activation pathway for the PECAM-1 NSC current. The kinetic characteristics of the PECAM-1\\u2013dependent current are similar to DAC/SOC NSC currents activated by intracellular calcium release (capacitative calcium entry), which are sensitive to trivalent cations such as lanthanum chloride (LaCl 3 ) ( Vaca and Kunze, 1995 ;  Parekh and Penner, 1997 ;  Kamouchi et al., 1999 ). Therefore, we evaluated the dependence on intracellular Ca 2+  release and the La 3+  sensitivity of H 2 O 2 -activated PECAM-1\\u2013dependent currents. RHP cells and HUVEC were dialyzed with CsCl pipette solution containing 10 mM EGTA and 0 mM Ca 2+  (with a calculated [Ca 2+ ] i  of <1 nM) and current recorded 10 min after break-in. Dialysis with this intracellular solution did not itself elicit large cation currents similar to those activated by H 2 O 2 , but subsequent exposure (after 3\\u20135 min) to 200 \\u03bcM H 2 O 2  activated currents similar to those observed in cells dialyzed with normal intracellular solution. H 2 O 2  currents were observed in 5/5 (100%) RHP cells and 3/3 (100%) HUVEC tested, whereas no currents were observed prior to H 2 O 2 . Moreover, the H 2 O 2  current was only partially inhibited in the presence of 100 \\u03bcM LaCl 3  ( n  = 3), whereas the current activated by thrombin (1.5 U/ml) was completely inhibited by 50 \\u03bcM LaCl 3  ( n  = 6). Interestingly, thrombin-induced capacitative currents mediated by DAC/SOC NSC channels manifest time course, reversal potential (Vrev = 0), mean peak current amplitude (730 \\u00b1 47pA), and current\\u2013voltage relationships quite similar to those observed for PECAM-1 currents. However, the differences in trivalent sensitivity, calcium gating, and phosphoinositide dependence suggest that the PECAM-1\\u2013dependent current is activated by an alternate signaling mechanism and through channels that are distinct from those mediating capacitative calcium entry ( O'Brien et al., 2001 ). In order to specifically examine the role of the PECAM-1 intracellular domain, REN cells stably transfected with a panel of PECAM-1 mutant constructs were examined in a series of patch clamp studies ( Fig. 7 ). The first construct, PITC, consists of the PECAM-1 extracellular domain fused to the nonhomologous ICAM-1 transmembrane and cytoplasmic domains. This fusion protein does not elicit cation current after antibody engagement ( O'Brien et al., 2001 ), but supports homophilic interaction and localizes to the cell\\u2013cell border in confluent monolayers (unpublished data). No currents were observed in voltage-clamped PITC cells puffed with 200 \\u03bcM H 2 O 2  ( n  = 7), suggesting that the PCD, and possibly the transmembrane domain, are necessary for current activation. A second construct, IL2PCD, was utilized to evaluate the role of the isolated PCD in current regulation. IL2PCD contains the nonhomologous interleuken (IL)-2 receptor (CD25) extracellular and transmembrane domains fused to an intact PECAM-1 cytoplasmic domain that can serve as a substrate for H 2 O 2 -induced tyrosine phosphorylation (unpublished data). In voltage-clamped IL2PCD cells puffed with 200 or 400 \\u03bcM H 2 O 2 , cation currents with kinetics identical to full-length PECAM-1 control cells were observed in 14/15 (93%) cells ( Fig. 7 ). These findings indicate that the PECAM-1 cytoplasmic domain is necessary and sufficient for H 2 O 2  current activation. In EC and RHP cells, PECAM-1 engagement yields calcium transients mediated by a NSC channel that is Src tyrosine kinase-dependent ( Gurubhagavatula et al., 1998 ;  O'Brien et al., 2001 ). Therefore, we sought to determine the relationship of tyrosine kinase activity to the PECAM-1\\u2013dependent H 2 O 2  current. RHP cells ( n  = 8) and HUVEC ( n  = 5) pretreated for 1 h with 10 \\u03bcM genistein and purified with 200 \\u03bcM H 2 O 2  failed to activate a current, suggesting a similar role for tyrosine kinase activity in regulation of this channel. To further refine this observation, voltage-clamped HUVEC and RHP cells were dialyzed through the patch pipette with 10 \\u03bcg/ml Sc-18, an Src-neutralizing rabbit polyclonal antibody specific for the Src family nonreceptor kinases ( Marrero et al., 1995 ). In HUVEC and RHP cells dialyzed with Sc-18, attenuated current was observed in 2/10 (20%) and 3/11 (27%) of cells exposed to 200 \\u03bcM H 2 O 2 , whereas H 2 O 2  failed to elicit any current in the remaining cells. By contrast, exposure of HUVEC and RHP cells dialyzed with an irrelevant rabbit polyclonal control antibody (anti-HSVTK) to H 2 O 2  evoked currents identical to nondialyzed cells in 10/12 (83%) and 4/5 (80%) of cells tested, respectively ( Fig. 8 ). These findings confirm that tyrosine kinase activation, including Src-family nonreceptor tyrosine kinases, is required for activation of PECAM-1\\u2013dependent cation signaling. To determine whether PECAM-1 is a substrate for H 2 O 2 -induced tyrosine phosphorylation, we exposed REN, RHP, and REN cells stably transfected with Y663F/Y686F, a full-length PECAM-1 construct with two Y-to-F point mutations ( Fig. 7 ) known to disrupt PECAM-1 tyrosine phosphorylation and phosphotyrosine-mediated protein\\u2013protein association ( Newman 1999 ), to 0.5 mM H 2 O 2  for 10 min. Anti\\u2013PECAM-1 immunoprecipitation and phosphotyrosine analysis of these lysates revealed that PECAM-1 is heavily tyrosine phosphorylated in RHP and HUVEC after H 2 O 2  exposure, predominantly on the Y663/Y686 cytoplasmic motif ( Fig. 9  A). Subsequently, we analyzed phosphorylation patterns in REN, RHP, and PITC under conditions identical to the patch clamp and Ca 2+  imaging experiments. After a 10-min H 2 O 2  exposure at 200 \\u03bcM, near the threshold dose required for cation signaling, full-length PECAM-1 phosphorylation was increased by 40% above expected baseline phosphorylation ( Lu et al., 1996 ,  1997 ), as seen by densitometric normalization to PECAM-1 levels ( Fig. 9  B). Notably, this concentration of H 2 O 2  appears to represent the threshold dose for PECAM-1 phosphorylation, as increasing H 2 O 2  doses yielded increasingly intense PECAM-1 tyrosine phosphorylation ( Fig. 9, A and B ). No phosphotyrosine signal was detected in PITC or untransfected REN cells ( Fig. 9  B), confirming that the intact PECAM-1 cytoplasmic domain may serve as a substrate for H 2 O 2 -induced tyrosine phosphorylation. To determine whether this motif represents a critical element of the PECAM-1 cytoplasmic domain for cation current signaling, we conducted patch clamp studies on the Y663F/Y686F cells. No cation currents were observed in voltage-clamped Y663F/Y686F cells puffed with 200 \\u03bcM H 2 O 2  ( n  = 8), suggesting that the PECAM-1 cytoplasmic Y663/Y686 SH2-interaction domain is necessary for current activation. Notably, Y663F/Y686F and PITC also serve as transfection controls demonstrating that current activation is not merely an artifact of transfection, selection, or heterologous protein expression. In order to test our hypothesis that PECAM-1 may play a role in oxidant-mediated signaling, we performed voltage clamp experiments on human umbilical vein EC (HUVEC), REN cells, and REN cells stably transfected with PECAM-1 (RHP). We have previously utilized REN cells, a mesothelioma cell line that does not express endogenous PECAM-1, as an EC model, and have found that many EC signaling processes may be reconstituted after PECAM-1 expression ( Gurubhagavatula et al., 1998 ;  O'Brien et al., 2001 ). Puffer pipette application of 200 \\u03bcM H 2 O 2  (a dose based on preliminary studies described below) to HUVEC and RHP cells voltage clamped at a holding potential of \\u201360 mV, elicited a large inward current in 93% of HUVEC ( n  = 15) with a peak amplitude of 648.4 \\u00b1 36.6 pA. A similar current (peak amplitude 682.5 \\u00b1 41.55 pA) was detected in 89% of RHP ( n  = 18), whereas no currents were observed in untransfected REN cells exposed to 200 ( n  = 16) or 400 \\u03bcM ( n  = 11) H 2 O 2  ( Fig. 1 ). H 2 O 2  currents were typically activated 30\\u201390 s after H 2 O 2  exposure and were of prolonged time course, features similar to cation currents evoked in HUVEC and RHP cells after PECAM-1 engagement ( O'Brien et al., 2001 ). The response to H 2 O 2  was concentration dependent in that RHP cells treated with H 2 O 2  at concentrations of 100 \\u03bcM or lower elicited current less consistently (though those responding manifested similar current kinetics). Cells that responded to 400 or 500 \\u03bcM H 2 O 2  displayed evident cell toxicity over 24 h (unpublished data), consistent with previous reports ( Barchowsky et al., 1994 ). Puffer application of vehicle alone did not elicit a current, further eliminating a potential role for shear stress\\u2013gated NSC channels or transient loss of seal in the response (unpublished data). To further confirm the role of PECAM-1 in both HUVEC and RHP cells, H 2 O 2  current responses were recorded in cells dialyzed with polyclonal anti\\u2013PECAM-1 cytoplasmic domain (PCD) antibody or control polyclonal antibody directed against herpes simplex virus thymidine kinase ([anti-] HSVTK). 200 \\u03bcM H 2 O 2  elicited currents with identical kinetics to nondialyzed cells in 83% of HUVEC ( n  = 12) and 80% RHP cells ( n  = 5) dialyzed with anti-HSVTK antibody. By contrast, in cells dialyzed with PCD, currents were completely blocked in 81% of HUVEC ( n  = 16) and 75% of RHP ( n  = 4) cells, and markedly attenuated in those cells in which currents were observed ( Fig. 2 ). The ion selectivity and voltage dependence of the PECAM-1 current were examined in HUVEC and RHP. The reversal potential of the evoked current was determined by voltage ramp (\\u221260 up to +50 mV for 500 ms) and voltage step (\\u221260 to +40 mV in 20 mV increments) protocols in both RHP cells ( Fig. 3 ) and HUVEC (unpublished data). The current\\u2013voltage relationship of the PECAM-1 current was ohmic, with no evidence of voltage-dependent gating or rectification. Similarly, initial trials with instantaneous voltage ramps to +100 mV showed no rectification (unpublished data). The reversal potential in standard physiologic solutions was \\u223c0 mV, close to the theoretical monovalent cation equilibrium potential of \\u22121.2 mV. Consistent with a NSC current, replacement of extracellular Na +  ions with impermeant Tris resulted in a left shift of the current reversal potential and a decrease in the slope conductance of the inward current. Conversely, asymmetric impermeant anion and symmetric cation solutions (CsAcetate pipette and CsCl bath) did not shift the reversal potential nor alter the slope conductance (unpublished data). To assess the permeability of PECAM-1\\u2013activated NSC channels to Ca 2+ , we examined ionic signaling in extracellular solution containing 100 mM Ca 2+  as the only permeant cation. As shown in  Fig. 4 , exposure of cells to 200 \\u03bcM H 2 O 2  activated a prolonged Ca 2+  current in 5/6 (83%) HUVEC and 6/6 (100%) RHP cells. The kinetics of this current were similar to those observed in monovalent cation solutions, and the current reversal potential was shifted to positive potentials, as predicted for Ca 2+ -permeant channels. These results are identical to those previously obtained after homophilic PECAM-1 engagement and anti\\u2013PECAM-1 antibody ligation ( O'Brien et al., 2001 ), suggesting that the same channel mediates both PECAM-1\\u2013dependent ionic signaling events. To further examine the role of PECAM-1 in mediating H 2 O 2 -activated calcium signaling, we measured H 2 O 2 -induced Ca 2+  fluorescence with laser scanning confocal microscopy in individual Fluo-4\\u2013loaded REN and RHP cells exposed to 200 \\u03bcM H 2 O 2 . As shown in  Fig. 5 , H 2 O 2  evoked a prolonged rise in intracellular calcium lasting about 5 min ( Fig. 5, A and B ), a time course quite similar to that of the PECAM-1\\u2013dependent cation current (compare  Figs. 1  and  5 ). Conversely, in REN cells, no increase in intracellular calcium was observed, confirming the requirement for PECAM-1 in this calcium response ( Fig. 5  B, inset). Addition of thrombin yielded calcium signals in both cell lines that were of rapid onset, but of much shorter duration, whereas buffer did not produce a rise in [Ca 2+ ] i  in either cell type ( Fig. 5  B). These findings suggest that EC PECAM-1 regulates the gating of a calcium-permeant, voltage-independent, NSC channel that mediates prolonged calcium signals observed in ECs after physiologic H 2 O 2  exposure. We have previously reported that PECAM-1 engagement activates a calcium-permeant channel that is independent of intracellular Ca 2+  store release ( O'Brien et al., 2001 ). We next sought to determine the relationship of the PECAM-1\\u2013dependent H 2 O 2  cation current to phosphoinositide-mediated intracellular calcium store release and to subsequently activated endothelial plasmalemmal nonspecific cation channels termed depletion-activated channel or store-operated channels (SOCs) that mediate capacitative calcium entry ( Vaca and Kunze, 1995 ;  Parekh and Penner, 1997 ;  Kamouchi et al., 1999 ). After dialysis with 5 mg/ml heparin, a compound known to act as an IP 3  receptor antagonist as well as a potential inhibitor of G protein receptor interaction ( Taylor and Broad, 1998 ), H 2 O 2 - and thrombin-induced currents were measured in RHP cells ( Fig. 6 ) and HUVEC (unpublished data). Treatment with 200 \\u03bcM H 2 O 2  evoked typical NSC currents in 8/9 (89%) RHP cells and 3/3 (100%) HUVEC, suggesting that the PECAM-1\\u2013activated NSC is neither IP 3  gated nor dependent on IP 3 -mediated intracellular calcium store release mechanisms, similar to findings after antibody engagement in whole-cell fluorimetry and voltage clamp studies ( O'Brien et al., 2001 ). Conversely, thrombin (1.5 U/ml), a G protein\\u2013coupled receptor agonist known to activate intracellular calcium store release and subsequent NSC activation in HUVEC and REN cells ( Gurubhagavatula et al., 1998 ;  O'Brien et al., 2001 ), activated a large inward current in control recordings ( n  = 9), but this current was completely blocked in cells dialyzed with heparin ( Fig. 6 , inset). Phosphoinositol turnover assays were also performed in RHP cells to confirm these findings as previously described ( Krymskaya et al., 1999 ;  O'Brien et al., 2001 ). Treatment with 200 \\u03bcM H 2 O 2  failed to elicit significant stimulation of phosphoinositide formation, whereas addition of thrombin yielded a robust increase in phosphoinositide turnover (unpublished data). These findings further support a phosphoinositide-independent activation pathway for the PECAM-1 NSC current. The kinetic characteristics of the PECAM-1\\u2013dependent current are similar to DAC/SOC NSC currents activated by intracellular calcium release (capacitative calcium entry), which are sensitive to trivalent cations such as lanthanum chloride (LaCl 3 ) ( Vaca and Kunze, 1995 ;  Parekh and Penner, 1997 ;  Kamouchi et al., 1999 ). Therefore, we evaluated the dependence on intracellular Ca 2+  release and the La 3+  sensitivity of H 2 O 2 -activated PECAM-1\\u2013dependent currents. RHP cells and HUVEC were dialyzed with CsCl pipette solution containing 10 mM EGTA and 0 mM Ca 2+  (with a calculated [Ca 2+ ] i  of <1 nM) and current recorded 10 min after break-in. Dialysis with this intracellular solution did not itself elicit large cation currents similar to those activated by H 2 O 2 , but subsequent exposure (after 3\\u20135 min) to 200 \\u03bcM H 2 O 2  activated currents similar to those observed in cells dialyzed with normal intracellular solution. H 2 O 2  currents were observed in 5/5 (100%) RHP cells and 3/3 (100%) HUVEC tested, whereas no currents were observed prior to H 2 O 2 . Moreover, the H 2 O 2  current was only partially inhibited in the presence of 100 \\u03bcM LaCl 3  ( n  = 3), whereas the current activated by thrombin (1.5 U/ml) was completely inhibited by 50 \\u03bcM LaCl 3  ( n  = 6). Interestingly, thrombin-induced capacitative currents mediated by DAC/SOC NSC channels manifest time course, reversal potential (Vrev = 0), mean peak current amplitude (730 \\u00b1 47pA), and current\\u2013voltage relationships quite similar to those observed for PECAM-1 currents. However, the differences in trivalent sensitivity, calcium gating, and phosphoinositide dependence suggest that the PECAM-1\\u2013dependent current is activated by an alternate signaling mechanism and through channels that are distinct from those mediating capacitative calcium entry ( O'Brien et al., 2001 ). In order to specifically examine the role of the PECAM-1 intracellular domain, REN cells stably transfected with a panel of PECAM-1 mutant constructs were examined in a series of patch clamp studies ( Fig. 7 ). The first construct, PITC, consists of the PECAM-1 extracellular domain fused to the nonhomologous ICAM-1 transmembrane and cytoplasmic domains. This fusion protein does not elicit cation current after antibody engagement ( O'Brien et al., 2001 ), but supports homophilic interaction and localizes to the cell\\u2013cell border in confluent monolayers (unpublished data). No currents were observed in voltage-clamped PITC cells puffed with 200 \\u03bcM H 2 O 2  ( n  = 7), suggesting that the PCD, and possibly the transmembrane domain, are necessary for current activation. A second construct, IL2PCD, was utilized to evaluate the role of the isolated PCD in current regulation. IL2PCD contains the nonhomologous interleuken (IL)-2 receptor (CD25) extracellular and transmembrane domains fused to an intact PECAM-1 cytoplasmic domain that can serve as a substrate for H 2 O 2 -induced tyrosine phosphorylation (unpublished data). In voltage-clamped IL2PCD cells puffed with 200 or 400 \\u03bcM H 2 O 2 , cation currents with kinetics identical to full-length PECAM-1 control cells were observed in 14/15 (93%) cells ( Fig. 7 ). These findings indicate that the PECAM-1 cytoplasmic domain is necessary and sufficient for H 2 O 2  current activation. In EC and RHP cells, PECAM-1 engagement yields calcium transients mediated by a NSC channel that is Src tyrosine kinase-dependent ( Gurubhagavatula et al., 1998 ;  O'Brien et al., 2001 ). Therefore, we sought to determine the relationship of tyrosine kinase activity to the PECAM-1\\u2013dependent H 2 O 2  current. RHP cells ( n  = 8) and HUVEC ( n  = 5) pretreated for 1 h with 10 \\u03bcM genistein and purified with 200 \\u03bcM H 2 O 2  failed to activate a current, suggesting a similar role for tyrosine kinase activity in regulation of this channel. To further refine this observation, voltage-clamped HUVEC and RHP cells were dialyzed through the patch pipette with 10 \\u03bcg/ml Sc-18, an Src-neutralizing rabbit polyclonal antibody specific for the Src family nonreceptor kinases ( Marrero et al., 1995 ). In HUVEC and RHP cells dialyzed with Sc-18, attenuated current was observed in 2/10 (20%) and 3/11 (27%) of cells exposed to 200 \\u03bcM H 2 O 2 , whereas H 2 O 2  failed to elicit any current in the remaining cells. By contrast, exposure of HUVEC and RHP cells dialyzed with an irrelevant rabbit polyclonal control antibody (anti-HSVTK) to H 2 O 2  evoked currents identical to nondialyzed cells in 10/12 (83%) and 4/5 (80%) of cells tested, respectively ( Fig. 8 ). These findings confirm that tyrosine kinase activation, including Src-family nonreceptor tyrosine kinases, is required for activation of PECAM-1\\u2013dependent cation signaling. To determine whether PECAM-1 is a substrate for H 2 O 2 -induced tyrosine phosphorylation, we exposed REN, RHP, and REN cells stably transfected with Y663F/Y686F, a full-length PECAM-1 construct with two Y-to-F point mutations ( Fig. 7 ) known to disrupt PECAM-1 tyrosine phosphorylation and phosphotyrosine-mediated protein\\u2013protein association ( Newman 1999 ), to 0.5 mM H 2 O 2  for 10 min. Anti\\u2013PECAM-1 immunoprecipitation and phosphotyrosine analysis of these lysates revealed that PECAM-1 is heavily tyrosine phosphorylated in RHP and HUVEC after H 2 O 2  exposure, predominantly on the Y663/Y686 cytoplasmic motif ( Fig. 9  A). Subsequently, we analyzed phosphorylation patterns in REN, RHP, and PITC under conditions identical to the patch clamp and Ca 2+  imaging experiments. After a 10-min H 2 O 2  exposure at 200 \\u03bcM, near the threshold dose required for cation signaling, full-length PECAM-1 phosphorylation was increased by 40% above expected baseline phosphorylation ( Lu et al., 1996 ,  1997 ), as seen by densitometric normalization to PECAM-1 levels ( Fig. 9  B). Notably, this concentration of H 2 O 2  appears to represent the threshold dose for PECAM-1 phosphorylation, as increasing H 2 O 2  doses yielded increasingly intense PECAM-1 tyrosine phosphorylation ( Fig. 9, A and B ). No phosphotyrosine signal was detected in PITC or untransfected REN cells ( Fig. 9  B), confirming that the intact PECAM-1 cytoplasmic domain may serve as a substrate for H 2 O 2 -induced tyrosine phosphorylation. To determine whether this motif represents a critical element of the PECAM-1 cytoplasmic domain for cation current signaling, we conducted patch clamp studies on the Y663F/Y686F cells. No cation currents were observed in voltage-clamped Y663F/Y686F cells puffed with 200 \\u03bcM H 2 O 2  ( n  = 8), suggesting that the PECAM-1 cytoplasmic Y663/Y686 SH2-interaction domain is necessary for current activation. Notably, Y663F/Y686F and PITC also serve as transfection controls demonstrating that current activation is not merely an artifact of transfection, selection, or heterologous protein expression. Reactive oxygen species such as hydrogen peroxide are critical mediators of physiologic and pathologic vascular responses after inflammatory, ischemic, and hyperoxic insults. Central to these effects are changes in EC [Ca 2+ ] i , although the molecular interactions underlying H 2 O 2  increases in [Ca 2+ ] i  are not known. Here we report that physiologically relevant concentrations of H 2 O 2  activate Ca 2+ -permeant plasmalemmal cation channels and that this ionic signaling requires PECAM-1 expression. Activation of this current requires the PECAM-1 cytoplasmic Y663/Y686 motif and Src family nonreceptor tyrosine kinase activity, but is independent of IP 3  receptor activation. This is a previously unknown EC function for PECAM-1 that may represent a widespread mechanism linking neutrophil\\u2013EC interactions and EC response in the context of oxidant injury and inflammation. H 2 O 2  has been proposed to function as an endothelial signal transduction intermediate at low micromolar concentrations and as a mediator of cell response to pathologic processes at higher concentrations. Although the signaling processes underlying these actions are not well understood, H 2 O 2  is known to induce tyrosine phosphorylation through activation of tyrosine kinases and inhibition of protein tyrosine phosphatases ( Barchowsky et al., 1995 ;  Carbajal and Schaeffer, 1998 ), and can activate a range of [Ca 2+ ] i  signal responses ( Suzucki et al., 1997 ;  Yang et al., 1999 ;  Lounsbury et al., 2000 ). We observed consistent activation of PECAM-1\\u2013dependent cation currents at concentrations of H 2 O 2  in excess of 100 \\u03bcM. At H 2 O 2  levels below 100 \\u03bcM, minimal Ca 2+  signaling has been reported, whereas sublethal concentrations exceeding 100 \\u03bcM manifest prolonged, low-amplitude [Ca 2+ ] i  transients involving influx of extracellular Ca 2+  ( Bowles et al., 2001 ). Cytotoxic concentrations exceeding 500 \\u03bcM result in the activation of phospholipases and release of Ca 2+  ions from internal calcium stores ( Schilling and Elliott, 1992 ;  Natarjan et al., 1993 ;  Barchowsky et al., 1994 ;  Doan et al., 1994 ;  Siflinger-Birnboim et al., 1996 ;  Yolk et al., 1997 ;  Hu et al., 1998 ;  Min et al., 1998 ). Hydrogen peroxide levels between 200 and 500 \\u03bcM, although not cytolytic ( Shasby et al., 1988 ), may yield substantial cell toxicity by inducing signal transduction pathways that ultimately lead to apoptosis ( Barchowsky et al., 1994 ;  Suhara et al., 1998 ). Our findings indicate that PECAM-1, a protein highly expressed in ECs, functions as a critical regulator of plasmalemmal ion channels that result in depolarization and Ca 2+  influx during sublethal hydrogen peroxide exposure. Although PECAM-1 is essential in regulating plasmalemmal cation influx, our results do not exclude other Ca 2+ -mobilizing actions of H 2 O 2 , particularly at higher H 2 O 2  concentrations. We did not observe consistent PECAM-1\\u2013dependent cation signaling or tyrosine phosphorylation at H 2 O 2  concentrations below 100\\u2013200 \\u03bcM, suggesting that this level of H 2 O 2  represents a threshold for PECAM-1\\u2013mediated events. Thus, low levels of H 2 O 2 , such as those endogenously produced as signaling intermediates during ICAM-dependent EC responses to leukocytes ( Wang and Doerschuk, 2000 ), may be PECAM-1 independent. Instead, PECAM-1 appears to function as a calcium-regulating protein at the moderate H 2 O 2  levels (100\\u2013400 \\u03bcM) that are generated during EC\\u2013neutrophil interaction in the context of inflammation ( Test and Weiss, 1984 ). However, an important caveat in interpreting experiments evaluating H 2 O 2  cellular responses is the substantial variability in effective dosage associated with media conditions, particularly serum concentration. For example, it has been reported that H 2 O 2  concentration\\u2013response relationships may be decreased tenfold relative to serum-free conditions that may support EC morphologic changes after exposure to H 2 O 2  levels as low as 20 \\u03bcM ( Bradley et al., 1995 ). Although we demonstrate that PECAM-1 is required for H 2 O 2  ionic signaling, the identity of the channel and potential linking molecules remain unknown. As the current kinetics, ionic selectivity, and associated [Ca 2+ ] i  transients in ECs are indistinguishable when activated by H 2 O 2  or PECAM-1 engagement ( Gurubhagavatula et al., 1998 ;  O'Brien et al., 2001 ), it is likely that these phenomena are associated with activation of the same channel(s). Similarly, the failure of PECAM-1 to reconstitute cation signaling in certain cell types suggests that PECAM-1 is not, itself, a channel component ( O'Brien et al., 2001 ). Recently, attention has been directed toward mammalian homologues of the  Drosophila  transient receptor potential and transient receptor potential\\u2013like proteins as possible components in multimeric channels activated by Ca 2+  store depletion ( Xu et al., 1997 ;  Moore et al., 1998 ;  Groschner et al., 1999 ). As we have previously reported ( O'Brien et al., 2001 ), there is a distinct similarity between PECAM-1\\u2013dependent currents and nonspecific cation currents observed in ECs after depletion of Ca 2+  stores ( Vaca and Kunze, 1995 ;  Parekh and Penner, 1997 ;  Kamouchi et al., 1999 ). Although the H 2 O 2 -activated current is kinetically similar to store activated currents ( Groschner et al., 1999 ), differences in La 3+  sensitivity, lack of gating associated with Ca 2+  depletion, and independence of IP 3  receptor activation ( Nilius et al., 1997 ;  Kamouchi et al., 1999 ) suggest important differences in channel permeability and gating. Our results are similar to those describing an H 2 O 2 -activated, store release\\u2013independent, plasmalemmal calcium\\u2013conducting channel in EC that is gated through an unknown mechanism ( Bowles et al., 2001 ). They are also consistent with reports of a tyrosine kinase\\u2013dependent NSC channel activated through engagement of basic FGF (bFGF) or insulin-like growth factor\\u20131 receptors in ECs ( Munaron and Pla, 2000 ). As tyrosine kinase activity is required for both receptor-dependent (bFGF or insulin-like growth factor\\u20131) and independent (H 2 O 2  and PECAM-1 ligation) EC Ca +2  signaling, it is likely that the tyrosine kinase substrate(s) responsible for channel activation are downstream of specific receptors, which are phosphorylated by either mechanism ( Fleming et al., 1996 ). Exposure to phosphatase inhibitors such as vanadate, pervanadate, and H 2 O 2  is known to yield selective tyrosine phosphorylation of the PECAM-1 cytoplasmic domain ( Sagawa et al., 1997 ). After tyrosine phosphorylation, PECAM-1 can interact with several SH2 domain\\u2013containing proteins, including SHP2, PI3kinase, and PLC-\\u03b3 ( Newman, 1999 ), though the physiologic relevance of these interactions in EC is not certain. Similarly, PECAM-1 can function as a reservoir for transcription-regulating proteins such as \\u03b2 and \\u03b3-catenin in a phosphorylation-dependent manner ( Ilan et al., 2000 ). Utilizing endothelia-like REN cells stably transfected with wild-type and mutant PECAM-1 constructs, we have begun to conduct a detailed analysis of PECAM-1 structure\\u2013function relationships that would be otherwise impossible in ECs. Where possible, we have confirmed these findings in HUVEC. Our data demonstrate that PECAM-1 and likely other participating proteins serve as substrates for H 2 O 2 -induced tyrosine phosphorylation, that the PECAM-1\\u2013regulated current requires Src family nonreceptor tyrosine kinase activity, and that the PECAM-1 cytoplasmic domain (and its incorporated Y663/Y686 phosphotyrosine motif) is necessary and sufficient for H 2 O 2 -induced current activation ( Fig. 10 ). In confluent cells, full-length PECAM-1, PITC, and Y663F/Y686F localize predominantly to cell\\u2013cell borders, whereas IL2PCD is diffusely distributed ( Sun et al., 2000 ). However, the techniques employed in this study do not allow conclusions to be drawn regarding PECAM-1 cell localization requirements for current regulation in confluent monolayers. Similarly, although H 2 O 2  activates tyrosine kinases including p60src in ECs ( Barchowsky et al., 1995 ;  Carbajal and Schaeffer, 1998 ), the specific tyrosine kinase(s) mediating H 2 O 2 -induced PECAM-1 phosphorylation remains an important unresolved question. Interestingly, preliminary data, beyond the scope of this manuscript, suggest that the PECAM-1 transmembrane domain may also play a role in modulating PECAM-1\\u2013dependent ionic signals in the absence of an intact cytoplasmic domain (unpublished data). This is consistent with findings that the isolated PECAM-1 transmembrane and cytoplasmic domains function in signal transduction after matrix metalloproteinase\\u2013induced extracellular domain cleavage ( Ilan et al., 2001 ). The identification of proteins that have a functional association with PECAM-1 in this response, as well as further delineation of specific tyrosine kinase subtypes underlying H 2 O 2 -induced PECAM-1 phosphorylation, will be important steps in further defining this unique signaling pathway. Modulation of endothelial [Ca 2+ ] i  is critical to a range of EC processes including neutrophil transmigration ( Huang et al., 1993 ), a key EC function regulated by PECAM-1. Exogenous H 2 O 2  stimulates neutrophil extravasation in vivo, which is blocked by anti\\u2013PECAM-1 antibodies ( Scalia and Lefer, 1998 ). Similarly, anti\\u2013PECAM-1 antibodies block neutrophil transmigration after inflammatory stimuli known to induce generation of reactive oxygen species ( Vaporciyan et al., 1993 ;  Chosay et al., 1998 ). Our finding that the PECAM-1 extracellular domain, a critical element in EC\\u2013neutrophil interaction during transmigration ( Nakada et al., 2000 ), is not involved in channel regulation, suggests that H 2 O 2  ionic signaling may not be sufficient to induce leukocyte transmigration. Thus, PECAM-1 may serve dual functions of adhesion (extracellular domain) and ionic signaling (cytoplasmic domain), both of which may be required to recapitulate PECAM-1\\u2013dependent transmigration. The role of PECAM-1\\u2013mediated [Ca 2+ ] i  signaling and the involvement of specific PECAM-1 domains in the regulation of neutrophil\\u2013endothelial interaction and in endothelial response to oxidant injury remains an important area for future investigation. Antibodies used include the following: mAb 4G6, a murine immunoglobin (IgG) directed against the PECAM-1 extracellular Ig loop six domain ( Yan et al., 1995 ); mAb 1.3, a murine IgG directed against the PECAM-1 extracellular domain, a gift of Dr. Peter Newman (Blood Center of Southeastern Wisconsin, Milwaukee, WI); PCD, a rabbit polyclonal antibody directed against the PECAM-1 cytoplasmic domain; Sc-18 (Santa Cruz Biotechnology), a rabbit polyclonal antibody found to specifically inhibit Src family kinases Src, Yes, Fyn, and Fgr ( Marrero et al., 1995 ); anti-HSVTK, rabbit polyclonal anti-HSVTK antibody ( Elshami et al., 1996 ); anti-IL2 receptor (CD25) mAb 3G10 (Boehringer Mannheim); and 4G10, an anti-phosphotyrosine murine mAb (Upstate Biotechnology). Purified antibodies were obtained by protein G affinity chromatography of hybridoma supernatants or serum. Active binding of antibodies was confirmed by flow cytometry. Reagent grade hydrogen peroxide (Sigma-Aldrich) was diluted in isotonic PBS or media on the day of experiments as indicated. PECAM-1 immunoprecipitation was performed on REN cell lysates incubated with mAb 4G6 or anti-CD25 mAb (Il2PCD) followed by incubation with protein A Sepharose (Amersham Pharmacia Biotech). Lysates were separated on 10% SDS polyacrylamide reducing gels (Invitrogen). Membranes were probed with anti-phosphotyrosine antibody (4G10), counterstained with HRP-conjugated donkey anti\\u2013mouse IgG (Cappel), and signals were visualized with ECL (New England Nuclear). Membranes were then stripped per the manufacturer (NEN Life Science Products) and reprobed with mAb 1.3 or PCD and counterstained with donkey anti\\u2013mouse or donkey anti\\u2013rabbit HRP-conjugated secondary antibody (Cappel). Signals were again detected with ECL. Image capture and densitometry analysis was performed on a Kodak Image Station as per the manufacturer. HUVEC (Clonetics) were cultured in M199 medium (Mediatech, Inc.) supplemented with 15% FBS, 100 \\u03bcg/ml EC growth factor (Clonetics), 100 ug/ml Heparin (Elkins-Sinn), and 2 mM  l -glutamine (GIBCO BRL). Only cells of passage six or less were used. PECAM-1 mutant constructs IL2PCD ( Sun et al., 2000 ), PITC, and Y663F/Y686F, a gift of Dr. P. Newman ( Fig. 7 ) were constructed with the sequence overlap extension technique as previously described ( Sun et al., 2000 ;  O'Brien et al., 2001 ). All full-length and mutant PECAM-1 constructs were subcloned into the pcDNA-neo vector. REN cells, a human mesothelioma cell line previously isolated in our laboratory ( Smythe et al., 1994 ), were grown in RPMI media (Mediatech) supplemented with 10% FBS and 2 mM  l -glutamine (R10 media). Stable polyclonal populations of REN cell PECAM-1 transfectants were generated by magnetic bead sorting (Dynal, Inc.) and selected in G418 (0.5 mg/ml) supplemented R10 media as previously described ( Gurubhagavatula et al., 1998 ). Expression was subsequently confirmed by flow cytometry using an EPICS Elite Flow cytometer (Coulter Corporation) as described ( O'Brien et al., 2001 ). All stably transfected cell lines uniformly expressed PECAM-1 at high levels relative to HUVEC ( Fig. 7 ). Cells were prepared for patch clamp experiments on round 12-mm glass coverslips (Fisher) coated with fibronectin (GIBCO BRL) or 1% gelatin (HUVEC). Treated coverslips were placed in 6-well plates and seeded for 18\\u201324 h with 100,000 cells/well in 3 ml media. Whole-cell currents were recorded utilizing standard whole-cell or perforated patch clamp methods as described previously ( O'Brien et al., 2001 ). Coverslips were transferred to a temperature-controlled chamber maintained at 35\\u00b0C (Brook Industries), and superfused at 1 ml/min with either HBSS (123 mM NaCl, 2 mM CaCl 2 , 1 mM KCl, 25 mM Hepes, and 15 mM glucose, pH 7.4), high-calcium solution (100 mM CaCl 2 , 10 mM glucose, 10 mM Hepes, pH 7.4), CsCl solution (130 mM CsCl, 10 mM glucose, 10 mM Hepes, pH 7.4), or Tris-Cl solution (123 mM Tris-HCl, 2 mM CaCl 2 , 1 mM KCl, 25 mM Hepes, and 15 mM glucose, pH 7.4) as indicated. Borosilicate glass electrodes (resistance 3\\u20135 Mohm) were filled with internal solution containing 130 mM CsCl, 1.2 mM MgCl 2 , 0.075 mM EGTA, 1 mM Mg-ATP, and 10 mM Hepes, except for asymmetric anion experiments in which 130 mM CsAcetate was substituted for CsCl as indicated. For perforated patch experiments, pipettes were dipped 1\\u20132 s in pipette solution then backfilled with pipette solution containing 200 mg/ml nystatin. After seal formation and establishment of whole-cell recording configuration, cells were voltage clamped at \\u201360 mV (Axopatch 200B; Axon Instruments). Records were sampled at 1 kHz and filtered at 100 Hz. Current reversal potentials were measured either by step or ramp (\\u221260 up to +100 mV applied every 30 s) depolarization protocols. Only current traces returning to baseline were considered. Hydrogen peroxide solution was puffed directly onto cells through puffer pipettes (Picospritzer; General Valve). EGTA, heparin, and antibody dialysis experiments were conducted with 10 mM EGTA (Sigma-Aldrich), 5 mg/ml heparin sulfate, or 10 \\u03bcg/ml of PCD, Sc-18, or anti-thymidine kinase antibody added to the pipette solution. LaCl 3  inhibition experiments were performed with varying concentrations of LaCl 3  (0\\u2013100 \\u03bcM) in HBSS extracellular solution puffed directly onto cells after agonist stimulation with H 2 O 2  or thrombin. Cells were equilibrated for 10 min after break-in before initiating experimental protocols ( Davis and Sharma, 1997 ;  Wang and Kotlikoff, 1997 ). Cells were incubated with 10 \\u03bcM Fluo-4 AM (Molecular Probes) for 10 min at room temperature in a recording chamber mounted on an inverted microscope (TE300; Nikon), and washed with HBSS extracellular solution for 30 min. Fluo-4 fluorescence was recorded using a laser scanning confocal head (Radiance 2000; Bio-Rad Laboratories) attached to an inverted microscope with a plan-apo 60\\u00d7 water immersion objective (1.2 n.a.; Nikon). Cells were excited with 488 nm light from a krypton/argon laser and x-y images (128 \\u00d7 30 pixels) recorded using Lasersharp software (Bio-Rad Laboratories) at a 44-ms interval. Images were analyzed using LaserPix version 4.2 software (Bio-Rad Laboratories). Fluorescence profiles were obtained by computing the mean fluorescence from a region of the cell (F) for each image and dividing this by the mean fluorescence of the cell prior to stimulation (F o ). Antibodies used include the following: mAb 4G6, a murine immunoglobin (IgG) directed against the PECAM-1 extracellular Ig loop six domain ( Yan et al., 1995 ); mAb 1.3, a murine IgG directed against the PECAM-1 extracellular domain, a gift of Dr. Peter Newman (Blood Center of Southeastern Wisconsin, Milwaukee, WI); PCD, a rabbit polyclonal antibody directed against the PECAM-1 cytoplasmic domain; Sc-18 (Santa Cruz Biotechnology), a rabbit polyclonal antibody found to specifically inhibit Src family kinases Src, Yes, Fyn, and Fgr ( Marrero et al., 1995 ); anti-HSVTK, rabbit polyclonal anti-HSVTK antibody ( Elshami et al., 1996 ); anti-IL2 receptor (CD25) mAb 3G10 (Boehringer Mannheim); and 4G10, an anti-phosphotyrosine murine mAb (Upstate Biotechnology). Purified antibodies were obtained by protein G affinity chromatography of hybridoma supernatants or serum. Active binding of antibodies was confirmed by flow cytometry. Reagent grade hydrogen peroxide (Sigma-Aldrich) was diluted in isotonic PBS or media on the day of experiments as indicated. PECAM-1 immunoprecipitation was performed on REN cell lysates incubated with mAb 4G6 or anti-CD25 mAb (Il2PCD) followed by incubation with protein A Sepharose (Amersham Pharmacia Biotech). Lysates were separated on 10% SDS polyacrylamide reducing gels (Invitrogen). Membranes were probed with anti-phosphotyrosine antibody (4G10), counterstained with HRP-conjugated donkey anti\\u2013mouse IgG (Cappel), and signals were visualized with ECL (New England Nuclear). Membranes were then stripped per the manufacturer (NEN Life Science Products) and reprobed with mAb 1.3 or PCD and counterstained with donkey anti\\u2013mouse or donkey anti\\u2013rabbit HRP-conjugated secondary antibody (Cappel). Signals were again detected with ECL. Image capture and densitometry analysis was performed on a Kodak Image Station as per the manufacturer. HUVEC (Clonetics) were cultured in M199 medium (Mediatech, Inc.) supplemented with 15% FBS, 100 \\u03bcg/ml EC growth factor (Clonetics), 100 ug/ml Heparin (Elkins-Sinn), and 2 mM  l -glutamine (GIBCO BRL). Only cells of passage six or less were used. PECAM-1 mutant constructs IL2PCD ( Sun et al., 2000 ), PITC, and Y663F/Y686F, a gift of Dr. P. Newman ( Fig. 7 ) were constructed with the sequence overlap extension technique as previously described ( Sun et al., 2000 ;  O'Brien et al., 2001 ). All full-length and mutant PECAM-1 constructs were subcloned into the pcDNA-neo vector. REN cells, a human mesothelioma cell line previously isolated in our laboratory ( Smythe et al., 1994 ), were grown in RPMI media (Mediatech) supplemented with 10% FBS and 2 mM  l -glutamine (R10 media). Stable polyclonal populations of REN cell PECAM-1 transfectants were generated by magnetic bead sorting (Dynal, Inc.) and selected in G418 (0.5 mg/ml) supplemented R10 media as previously described ( Gurubhagavatula et al., 1998 ). Expression was subsequently confirmed by flow cytometry using an EPICS Elite Flow cytometer (Coulter Corporation) as described ( O'Brien et al., 2001 ). All stably transfected cell lines uniformly expressed PECAM-1 at high levels relative to HUVEC ( Fig. 7 ). Cells were prepared for patch clamp experiments on round 12-mm glass coverslips (Fisher) coated with fibronectin (GIBCO BRL) or 1% gelatin (HUVEC). Treated coverslips were placed in 6-well plates and seeded for 18\\u201324 h with 100,000 cells/well in 3 ml media. Whole-cell currents were recorded utilizing standard whole-cell or perforated patch clamp methods as described previously ( O'Brien et al., 2001 ). Coverslips were transferred to a temperature-controlled chamber maintained at 35\\u00b0C (Brook Industries), and superfused at 1 ml/min with either HBSS (123 mM NaCl, 2 mM CaCl 2 , 1 mM KCl, 25 mM Hepes, and 15 mM glucose, pH 7.4), high-calcium solution (100 mM CaCl 2 , 10 mM glucose, 10 mM Hepes, pH 7.4), CsCl solution (130 mM CsCl, 10 mM glucose, 10 mM Hepes, pH 7.4), or Tris-Cl solution (123 mM Tris-HCl, 2 mM CaCl 2 , 1 mM KCl, 25 mM Hepes, and 15 mM glucose, pH 7.4) as indicated. Borosilicate glass electrodes (resistance 3\\u20135 Mohm) were filled with internal solution containing 130 mM CsCl, 1.2 mM MgCl 2 , 0.075 mM EGTA, 1 mM Mg-ATP, and 10 mM Hepes, except for asymmetric anion experiments in which 130 mM CsAcetate was substituted for CsCl as indicated. For perforated patch experiments, pipettes were dipped 1\\u20132 s in pipette solution then backfilled with pipette solution containing 200 mg/ml nystatin. After seal formation and establishment of whole-cell recording configuration, cells were voltage clamped at \\u201360 mV (Axopatch 200B; Axon Instruments). Records were sampled at 1 kHz and filtered at 100 Hz. Current reversal potentials were measured either by step or ramp (\\u221260 up to +100 mV applied every 30 s) depolarization protocols. Only current traces returning to baseline were considered. Hydrogen peroxide solution was puffed directly onto cells through puffer pipettes (Picospritzer; General Valve). EGTA, heparin, and antibody dialysis experiments were conducted with 10 mM EGTA (Sigma-Aldrich), 5 mg/ml heparin sulfate, or 10 \\u03bcg/ml of PCD, Sc-18, or anti-thymidine kinase antibody added to the pipette solution. LaCl 3  inhibition experiments were performed with varying concentrations of LaCl 3  (0\\u2013100 \\u03bcM) in HBSS extracellular solution puffed directly onto cells after agonist stimulation with H 2 O 2  or thrombin. Cells were equilibrated for 10 min after break-in before initiating experimental protocols ( Davis and Sharma, 1997 ;  Wang and Kotlikoff, 1997 ). Cells were incubated with 10 \\u03bcM Fluo-4 AM (Molecular Probes) for 10 min at room temperature in a recording chamber mounted on an inverted microscope (TE300; Nikon), and washed with HBSS extracellular solution for 30 min. Fluo-4 fluorescence was recorded using a laser scanning confocal head (Radiance 2000; Bio-Rad Laboratories) attached to an inverted microscope with a plan-apo 60\\u00d7 water immersion objective (1.2 n.a.; Nikon). Cells were excited with 488 nm light from a krypton/argon laser and x-y images (128 \\u00d7 30 pixels) recorded using Lasersharp software (Bio-Rad Laboratories) at a 44-ms interval. Images were analyzed using LaserPix version 4.2 software (Bio-Rad Laboratories). Fluorescence profiles were obtained by computing the mean fluorescence from a region of the cell (F) for each image and dividing this by the mean fluorescence of the cell prior to stimulation (F o ). \",\n          \"Spontaneous regression of a malignant tumor is very rare. It has been estimated that this occurs less than once in 60,000 to 100,000 people with cancer and about 20 such cases have been reported annually ( 1 ,  2 ). Spontaneous regression of a malignant tumor has often noted in hypernephroma, neuroblastoma, malignant melanoma, choriocarcinoma, bladder cancer, but this is extremely rare for stomach cancer ( 3 ). We recently experienced a case of spontaneous regression of advanced gastric cancer, Borrmann type IV, and we report on our case along with a review of the related literature. An 84 yr-old male with a history of nausea and vomiting for the previous 3 weeks was admitted to our hospital. He used no medicine or herbal material except for an oral hypoglycemic agent for diabetes mellitus for 10 yr. His vital signs were stable and the physical examination revealed a palpable mass at the epigastrium. On laboratory examination, the WBC was 7,630/\\u00b5L, the hemoglobin level was 14.2 g/dL, platelet 194,000/\\u00b5L, fasting glucose 140 mg/dL, blood urea nitrogen 24.6 mg/dL, creatinine 1.25 mg/dL, total protein 6.62 g/dL, albumin 3.49 g/dL, AST 18 IU/L, ALT 13 IU/L, total bilirubin 0.41 mg/dL, direct bilirubin 0.18 mg/dL, alkaline phosphatase 126 IU/L, \\u03b3-GTP 11 IU/L, the prothrombin time (INR) was 1.0 and the activated partial thromboplastin time was 23.0 sec. Esophagogastroduodenoscopy (EGD) showed rugal fold thickening and widening along the whole length of the gastric body. The mucosa was congested and it looked dysplastic. When touched with biopsy forceps, the mucosa showed friability and bleeding to easy touch ( Fig. 1 ). Since the time of discharge, he took famotidine and itopride HCL without any other medication or herbal medication and he revisited our hospital for follow up 6 months later. EGD showed that the gastric folds were well distended by air inflation and the mucosa looked normal at the previously dysplastic and friable areas from the cardia to the lower body ( Fig. 2 ). Abdominal computerized tomography (CT) revealed diffuse irregular wall thickening and mucosal enhancement from the cardia to the lower body. Perigastric infiltration was present, yet lymph node enlargement or distant metastatic lesion was not identified ( Fig. 3A ). Multiple pieces of mucosal biopsy were obtained and the histological examination showed poorly differentiated adenocarcinoma ( Fig. 4A-C ). Malignant cells were stained at Periodic acid Schiff (PAS) stain and immunohistochemical stain was positive for cytokeratin AE1/AE3 and negative for LCA. Finally, we diagnosed the patient as suffering with advanced gastric cancer, Borrmann type IV, stage II (T3N0M0) and we planned surgical resection. But patient refused surgical resection and chemotherapy on consolation and then he was discharged. Abdominal CT revealed decreased wall thickening of the body compare to that of 6 months ago. The perigastric infiltration had regressed ( Fig. 3B ). The histology showed no evidence of malignancy ( Fig. 4D ). We examed DNA test (DNA fingerprinting) using autosomal short tandem repeat to know whether the two biopsy specimens were from the same patient. And the result of DNA test revealed that the DNA of two specimens were identical. The patient is alive without any digestive symptoms 14 months after the follow up exam. According to Everson's definition, spontaneous regression of malignant tumor is partial or complete disappearance of a malignant tumor in the absence of treatment that is ordinarily considered to be capable of producing regression. Further, the regression need not progress to complete disappearance of the tumor ( 4 ). This is very rare phenomenon. Of the 741 cases of spontaneous regression of malignant tumor from the year 1900 year to 1987, nine types of tumor (hypernephroma, malignant melanoma, neuroblastoma, retinoblastoma, leukemia, lymphoma, breast cancer, bladder cancer and choriocarcinoma) were noted in 511 cases (69%) and stomach cancer was noted in 17 cases (2.29%) ( 3 ). Regarding gastric cancer, 19 cases ( 5 ) of spontaneous regression have been reported. Spontaneous regression of malignant tumor is very rare and its cause is mostly ambiguous, so it is very difficult to determine the scientific mechanism. Yet the mechanisms suggested till now are a hormonal or immunological mechanism, operative trauma, necrosis or vascular insufficiency, infection and a psychological mechanism ( 1 ,  3 ). A hormonal mechanism was suggested because the onset of a pregnancy ( 6 ) or the cessation of ingestion of oral contraceptives was associated with the onset of a spontaneous regression of tumor ( 7 ,  8 ). In many cases of renal cell carcinoma, the remaining tumor mass regressed after the primary tumor or metastases had been surgically resected ( 9 ,  10 ), so we can presume that host immune system can destroy remaining tumor after the removal of the tumor burden ( 11 ). Other important mechanism is a febrile response associated with infection ( 12 ,  13 ). Of the patients with lung cancer, the 5-yr survival is increased in patients who are complicated with postoperative empyema ( 14 ). It was also reported that a patient with hepatocellular carcinoma developed severe cholangitis due to choledocholithiasis, and then hepatectomy was performed but the resected tumor demonstrated complete necrosis ( 15 ). In addition, some authors suggested a psychological factor ( 16 ) as the possible mechanism, but the evidence on this is still not clear. At first, we suspected that the biopsy specimens could be mistaken. But, it was proven that the DNA of two specimens were identical according to the results of DNA fingerprinting. And the patient had no twin brother. So we could conclude that it was the spontaneous regression of advanced gastric cancer. In this case, we could not identify any possible reason for the spontaneous regression of advanced gastric cancer. We herein report this very rare case, with a review of the related literature, in which advanced gastric cancer spontaneously regressed in 6 months without any clear cause. \"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"figures\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 11457,\n        \"samples\": [\n          \"Summary of the studies performed on circulating FASN levels. A : Linear association between circulating FASN concentration and insulin sensitivity according to glucose tolerance status ( upper panel : normal glucose tolerance;  lower panel : altered glucose tolerance). Although the absolute value of circulating FASN is reported, the statistical analysis was performed using its log-transformed value.  B : Linear association between log circulating FASN concentration and insulin sensitivity in all subjects (solid line with 95% CI) and according to glucose tolerance status (dotted lines). The association was also significant in each subgroup separately ( r  = \\u22120.43,  P  = 0.01, in subjects with normal glucose tolerance [NGT] and  r  = \\u22120.39,  P  = 0.02, in subjects with type 2 diabetes). A : Linear association between log circulating FASN concentration and FASN gene expression in visceral adipose tissue in subjects with type 2 diabetes.  B : Western blot showing the fragment recognized by the antibody used in the enzyme-linked immunosorbent assay in insulin-resistant (lanes 1\\u20135) and insulin-sensitive (lanes 6\\u201310) subjects. Previous experience indicates that FASN is an ordered cellular response of a living cell that actively extrudes excess intracellular FASN molecules from the cell (reference 10). The extracellular FASN molecules are present in various immunoreactive forms and show different expression patterns. In serum of breast cancer patients, the FASN is a small molecule similar to the form in breast cancer cell lysate. We here substantiate that a 150 kDa fragment is preferentially detected in serum from subjects with insulin resistance. A : Change in log circulating FASN concentration after weight loss in all subjects ( left panel ) and according to glucose tolerance status ( right panels ).  B : Change in circulating FASN in healthy lean young men after 3-month physical training\\u2013induced improvement of insulin action. * P  = 0.05 compared with baseline.  C : Change in circulating FASN after surgery-induced weight loss. ** P  = 0.01. A : Linear association between circulating FASN concentration and insulin sensitivity according to glucose tolerance status ( upper panel : normal glucose tolerance;  lower panel : altered glucose tolerance). Although the absolute value of circulating FASN is reported, the statistical analysis was performed using its log-transformed value.  B : Linear association between log circulating FASN concentration and insulin sensitivity in all subjects (solid line with 95% CI) and according to glucose tolerance status (dotted lines). The association was also significant in each subgroup separately ( r  = \\u22120.43,  P  = 0.01, in subjects with normal glucose tolerance [NGT] and  r  = \\u22120.39,  P  = 0.02, in subjects with type 2 diabetes). A : Linear association between log circulating FASN concentration and FASN gene expression in visceral adipose tissue in subjects with type 2 diabetes.  B : Western blot showing the fragment recognized by the antibody used in the enzyme-linked immunosorbent assay in insulin-resistant (lanes 1\\u20135) and insulin-sensitive (lanes 6\\u201310) subjects. Previous experience indicates that FASN is an ordered cellular response of a living cell that actively extrudes excess intracellular FASN molecules from the cell (reference 10). The extracellular FASN molecules are present in various immunoreactive forms and show different expression patterns. In serum of breast cancer patients, the FASN is a small molecule similar to the form in breast cancer cell lysate. We here substantiate that a 150 kDa fragment is preferentially detected in serum from subjects with insulin resistance. A : Change in log circulating FASN concentration after weight loss in all subjects ( left panel ) and according to glucose tolerance status ( right panels ).  B : Change in circulating FASN in healthy lean young men after 3-month physical training\\u2013induced improvement of insulin action. * P  = 0.05 compared with baseline.  C : Change in circulating FASN after surgery-induced weight loss. ** P  = 0.01. \",\n          \"Heterochromatin forms a compartment at the nuclear and nucleolar periphery in human fibroblasts and muscle cells.  (A) Late-replicating DNA (labeled by BrdU; green) concentrates at the nuclear periphery, whereas SC35 domains rich in RNA metabolic factors (red) punctuate the nuclear interior. (B) Heterochromatin, primarily at the nuclear periphery and nucleolus, is delineated as the DAPI region (blue) devoid of hnRNA (red). Inset shows magnification of the periphery. The image was deconvolved to minimize out-of-focus light. (C) This myotube highlights the prominent peripheral heterochromatin compartment in five nuclei as described in B. (D) The inactive X chromosome marked by the accumulation of Xist RNA (green) is localized either at the nucleolus or at the nuclear periphery (E). Interphase telomere distribution.  (A) Mammalian telomeres (red) distribute widely (blue, nucleolus; green, SC35 domains) with most avoiding the peripheral heterochromatin compartment. (B) Through focal series montage demonstrating the distribution of telomeres through the nucleus. Sections are 300 nm apart along the Z axis from the coverslip (top left) to the top of the cell (bottom right). The series was captured in wide field and deconvolved using a constrained iterative algorithm. Red, telomeres; green, nucleopores; blue, SC35 domains. The 4q telomere favors the heterochromatic compartment and, in particular, the nuclear periphery.  This specific localization is striking when compared to other loci on chromosome 4 and to other telomeres. (A) The 4q35 (red, arrows) is found at either the nuclear periphery or next to the nucleolus as demarcated by fibrillarin (blue) in this fibroblast. (B) 4q35 (red, arrows) localizes within the peripheral rim (DAPI, blue) defined by depletion of hnRNA (green). (C) The peripheral localization of 4q (red) contrasts with the internal localization of 4p (green). Lamin A and nucleoli were detected simultaneously (blue). (D) The centromere of chromosome 4 (green) favors the nucleolus (blue), whereas the 4q telomere (red) is oriented at the periphery. (E) Analysis of three 4q subtelomeric probes reveals no preferred ordering between these three loci with respect to the periphery. RP11-279K24 (red) is more peripheral than RP11-597P9 (green) in one allele in contrast to the loci at the other allele. (F) Likewise, RP11-279K24 (red) is more peripheral than D4Z4 (green) in one allele in contrast to the other. (G) Chromosome 17q (red) and 17p (green) avoid the nuclear periphery and are often in the vicinity of the nucleolus (blue), particularly in myotubes. (H) The 17q (red) telomere frequently colocalizes with SC35 domains (blue) in a myoblast. (I) Quantitative distributions of total or specific telomeres (or centromere) in myoblasts. We analyzed 1,000 total telomeres, >800 4q, 200 4p, 200 4cen, 100 17p, and 100 17q signals. The spatial orientation of chromosomes 4 and 17 correlates with the density of expression along their axes.  (A) Model depicting the spatial arrangement of chromosomes 4 and 17. (B) Distribution of gene density and expression for each chromosome (data reproduced with permission from  Versteeg et al., 2003 ). In FSHD patient muscle, truncation of D4Z4 repeats does not significantly alter 4q35 localization to the heterochromatic rim.  (A) Cytogenetic preparation of FSHD lymphoblastoid cells demonstrating different intensities of D4Z4 signals (green) at each 4q35 allele (red), permitting the mutant and wild-type alleles to be distinguished. (B) Both 4q35 alleles (green) in a mutant FSHD myoblast remain at the nuclear periphery depleted of hnRNA (red). (C) Localization of wild-type (arrow) and mutant (arrowhead) alleles in a FSHD myoblast using the same probes as in A. DAPI (blue) delineates the nucleus. (D) Quantitation of the localization of mutant (mut) versus wild-type (wt) allele in three FSHD myoblast cell lines (GM 17731, GM17899, and GM17869A) and in a normal myoblast line (50MB-1) before and after differentiation. 100 cells analyzed per sample. Localization of 4q35 (red) in a normal (E) and a FSHD myotube (F) with more intense D4Z4 signal (green) demarcating wild type versus weaker mutant 4q35 allele. Red and green signals from a z-stack were projected onto a single plane of blue signal. In the FSHD myotube, lamin A and fibrillarin were simultaneously detected by Alexa 350. (G) The number of cells (of 100 analyzed) with both 4q35 alleles dissociated from the periphery is rare regardless of D4Z4 copy number and may correlate with the age of the patient (see online supplemental material, available at  http://www.jcb.org/cgi/content/full/jcb.200403128/DC1 ). Typically, both alleles (red) lie just below the nuclear envelope stained for nucleopores (green) in normal muscle (H; blue, nucleoli) and FSHD muscle (I; blue, sc35). Positioning of 4q35 at the nuclear periphery resists perturbation.  (A) Human 4q35 locus (green) in the foreign nucleoplasm of a mouse somatic cell hybrid. AT-rich mouse chromocenters (centromeres) are apparent by DAPI (blue). (B) The nuclear and nucleolar periphery are favored over mouse chromocenters as preferred sites for 4q35 localization (100 cells analyzed). (C) Quantitation of nuclear positions of translocated loci 4q35 and Xp21 (analysis of 150 cells per sample). As discussed by  Kosak et al. (2002) , 50% peripheral by two-dimensional analysis in freely rotated suspension cells corresponds to \\u223c95% peripheral localization in three-dimensional analysis. (D) Ideogram of the normal and the derivative chromosomes with the relative positions of the probes used in this work. (E) Normal female lymphocyte line and translocation cell line (F) with Xp21 marking the X chromosomes (red) and Xist RNA (green) identifying the inactive X. (G) Translocation cell line with Xp21 (red) on both X chromosomes and a 4q35 marker (green) identifying the derivative X. (H and I) Translocation cell line with 4q35 (green) on both chromosome 4s and D4Z4 (red) identifying the normal 4. Subtelomeres containing D4Z4 repeats tend to localize to the heterochromatic compartment in both fibroblasts and muscle.  (A) Ideogram depicting the known positions and rough size of D4Z4 (red) and \\u03b2 satellite repeats (green;  Lyle et al., 1995 ;  Lemmers et al., 2002 ). (B) Both 4q (red) and 10q (green) favor the nuclear or nucleolar periphery (blue) in fibroblasts. (C) 10q (green) near the nuclear or nucleolar periphery (blue) in myoblasts. (D) A maximal intensity projection showing all D4Z4 signals (green) in a three-dimensional stack in muscle (red, 4q). D4Z4 telomeres cluster at the nucleolus (blue, arrow). (E) Quantitative summary showing that the distribution of the D4Z4 subtelomere family is a distinctive subset from that of total telomeres (in myoblasts). We analyzed >800 4q, 100 10q, 100 D4Z4, and 1,000 total telomere signals. Heterochromatin forms a compartment at the nuclear and nucleolar periphery in human fibroblasts and muscle cells.  (A) Late-replicating DNA (labeled by BrdU; green) concentrates at the nuclear periphery, whereas SC35 domains rich in RNA metabolic factors (red) punctuate the nuclear interior. (B) Heterochromatin, primarily at the nuclear periphery and nucleolus, is delineated as the DAPI region (blue) devoid of hnRNA (red). Inset shows magnification of the periphery. The image was deconvolved to minimize out-of-focus light. (C) This myotube highlights the prominent peripheral heterochromatin compartment in five nuclei as described in B. (D) The inactive X chromosome marked by the accumulation of Xist RNA (green) is localized either at the nucleolus or at the nuclear periphery (E). Interphase telomere distribution.  (A) Mammalian telomeres (red) distribute widely (blue, nucleolus; green, SC35 domains) with most avoiding the peripheral heterochromatin compartment. (B) Through focal series montage demonstrating the distribution of telomeres through the nucleus. Sections are 300 nm apart along the Z axis from the coverslip (top left) to the top of the cell (bottom right). The series was captured in wide field and deconvolved using a constrained iterative algorithm. Red, telomeres; green, nucleopores; blue, SC35 domains. The 4q telomere favors the heterochromatic compartment and, in particular, the nuclear periphery.  This specific localization is striking when compared to other loci on chromosome 4 and to other telomeres. (A) The 4q35 (red, arrows) is found at either the nuclear periphery or next to the nucleolus as demarcated by fibrillarin (blue) in this fibroblast. (B) 4q35 (red, arrows) localizes within the peripheral rim (DAPI, blue) defined by depletion of hnRNA (green). (C) The peripheral localization of 4q (red) contrasts with the internal localization of 4p (green). Lamin A and nucleoli were detected simultaneously (blue). (D) The centromere of chromosome 4 (green) favors the nucleolus (blue), whereas the 4q telomere (red) is oriented at the periphery. (E) Analysis of three 4q subtelomeric probes reveals no preferred ordering between these three loci with respect to the periphery. RP11-279K24 (red) is more peripheral than RP11-597P9 (green) in one allele in contrast to the loci at the other allele. (F) Likewise, RP11-279K24 (red) is more peripheral than D4Z4 (green) in one allele in contrast to the other. (G) Chromosome 17q (red) and 17p (green) avoid the nuclear periphery and are often in the vicinity of the nucleolus (blue), particularly in myotubes. (H) The 17q (red) telomere frequently colocalizes with SC35 domains (blue) in a myoblast. (I) Quantitative distributions of total or specific telomeres (or centromere) in myoblasts. We analyzed 1,000 total telomeres, >800 4q, 200 4p, 200 4cen, 100 17p, and 100 17q signals. The spatial orientation of chromosomes 4 and 17 correlates with the density of expression along their axes.  (A) Model depicting the spatial arrangement of chromosomes 4 and 17. (B) Distribution of gene density and expression for each chromosome (data reproduced with permission from  Versteeg et al., 2003 ). In FSHD patient muscle, truncation of D4Z4 repeats does not significantly alter 4q35 localization to the heterochromatic rim.  (A) Cytogenetic preparation of FSHD lymphoblastoid cells demonstrating different intensities of D4Z4 signals (green) at each 4q35 allele (red), permitting the mutant and wild-type alleles to be distinguished. (B) Both 4q35 alleles (green) in a mutant FSHD myoblast remain at the nuclear periphery depleted of hnRNA (red). (C) Localization of wild-type (arrow) and mutant (arrowhead) alleles in a FSHD myoblast using the same probes as in A. DAPI (blue) delineates the nucleus. (D) Quantitation of the localization of mutant (mut) versus wild-type (wt) allele in three FSHD myoblast cell lines (GM 17731, GM17899, and GM17869A) and in a normal myoblast line (50MB-1) before and after differentiation. 100 cells analyzed per sample. Localization of 4q35 (red) in a normal (E) and a FSHD myotube (F) with more intense D4Z4 signal (green) demarcating wild type versus weaker mutant 4q35 allele. Red and green signals from a z-stack were projected onto a single plane of blue signal. In the FSHD myotube, lamin A and fibrillarin were simultaneously detected by Alexa 350. (G) The number of cells (of 100 analyzed) with both 4q35 alleles dissociated from the periphery is rare regardless of D4Z4 copy number and may correlate with the age of the patient (see online supplemental material, available at  http://www.jcb.org/cgi/content/full/jcb.200403128/DC1 ). Typically, both alleles (red) lie just below the nuclear envelope stained for nucleopores (green) in normal muscle (H; blue, nucleoli) and FSHD muscle (I; blue, sc35). Positioning of 4q35 at the nuclear periphery resists perturbation.  (A) Human 4q35 locus (green) in the foreign nucleoplasm of a mouse somatic cell hybrid. AT-rich mouse chromocenters (centromeres) are apparent by DAPI (blue). (B) The nuclear and nucleolar periphery are favored over mouse chromocenters as preferred sites for 4q35 localization (100 cells analyzed). (C) Quantitation of nuclear positions of translocated loci 4q35 and Xp21 (analysis of 150 cells per sample). As discussed by  Kosak et al. (2002) , 50% peripheral by two-dimensional analysis in freely rotated suspension cells corresponds to \\u223c95% peripheral localization in three-dimensional analysis. (D) Ideogram of the normal and the derivative chromosomes with the relative positions of the probes used in this work. (E) Normal female lymphocyte line and translocation cell line (F) with Xp21 marking the X chromosomes (red) and Xist RNA (green) identifying the inactive X. (G) Translocation cell line with Xp21 (red) on both X chromosomes and a 4q35 marker (green) identifying the derivative X. (H and I) Translocation cell line with 4q35 (green) on both chromosome 4s and D4Z4 (red) identifying the normal 4. Subtelomeres containing D4Z4 repeats tend to localize to the heterochromatic compartment in both fibroblasts and muscle.  (A) Ideogram depicting the known positions and rough size of D4Z4 (red) and \\u03b2 satellite repeats (green;  Lyle et al., 1995 ;  Lemmers et al., 2002 ). (B) Both 4q (red) and 10q (green) favor the nuclear or nucleolar periphery (blue) in fibroblasts. (C) 10q (green) near the nuclear or nucleolar periphery (blue) in myoblasts. (D) A maximal intensity projection showing all D4Z4 signals (green) in a three-dimensional stack in muscle (red, 4q). D4Z4 telomeres cluster at the nucleolus (blue, arrow). (E) Quantitative summary showing that the distribution of the D4Z4 subtelomere family is a distinctive subset from that of total telomeres (in myoblasts). We analyzed >800 4q, 100 10q, 100 D4Z4, and 1,000 total telomere signals. \",\n          \"Deletion of \\n YCA1 \\n does not prevent accumulation of mammalian caspase inhibitor FITC-VAD-FMK or cell death in the  cdc13-1  mutant.  Wild-type,  yca1 \\u0394,  cdc13-1 , and  yca1 \\u0394  cdc13-1  strains were incubated at 24\\u00b0C or at nonpermissive 37\\u00b0C for 24 h. (A) Caspase activation was assayed by staining with 10 \\u03bcM FITC-VAD-FMK and measurement of fluorescence was done by flow cytometry. Black line, 24\\u00b0C; red line, 37\\u00b0C. (B) Viability was analyzed by serial 10-fold dilutions spotted onto YPD plates and incubated for 3 d at 24\\u00b0C. Cell death and binding of caspase inhibitors in the \\n cdc13-1 \\n mutant at nonpermissive temperature.  (A)  cdc13-1  cells in early logarithmic phase were shifted from 24 to 37\\u00b0C for 24 h and then stained with 5 \\u03bcg/ml propidium iodide (PI) and/or 10 \\u03bcM FITC-VAD-FMK before analysis by flow cytometry. Four distinct subpopulations (a\\u2013d) can be identified on these scatter plots as described in the text. (B) Presence of caspase inhibitor Z-VAD-FMK does not block binding of FITC-VAD-FMK or cell death in the  cdc13-1  mutant. (C) Dead  cdc13-1  cells are unspecifically labeled with caspase inhibitor substrate FITC-DEVD-FMK, uncleavable by Yca1 protease ( Madeo et al., 2002a ). Dead yeast cells unspecifically bind fluorescein and FITC or FITC-conjugated caspase inhibitor FITC-VAD-FMK.  (A) After 24-h incubation at 37\\u00b0C, the  cdc13-1  mutant cells were stained with 10 \\u03bcM fluorescein or FITC and subjected to flow cytometry analysis. (B) Exponentially growing wild-type cells were either heated to 80\\u00b0C for 5 min (Dead cells) or maintained at 24\\u00b0C (Live cells). Cells from both cultures were subsequently labeled with PI and caspase inhibitor FITC-VAD-FMK. Green (FITC) and red (PI) fluorescence of stained cells was measured by flow cytometry. Staining of  cdc13-1  after incubation at the restrictive temperature and heat-killed wild-type yeast cells with DHR123.  (A)  cdc13-1  cells were incubated at 37\\u00b0C overnight in the presence or absence of the antioxidant N-acetylcysteine (NAC), stained with DHR123 and PI, and analyzed by flow cytometry. DHR123 staining reveals two populations of green fluorescence\\u2013positive cells: dead cells show intermediate green fluorescence, whereas a population with bright green fluorescence represents cells producing ROS. Control experiments with wild-type and  cdc13-1  cells at 24\\u00b0C revealed only a single nonfluorescent population restricted to the bottom left quadrant (not depicted). (B) Expression of the mammalian antiapoptotic protein Bcl-x L  in  cdc13-1  cells decreases ROS production but does not suppress death after 8-h incubation at 37\\u00b0C. Functional expression of Bcl-x L  in yeast cells was confirmed by rescuing toxicity of Bax expression. (C) Heat-killed wild-type cells show intermediate green fluorescence after staining with DHR123. Live and dead cells were prepared as described for  Fig. 3  and stained with DHR123 (blue line) or left unstained (red line). Addition of sorbitol to media increases survival of  cdc13-1  cells at the restrictive temperature. \\n cdc13-1  and wild-type cells were grown in YPD medium in the presence or absence of 1.2 M sorbitol at 37\\u00b0C for 24 h. Serial 10-fold dilutions were spotted on a YPD plate and incubated for 3 d at 24\\u00b0C. Deletion of \\n YCA1 \\n does not prevent accumulation of mammalian caspase inhibitor FITC-VAD-FMK or cell death in the  cdc13-1  mutant.  Wild-type,  yca1 \\u0394,  cdc13-1 , and  yca1 \\u0394  cdc13-1  strains were incubated at 24\\u00b0C or at nonpermissive 37\\u00b0C for 24 h. (A) Caspase activation was assayed by staining with 10 \\u03bcM FITC-VAD-FMK and measurement of fluorescence was done by flow cytometry. Black line, 24\\u00b0C; red line, 37\\u00b0C. (B) Viability was analyzed by serial 10-fold dilutions spotted onto YPD plates and incubated for 3 d at 24\\u00b0C. Cell death and binding of caspase inhibitors in the \\n cdc13-1 \\n mutant at nonpermissive temperature.  (A)  cdc13-1  cells in early logarithmic phase were shifted from 24 to 37\\u00b0C for 24 h and then stained with 5 \\u03bcg/ml propidium iodide (PI) and/or 10 \\u03bcM FITC-VAD-FMK before analysis by flow cytometry. Four distinct subpopulations (a\\u2013d) can be identified on these scatter plots as described in the text. (B) Presence of caspase inhibitor Z-VAD-FMK does not block binding of FITC-VAD-FMK or cell death in the  cdc13-1  mutant. (C) Dead  cdc13-1  cells are unspecifically labeled with caspase inhibitor substrate FITC-DEVD-FMK, uncleavable by Yca1 protease ( Madeo et al., 2002a ). Dead yeast cells unspecifically bind fluorescein and FITC or FITC-conjugated caspase inhibitor FITC-VAD-FMK.  (A) After 24-h incubation at 37\\u00b0C, the  cdc13-1  mutant cells were stained with 10 \\u03bcM fluorescein or FITC and subjected to flow cytometry analysis. (B) Exponentially growing wild-type cells were either heated to 80\\u00b0C for 5 min (Dead cells) or maintained at 24\\u00b0C (Live cells). Cells from both cultures were subsequently labeled with PI and caspase inhibitor FITC-VAD-FMK. Green (FITC) and red (PI) fluorescence of stained cells was measured by flow cytometry. Staining of  cdc13-1  after incubation at the restrictive temperature and heat-killed wild-type yeast cells with DHR123.  (A)  cdc13-1  cells were incubated at 37\\u00b0C overnight in the presence or absence of the antioxidant N-acetylcysteine (NAC), stained with DHR123 and PI, and analyzed by flow cytometry. DHR123 staining reveals two populations of green fluorescence\\u2013positive cells: dead cells show intermediate green fluorescence, whereas a population with bright green fluorescence represents cells producing ROS. Control experiments with wild-type and  cdc13-1  cells at 24\\u00b0C revealed only a single nonfluorescent population restricted to the bottom left quadrant (not depicted). (B) Expression of the mammalian antiapoptotic protein Bcl-x L  in  cdc13-1  cells decreases ROS production but does not suppress death after 8-h incubation at 37\\u00b0C. Functional expression of Bcl-x L  in yeast cells was confirmed by rescuing toxicity of Bax expression. (C) Heat-killed wild-type cells show intermediate green fluorescence after staining with DHR123. Live and dead cells were prepared as described for  Fig. 3  and stained with DHR123 (blue line) or left unstained (red line). Addition of sorbitol to media increases survival of  cdc13-1  cells at the restrictive temperature. \\n cdc13-1  and wild-type cells were grown in YPD medium in the presence or absence of 1.2 M sorbitol at 37\\u00b0C for 24 h. Serial 10-fold dilutions were spotted on a YPD plate and incubated for 3 d at 24\\u00b0C. \"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"tables\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 5793,\n        \"samples\": [\n          \"Average in-plane strain, (\\u03b3 11  + \\u03b3 22 )/2 at  x  = (0, 0, 0) from point QD at  y  = (0, 0,  d ) \",\n          \"Baseline characteristics of study participants based on hypertriglyceridemic waist phenotypes Comparison of lipid profile based on hypertriglyceridemic phenotype Body fat distributions based on hypertriglyceridemic waist phenotypes \",\n          \"Characteristics of the study population at the time of enrolment ( n  = 24) Echocardiogram and intracardiac electrogram (IEGM)-optimized delay values Optimal VV delay distribution at rest and during exercise, according to echocardiogram and IEGM optimization Echocardiogram and intracardiac electrogram (IEGM)-optimized delay values Optimal VV delay distribution at rest and during exercise, according to echocardiogram and IEGM optimization \"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"lists\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": null,\n        \"min\": null,\n        \"max\": null,\n        \"num_unique_values\": 0,\n        \"samples\": [],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 86
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [],
      "metadata": {
        "id": "sZyGZLO_IVN7"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import pandas as pd\n",
        "\n",
        "# Select only the 'abstract' and 'paragraphs' columns\n",
        "df_filtered = csv[['abstract', 'paragraphs']]\n",
        "\n",
        "# Save the filtered dataframe to a new CSV or any other format\n",
        "df_filtered.to_csv('filtered_data1.csv', index=False)\n",
        "\n",
        "# Display the first few rows of the filtered data to verify\n"
      ],
      "metadata": {
        "id": "Js1aKuNKJru3"
      },
      "execution_count": 87,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "df_filtered.head()\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 207
        },
        "id": "k-olazTQJyh4",
        "outputId": "5738f5a2-36a1-4407-b97d-c7393fc72e56"
      },
      "execution_count": 88,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "                                            abstract  \\\n",
              "0                                                NaN   \n",
              "1                                                NaN   \n",
              "2  OBJECTIVE To investigate the associations of p...   \n",
              "3  Bronchial provocation testing uses a variety o...   \n",
              "4  These experiments justify the following genera...   \n",
              "\n",
              "                                          paragraphs  \n",
              "0                                                NaN  \n",
              "1  RNAi can unite the precision view from microsc...  \n",
              "2  To investigate the associations of plasma leve...  \n",
              "3  Bronchial provocation testing may be employed ...  \n",
              "4                                                NaN  "
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-d329218b-a776-402c-8a52-41ad291fd353\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>abstract</th>\n",
              "      <th>paragraphs</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>NaN</td>\n",
              "      <td>RNAi can unite the precision view from microsc...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>OBJECTIVE To investigate the associations of p...</td>\n",
              "      <td>To investigate the associations of plasma leve...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>Bronchial provocation testing uses a variety o...</td>\n",
              "      <td>Bronchial provocation testing may be employed ...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>These experiments justify the following genera...</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-d329218b-a776-402c-8a52-41ad291fd353')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-d329218b-a776-402c-8a52-41ad291fd353 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-d329218b-a776-402c-8a52-41ad291fd353');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-f0e28db4-faa8-47fd-9632-67cc108a9b61\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-f0e28db4-faa8-47fd-9632-67cc108a9b61')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-f0e28db4-faa8-47fd-9632-67cc108a9b61 button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "df_filtered",
              "summary": "{\n  \"name\": \"df_filtered\",\n  \"rows\": 76244,\n  \"fields\": [\n    {\n      \"column\": \"abstract\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 68310,\n        \"samples\": [\n          \"In this paper we describe a 66-kD protein that co-purifies with intermediate filaments from rat optic nerve and spinal cord but can be separated further by ion-exchange chromatography. This protein is distinct from the 68-kD neurofilament subunit protein as judged by isoelectric focusing, immunoblotting, peptide mapping, and tests of polymerization competence. This protein is avidly recognized by the monoclonal anti-intermediate filament antigen antibody, previously demonstrated to recognize a common antigenic determinant in all five known classes of intermediate filaments. Also, when isolated this protein binds to various intermediate filament subunit proteins, which suggests an in vivo interaction with the intermediate filament cytoskeleton, and it appears to be axonally transported in the rat optic nerve. Because of this ability to bind to intermediate filaments in situ and in vitro we have named this protein alpha-internexin. A possible functional role for the protein in organizing filament assembly and distribution is discussed.\",\n          \"Discrete fluctuations in conductance of lipid bilayer membranes may be observed during the initial stages of membrane interaction with EIM (\\\"excitability inducing material\\\"), during destruction of the EIM conductance by proteolysis, and during the potential-dependent transitions between low and high conductance states in the \\\"excitable\\\" membranes. The discrete conductance steps observed during the initial reaction of EIM with the lipid membranes are remarkably uniform, even in membranes of widely varying lipid composition. They range only from 2 to 6 x 10 -10  ohm -1  and average 4 x 10 -10  ohm -1 . Steps found during destruction of the EIM conductance by proteolysis are somewhat smaller. The transition between high conductance and low conductance states may involve steps as small as 0.5 x 10 -10  ohm -1 . These phenomena are consistent with the formation of a stable protein bridge across the lipid membrane to provide a polar channel for the transport of cations. T6he uniform conductance fluctuations observed during the formation of these macromolecular channels may indicate that the ions in a conductive channel, in its open state, are largely protected from the influence of the polar groups of the membrane lipids. Potential-dependent changes in conductance may be due to configurational or positional changes in the protein channel. Differences in lipid-lipid and lipid-macromolecule interactions may account for the variations in switching kinetics in various membrane systems.\",\n          \"Uncoupling proteins 2 and 3 (UCP2/3) are essential for mitochondrial Ca 2+  uptake but both proteins exhibit distinct activities in regard to the source and mode of Ca 2+  mobilization. In the present work, structural determinants of their contribution to mitochondrial Ca 2+  uptake were explored. Previous findings indicate the importance of the intermembrane loop 2 (IML2) for the contribution of UCP2/3. Thus, the IML2 of UCP2/3 was substituted by that of UCP1. These chimeras had no activity in mitochondrial uptake of intracellularly released Ca 2+ , while they mimicked the wild-type proteins by potentiating mitochondrial sequestration of entering Ca 2+ . Alignment of the IML2 sequences revealed that UCP1, UCP2 and UCP3 share a basic amino acid in positions 163, 164 and 167, while only UCP2 and UCP3 contain a second basic residue in positions 168 and 171, respectively. Accordingly, mutants of UCP3 in positions 167 and 171/172 were made. In permeabilized cells, these mutants exhibited distinct Ca 2+  sensitivities in regard to mitochondrial Ca 2+  sequestration. In intact cells, these mutants established different activities in mitochondrial uptake of either intracellularly released (UCP3 R171,E172 ) or entering (UCP3 R167 ) Ca 2+ . Our data demonstrate that distinct sites in the IML2 of UCP3 effect mitochondrial uptake of high and low Ca 2+  signals.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"paragraphs\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 34792,\n        \"samples\": [\n          \"C57BL/6 (H-2 b ) male mice and 129/SEV and IFN-\\u03b3 receptor knockout (IFN-\\u03b3R KO; reference  17 ) mice of both sexes were either purchased from The  Jackson Laboratory  (Bar Harbor, ME) or else bred in our facility and used at 2\\u201312 mo of age. LCMV, strain Armstrong, an RNA virus in the Old World arenavirus family, was propagated in BHK21 baby hamster kidney cells ( 18 ). The WR strain of VV, a DNA virus in the poxvirus family ( 11 ), and the AN3739 strain of PV, an RNA virus in the New World arenavirus family only distantly related to LCMV, were propagated in L929 or BHK21 cells ( 11 ,  18 ,  19 ). MCMV, strain Smith, a DNA virus in the herpesvirus family, was obtained from salivary glands of infected BALB/c mice ( 20 ). For acute virus infections, mice were injected intraperitoneally with 4 \\u00d7 10 4  PFU of LCMV, 2 \\u00d7 10 4 , 4 \\u00d7 10 5 , or 10 6  PFU of VV, 10 6  PFU of PV, or 10 4  or 10 5  PFU of MCMV. KO (H-2 b ), an SV40-transformed kidney cell line derived from a C57BL/6 mouse ( 21 ) and provided to us by Dr. Satvir Tevethia (Pennsylvania State Medical Center, Hershey, PA), was propagated in DMEM ( GIBCO BRL , Gaithersburg, MD) supplemented with 5 \\u00d7 10 \\u22125  M 2-ME. L929 (H-2 k ), a continuous liver cell line derived from C3H mice, MC57G (H-2 b ), a methylcholanthrene-induced fibroblast cell line from C57BL/6 mice, early passage mouse embryo fibroblast (MEF) cells from C57BL/6 mice, and ATCC Vero cells were propagated in Eagle's MEM ( GIBCO BRL ). KO and MC57G cells were infected with LCMV and PV at a multiplicity of infection (MOI) of 0.1\\u20130.2 PFU/cell and incubated for 2 d at 37\\u00b0C. All cell lines were supplemented with 100 U/ml penicillin G, 100 \\u03bcg/ml streptomycin sulfate, 2 mM  l -glutamine, 10 mM Hepes, and 10% heat-inactivated (56\\u00b0C, 30 min) fetal bovine serum (FBS;  Sigma Chemical Co. , St. Louis, MO). Mice were immunized with a sublethal dose of one virus. After the rise and fall of the acute T cell response and when the immune system had returned to homeostasis (usually 6 wk or longer), the mouse was challenged with the second virus. At 3\\u20135 d after the second virus infection, the spleen, fat pads, and/or liver were harvested, homogenized, and titrated for virus plaques, as previously described ( 22 ). In mice infected with LCMV only the spleens were titrated, as this strain of LCMV does not replicate in the liver or fat pads. These viruses were used either in crude tissue culture supernatants or were purified over sucrose gradients and diluted in PBS. Control naive mice were either left uninjected or injected with tissue culture media or with culture media sedimented like virus over a sucrose gradient, and there was no significant difference in virus titer when any one of these methods was compared with another. The control mice were always age matched to the experimental group and housed under exactly the same pathogen-free conditions as the experimental group for the identical time period. All mice used were healthy with no evidence of any underlying disease. Peritoneal cells from individual mice were stained for intracellular IFN-\\u03b3 based on a method previously described ( 23 ). Peritoneal cells at 2 \\u00d7 10 6  per tube were stimulated with PMA ( Sigma Chemical Co. ) at 50 ng/ml and ionomycin ( Sigma Chemical Co. ) at 500 ng/ml for 2 h at 37\\u00b0C. Then brefeldin A ( Sigma Chemical Co. ) was added at 10 \\u03bcg/ml for 2 h at 37\\u00b0C enabling intracellular proteins to accumulate. After the cells were washed with FACS \\u00ae  buffer (PBS with 2% FBS and 0.02% sodium azide) they were incubated with anti-Fc \\u03b3II receptor antibody ( PharMingen , San Diego, CA) as a blocking antibody, for 10 min at 4\\u00b0C. Then tricolor-conjugated anti\\u2013 mouse CD4 (clone CT-CD4; Caltag Laboratories, San Francisco, CA) or CD8 (clone CT-CD8\\u03b1; Caltag Laboratories) was added for 30 min at 4\\u00b0C. After two washes with FACS \\u00ae  buffer the cells were fixed with 200 \\u03bcl of 2% paraformaldehyde for 20 min at room temperature. They were then washed with FACS \\u00ae  buffer containing 0.5% saponin ( Sigma Chemical Co. ), a permeabilizing agent, and then incubated in this buffer for another 10 min at room temperature before adding PE-conjugated rat anti\\u2013mouse anti-IFN-\\u03b3 mAb ( PharMingen ) or control PE-conjugated rat IgG 1  isotype for 30 min at room temperature. They were then washed two more times with FACS \\u00ae  buffer with saponin and then one time with FACS \\u00ae  buffer before analysis on either a FACS \\u00ae  440 ( Becton Dickinson  & Co., Sparks, MD) or FACStar \\u00ae  Plus ( Becton Dickinson  & Co.). The total number of CD8 +  and CD4 +  or the total number of CD8 +  and CD4 +  cells secreting IFN-\\u03b3 was determined by multiplying the percentage times the total cell yield from each mouse. The total CD4 +  percentages may be an underestimate as these cells tend to downregulate the CD4 +  receptor on stimulation with PMA and ionomycin. That was not a problem with the CD8 +  cells. Adult 129/SEV and IFN-\\u03b3 receptor knockout mice (IFN-\\u03b3R KO) mice were inoculated intraperitoneally with LCMV, and at the indicated times after infection spleens were harvested. The LCMV-specific precursor frequency (p/f) per CD8 +  cell was quantified by LDAs of unsorted cells ( 12 ,  24 ). The percentage of CD8 +  T cells was determined by fluorescent antibody staining and FACS \\u00ae  analysis, as previously described ( 12 ,  24 ). The limiting dilution assays used a previously described method ( 12 ,  24 ). In brief, splenic lymphocytes from infected mice were harvested and titrated in U-bottomed, 96-well plates with 24 replicates at each dilution. They were stimulated with virus-infected PECs (3\\u20134 \\u00d7 10 4 /well) and supplemented with irradiated splenic feeders (1\\u20132 \\u00d7 10 5 /well) and growth factors provided by using a 16% culture supernatant from IL-2\\u2013secreting, gibbon lymphoma tumor cell line MLA.144 (American Type Culture Collection, Rockville, MD; reference  25 ). After 4 d, the cultures were fed with 10 4  irradiated, virus-infected PEC. On days 7\\u20138 of culture, individual wells were split twofold and assayed for cytolytic function on LCMV-infected or uninfected syngeneic target cells (KO) using a modified  51 Cr-release assay.  51 Cr-labeled targets (5 \\u00d7 10 3 ) were added to all wells, and after an 8\\u201310 h incubation at 37\\u00b0C the supernatant was harvested. Positive wells were defined as those wells whose  51 Cr-release exceeded the mean spontaneous release by >3 standard deviations. All wells that lysed uninfected syngeneic targets were eliminated from the analysis. Frequencies were calculated using \\u03c7 2  analysis according to Taswell ( 26 ) on a computer program kindly provided by Dr. Richard Miller (University of Michigan, Ann Arbor, MI). Cell-mediated cytotoxicity was determined using a standard microcytotoxicity (CTL) assay ( 19 ). Varying numbers of effector leukocytes were plated in triplicate to achieve the desired E/T ratio.  51 Cr-labeled MC57G target cells (5 \\u00d7 10 3 ), either uninfected or infected with virus, were added to all wells, and after a 6 h incubation at 37\\u00b0C the supernatant was harvested and counted. Data are expressed as percent specific  51 Cr-release = 100 \\u00d7 ([experimental cpm \\u2212 spontaneous cpm]/[maximum release cpm \\u2212 spontaneous release cpm]). Lytic units were calculated using exponential fit method ( 27 ) provided by software from Proteins International (Rochester Hills, MI). One lytic unit was defined as the number of effector cells required to lyse 5% of a population of 5 \\u00d7 10 3  targets in a 6 h CTL assay. A low percentage lysis was used for these calculations because some of the samples examined had relatively low levels of killing, and we wished to make quantitative comparisons within a wide range of very low and high CTL activity. The spontaneous release for each target used in these assays was <20%. C57BL/6 mice were injected intravenously with one spleen equivalent (\\u223c5\\u201310 \\u00d7 10 7  splenocytes) of immunologically naive (nonimmune) leukocytes or LCMV- immune (>6 wk after infection) leukocytes and challenged I.P. with PV (10 6  PFU) or VV (10 6  PFU). In some experiments LCMV-immune leukocytes were treated with anti-CD8 (rat anti\\u2013mouse CD8 mAb, clone Lyt 2.43; reference  28 ) and complement (rabbit serum) to remove CD8 +  T cells ( 29 ) or with anti-CD4 (rat anti\\u2013mouse CD4 mAb, clone GK1.5) and complement (guinea pig serum) to remove CD4 +  T cells before transfer. Spleens were harvested, homogenized, and plaque-assayed for virus titer at 3\\u20134 d after infection. The effectiveness of the CD4 +  or CD8 +  T cell depletion was monitored by staining with anti\\u2013 mouse CD4 (clone L3T4 RM4-4;  PharMingen ) or anti\\u2013mouse CD8 (clone CD8a Ly-2(53-6.7);  PharMingen ; reference  3 ). FACS \\u00ae  analyses showed that the CD4 +  or CD8 +  T cell contamination was <2%. The infiltrating leukocytes of VV-challenged LCMV-immune mice were isolated from the fat pads by mincing and digesting with collagenase B (200 mg/ml) in MEM media plus 4% BSA for 1 h at 37\\u00b0C, and then the lymphocytes were separated over Lympholite M (Cedarlane Labs., Hornby, Ontario, Canada) and analyzed. The number of PFU was determined by plaque assay using a 10% homogenate of tissue taken from individual mice and 10-fold dilutions of this homogenate on the appropriate cell line, ATCC vero cells for LCMV, PV, VV, and MEF cells for MCMV. Results were expressed as the geometric mean titers, i.e., the arithmetic averages of the logs for four or five separate animals titrated for virus individually plus or minus the SEM. Titers reported are log 10  PFU per whole spleen, liver, and both abdominal fat pads. The level of acute fatty necrosis was scored visually based on the severity of disease from level 0 to 7. Levels 1 and 2 represent very mild to mild disease with a few white necrotic spots on one or both lower abdominal fat pads; levels 3 and 4 represent mildly moderate and moderate with larger patches of necrosis of the lower abdominal fat pads and extension into the upper left quadrant fat pad around the spleen; and levels 5 and 6 represent moderately severe to severe with very extensive large patches of necrosis on the lower abdominal fat pads and spotty fatty necrosis throughout omental fat pads as well as the splenic fat pad; level 7 represents very severe disease with such severe fatty necrosis that the organs are adherent to each other. C57BL/6 (H-2 b ) male mice and 129/SEV and IFN-\\u03b3 receptor knockout (IFN-\\u03b3R KO; reference  17 ) mice of both sexes were either purchased from The  Jackson Laboratory  (Bar Harbor, ME) or else bred in our facility and used at 2\\u201312 mo of age. LCMV, strain Armstrong, an RNA virus in the Old World arenavirus family, was propagated in BHK21 baby hamster kidney cells ( 18 ). The WR strain of VV, a DNA virus in the poxvirus family ( 11 ), and the AN3739 strain of PV, an RNA virus in the New World arenavirus family only distantly related to LCMV, were propagated in L929 or BHK21 cells ( 11 ,  18 ,  19 ). MCMV, strain Smith, a DNA virus in the herpesvirus family, was obtained from salivary glands of infected BALB/c mice ( 20 ). For acute virus infections, mice were injected intraperitoneally with 4 \\u00d7 10 4  PFU of LCMV, 2 \\u00d7 10 4 , 4 \\u00d7 10 5 , or 10 6  PFU of VV, 10 6  PFU of PV, or 10 4  or 10 5  PFU of MCMV. KO (H-2 b ), an SV40-transformed kidney cell line derived from a C57BL/6 mouse ( 21 ) and provided to us by Dr. Satvir Tevethia (Pennsylvania State Medical Center, Hershey, PA), was propagated in DMEM ( GIBCO BRL , Gaithersburg, MD) supplemented with 5 \\u00d7 10 \\u22125  M 2-ME. L929 (H-2 k ), a continuous liver cell line derived from C3H mice, MC57G (H-2 b ), a methylcholanthrene-induced fibroblast cell line from C57BL/6 mice, early passage mouse embryo fibroblast (MEF) cells from C57BL/6 mice, and ATCC Vero cells were propagated in Eagle's MEM ( GIBCO BRL ). KO and MC57G cells were infected with LCMV and PV at a multiplicity of infection (MOI) of 0.1\\u20130.2 PFU/cell and incubated for 2 d at 37\\u00b0C. All cell lines were supplemented with 100 U/ml penicillin G, 100 \\u03bcg/ml streptomycin sulfate, 2 mM  l -glutamine, 10 mM Hepes, and 10% heat-inactivated (56\\u00b0C, 30 min) fetal bovine serum (FBS;  Sigma Chemical Co. , St. Louis, MO). Mice were immunized with a sublethal dose of one virus. After the rise and fall of the acute T cell response and when the immune system had returned to homeostasis (usually 6 wk or longer), the mouse was challenged with the second virus. At 3\\u20135 d after the second virus infection, the spleen, fat pads, and/or liver were harvested, homogenized, and titrated for virus plaques, as previously described ( 22 ). In mice infected with LCMV only the spleens were titrated, as this strain of LCMV does not replicate in the liver or fat pads. These viruses were used either in crude tissue culture supernatants or were purified over sucrose gradients and diluted in PBS. Control naive mice were either left uninjected or injected with tissue culture media or with culture media sedimented like virus over a sucrose gradient, and there was no significant difference in virus titer when any one of these methods was compared with another. The control mice were always age matched to the experimental group and housed under exactly the same pathogen-free conditions as the experimental group for the identical time period. All mice used were healthy with no evidence of any underlying disease. Peritoneal cells from individual mice were stained for intracellular IFN-\\u03b3 based on a method previously described ( 23 ). Peritoneal cells at 2 \\u00d7 10 6  per tube were stimulated with PMA ( Sigma Chemical Co. ) at 50 ng/ml and ionomycin ( Sigma Chemical Co. ) at 500 ng/ml for 2 h at 37\\u00b0C. Then brefeldin A ( Sigma Chemical Co. ) was added at 10 \\u03bcg/ml for 2 h at 37\\u00b0C enabling intracellular proteins to accumulate. After the cells were washed with FACS \\u00ae  buffer (PBS with 2% FBS and 0.02% sodium azide) they were incubated with anti-Fc \\u03b3II receptor antibody ( PharMingen , San Diego, CA) as a blocking antibody, for 10 min at 4\\u00b0C. Then tricolor-conjugated anti\\u2013 mouse CD4 (clone CT-CD4; Caltag Laboratories, San Francisco, CA) or CD8 (clone CT-CD8\\u03b1; Caltag Laboratories) was added for 30 min at 4\\u00b0C. After two washes with FACS \\u00ae  buffer the cells were fixed with 200 \\u03bcl of 2% paraformaldehyde for 20 min at room temperature. They were then washed with FACS \\u00ae  buffer containing 0.5% saponin ( Sigma Chemical Co. ), a permeabilizing agent, and then incubated in this buffer for another 10 min at room temperature before adding PE-conjugated rat anti\\u2013mouse anti-IFN-\\u03b3 mAb ( PharMingen ) or control PE-conjugated rat IgG 1  isotype for 30 min at room temperature. They were then washed two more times with FACS \\u00ae  buffer with saponin and then one time with FACS \\u00ae  buffer before analysis on either a FACS \\u00ae  440 ( Becton Dickinson  & Co., Sparks, MD) or FACStar \\u00ae  Plus ( Becton Dickinson  & Co.). The total number of CD8 +  and CD4 +  or the total number of CD8 +  and CD4 +  cells secreting IFN-\\u03b3 was determined by multiplying the percentage times the total cell yield from each mouse. The total CD4 +  percentages may be an underestimate as these cells tend to downregulate the CD4 +  receptor on stimulation with PMA and ionomycin. That was not a problem with the CD8 +  cells. Adult 129/SEV and IFN-\\u03b3 receptor knockout mice (IFN-\\u03b3R KO) mice were inoculated intraperitoneally with LCMV, and at the indicated times after infection spleens were harvested. The LCMV-specific precursor frequency (p/f) per CD8 +  cell was quantified by LDAs of unsorted cells ( 12 ,  24 ). The percentage of CD8 +  T cells was determined by fluorescent antibody staining and FACS \\u00ae  analysis, as previously described ( 12 ,  24 ). The limiting dilution assays used a previously described method ( 12 ,  24 ). In brief, splenic lymphocytes from infected mice were harvested and titrated in U-bottomed, 96-well plates with 24 replicates at each dilution. They were stimulated with virus-infected PECs (3\\u20134 \\u00d7 10 4 /well) and supplemented with irradiated splenic feeders (1\\u20132 \\u00d7 10 5 /well) and growth factors provided by using a 16% culture supernatant from IL-2\\u2013secreting, gibbon lymphoma tumor cell line MLA.144 (American Type Culture Collection, Rockville, MD; reference  25 ). After 4 d, the cultures were fed with 10 4  irradiated, virus-infected PEC. On days 7\\u20138 of culture, individual wells were split twofold and assayed for cytolytic function on LCMV-infected or uninfected syngeneic target cells (KO) using a modified  51 Cr-release assay.  51 Cr-labeled targets (5 \\u00d7 10 3 ) were added to all wells, and after an 8\\u201310 h incubation at 37\\u00b0C the supernatant was harvested. Positive wells were defined as those wells whose  51 Cr-release exceeded the mean spontaneous release by >3 standard deviations. All wells that lysed uninfected syngeneic targets were eliminated from the analysis. Frequencies were calculated using \\u03c7 2  analysis according to Taswell ( 26 ) on a computer program kindly provided by Dr. Richard Miller (University of Michigan, Ann Arbor, MI). Cell-mediated cytotoxicity was determined using a standard microcytotoxicity (CTL) assay ( 19 ). Varying numbers of effector leukocytes were plated in triplicate to achieve the desired E/T ratio.  51 Cr-labeled MC57G target cells (5 \\u00d7 10 3 ), either uninfected or infected with virus, were added to all wells, and after a 6 h incubation at 37\\u00b0C the supernatant was harvested and counted. Data are expressed as percent specific  51 Cr-release = 100 \\u00d7 ([experimental cpm \\u2212 spontaneous cpm]/[maximum release cpm \\u2212 spontaneous release cpm]). Lytic units were calculated using exponential fit method ( 27 ) provided by software from Proteins International (Rochester Hills, MI). One lytic unit was defined as the number of effector cells required to lyse 5% of a population of 5 \\u00d7 10 3  targets in a 6 h CTL assay. A low percentage lysis was used for these calculations because some of the samples examined had relatively low levels of killing, and we wished to make quantitative comparisons within a wide range of very low and high CTL activity. The spontaneous release for each target used in these assays was <20%. C57BL/6 mice were injected intravenously with one spleen equivalent (\\u223c5\\u201310 \\u00d7 10 7  splenocytes) of immunologically naive (nonimmune) leukocytes or LCMV- immune (>6 wk after infection) leukocytes and challenged I.P. with PV (10 6  PFU) or VV (10 6  PFU). In some experiments LCMV-immune leukocytes were treated with anti-CD8 (rat anti\\u2013mouse CD8 mAb, clone Lyt 2.43; reference  28 ) and complement (rabbit serum) to remove CD8 +  T cells ( 29 ) or with anti-CD4 (rat anti\\u2013mouse CD4 mAb, clone GK1.5) and complement (guinea pig serum) to remove CD4 +  T cells before transfer. Spleens were harvested, homogenized, and plaque-assayed for virus titer at 3\\u20134 d after infection. The effectiveness of the CD4 +  or CD8 +  T cell depletion was monitored by staining with anti\\u2013 mouse CD4 (clone L3T4 RM4-4;  PharMingen ) or anti\\u2013mouse CD8 (clone CD8a Ly-2(53-6.7);  PharMingen ; reference  3 ). FACS \\u00ae  analyses showed that the CD4 +  or CD8 +  T cell contamination was <2%. The infiltrating leukocytes of VV-challenged LCMV-immune mice were isolated from the fat pads by mincing and digesting with collagenase B (200 mg/ml) in MEM media plus 4% BSA for 1 h at 37\\u00b0C, and then the lymphocytes were separated over Lympholite M (Cedarlane Labs., Hornby, Ontario, Canada) and analyzed. The number of PFU was determined by plaque assay using a 10% homogenate of tissue taken from individual mice and 10-fold dilutions of this homogenate on the appropriate cell line, ATCC vero cells for LCMV, PV, VV, and MEF cells for MCMV. Results were expressed as the geometric mean titers, i.e., the arithmetic averages of the logs for four or five separate animals titrated for virus individually plus or minus the SEM. Titers reported are log 10  PFU per whole spleen, liver, and both abdominal fat pads. The level of acute fatty necrosis was scored visually based on the severity of disease from level 0 to 7. Levels 1 and 2 represent very mild to mild disease with a few white necrotic spots on one or both lower abdominal fat pads; levels 3 and 4 represent mildly moderate and moderate with larger patches of necrosis of the lower abdominal fat pads and extension into the upper left quadrant fat pad around the spleen; and levels 5 and 6 represent moderately severe to severe with very extensive large patches of necrosis on the lower abdominal fat pads and spotty fatty necrosis throughout omental fat pads as well as the splenic fat pad; level 7 represents very severe disease with such severe fatty necrosis that the organs are adherent to each other. Table  1  shows viral titration data from experiments in which mice immune to one virus were challenged with any of a panel of heterologous viruses. Prior immunity to LCMV, PV or MCMV resulted in significant 10\\u2013300-fold lower VV titers in three major organs (spleen, fat pads, and liver) at 3\\u20135 d after VV infection. Prior immunity to LCMV also resulted in a significant 5\\u201313-fold lower PV titers 3\\u20134 d after PV infection, while immunity to MCMV provided a less dramatic but still significant 2.5-fold decrease in PV titer of the fat pads. Immunity to MCMV resulted in a significant sixfold lower virus titer in the spleen of LCMV-infected mice; immunity to PV lowered the splenic LCMV titer only 2.5-fold (the Armstrong strain of LCMV does not replicate in the liver or fat pad). Immunity to LCMV and PV resulted in a four- and fivefold decrease, respectively, in liver titers in MCMV-infected mice. In contrast, immunity to VV did not significantly lower LCMV, PV, or MCMV titers. Prior immunity to LCMV or PV significantly lowered VV titers over a 2 log 10  range of challenge virus dose (Table  2 ). This protective effect of prior immunity lasted for long periods of time after the original virus infection, as mice either 8 or 29 wk after LCMV infection demonstrated equal efficiency at resisting PV challenge (Fig.  1 \\n A ) as did mice either 6 or 19 wk after MCMV infection challenged with VV (Fig.  1 \\n B ). These studies suggest that protective immunity between these four heterologous viruses is a common occurrence, but it is not universal to all virus combinations, as there is a selective protective effect dependent on the sequence of virus infections. These experiments were initially done with standard virus stocks derived from serum-containing tissue culture supernatants but then were performed with purified virus stocks to determine if cross-reactive responses to cellular contaminants or FBS were influencing the results. Impurities such as FBS and cellular debris were removed by purification of the viruses over sucrose gradients, followed by substantial 20\\u2013100-fold dilutions in PBS before inoculation of mice. Use of purified and diluted viruses eliminated any anamnestic response to FBS or cellular contaminants. With elispot assays we did detect FBS-specific responses of CD4 +  cells to IFN-\\u03b3 and IL-4 when inoculations were done with virus prepared in media containing 10% FBS (data not shown), but no such responses were detected when purified or diluted virus preparations were used. For the protective immunity experiments, however, the conventional virus stocks and purified viruses produced similar results, so all the results were pooled for the analyses in Table  1 . It should be further noted that the virus stocks were grown in different cell lines; LCMV was grown in BHK cells, PV was grown in BHK or L929 cells, and VV was grown in NCTC929 cells or L929 cells (11 and 12). MCMV was derived directly from mouse salivary glands ( 11 ). Also, serum from LCMV-, PV-, and VV-immune mice infected with the unpurified virus stocks did not demonstrate any antibody production against L929 or BHK cell surface molecules by FACS \\u00ae  analysis (data not shown). All this information makes it highly unlikely that a cellular contaminant influenced the protective immunity. Adoptive cell transfer of LCMV-immune splenocytes into naive mice resulted in five- to eightfold more rapid virus clearance in the spleen and fat pads, on VV or PV challenge than in mice adoptively reconstituted with naive splenocytes (Fig.  2 ). These results supported the concept that the protective heterologous immunity against acute VV and PV infections was dependent on immune lymphocytes. Depletion of either CD8 +  (Fig.  2 ,  A  and  C ) or CD4 +  (Fig.  2 ,  B  and  D ) T cells before the cell transfer resulted in a loss of this protective effect, suggesting that both memory T cell types were required. Studies were done to examine whether CD4 +  memory T cells were required for reactivation of the LCMV-specific memory CTL in mice challenged acutely with PV. Depletion of CD4 +  T cells in vivo by inoculation with anti-CD4 mAb did not have a significant effect on PV-specific CTL induction in PV-infected control or LCMV-immune mice. In LCMV-immune mice challenged with PV, the reactivated LCMV-specific bulk CTL activity ( 3 ) in the control group was 6.3 \\u00b1 1.4 lytic units (LU; calculated as 5% lysis of 5 \\u00d7 10 3  targets) per 10 6  splenocytes ( n  = 18) in bulk assays, but in the CD4 +  T cell\\u2013 depleted mice it was dramatically lower, at <1.2 \\u00b1 0.1 LU per 10 6  splenocytes ( n  = 6). All of these mice were depleted of NK cells with mAb to NK1.1 in order to reduce background cytotoxicity to better resolve the virus-specific killing ( 12 ). These results collectively suggest that CD4 +  T cells were required for the heterologous activation of CD8 +  CTL and that both CD4 +  and CD8 +  memory T cells were required for protective immunity. A property of memory T cells is their ability to produce much higher levels of cytokines than do naive cells after stimulation ( 14 ). Therefore, we hypothesized that a heterologous virus that stimulated memory T cells would elicit a potent IFN-\\u03b3 response, a cytokine to which VV is known to be extremely sensitive ( 17 ,  30 ). Analysis by ELISA assay of IFN-\\u03b3 concentrations (Fig.  3 \\n I ) in the peritoneal fluid of LCMV-immune mice at day 3 after VV infection (16,115 \\u00b1 4,262 pg/ml,  n  = 12) revealed >1,074-fold higher titers of IFN-\\u03b3 than in uninfected naive mice (<15 \\u00b1 0 pg/ml,  n  = 8) or LCMV-immune mice (<15 \\u00b1 0 pg/ml,  n  = 8) and 5.4-fold (2,979 \\u00b1 1,039 pg/ml;  n  = 12; Student's  t  test,  P  = 0.007) more than control mice challenged with VV. LCMV-immune mice challenged with PV did not show this enhanced production of IFN-\\u03b3 (Fig.  3 \\n II ) nor did VV-immune mice challenged with LCMV (data not shown). We then examined the peritoneal cavity more closely, as this is the site of initial VV replication where the most significant impact on early virus replication could occur. FACS \\u00ae  staining of PECs resolved a significant 8- and 2.7-fold enhanced recruitment of CD8 +  and CD4 +  T cells, respectively, in the LCMV-immune versus naive mice 3 d after challenged with VV (Fig.  4 ,  A  and  B ). Furthermore, the peritoneal T cells in the LCMV-immune mice 3 d after challenge with VV were predominantly of the memory phenotype, CD44 hi . Basing our cutoff value for CD44 hi  (FITC-conjugated anti-CD44;  PharMingen ) at a mean fluorescence of 12.6 units as determined on naive splenocytes, it was found that 92 \\u00b1 5% ( n  = 3) of the peritoneal CD8 +  cells and 75 \\u00b1 19% ( n  = 3) of the peritoneal CD4 +  cells were CD44 hi  memory phenotype. The CD8 +  cells had a mean fluorescence of 182 \\u00b1 40 and the CD4 +  cells had a mean fluorescence of 177 \\u00b1 58. These cells were examined for expression of intracellular IFN-\\u03b3 after treatment with PMA and ionomycin. There was a significant 10- and 2.7-fold increase in the number of IFN-\\u03b3\\u2013producing CD8 +  and CD4 +  T cells, respectively, in LCMV-immune mice versus naive mice challenged with VV (Fig.  4 ,  A  and  B ). To further examine whether IFN-\\u03b3 was playing a role in the protective effect against heterologous viruses, we initially treated LCMV-immune mice with anti-IFN-\\u03b3 antibodies, as previously described ( 31 ). In brief, we gave 0.2 ml of a 1/5 dilution of anti-IFN-\\u03b3 ascites intraperitoneally daily, in combination with a similar intravenous dose on day 0 and day 3 of VV infection. In the first experiment at 4 d after VV infection there was a 20-fold increase in the VV fat pad titer in the anti-IFN-\\u03b3\\u2013treated group (LCMV-immune + VV, 2.77 \\u00b1 0.23 PFU/ml vs. LCMV-immune + VV + anti-IFN-\\u03b3, 4.07 \\u00b1 0.45 PFU/ml;  n  = 5;  P  = 0.04, Student's  t  test). In the second experiment at day 6 after VV infection there was a 13-fold increase in the anti-IFN-\\u03b3\\u2013treated group (LCMV-immune + VV, 4.23 \\u00b1 0.61 PFU/ml vs. LCMV-immune + VV + anti\\u2013IFN-\\u03b3, 5.31 \\u00b1 0.22;  n  = 5;  P  = 0.006, Student's  t  test). To further confirm this observation, we extended these studies into IFN-\\u03b3 receptor knockout (IFN-\\u03b3R KO) mice. Since the IFN-\\u03b3R KO mice were on the 129/SEV background instead of the C57BL/6 background, normal 129/ SEV mice were used as controls (Fig.  5 ). The 129/SEV and 129 IFN-\\u03b3R KO mice developed equivalent high frequency LCMV-specific memory pCTL, as shown by limiting dilution assays (Fig.  5 ,  A  and  B ). 129/SEV mice immune to LCMV developed protective heterologous immunity to PV and VV, similar to that in the C57BL/6 mice. However, IFN-\\u03b3R KO mice immune to LCMV did not resist VV infection, confirming a role for IFN-\\u03b3 in the control of VV by LCMV-induced memory T cells (Fig.  6 ,  A  and  B ). Similar experiments demonstrated that IFN-\\u03b3 was not required for the protective effect against PV (Fig.  6 ,  C  and  D ), suggesting the possibility that other cytokines or even direct cytotoxicity by the cross-reactive CTL may be effective. A role for cytotoxicity could not be easily addressed in these systems, as cytotoxicity-deficient perforin KO mice develop persistent LCMV infections instead of memory T cell responses ( 32 ). Substantial evidence has been provided that T cells can not only be mediators of protective immunity, but also of immunopathology ( 10 ,  33 \\u2013 37 ). Our studies described below of heterologous immunity between viruses indicate that a previous virus infection may predispose a host to a much more severe disease upon infection with an unrelated heterologous virus. In general accordance with an earlier report ( 22 ), mice immunized with LCMV, unlike naive mice, developed severe immunopathology consisting of extensive acute fat necrosis (AFN) with a cellular infiltrate in visceral fat pads 5\\u20137 d after VV challenge (Fig.  7 ). In contrast, prior immunity to PV or MCMV, although protective against VV challenge, did not result in AFN on VV infection. The leukocyte infiltrate within the abdominal fat pads contained 19% \\u00b1 7 (SEM;  n  = 2) CD4 +  T cells and 23% \\u00b1 5 (SEM;  n  = 2) CD8 +  T cells and mediated CTL activity against both VV- (8 \\u00b1 4 lytic units per 10 6  cells;  n  = 5) and LCMV- (9.4 \\u00b1 6.9 lytic units per 10 6  cells;  n  = 5) infected targets. The CTL activity segregated to the CD8 +  T cell population on sorting (data not shown). Therefore, these fat pads contained infiltrates of activated CTL specific not only for VV but also for LCMV. A reactivation of the LCMV-specific memory splenic CTL by VV has previously been documented ( 11 ,  12 ). Adoptive transfer studies showed that splenocytes from LCMV-immune mice not only provided some protective immunity against VV synthesis, but also induced rather severe acute fatty necrosis (Fig.  8 \\n A ). This immunopathology did not occur in mice lacking IFN-\\u03b3 receptors (Fig.  8 \\n B ). Further, LCMV-immune mice acutely infected with VV and treated with anti-IFN-\\u03b3 daily had decreased evidence of AFN (level of AFN: LCMV-immune + VV, 4 \\u00b1 1.2; LCMV-immune + VV + anti-IFN-\\u03b3, 1 \\u00b1 0.5;  n  = 5). As shown above, LCMV-immune mice challenged with VV (Fig.  3 \\n I ) had higher levels of IFN-\\u03b3 in the peritoneal fluid than did naive mice challenged with VV. This enhanced IFN-\\u03b3 response in immune mice may thus play a role in protective immunity as well as in the induction of immunopathology. In this sequence of viruses the immunological contribution of the LCMV-specific memory cells may result in an overwhelming delayed-type hypersensitivity-like cell-mediated immune response, resulting in enhanced pathogenesis, even though the VV is cleared more rapidly. Table  1  shows viral titration data from experiments in which mice immune to one virus were challenged with any of a panel of heterologous viruses. Prior immunity to LCMV, PV or MCMV resulted in significant 10\\u2013300-fold lower VV titers in three major organs (spleen, fat pads, and liver) at 3\\u20135 d after VV infection. Prior immunity to LCMV also resulted in a significant 5\\u201313-fold lower PV titers 3\\u20134 d after PV infection, while immunity to MCMV provided a less dramatic but still significant 2.5-fold decrease in PV titer of the fat pads. Immunity to MCMV resulted in a significant sixfold lower virus titer in the spleen of LCMV-infected mice; immunity to PV lowered the splenic LCMV titer only 2.5-fold (the Armstrong strain of LCMV does not replicate in the liver or fat pad). Immunity to LCMV and PV resulted in a four- and fivefold decrease, respectively, in liver titers in MCMV-infected mice. In contrast, immunity to VV did not significantly lower LCMV, PV, or MCMV titers. Prior immunity to LCMV or PV significantly lowered VV titers over a 2 log 10  range of challenge virus dose (Table  2 ). This protective effect of prior immunity lasted for long periods of time after the original virus infection, as mice either 8 or 29 wk after LCMV infection demonstrated equal efficiency at resisting PV challenge (Fig.  1 \\n A ) as did mice either 6 or 19 wk after MCMV infection challenged with VV (Fig.  1 \\n B ). These studies suggest that protective immunity between these four heterologous viruses is a common occurrence, but it is not universal to all virus combinations, as there is a selective protective effect dependent on the sequence of virus infections. These experiments were initially done with standard virus stocks derived from serum-containing tissue culture supernatants but then were performed with purified virus stocks to determine if cross-reactive responses to cellular contaminants or FBS were influencing the results. Impurities such as FBS and cellular debris were removed by purification of the viruses over sucrose gradients, followed by substantial 20\\u2013100-fold dilutions in PBS before inoculation of mice. Use of purified and diluted viruses eliminated any anamnestic response to FBS or cellular contaminants. With elispot assays we did detect FBS-specific responses of CD4 +  cells to IFN-\\u03b3 and IL-4 when inoculations were done with virus prepared in media containing 10% FBS (data not shown), but no such responses were detected when purified or diluted virus preparations were used. For the protective immunity experiments, however, the conventional virus stocks and purified viruses produced similar results, so all the results were pooled for the analyses in Table  1 . It should be further noted that the virus stocks were grown in different cell lines; LCMV was grown in BHK cells, PV was grown in BHK or L929 cells, and VV was grown in NCTC929 cells or L929 cells (11 and 12). MCMV was derived directly from mouse salivary glands ( 11 ). Also, serum from LCMV-, PV-, and VV-immune mice infected with the unpurified virus stocks did not demonstrate any antibody production against L929 or BHK cell surface molecules by FACS \\u00ae  analysis (data not shown). All this information makes it highly unlikely that a cellular contaminant influenced the protective immunity. Adoptive cell transfer of LCMV-immune splenocytes into naive mice resulted in five- to eightfold more rapid virus clearance in the spleen and fat pads, on VV or PV challenge than in mice adoptively reconstituted with naive splenocytes (Fig.  2 ). These results supported the concept that the protective heterologous immunity against acute VV and PV infections was dependent on immune lymphocytes. Depletion of either CD8 +  (Fig.  2 ,  A  and  C ) or CD4 +  (Fig.  2 ,  B  and  D ) T cells before the cell transfer resulted in a loss of this protective effect, suggesting that both memory T cell types were required. Studies were done to examine whether CD4 +  memory T cells were required for reactivation of the LCMV-specific memory CTL in mice challenged acutely with PV. Depletion of CD4 +  T cells in vivo by inoculation with anti-CD4 mAb did not have a significant effect on PV-specific CTL induction in PV-infected control or LCMV-immune mice. In LCMV-immune mice challenged with PV, the reactivated LCMV-specific bulk CTL activity ( 3 ) in the control group was 6.3 \\u00b1 1.4 lytic units (LU; calculated as 5% lysis of 5 \\u00d7 10 3  targets) per 10 6  splenocytes ( n  = 18) in bulk assays, but in the CD4 +  T cell\\u2013 depleted mice it was dramatically lower, at <1.2 \\u00b1 0.1 LU per 10 6  splenocytes ( n  = 6). All of these mice were depleted of NK cells with mAb to NK1.1 in order to reduce background cytotoxicity to better resolve the virus-specific killing ( 12 ). These results collectively suggest that CD4 +  T cells were required for the heterologous activation of CD8 +  CTL and that both CD4 +  and CD8 +  memory T cells were required for protective immunity. A property of memory T cells is their ability to produce much higher levels of cytokines than do naive cells after stimulation ( 14 ). Therefore, we hypothesized that a heterologous virus that stimulated memory T cells would elicit a potent IFN-\\u03b3 response, a cytokine to which VV is known to be extremely sensitive ( 17 ,  30 ). Analysis by ELISA assay of IFN-\\u03b3 concentrations (Fig.  3 \\n I ) in the peritoneal fluid of LCMV-immune mice at day 3 after VV infection (16,115 \\u00b1 4,262 pg/ml,  n  = 12) revealed >1,074-fold higher titers of IFN-\\u03b3 than in uninfected naive mice (<15 \\u00b1 0 pg/ml,  n  = 8) or LCMV-immune mice (<15 \\u00b1 0 pg/ml,  n  = 8) and 5.4-fold (2,979 \\u00b1 1,039 pg/ml;  n  = 12; Student's  t  test,  P  = 0.007) more than control mice challenged with VV. LCMV-immune mice challenged with PV did not show this enhanced production of IFN-\\u03b3 (Fig.  3 \\n II ) nor did VV-immune mice challenged with LCMV (data not shown). We then examined the peritoneal cavity more closely, as this is the site of initial VV replication where the most significant impact on early virus replication could occur. FACS \\u00ae  staining of PECs resolved a significant 8- and 2.7-fold enhanced recruitment of CD8 +  and CD4 +  T cells, respectively, in the LCMV-immune versus naive mice 3 d after challenged with VV (Fig.  4 ,  A  and  B ). Furthermore, the peritoneal T cells in the LCMV-immune mice 3 d after challenge with VV were predominantly of the memory phenotype, CD44 hi . Basing our cutoff value for CD44 hi  (FITC-conjugated anti-CD44;  PharMingen ) at a mean fluorescence of 12.6 units as determined on naive splenocytes, it was found that 92 \\u00b1 5% ( n  = 3) of the peritoneal CD8 +  cells and 75 \\u00b1 19% ( n  = 3) of the peritoneal CD4 +  cells were CD44 hi  memory phenotype. The CD8 +  cells had a mean fluorescence of 182 \\u00b1 40 and the CD4 +  cells had a mean fluorescence of 177 \\u00b1 58. These cells were examined for expression of intracellular IFN-\\u03b3 after treatment with PMA and ionomycin. There was a significant 10- and 2.7-fold increase in the number of IFN-\\u03b3\\u2013producing CD8 +  and CD4 +  T cells, respectively, in LCMV-immune mice versus naive mice challenged with VV (Fig.  4 ,  A  and  B ). To further examine whether IFN-\\u03b3 was playing a role in the protective effect against heterologous viruses, we initially treated LCMV-immune mice with anti-IFN-\\u03b3 antibodies, as previously described ( 31 ). In brief, we gave 0.2 ml of a 1/5 dilution of anti-IFN-\\u03b3 ascites intraperitoneally daily, in combination with a similar intravenous dose on day 0 and day 3 of VV infection. In the first experiment at 4 d after VV infection there was a 20-fold increase in the VV fat pad titer in the anti-IFN-\\u03b3\\u2013treated group (LCMV-immune + VV, 2.77 \\u00b1 0.23 PFU/ml vs. LCMV-immune + VV + anti-IFN-\\u03b3, 4.07 \\u00b1 0.45 PFU/ml;  n  = 5;  P  = 0.04, Student's  t  test). In the second experiment at day 6 after VV infection there was a 13-fold increase in the anti-IFN-\\u03b3\\u2013treated group (LCMV-immune + VV, 4.23 \\u00b1 0.61 PFU/ml vs. LCMV-immune + VV + anti\\u2013IFN-\\u03b3, 5.31 \\u00b1 0.22;  n  = 5;  P  = 0.006, Student's  t  test). To further confirm this observation, we extended these studies into IFN-\\u03b3 receptor knockout (IFN-\\u03b3R KO) mice. Since the IFN-\\u03b3R KO mice were on the 129/SEV background instead of the C57BL/6 background, normal 129/ SEV mice were used as controls (Fig.  5 ). The 129/SEV and 129 IFN-\\u03b3R KO mice developed equivalent high frequency LCMV-specific memory pCTL, as shown by limiting dilution assays (Fig.  5 ,  A  and  B ). 129/SEV mice immune to LCMV developed protective heterologous immunity to PV and VV, similar to that in the C57BL/6 mice. However, IFN-\\u03b3R KO mice immune to LCMV did not resist VV infection, confirming a role for IFN-\\u03b3 in the control of VV by LCMV-induced memory T cells (Fig.  6 ,  A  and  B ). Similar experiments demonstrated that IFN-\\u03b3 was not required for the protective effect against PV (Fig.  6 ,  C  and  D ), suggesting the possibility that other cytokines or even direct cytotoxicity by the cross-reactive CTL may be effective. A role for cytotoxicity could not be easily addressed in these systems, as cytotoxicity-deficient perforin KO mice develop persistent LCMV infections instead of memory T cell responses ( 32 ). Substantial evidence has been provided that T cells can not only be mediators of protective immunity, but also of immunopathology ( 10 ,  33 \\u2013 37 ). Our studies described below of heterologous immunity between viruses indicate that a previous virus infection may predispose a host to a much more severe disease upon infection with an unrelated heterologous virus. In general accordance with an earlier report ( 22 ), mice immunized with LCMV, unlike naive mice, developed severe immunopathology consisting of extensive acute fat necrosis (AFN) with a cellular infiltrate in visceral fat pads 5\\u20137 d after VV challenge (Fig.  7 ). In contrast, prior immunity to PV or MCMV, although protective against VV challenge, did not result in AFN on VV infection. The leukocyte infiltrate within the abdominal fat pads contained 19% \\u00b1 7 (SEM;  n  = 2) CD4 +  T cells and 23% \\u00b1 5 (SEM;  n  = 2) CD8 +  T cells and mediated CTL activity against both VV- (8 \\u00b1 4 lytic units per 10 6  cells;  n  = 5) and LCMV- (9.4 \\u00b1 6.9 lytic units per 10 6  cells;  n  = 5) infected targets. The CTL activity segregated to the CD8 +  T cell population on sorting (data not shown). Therefore, these fat pads contained infiltrates of activated CTL specific not only for VV but also for LCMV. A reactivation of the LCMV-specific memory splenic CTL by VV has previously been documented ( 11 ,  12 ). Adoptive transfer studies showed that splenocytes from LCMV-immune mice not only provided some protective immunity against VV synthesis, but also induced rather severe acute fatty necrosis (Fig.  8 \\n A ). This immunopathology did not occur in mice lacking IFN-\\u03b3 receptors (Fig.  8 \\n B ). Further, LCMV-immune mice acutely infected with VV and treated with anti-IFN-\\u03b3 daily had decreased evidence of AFN (level of AFN: LCMV-immune + VV, 4 \\u00b1 1.2; LCMV-immune + VV + anti-IFN-\\u03b3, 1 \\u00b1 0.5;  n  = 5). As shown above, LCMV-immune mice challenged with VV (Fig.  3 \\n I ) had higher levels of IFN-\\u03b3 in the peritoneal fluid than did naive mice challenged with VV. This enhanced IFN-\\u03b3 response in immune mice may thus play a role in protective immunity as well as in the induction of immunopathology. In this sequence of viruses the immunological contribution of the LCMV-specific memory cells may result in an overwhelming delayed-type hypersensitivity-like cell-mediated immune response, resulting in enhanced pathogenesis, even though the VV is cleared more rapidly. Most models for the immune response to viral infections have used infected animals sheltered from other environmental pathogens, a rather unnatural circumstance. In this study, we show that these sheltered mice responded quite differently to infection than did mice previously exposed to other pathogens, as early protective immunity can be mediated by memory T cells generated by a previous heterologous viral infection. Our results would suggest that both memory CD4 +  and CD8 +  T cells can be reactivated, as both populations were required for the protective immunity to occur in the virus combinations tested. For instance, in LCMV-immune versus a naive host challenged with VV, both memory cell types were recruited early in infection into the peritoneal cavity, the site of virus challenge, and were producing significantly more IFN-\\u03b3, a cytokine to which VV is very sensitive ( 17 ,  30 ). We have not examined the role of memory B cells in this system but clearly memory T cells are required for this type of protective immunity. Although T cells are generally thought to be highly specific for peptide\\u2013MHC complexes, more extensive investigations have shown that a single T cell clone can recognize two unrelated peptide sequences on the same protein, two different proteins of the same virus, and two different proteins from two unrelated viruses ( 38 \\u2013 40 ). It is not surprising that peptides from a second viral infection could cross-react with and stimulate some of the many clones of T cells comprising the memory pool specific for another virus. We have reported that memory CD8 +  T cells can often cross-react between heterologous viruses ( 12 ), and such cross- reactive T cells may contribute to this more rapid early resistance to a second infection. The T cell repertoire to each new infection would be altered as cross-reactive memory cells join in the response, resembling in some ways the original antigenic sin network of antibody responses to different strains of influenza virus, where primary immunity is boosted not by a homologous but, by a cross-reacting vaccine of a different influenza virus serotype, and some of the newly formed antibodies react better with the primary virus than with the virus actually eliciting the response ( 41 ). Memory T cells are known to express high levels of adhesion molecules ( 8 \\u2013 10 ) and to be easier to stimulate ( 13 \\u2013 16 ) than naive cells, making them susceptible to low affinity cross-reactive TcR triggering. Some reports have suggested that memory T cells can be stimulated nonspecifically by cytokines ( 5 ,  6 ,  42 ), and may be cross-activated by an infection that does not require stimulation through their T cell receptor by cross-reactive viral peptides. However, data from our laboratory suggests that memory T cells do not nonspecifically proliferate in response to an antigen to which there are no cross-reactive TcR epitopes detectable ( 43 ), and our experiments on heterologous immunity between viruses argue on behalf of cross-reactive memory cells rather than nonspecific activation effects, because the specific sequence of virus infections determines if there is any protective immunity. Nevertheless, it is possible there is some as yet unknown selective mechanism for cross-activation of different memory T cell populations. The contribution of memory cells specific for one virus to the natural resistance against another virus is presently not predictable and is not necessarily reciprocal. If the immunodominant memory T cell response to an earlier virus could recognize a cross-reactive determinant on the challenge virus, the challenge virus would find a large pool of memory T cells capable of a rapid response. Should the challenge virus not be recognized by part of the immunodominant memory T cell repertoire, then less of a protective effect may ensue. Individuals vary considerably in their responses to viral infections, ranging from subclinical to severe. There are many factors that contribute to this variation in responsiveness, including the dose and route of infection, as well as the physiological state and genetic background of the host. In this study, we show that memory T cells specific to unrelated viruses may also contribute to the host's primary response to a second virus. The beneficial effect of these early protective memory T cells is to slow down the spread of infection, much like a natural immune-mediated response, allowing time for more suitable high affinity antigen-specific T and B cell responses to develop. It is reasonable to expect that this level of resistance may be the difference between clinical and subclinical infections or lethal and nonlethal infections. The detrimental immunopathological effect of memory T cells leads us to question whether they may play a role in human infections like with Epstein-Barr virus or varicella-zoster virus that lead to much more severe disease in young adults than in young children, who presumably have a more restricted memory T cell repertoire ( 44 ,  45 ). Reactivation of memory T cells may also play a role in exacerbation of autoimmune diseases, such as multiple sclerosis or diabetes. For example, in an experimental model for chronic virus-induced CNS autoimmunity, disease may be induced by infection with a virus sharing epitopes with a protein expressed in oligodendrocytes, and this T cell\\u2013dependent disease could be exacerbated by a second infection with an unrelated virus ( 46 ). Thus, alterations in acute responses caused by the memory T cell modulations induced by the host's history of prior infections may have the paradoxical effect of producing both beneficial protective heterologous immunity and/or immunopathology. \",\n          \"Reactive oxygen species (H 2 O 2 ,  \\u2022 O 2 \\n -,\\u2022 OH) are important mediators of endothelial cell (EC) *  injury and likely play a central role in cardiovascular, cerebrovascular, and pulmonary syndromes such as myocardial infarction, ischemic stroke, acute respiratory distress syndrome, and acute organ transplant rejection ( Lounsbury et al., 2000 ). H 2 O 2  appears to be particularly important in the context of neutrophil\\u2013endothelial interaction in inflammation ( Martin, 1984 ;  Lewis et al., 1988 ;  Gasic et al., 1991 ), during which local H 2 O 2  levels may exceed 100\\u2013400 \\u03bcM ( Test and Weiss, 1984 ). H 2 O 2  causes cell depolarization ( Matoba et al., 2000 ) and increases cytosolic-free Ca 2+  concentration ([Ca 2+ ] i ) in EC through a range of concentration-dependent [Ca 2+ ] i  responses, including Ca 2+  influx into EC ( Bowles et al., 2001 ) and Ca 2+  release from intracellular stores ( Schilling and Elliott, 1992 ;  Doan et al., 1994 ;  Siflinger-Birnboim et al., 1996 ;  Yolk et al., 1997 ;  Hu et al., 1998 ). This rise in [Ca 2+ ] i  triggers the release of platelet-activating factor ( Lewis et al., 1988 ) and PGI 2  ( Harlan and Callahan, 1984 ), endothelial retraction, adhesion molecule redistribution ( Bradley et al., 1995 ), EC permeability ( Siflinger-Birnboim et al., 1996 ;  Carbajal and Schaeffer, 1998 ), gene transcription ( Barchowsky et al., 1995 ), and apoptosis ( Martin, 1984 ;  Suzucki et al., 1997 ). H 2 O 2  can also activate tyrosine kinases and inhibit protein tyrosine phosphatases in ECs ( Barchowsky et al., 1995 ;  Carbajal and Schaeffer, 1998 ). Despite the recognized importance of these EC responses to oxidant exposure, the key regulatory proteins modulating EC calcium responses remain largely unknown. One potential participant in this regulation is platelet endothelial adhesion molecule (PECAM)-1/CD-31, a 130-kD type 1 membrane glycoprotein belonging to the Ig superfamily of CAMs that plays an important role in the modulation of EC\\u2013neutrophil interaction and transmigration in inflammation ( Vaporciyan et al., 1993 ;  Nakada et al., 2000 ) and oxidant injury ( Scalia and Lefer, 1998 ). PECAM-1, which functions as both an adhesion and signaling protein, is highly expressed on ECs where it localizes to cell\\u2013cell borders, as well as on leukocytes and platelets ( Newman, 1997 ). Recently, it has been shown that engagement of PECAM-1 elicits prolonged EC calcium transients through a calcium-permeant, plasmalemmel nonselective cation (NSC) channel. Activation of this channel is dependent on Src family kinase activity, but independent of inositol trisphosphate (IP 3 )-mediated store release or phosphoinositide turnover ( Gurubhagavatula et al., 1998 ;  O'Brien et al., 2001 ). In addition to a large extracellular domain containing six Ig-like loops, PECAM-1 has a 118\\u2013amino acid cytoplasmic domain containing a dual tyrosine SH2 binding motif (Y663/Y686). Under a range of conditions, including phosphatase inhibition ( Jackson et al., 1997 ;  Sagawa et al., 1997 ), Src and Csk overexpression ( Lu et al., 1997 ;  Cao et al., 1998 ), and anti\\u2013PECAM-1 antibody engagement ( Varon et al., 1998 ), this motif is selectively tyrosine phosphorylated. Once phosphorylated, PECAM-1 can associate with SH2 domain proteins such as SHP-2, SHP-1, PLC\\u03b3, and PI3-kinase, although the physiologic relevance of these interactions in EC are unknown ( Newman, 1999 ). In light of the evidence that PECAM-1 participates in ionic signaling and may serve as a substrate for phosphotyrosine modification, we hypothesized that PECAM-1 might participate in the regulation of oxidant-mediated calcium signals in ECs. In order to test our hypothesis that PECAM-1 may play a role in oxidant-mediated signaling, we performed voltage clamp experiments on human umbilical vein EC (HUVEC), REN cells, and REN cells stably transfected with PECAM-1 (RHP). We have previously utilized REN cells, a mesothelioma cell line that does not express endogenous PECAM-1, as an EC model, and have found that many EC signaling processes may be reconstituted after PECAM-1 expression ( Gurubhagavatula et al., 1998 ;  O'Brien et al., 2001 ). Puffer pipette application of 200 \\u03bcM H 2 O 2  (a dose based on preliminary studies described below) to HUVEC and RHP cells voltage clamped at a holding potential of \\u201360 mV, elicited a large inward current in 93% of HUVEC ( n  = 15) with a peak amplitude of 648.4 \\u00b1 36.6 pA. A similar current (peak amplitude 682.5 \\u00b1 41.55 pA) was detected in 89% of RHP ( n  = 18), whereas no currents were observed in untransfected REN cells exposed to 200 ( n  = 16) or 400 \\u03bcM ( n  = 11) H 2 O 2  ( Fig. 1 ). H 2 O 2  currents were typically activated 30\\u201390 s after H 2 O 2  exposure and were of prolonged time course, features similar to cation currents evoked in HUVEC and RHP cells after PECAM-1 engagement ( O'Brien et al., 2001 ). The response to H 2 O 2  was concentration dependent in that RHP cells treated with H 2 O 2  at concentrations of 100 \\u03bcM or lower elicited current less consistently (though those responding manifested similar current kinetics). Cells that responded to 400 or 500 \\u03bcM H 2 O 2  displayed evident cell toxicity over 24 h (unpublished data), consistent with previous reports ( Barchowsky et al., 1994 ). Puffer application of vehicle alone did not elicit a current, further eliminating a potential role for shear stress\\u2013gated NSC channels or transient loss of seal in the response (unpublished data). To further confirm the role of PECAM-1 in both HUVEC and RHP cells, H 2 O 2  current responses were recorded in cells dialyzed with polyclonal anti\\u2013PECAM-1 cytoplasmic domain (PCD) antibody or control polyclonal antibody directed against herpes simplex virus thymidine kinase ([anti-] HSVTK). 200 \\u03bcM H 2 O 2  elicited currents with identical kinetics to nondialyzed cells in 83% of HUVEC ( n  = 12) and 80% RHP cells ( n  = 5) dialyzed with anti-HSVTK antibody. By contrast, in cells dialyzed with PCD, currents were completely blocked in 81% of HUVEC ( n  = 16) and 75% of RHP ( n  = 4) cells, and markedly attenuated in those cells in which currents were observed ( Fig. 2 ). The ion selectivity and voltage dependence of the PECAM-1 current were examined in HUVEC and RHP. The reversal potential of the evoked current was determined by voltage ramp (\\u221260 up to +50 mV for 500 ms) and voltage step (\\u221260 to +40 mV in 20 mV increments) protocols in both RHP cells ( Fig. 3 ) and HUVEC (unpublished data). The current\\u2013voltage relationship of the PECAM-1 current was ohmic, with no evidence of voltage-dependent gating or rectification. Similarly, initial trials with instantaneous voltage ramps to +100 mV showed no rectification (unpublished data). The reversal potential in standard physiologic solutions was \\u223c0 mV, close to the theoretical monovalent cation equilibrium potential of \\u22121.2 mV. Consistent with a NSC current, replacement of extracellular Na +  ions with impermeant Tris resulted in a left shift of the current reversal potential and a decrease in the slope conductance of the inward current. Conversely, asymmetric impermeant anion and symmetric cation solutions (CsAcetate pipette and CsCl bath) did not shift the reversal potential nor alter the slope conductance (unpublished data). To assess the permeability of PECAM-1\\u2013activated NSC channels to Ca 2+ , we examined ionic signaling in extracellular solution containing 100 mM Ca 2+  as the only permeant cation. As shown in  Fig. 4 , exposure of cells to 200 \\u03bcM H 2 O 2  activated a prolonged Ca 2+  current in 5/6 (83%) HUVEC and 6/6 (100%) RHP cells. The kinetics of this current were similar to those observed in monovalent cation solutions, and the current reversal potential was shifted to positive potentials, as predicted for Ca 2+ -permeant channels. These results are identical to those previously obtained after homophilic PECAM-1 engagement and anti\\u2013PECAM-1 antibody ligation ( O'Brien et al., 2001 ), suggesting that the same channel mediates both PECAM-1\\u2013dependent ionic signaling events. To further examine the role of PECAM-1 in mediating H 2 O 2 -activated calcium signaling, we measured H 2 O 2 -induced Ca 2+  fluorescence with laser scanning confocal microscopy in individual Fluo-4\\u2013loaded REN and RHP cells exposed to 200 \\u03bcM H 2 O 2 . As shown in  Fig. 5 , H 2 O 2  evoked a prolonged rise in intracellular calcium lasting about 5 min ( Fig. 5, A and B ), a time course quite similar to that of the PECAM-1\\u2013dependent cation current (compare  Figs. 1  and  5 ). Conversely, in REN cells, no increase in intracellular calcium was observed, confirming the requirement for PECAM-1 in this calcium response ( Fig. 5  B, inset). Addition of thrombin yielded calcium signals in both cell lines that were of rapid onset, but of much shorter duration, whereas buffer did not produce a rise in [Ca 2+ ] i  in either cell type ( Fig. 5  B). These findings suggest that EC PECAM-1 regulates the gating of a calcium-permeant, voltage-independent, NSC channel that mediates prolonged calcium signals observed in ECs after physiologic H 2 O 2  exposure. We have previously reported that PECAM-1 engagement activates a calcium-permeant channel that is independent of intracellular Ca 2+  store release ( O'Brien et al., 2001 ). We next sought to determine the relationship of the PECAM-1\\u2013dependent H 2 O 2  cation current to phosphoinositide-mediated intracellular calcium store release and to subsequently activated endothelial plasmalemmal nonspecific cation channels termed depletion-activated channel or store-operated channels (SOCs) that mediate capacitative calcium entry ( Vaca and Kunze, 1995 ;  Parekh and Penner, 1997 ;  Kamouchi et al., 1999 ). After dialysis with 5 mg/ml heparin, a compound known to act as an IP 3  receptor antagonist as well as a potential inhibitor of G protein receptor interaction ( Taylor and Broad, 1998 ), H 2 O 2 - and thrombin-induced currents were measured in RHP cells ( Fig. 6 ) and HUVEC (unpublished data). Treatment with 200 \\u03bcM H 2 O 2  evoked typical NSC currents in 8/9 (89%) RHP cells and 3/3 (100%) HUVEC, suggesting that the PECAM-1\\u2013activated NSC is neither IP 3  gated nor dependent on IP 3 -mediated intracellular calcium store release mechanisms, similar to findings after antibody engagement in whole-cell fluorimetry and voltage clamp studies ( O'Brien et al., 2001 ). Conversely, thrombin (1.5 U/ml), a G protein\\u2013coupled receptor agonist known to activate intracellular calcium store release and subsequent NSC activation in HUVEC and REN cells ( Gurubhagavatula et al., 1998 ;  O'Brien et al., 2001 ), activated a large inward current in control recordings ( n  = 9), but this current was completely blocked in cells dialyzed with heparin ( Fig. 6 , inset). Phosphoinositol turnover assays were also performed in RHP cells to confirm these findings as previously described ( Krymskaya et al., 1999 ;  O'Brien et al., 2001 ). Treatment with 200 \\u03bcM H 2 O 2  failed to elicit significant stimulation of phosphoinositide formation, whereas addition of thrombin yielded a robust increase in phosphoinositide turnover (unpublished data). These findings further support a phosphoinositide-independent activation pathway for the PECAM-1 NSC current. The kinetic characteristics of the PECAM-1\\u2013dependent current are similar to DAC/SOC NSC currents activated by intracellular calcium release (capacitative calcium entry), which are sensitive to trivalent cations such as lanthanum chloride (LaCl 3 ) ( Vaca and Kunze, 1995 ;  Parekh and Penner, 1997 ;  Kamouchi et al., 1999 ). Therefore, we evaluated the dependence on intracellular Ca 2+  release and the La 3+  sensitivity of H 2 O 2 -activated PECAM-1\\u2013dependent currents. RHP cells and HUVEC were dialyzed with CsCl pipette solution containing 10 mM EGTA and 0 mM Ca 2+  (with a calculated [Ca 2+ ] i  of <1 nM) and current recorded 10 min after break-in. Dialysis with this intracellular solution did not itself elicit large cation currents similar to those activated by H 2 O 2 , but subsequent exposure (after 3\\u20135 min) to 200 \\u03bcM H 2 O 2  activated currents similar to those observed in cells dialyzed with normal intracellular solution. H 2 O 2  currents were observed in 5/5 (100%) RHP cells and 3/3 (100%) HUVEC tested, whereas no currents were observed prior to H 2 O 2 . Moreover, the H 2 O 2  current was only partially inhibited in the presence of 100 \\u03bcM LaCl 3  ( n  = 3), whereas the current activated by thrombin (1.5 U/ml) was completely inhibited by 50 \\u03bcM LaCl 3  ( n  = 6). Interestingly, thrombin-induced capacitative currents mediated by DAC/SOC NSC channels manifest time course, reversal potential (Vrev = 0), mean peak current amplitude (730 \\u00b1 47pA), and current\\u2013voltage relationships quite similar to those observed for PECAM-1 currents. However, the differences in trivalent sensitivity, calcium gating, and phosphoinositide dependence suggest that the PECAM-1\\u2013dependent current is activated by an alternate signaling mechanism and through channels that are distinct from those mediating capacitative calcium entry ( O'Brien et al., 2001 ). In order to specifically examine the role of the PECAM-1 intracellular domain, REN cells stably transfected with a panel of PECAM-1 mutant constructs were examined in a series of patch clamp studies ( Fig. 7 ). The first construct, PITC, consists of the PECAM-1 extracellular domain fused to the nonhomologous ICAM-1 transmembrane and cytoplasmic domains. This fusion protein does not elicit cation current after antibody engagement ( O'Brien et al., 2001 ), but supports homophilic interaction and localizes to the cell\\u2013cell border in confluent monolayers (unpublished data). No currents were observed in voltage-clamped PITC cells puffed with 200 \\u03bcM H 2 O 2  ( n  = 7), suggesting that the PCD, and possibly the transmembrane domain, are necessary for current activation. A second construct, IL2PCD, was utilized to evaluate the role of the isolated PCD in current regulation. IL2PCD contains the nonhomologous interleuken (IL)-2 receptor (CD25) extracellular and transmembrane domains fused to an intact PECAM-1 cytoplasmic domain that can serve as a substrate for H 2 O 2 -induced tyrosine phosphorylation (unpublished data). In voltage-clamped IL2PCD cells puffed with 200 or 400 \\u03bcM H 2 O 2 , cation currents with kinetics identical to full-length PECAM-1 control cells were observed in 14/15 (93%) cells ( Fig. 7 ). These findings indicate that the PECAM-1 cytoplasmic domain is necessary and sufficient for H 2 O 2  current activation. In EC and RHP cells, PECAM-1 engagement yields calcium transients mediated by a NSC channel that is Src tyrosine kinase-dependent ( Gurubhagavatula et al., 1998 ;  O'Brien et al., 2001 ). Therefore, we sought to determine the relationship of tyrosine kinase activity to the PECAM-1\\u2013dependent H 2 O 2  current. RHP cells ( n  = 8) and HUVEC ( n  = 5) pretreated for 1 h with 10 \\u03bcM genistein and purified with 200 \\u03bcM H 2 O 2  failed to activate a current, suggesting a similar role for tyrosine kinase activity in regulation of this channel. To further refine this observation, voltage-clamped HUVEC and RHP cells were dialyzed through the patch pipette with 10 \\u03bcg/ml Sc-18, an Src-neutralizing rabbit polyclonal antibody specific for the Src family nonreceptor kinases ( Marrero et al., 1995 ). In HUVEC and RHP cells dialyzed with Sc-18, attenuated current was observed in 2/10 (20%) and 3/11 (27%) of cells exposed to 200 \\u03bcM H 2 O 2 , whereas H 2 O 2  failed to elicit any current in the remaining cells. By contrast, exposure of HUVEC and RHP cells dialyzed with an irrelevant rabbit polyclonal control antibody (anti-HSVTK) to H 2 O 2  evoked currents identical to nondialyzed cells in 10/12 (83%) and 4/5 (80%) of cells tested, respectively ( Fig. 8 ). These findings confirm that tyrosine kinase activation, including Src-family nonreceptor tyrosine kinases, is required for activation of PECAM-1\\u2013dependent cation signaling. To determine whether PECAM-1 is a substrate for H 2 O 2 -induced tyrosine phosphorylation, we exposed REN, RHP, and REN cells stably transfected with Y663F/Y686F, a full-length PECAM-1 construct with two Y-to-F point mutations ( Fig. 7 ) known to disrupt PECAM-1 tyrosine phosphorylation and phosphotyrosine-mediated protein\\u2013protein association ( Newman 1999 ), to 0.5 mM H 2 O 2  for 10 min. Anti\\u2013PECAM-1 immunoprecipitation and phosphotyrosine analysis of these lysates revealed that PECAM-1 is heavily tyrosine phosphorylated in RHP and HUVEC after H 2 O 2  exposure, predominantly on the Y663/Y686 cytoplasmic motif ( Fig. 9  A). Subsequently, we analyzed phosphorylation patterns in REN, RHP, and PITC under conditions identical to the patch clamp and Ca 2+  imaging experiments. After a 10-min H 2 O 2  exposure at 200 \\u03bcM, near the threshold dose required for cation signaling, full-length PECAM-1 phosphorylation was increased by 40% above expected baseline phosphorylation ( Lu et al., 1996 ,  1997 ), as seen by densitometric normalization to PECAM-1 levels ( Fig. 9  B). Notably, this concentration of H 2 O 2  appears to represent the threshold dose for PECAM-1 phosphorylation, as increasing H 2 O 2  doses yielded increasingly intense PECAM-1 tyrosine phosphorylation ( Fig. 9, A and B ). No phosphotyrosine signal was detected in PITC or untransfected REN cells ( Fig. 9  B), confirming that the intact PECAM-1 cytoplasmic domain may serve as a substrate for H 2 O 2 -induced tyrosine phosphorylation. To determine whether this motif represents a critical element of the PECAM-1 cytoplasmic domain for cation current signaling, we conducted patch clamp studies on the Y663F/Y686F cells. No cation currents were observed in voltage-clamped Y663F/Y686F cells puffed with 200 \\u03bcM H 2 O 2  ( n  = 8), suggesting that the PECAM-1 cytoplasmic Y663/Y686 SH2-interaction domain is necessary for current activation. Notably, Y663F/Y686F and PITC also serve as transfection controls demonstrating that current activation is not merely an artifact of transfection, selection, or heterologous protein expression. In order to test our hypothesis that PECAM-1 may play a role in oxidant-mediated signaling, we performed voltage clamp experiments on human umbilical vein EC (HUVEC), REN cells, and REN cells stably transfected with PECAM-1 (RHP). We have previously utilized REN cells, a mesothelioma cell line that does not express endogenous PECAM-1, as an EC model, and have found that many EC signaling processes may be reconstituted after PECAM-1 expression ( Gurubhagavatula et al., 1998 ;  O'Brien et al., 2001 ). Puffer pipette application of 200 \\u03bcM H 2 O 2  (a dose based on preliminary studies described below) to HUVEC and RHP cells voltage clamped at a holding potential of \\u201360 mV, elicited a large inward current in 93% of HUVEC ( n  = 15) with a peak amplitude of 648.4 \\u00b1 36.6 pA. A similar current (peak amplitude 682.5 \\u00b1 41.55 pA) was detected in 89% of RHP ( n  = 18), whereas no currents were observed in untransfected REN cells exposed to 200 ( n  = 16) or 400 \\u03bcM ( n  = 11) H 2 O 2  ( Fig. 1 ). H 2 O 2  currents were typically activated 30\\u201390 s after H 2 O 2  exposure and were of prolonged time course, features similar to cation currents evoked in HUVEC and RHP cells after PECAM-1 engagement ( O'Brien et al., 2001 ). The response to H 2 O 2  was concentration dependent in that RHP cells treated with H 2 O 2  at concentrations of 100 \\u03bcM or lower elicited current less consistently (though those responding manifested similar current kinetics). Cells that responded to 400 or 500 \\u03bcM H 2 O 2  displayed evident cell toxicity over 24 h (unpublished data), consistent with previous reports ( Barchowsky et al., 1994 ). Puffer application of vehicle alone did not elicit a current, further eliminating a potential role for shear stress\\u2013gated NSC channels or transient loss of seal in the response (unpublished data). To further confirm the role of PECAM-1 in both HUVEC and RHP cells, H 2 O 2  current responses were recorded in cells dialyzed with polyclonal anti\\u2013PECAM-1 cytoplasmic domain (PCD) antibody or control polyclonal antibody directed against herpes simplex virus thymidine kinase ([anti-] HSVTK). 200 \\u03bcM H 2 O 2  elicited currents with identical kinetics to nondialyzed cells in 83% of HUVEC ( n  = 12) and 80% RHP cells ( n  = 5) dialyzed with anti-HSVTK antibody. By contrast, in cells dialyzed with PCD, currents were completely blocked in 81% of HUVEC ( n  = 16) and 75% of RHP ( n  = 4) cells, and markedly attenuated in those cells in which currents were observed ( Fig. 2 ). The ion selectivity and voltage dependence of the PECAM-1 current were examined in HUVEC and RHP. The reversal potential of the evoked current was determined by voltage ramp (\\u221260 up to +50 mV for 500 ms) and voltage step (\\u221260 to +40 mV in 20 mV increments) protocols in both RHP cells ( Fig. 3 ) and HUVEC (unpublished data). The current\\u2013voltage relationship of the PECAM-1 current was ohmic, with no evidence of voltage-dependent gating or rectification. Similarly, initial trials with instantaneous voltage ramps to +100 mV showed no rectification (unpublished data). The reversal potential in standard physiologic solutions was \\u223c0 mV, close to the theoretical monovalent cation equilibrium potential of \\u22121.2 mV. Consistent with a NSC current, replacement of extracellular Na +  ions with impermeant Tris resulted in a left shift of the current reversal potential and a decrease in the slope conductance of the inward current. Conversely, asymmetric impermeant anion and symmetric cation solutions (CsAcetate pipette and CsCl bath) did not shift the reversal potential nor alter the slope conductance (unpublished data). To assess the permeability of PECAM-1\\u2013activated NSC channels to Ca 2+ , we examined ionic signaling in extracellular solution containing 100 mM Ca 2+  as the only permeant cation. As shown in  Fig. 4 , exposure of cells to 200 \\u03bcM H 2 O 2  activated a prolonged Ca 2+  current in 5/6 (83%) HUVEC and 6/6 (100%) RHP cells. The kinetics of this current were similar to those observed in monovalent cation solutions, and the current reversal potential was shifted to positive potentials, as predicted for Ca 2+ -permeant channels. These results are identical to those previously obtained after homophilic PECAM-1 engagement and anti\\u2013PECAM-1 antibody ligation ( O'Brien et al., 2001 ), suggesting that the same channel mediates both PECAM-1\\u2013dependent ionic signaling events. To further examine the role of PECAM-1 in mediating H 2 O 2 -activated calcium signaling, we measured H 2 O 2 -induced Ca 2+  fluorescence with laser scanning confocal microscopy in individual Fluo-4\\u2013loaded REN and RHP cells exposed to 200 \\u03bcM H 2 O 2 . As shown in  Fig. 5 , H 2 O 2  evoked a prolonged rise in intracellular calcium lasting about 5 min ( Fig. 5, A and B ), a time course quite similar to that of the PECAM-1\\u2013dependent cation current (compare  Figs. 1  and  5 ). Conversely, in REN cells, no increase in intracellular calcium was observed, confirming the requirement for PECAM-1 in this calcium response ( Fig. 5  B, inset). Addition of thrombin yielded calcium signals in both cell lines that were of rapid onset, but of much shorter duration, whereas buffer did not produce a rise in [Ca 2+ ] i  in either cell type ( Fig. 5  B). These findings suggest that EC PECAM-1 regulates the gating of a calcium-permeant, voltage-independent, NSC channel that mediates prolonged calcium signals observed in ECs after physiologic H 2 O 2  exposure. We have previously reported that PECAM-1 engagement activates a calcium-permeant channel that is independent of intracellular Ca 2+  store release ( O'Brien et al., 2001 ). We next sought to determine the relationship of the PECAM-1\\u2013dependent H 2 O 2  cation current to phosphoinositide-mediated intracellular calcium store release and to subsequently activated endothelial plasmalemmal nonspecific cation channels termed depletion-activated channel or store-operated channels (SOCs) that mediate capacitative calcium entry ( Vaca and Kunze, 1995 ;  Parekh and Penner, 1997 ;  Kamouchi et al., 1999 ). After dialysis with 5 mg/ml heparin, a compound known to act as an IP 3  receptor antagonist as well as a potential inhibitor of G protein receptor interaction ( Taylor and Broad, 1998 ), H 2 O 2 - and thrombin-induced currents were measured in RHP cells ( Fig. 6 ) and HUVEC (unpublished data). Treatment with 200 \\u03bcM H 2 O 2  evoked typical NSC currents in 8/9 (89%) RHP cells and 3/3 (100%) HUVEC, suggesting that the PECAM-1\\u2013activated NSC is neither IP 3  gated nor dependent on IP 3 -mediated intracellular calcium store release mechanisms, similar to findings after antibody engagement in whole-cell fluorimetry and voltage clamp studies ( O'Brien et al., 2001 ). Conversely, thrombin (1.5 U/ml), a G protein\\u2013coupled receptor agonist known to activate intracellular calcium store release and subsequent NSC activation in HUVEC and REN cells ( Gurubhagavatula et al., 1998 ;  O'Brien et al., 2001 ), activated a large inward current in control recordings ( n  = 9), but this current was completely blocked in cells dialyzed with heparin ( Fig. 6 , inset). Phosphoinositol turnover assays were also performed in RHP cells to confirm these findings as previously described ( Krymskaya et al., 1999 ;  O'Brien et al., 2001 ). Treatment with 200 \\u03bcM H 2 O 2  failed to elicit significant stimulation of phosphoinositide formation, whereas addition of thrombin yielded a robust increase in phosphoinositide turnover (unpublished data). These findings further support a phosphoinositide-independent activation pathway for the PECAM-1 NSC current. The kinetic characteristics of the PECAM-1\\u2013dependent current are similar to DAC/SOC NSC currents activated by intracellular calcium release (capacitative calcium entry), which are sensitive to trivalent cations such as lanthanum chloride (LaCl 3 ) ( Vaca and Kunze, 1995 ;  Parekh and Penner, 1997 ;  Kamouchi et al., 1999 ). Therefore, we evaluated the dependence on intracellular Ca 2+  release and the La 3+  sensitivity of H 2 O 2 -activated PECAM-1\\u2013dependent currents. RHP cells and HUVEC were dialyzed with CsCl pipette solution containing 10 mM EGTA and 0 mM Ca 2+  (with a calculated [Ca 2+ ] i  of <1 nM) and current recorded 10 min after break-in. Dialysis with this intracellular solution did not itself elicit large cation currents similar to those activated by H 2 O 2 , but subsequent exposure (after 3\\u20135 min) to 200 \\u03bcM H 2 O 2  activated currents similar to those observed in cells dialyzed with normal intracellular solution. H 2 O 2  currents were observed in 5/5 (100%) RHP cells and 3/3 (100%) HUVEC tested, whereas no currents were observed prior to H 2 O 2 . Moreover, the H 2 O 2  current was only partially inhibited in the presence of 100 \\u03bcM LaCl 3  ( n  = 3), whereas the current activated by thrombin (1.5 U/ml) was completely inhibited by 50 \\u03bcM LaCl 3  ( n  = 6). Interestingly, thrombin-induced capacitative currents mediated by DAC/SOC NSC channels manifest time course, reversal potential (Vrev = 0), mean peak current amplitude (730 \\u00b1 47pA), and current\\u2013voltage relationships quite similar to those observed for PECAM-1 currents. However, the differences in trivalent sensitivity, calcium gating, and phosphoinositide dependence suggest that the PECAM-1\\u2013dependent current is activated by an alternate signaling mechanism and through channels that are distinct from those mediating capacitative calcium entry ( O'Brien et al., 2001 ). In order to specifically examine the role of the PECAM-1 intracellular domain, REN cells stably transfected with a panel of PECAM-1 mutant constructs were examined in a series of patch clamp studies ( Fig. 7 ). The first construct, PITC, consists of the PECAM-1 extracellular domain fused to the nonhomologous ICAM-1 transmembrane and cytoplasmic domains. This fusion protein does not elicit cation current after antibody engagement ( O'Brien et al., 2001 ), but supports homophilic interaction and localizes to the cell\\u2013cell border in confluent monolayers (unpublished data). No currents were observed in voltage-clamped PITC cells puffed with 200 \\u03bcM H 2 O 2  ( n  = 7), suggesting that the PCD, and possibly the transmembrane domain, are necessary for current activation. A second construct, IL2PCD, was utilized to evaluate the role of the isolated PCD in current regulation. IL2PCD contains the nonhomologous interleuken (IL)-2 receptor (CD25) extracellular and transmembrane domains fused to an intact PECAM-1 cytoplasmic domain that can serve as a substrate for H 2 O 2 -induced tyrosine phosphorylation (unpublished data). In voltage-clamped IL2PCD cells puffed with 200 or 400 \\u03bcM H 2 O 2 , cation currents with kinetics identical to full-length PECAM-1 control cells were observed in 14/15 (93%) cells ( Fig. 7 ). These findings indicate that the PECAM-1 cytoplasmic domain is necessary and sufficient for H 2 O 2  current activation. In EC and RHP cells, PECAM-1 engagement yields calcium transients mediated by a NSC channel that is Src tyrosine kinase-dependent ( Gurubhagavatula et al., 1998 ;  O'Brien et al., 2001 ). Therefore, we sought to determine the relationship of tyrosine kinase activity to the PECAM-1\\u2013dependent H 2 O 2  current. RHP cells ( n  = 8) and HUVEC ( n  = 5) pretreated for 1 h with 10 \\u03bcM genistein and purified with 200 \\u03bcM H 2 O 2  failed to activate a current, suggesting a similar role for tyrosine kinase activity in regulation of this channel. To further refine this observation, voltage-clamped HUVEC and RHP cells were dialyzed through the patch pipette with 10 \\u03bcg/ml Sc-18, an Src-neutralizing rabbit polyclonal antibody specific for the Src family nonreceptor kinases ( Marrero et al., 1995 ). In HUVEC and RHP cells dialyzed with Sc-18, attenuated current was observed in 2/10 (20%) and 3/11 (27%) of cells exposed to 200 \\u03bcM H 2 O 2 , whereas H 2 O 2  failed to elicit any current in the remaining cells. By contrast, exposure of HUVEC and RHP cells dialyzed with an irrelevant rabbit polyclonal control antibody (anti-HSVTK) to H 2 O 2  evoked currents identical to nondialyzed cells in 10/12 (83%) and 4/5 (80%) of cells tested, respectively ( Fig. 8 ). These findings confirm that tyrosine kinase activation, including Src-family nonreceptor tyrosine kinases, is required for activation of PECAM-1\\u2013dependent cation signaling. To determine whether PECAM-1 is a substrate for H 2 O 2 -induced tyrosine phosphorylation, we exposed REN, RHP, and REN cells stably transfected with Y663F/Y686F, a full-length PECAM-1 construct with two Y-to-F point mutations ( Fig. 7 ) known to disrupt PECAM-1 tyrosine phosphorylation and phosphotyrosine-mediated protein\\u2013protein association ( Newman 1999 ), to 0.5 mM H 2 O 2  for 10 min. Anti\\u2013PECAM-1 immunoprecipitation and phosphotyrosine analysis of these lysates revealed that PECAM-1 is heavily tyrosine phosphorylated in RHP and HUVEC after H 2 O 2  exposure, predominantly on the Y663/Y686 cytoplasmic motif ( Fig. 9  A). Subsequently, we analyzed phosphorylation patterns in REN, RHP, and PITC under conditions identical to the patch clamp and Ca 2+  imaging experiments. After a 10-min H 2 O 2  exposure at 200 \\u03bcM, near the threshold dose required for cation signaling, full-length PECAM-1 phosphorylation was increased by 40% above expected baseline phosphorylation ( Lu et al., 1996 ,  1997 ), as seen by densitometric normalization to PECAM-1 levels ( Fig. 9  B). Notably, this concentration of H 2 O 2  appears to represent the threshold dose for PECAM-1 phosphorylation, as increasing H 2 O 2  doses yielded increasingly intense PECAM-1 tyrosine phosphorylation ( Fig. 9, A and B ). No phosphotyrosine signal was detected in PITC or untransfected REN cells ( Fig. 9  B), confirming that the intact PECAM-1 cytoplasmic domain may serve as a substrate for H 2 O 2 -induced tyrosine phosphorylation. To determine whether this motif represents a critical element of the PECAM-1 cytoplasmic domain for cation current signaling, we conducted patch clamp studies on the Y663F/Y686F cells. No cation currents were observed in voltage-clamped Y663F/Y686F cells puffed with 200 \\u03bcM H 2 O 2  ( n  = 8), suggesting that the PECAM-1 cytoplasmic Y663/Y686 SH2-interaction domain is necessary for current activation. Notably, Y663F/Y686F and PITC also serve as transfection controls demonstrating that current activation is not merely an artifact of transfection, selection, or heterologous protein expression. Reactive oxygen species such as hydrogen peroxide are critical mediators of physiologic and pathologic vascular responses after inflammatory, ischemic, and hyperoxic insults. Central to these effects are changes in EC [Ca 2+ ] i , although the molecular interactions underlying H 2 O 2  increases in [Ca 2+ ] i  are not known. Here we report that physiologically relevant concentrations of H 2 O 2  activate Ca 2+ -permeant plasmalemmal cation channels and that this ionic signaling requires PECAM-1 expression. Activation of this current requires the PECAM-1 cytoplasmic Y663/Y686 motif and Src family nonreceptor tyrosine kinase activity, but is independent of IP 3  receptor activation. This is a previously unknown EC function for PECAM-1 that may represent a widespread mechanism linking neutrophil\\u2013EC interactions and EC response in the context of oxidant injury and inflammation. H 2 O 2  has been proposed to function as an endothelial signal transduction intermediate at low micromolar concentrations and as a mediator of cell response to pathologic processes at higher concentrations. Although the signaling processes underlying these actions are not well understood, H 2 O 2  is known to induce tyrosine phosphorylation through activation of tyrosine kinases and inhibition of protein tyrosine phosphatases ( Barchowsky et al., 1995 ;  Carbajal and Schaeffer, 1998 ), and can activate a range of [Ca 2+ ] i  signal responses ( Suzucki et al., 1997 ;  Yang et al., 1999 ;  Lounsbury et al., 2000 ). We observed consistent activation of PECAM-1\\u2013dependent cation currents at concentrations of H 2 O 2  in excess of 100 \\u03bcM. At H 2 O 2  levels below 100 \\u03bcM, minimal Ca 2+  signaling has been reported, whereas sublethal concentrations exceeding 100 \\u03bcM manifest prolonged, low-amplitude [Ca 2+ ] i  transients involving influx of extracellular Ca 2+  ( Bowles et al., 2001 ). Cytotoxic concentrations exceeding 500 \\u03bcM result in the activation of phospholipases and release of Ca 2+  ions from internal calcium stores ( Schilling and Elliott, 1992 ;  Natarjan et al., 1993 ;  Barchowsky et al., 1994 ;  Doan et al., 1994 ;  Siflinger-Birnboim et al., 1996 ;  Yolk et al., 1997 ;  Hu et al., 1998 ;  Min et al., 1998 ). Hydrogen peroxide levels between 200 and 500 \\u03bcM, although not cytolytic ( Shasby et al., 1988 ), may yield substantial cell toxicity by inducing signal transduction pathways that ultimately lead to apoptosis ( Barchowsky et al., 1994 ;  Suhara et al., 1998 ). Our findings indicate that PECAM-1, a protein highly expressed in ECs, functions as a critical regulator of plasmalemmal ion channels that result in depolarization and Ca 2+  influx during sublethal hydrogen peroxide exposure. Although PECAM-1 is essential in regulating plasmalemmal cation influx, our results do not exclude other Ca 2+ -mobilizing actions of H 2 O 2 , particularly at higher H 2 O 2  concentrations. We did not observe consistent PECAM-1\\u2013dependent cation signaling or tyrosine phosphorylation at H 2 O 2  concentrations below 100\\u2013200 \\u03bcM, suggesting that this level of H 2 O 2  represents a threshold for PECAM-1\\u2013mediated events. Thus, low levels of H 2 O 2 , such as those endogenously produced as signaling intermediates during ICAM-dependent EC responses to leukocytes ( Wang and Doerschuk, 2000 ), may be PECAM-1 independent. Instead, PECAM-1 appears to function as a calcium-regulating protein at the moderate H 2 O 2  levels (100\\u2013400 \\u03bcM) that are generated during EC\\u2013neutrophil interaction in the context of inflammation ( Test and Weiss, 1984 ). However, an important caveat in interpreting experiments evaluating H 2 O 2  cellular responses is the substantial variability in effective dosage associated with media conditions, particularly serum concentration. For example, it has been reported that H 2 O 2  concentration\\u2013response relationships may be decreased tenfold relative to serum-free conditions that may support EC morphologic changes after exposure to H 2 O 2  levels as low as 20 \\u03bcM ( Bradley et al., 1995 ). Although we demonstrate that PECAM-1 is required for H 2 O 2  ionic signaling, the identity of the channel and potential linking molecules remain unknown. As the current kinetics, ionic selectivity, and associated [Ca 2+ ] i  transients in ECs are indistinguishable when activated by H 2 O 2  or PECAM-1 engagement ( Gurubhagavatula et al., 1998 ;  O'Brien et al., 2001 ), it is likely that these phenomena are associated with activation of the same channel(s). Similarly, the failure of PECAM-1 to reconstitute cation signaling in certain cell types suggests that PECAM-1 is not, itself, a channel component ( O'Brien et al., 2001 ). Recently, attention has been directed toward mammalian homologues of the  Drosophila  transient receptor potential and transient receptor potential\\u2013like proteins as possible components in multimeric channels activated by Ca 2+  store depletion ( Xu et al., 1997 ;  Moore et al., 1998 ;  Groschner et al., 1999 ). As we have previously reported ( O'Brien et al., 2001 ), there is a distinct similarity between PECAM-1\\u2013dependent currents and nonspecific cation currents observed in ECs after depletion of Ca 2+  stores ( Vaca and Kunze, 1995 ;  Parekh and Penner, 1997 ;  Kamouchi et al., 1999 ). Although the H 2 O 2 -activated current is kinetically similar to store activated currents ( Groschner et al., 1999 ), differences in La 3+  sensitivity, lack of gating associated with Ca 2+  depletion, and independence of IP 3  receptor activation ( Nilius et al., 1997 ;  Kamouchi et al., 1999 ) suggest important differences in channel permeability and gating. Our results are similar to those describing an H 2 O 2 -activated, store release\\u2013independent, plasmalemmal calcium\\u2013conducting channel in EC that is gated through an unknown mechanism ( Bowles et al., 2001 ). They are also consistent with reports of a tyrosine kinase\\u2013dependent NSC channel activated through engagement of basic FGF (bFGF) or insulin-like growth factor\\u20131 receptors in ECs ( Munaron and Pla, 2000 ). As tyrosine kinase activity is required for both receptor-dependent (bFGF or insulin-like growth factor\\u20131) and independent (H 2 O 2  and PECAM-1 ligation) EC Ca +2  signaling, it is likely that the tyrosine kinase substrate(s) responsible for channel activation are downstream of specific receptors, which are phosphorylated by either mechanism ( Fleming et al., 1996 ). Exposure to phosphatase inhibitors such as vanadate, pervanadate, and H 2 O 2  is known to yield selective tyrosine phosphorylation of the PECAM-1 cytoplasmic domain ( Sagawa et al., 1997 ). After tyrosine phosphorylation, PECAM-1 can interact with several SH2 domain\\u2013containing proteins, including SHP2, PI3kinase, and PLC-\\u03b3 ( Newman, 1999 ), though the physiologic relevance of these interactions in EC is not certain. Similarly, PECAM-1 can function as a reservoir for transcription-regulating proteins such as \\u03b2 and \\u03b3-catenin in a phosphorylation-dependent manner ( Ilan et al., 2000 ). Utilizing endothelia-like REN cells stably transfected with wild-type and mutant PECAM-1 constructs, we have begun to conduct a detailed analysis of PECAM-1 structure\\u2013function relationships that would be otherwise impossible in ECs. Where possible, we have confirmed these findings in HUVEC. Our data demonstrate that PECAM-1 and likely other participating proteins serve as substrates for H 2 O 2 -induced tyrosine phosphorylation, that the PECAM-1\\u2013regulated current requires Src family nonreceptor tyrosine kinase activity, and that the PECAM-1 cytoplasmic domain (and its incorporated Y663/Y686 phosphotyrosine motif) is necessary and sufficient for H 2 O 2 -induced current activation ( Fig. 10 ). In confluent cells, full-length PECAM-1, PITC, and Y663F/Y686F localize predominantly to cell\\u2013cell borders, whereas IL2PCD is diffusely distributed ( Sun et al., 2000 ). However, the techniques employed in this study do not allow conclusions to be drawn regarding PECAM-1 cell localization requirements for current regulation in confluent monolayers. Similarly, although H 2 O 2  activates tyrosine kinases including p60src in ECs ( Barchowsky et al., 1995 ;  Carbajal and Schaeffer, 1998 ), the specific tyrosine kinase(s) mediating H 2 O 2 -induced PECAM-1 phosphorylation remains an important unresolved question. Interestingly, preliminary data, beyond the scope of this manuscript, suggest that the PECAM-1 transmembrane domain may also play a role in modulating PECAM-1\\u2013dependent ionic signals in the absence of an intact cytoplasmic domain (unpublished data). This is consistent with findings that the isolated PECAM-1 transmembrane and cytoplasmic domains function in signal transduction after matrix metalloproteinase\\u2013induced extracellular domain cleavage ( Ilan et al., 2001 ). The identification of proteins that have a functional association with PECAM-1 in this response, as well as further delineation of specific tyrosine kinase subtypes underlying H 2 O 2 -induced PECAM-1 phosphorylation, will be important steps in further defining this unique signaling pathway. Modulation of endothelial [Ca 2+ ] i  is critical to a range of EC processes including neutrophil transmigration ( Huang et al., 1993 ), a key EC function regulated by PECAM-1. Exogenous H 2 O 2  stimulates neutrophil extravasation in vivo, which is blocked by anti\\u2013PECAM-1 antibodies ( Scalia and Lefer, 1998 ). Similarly, anti\\u2013PECAM-1 antibodies block neutrophil transmigration after inflammatory stimuli known to induce generation of reactive oxygen species ( Vaporciyan et al., 1993 ;  Chosay et al., 1998 ). Our finding that the PECAM-1 extracellular domain, a critical element in EC\\u2013neutrophil interaction during transmigration ( Nakada et al., 2000 ), is not involved in channel regulation, suggests that H 2 O 2  ionic signaling may not be sufficient to induce leukocyte transmigration. Thus, PECAM-1 may serve dual functions of adhesion (extracellular domain) and ionic signaling (cytoplasmic domain), both of which may be required to recapitulate PECAM-1\\u2013dependent transmigration. The role of PECAM-1\\u2013mediated [Ca 2+ ] i  signaling and the involvement of specific PECAM-1 domains in the regulation of neutrophil\\u2013endothelial interaction and in endothelial response to oxidant injury remains an important area for future investigation. Antibodies used include the following: mAb 4G6, a murine immunoglobin (IgG) directed against the PECAM-1 extracellular Ig loop six domain ( Yan et al., 1995 ); mAb 1.3, a murine IgG directed against the PECAM-1 extracellular domain, a gift of Dr. Peter Newman (Blood Center of Southeastern Wisconsin, Milwaukee, WI); PCD, a rabbit polyclonal antibody directed against the PECAM-1 cytoplasmic domain; Sc-18 (Santa Cruz Biotechnology), a rabbit polyclonal antibody found to specifically inhibit Src family kinases Src, Yes, Fyn, and Fgr ( Marrero et al., 1995 ); anti-HSVTK, rabbit polyclonal anti-HSVTK antibody ( Elshami et al., 1996 ); anti-IL2 receptor (CD25) mAb 3G10 (Boehringer Mannheim); and 4G10, an anti-phosphotyrosine murine mAb (Upstate Biotechnology). Purified antibodies were obtained by protein G affinity chromatography of hybridoma supernatants or serum. Active binding of antibodies was confirmed by flow cytometry. Reagent grade hydrogen peroxide (Sigma-Aldrich) was diluted in isotonic PBS or media on the day of experiments as indicated. PECAM-1 immunoprecipitation was performed on REN cell lysates incubated with mAb 4G6 or anti-CD25 mAb (Il2PCD) followed by incubation with protein A Sepharose (Amersham Pharmacia Biotech). Lysates were separated on 10% SDS polyacrylamide reducing gels (Invitrogen). Membranes were probed with anti-phosphotyrosine antibody (4G10), counterstained with HRP-conjugated donkey anti\\u2013mouse IgG (Cappel), and signals were visualized with ECL (New England Nuclear). Membranes were then stripped per the manufacturer (NEN Life Science Products) and reprobed with mAb 1.3 or PCD and counterstained with donkey anti\\u2013mouse or donkey anti\\u2013rabbit HRP-conjugated secondary antibody (Cappel). Signals were again detected with ECL. Image capture and densitometry analysis was performed on a Kodak Image Station as per the manufacturer. HUVEC (Clonetics) were cultured in M199 medium (Mediatech, Inc.) supplemented with 15% FBS, 100 \\u03bcg/ml EC growth factor (Clonetics), 100 ug/ml Heparin (Elkins-Sinn), and 2 mM  l -glutamine (GIBCO BRL). Only cells of passage six or less were used. PECAM-1 mutant constructs IL2PCD ( Sun et al., 2000 ), PITC, and Y663F/Y686F, a gift of Dr. P. Newman ( Fig. 7 ) were constructed with the sequence overlap extension technique as previously described ( Sun et al., 2000 ;  O'Brien et al., 2001 ). All full-length and mutant PECAM-1 constructs were subcloned into the pcDNA-neo vector. REN cells, a human mesothelioma cell line previously isolated in our laboratory ( Smythe et al., 1994 ), were grown in RPMI media (Mediatech) supplemented with 10% FBS and 2 mM  l -glutamine (R10 media). Stable polyclonal populations of REN cell PECAM-1 transfectants were generated by magnetic bead sorting (Dynal, Inc.) and selected in G418 (0.5 mg/ml) supplemented R10 media as previously described ( Gurubhagavatula et al., 1998 ). Expression was subsequently confirmed by flow cytometry using an EPICS Elite Flow cytometer (Coulter Corporation) as described ( O'Brien et al., 2001 ). All stably transfected cell lines uniformly expressed PECAM-1 at high levels relative to HUVEC ( Fig. 7 ). Cells were prepared for patch clamp experiments on round 12-mm glass coverslips (Fisher) coated with fibronectin (GIBCO BRL) or 1% gelatin (HUVEC). Treated coverslips were placed in 6-well plates and seeded for 18\\u201324 h with 100,000 cells/well in 3 ml media. Whole-cell currents were recorded utilizing standard whole-cell or perforated patch clamp methods as described previously ( O'Brien et al., 2001 ). Coverslips were transferred to a temperature-controlled chamber maintained at 35\\u00b0C (Brook Industries), and superfused at 1 ml/min with either HBSS (123 mM NaCl, 2 mM CaCl 2 , 1 mM KCl, 25 mM Hepes, and 15 mM glucose, pH 7.4), high-calcium solution (100 mM CaCl 2 , 10 mM glucose, 10 mM Hepes, pH 7.4), CsCl solution (130 mM CsCl, 10 mM glucose, 10 mM Hepes, pH 7.4), or Tris-Cl solution (123 mM Tris-HCl, 2 mM CaCl 2 , 1 mM KCl, 25 mM Hepes, and 15 mM glucose, pH 7.4) as indicated. Borosilicate glass electrodes (resistance 3\\u20135 Mohm) were filled with internal solution containing 130 mM CsCl, 1.2 mM MgCl 2 , 0.075 mM EGTA, 1 mM Mg-ATP, and 10 mM Hepes, except for asymmetric anion experiments in which 130 mM CsAcetate was substituted for CsCl as indicated. For perforated patch experiments, pipettes were dipped 1\\u20132 s in pipette solution then backfilled with pipette solution containing 200 mg/ml nystatin. After seal formation and establishment of whole-cell recording configuration, cells were voltage clamped at \\u201360 mV (Axopatch 200B; Axon Instruments). Records were sampled at 1 kHz and filtered at 100 Hz. Current reversal potentials were measured either by step or ramp (\\u221260 up to +100 mV applied every 30 s) depolarization protocols. Only current traces returning to baseline were considered. Hydrogen peroxide solution was puffed directly onto cells through puffer pipettes (Picospritzer; General Valve). EGTA, heparin, and antibody dialysis experiments were conducted with 10 mM EGTA (Sigma-Aldrich), 5 mg/ml heparin sulfate, or 10 \\u03bcg/ml of PCD, Sc-18, or anti-thymidine kinase antibody added to the pipette solution. LaCl 3  inhibition experiments were performed with varying concentrations of LaCl 3  (0\\u2013100 \\u03bcM) in HBSS extracellular solution puffed directly onto cells after agonist stimulation with H 2 O 2  or thrombin. Cells were equilibrated for 10 min after break-in before initiating experimental protocols ( Davis and Sharma, 1997 ;  Wang and Kotlikoff, 1997 ). Cells were incubated with 10 \\u03bcM Fluo-4 AM (Molecular Probes) for 10 min at room temperature in a recording chamber mounted on an inverted microscope (TE300; Nikon), and washed with HBSS extracellular solution for 30 min. Fluo-4 fluorescence was recorded using a laser scanning confocal head (Radiance 2000; Bio-Rad Laboratories) attached to an inverted microscope with a plan-apo 60\\u00d7 water immersion objective (1.2 n.a.; Nikon). Cells were excited with 488 nm light from a krypton/argon laser and x-y images (128 \\u00d7 30 pixels) recorded using Lasersharp software (Bio-Rad Laboratories) at a 44-ms interval. Images were analyzed using LaserPix version 4.2 software (Bio-Rad Laboratories). Fluorescence profiles were obtained by computing the mean fluorescence from a region of the cell (F) for each image and dividing this by the mean fluorescence of the cell prior to stimulation (F o ). Antibodies used include the following: mAb 4G6, a murine immunoglobin (IgG) directed against the PECAM-1 extracellular Ig loop six domain ( Yan et al., 1995 ); mAb 1.3, a murine IgG directed against the PECAM-1 extracellular domain, a gift of Dr. Peter Newman (Blood Center of Southeastern Wisconsin, Milwaukee, WI); PCD, a rabbit polyclonal antibody directed against the PECAM-1 cytoplasmic domain; Sc-18 (Santa Cruz Biotechnology), a rabbit polyclonal antibody found to specifically inhibit Src family kinases Src, Yes, Fyn, and Fgr ( Marrero et al., 1995 ); anti-HSVTK, rabbit polyclonal anti-HSVTK antibody ( Elshami et al., 1996 ); anti-IL2 receptor (CD25) mAb 3G10 (Boehringer Mannheim); and 4G10, an anti-phosphotyrosine murine mAb (Upstate Biotechnology). Purified antibodies were obtained by protein G affinity chromatography of hybridoma supernatants or serum. Active binding of antibodies was confirmed by flow cytometry. Reagent grade hydrogen peroxide (Sigma-Aldrich) was diluted in isotonic PBS or media on the day of experiments as indicated. PECAM-1 immunoprecipitation was performed on REN cell lysates incubated with mAb 4G6 or anti-CD25 mAb (Il2PCD) followed by incubation with protein A Sepharose (Amersham Pharmacia Biotech). Lysates were separated on 10% SDS polyacrylamide reducing gels (Invitrogen). Membranes were probed with anti-phosphotyrosine antibody (4G10), counterstained with HRP-conjugated donkey anti\\u2013mouse IgG (Cappel), and signals were visualized with ECL (New England Nuclear). Membranes were then stripped per the manufacturer (NEN Life Science Products) and reprobed with mAb 1.3 or PCD and counterstained with donkey anti\\u2013mouse or donkey anti\\u2013rabbit HRP-conjugated secondary antibody (Cappel). Signals were again detected with ECL. Image capture and densitometry analysis was performed on a Kodak Image Station as per the manufacturer. HUVEC (Clonetics) were cultured in M199 medium (Mediatech, Inc.) supplemented with 15% FBS, 100 \\u03bcg/ml EC growth factor (Clonetics), 100 ug/ml Heparin (Elkins-Sinn), and 2 mM  l -glutamine (GIBCO BRL). Only cells of passage six or less were used. PECAM-1 mutant constructs IL2PCD ( Sun et al., 2000 ), PITC, and Y663F/Y686F, a gift of Dr. P. Newman ( Fig. 7 ) were constructed with the sequence overlap extension technique as previously described ( Sun et al., 2000 ;  O'Brien et al., 2001 ). All full-length and mutant PECAM-1 constructs were subcloned into the pcDNA-neo vector. REN cells, a human mesothelioma cell line previously isolated in our laboratory ( Smythe et al., 1994 ), were grown in RPMI media (Mediatech) supplemented with 10% FBS and 2 mM  l -glutamine (R10 media). Stable polyclonal populations of REN cell PECAM-1 transfectants were generated by magnetic bead sorting (Dynal, Inc.) and selected in G418 (0.5 mg/ml) supplemented R10 media as previously described ( Gurubhagavatula et al., 1998 ). Expression was subsequently confirmed by flow cytometry using an EPICS Elite Flow cytometer (Coulter Corporation) as described ( O'Brien et al., 2001 ). All stably transfected cell lines uniformly expressed PECAM-1 at high levels relative to HUVEC ( Fig. 7 ). Cells were prepared for patch clamp experiments on round 12-mm glass coverslips (Fisher) coated with fibronectin (GIBCO BRL) or 1% gelatin (HUVEC). Treated coverslips were placed in 6-well plates and seeded for 18\\u201324 h with 100,000 cells/well in 3 ml media. Whole-cell currents were recorded utilizing standard whole-cell or perforated patch clamp methods as described previously ( O'Brien et al., 2001 ). Coverslips were transferred to a temperature-controlled chamber maintained at 35\\u00b0C (Brook Industries), and superfused at 1 ml/min with either HBSS (123 mM NaCl, 2 mM CaCl 2 , 1 mM KCl, 25 mM Hepes, and 15 mM glucose, pH 7.4), high-calcium solution (100 mM CaCl 2 , 10 mM glucose, 10 mM Hepes, pH 7.4), CsCl solution (130 mM CsCl, 10 mM glucose, 10 mM Hepes, pH 7.4), or Tris-Cl solution (123 mM Tris-HCl, 2 mM CaCl 2 , 1 mM KCl, 25 mM Hepes, and 15 mM glucose, pH 7.4) as indicated. Borosilicate glass electrodes (resistance 3\\u20135 Mohm) were filled with internal solution containing 130 mM CsCl, 1.2 mM MgCl 2 , 0.075 mM EGTA, 1 mM Mg-ATP, and 10 mM Hepes, except for asymmetric anion experiments in which 130 mM CsAcetate was substituted for CsCl as indicated. For perforated patch experiments, pipettes were dipped 1\\u20132 s in pipette solution then backfilled with pipette solution containing 200 mg/ml nystatin. After seal formation and establishment of whole-cell recording configuration, cells were voltage clamped at \\u201360 mV (Axopatch 200B; Axon Instruments). Records were sampled at 1 kHz and filtered at 100 Hz. Current reversal potentials were measured either by step or ramp (\\u221260 up to +100 mV applied every 30 s) depolarization protocols. Only current traces returning to baseline were considered. Hydrogen peroxide solution was puffed directly onto cells through puffer pipettes (Picospritzer; General Valve). EGTA, heparin, and antibody dialysis experiments were conducted with 10 mM EGTA (Sigma-Aldrich), 5 mg/ml heparin sulfate, or 10 \\u03bcg/ml of PCD, Sc-18, or anti-thymidine kinase antibody added to the pipette solution. LaCl 3  inhibition experiments were performed with varying concentrations of LaCl 3  (0\\u2013100 \\u03bcM) in HBSS extracellular solution puffed directly onto cells after agonist stimulation with H 2 O 2  or thrombin. Cells were equilibrated for 10 min after break-in before initiating experimental protocols ( Davis and Sharma, 1997 ;  Wang and Kotlikoff, 1997 ). Cells were incubated with 10 \\u03bcM Fluo-4 AM (Molecular Probes) for 10 min at room temperature in a recording chamber mounted on an inverted microscope (TE300; Nikon), and washed with HBSS extracellular solution for 30 min. Fluo-4 fluorescence was recorded using a laser scanning confocal head (Radiance 2000; Bio-Rad Laboratories) attached to an inverted microscope with a plan-apo 60\\u00d7 water immersion objective (1.2 n.a.; Nikon). Cells were excited with 488 nm light from a krypton/argon laser and x-y images (128 \\u00d7 30 pixels) recorded using Lasersharp software (Bio-Rad Laboratories) at a 44-ms interval. Images were analyzed using LaserPix version 4.2 software (Bio-Rad Laboratories). Fluorescence profiles were obtained by computing the mean fluorescence from a region of the cell (F) for each image and dividing this by the mean fluorescence of the cell prior to stimulation (F o ). \",\n          \"Spontaneous regression of a malignant tumor is very rare. It has been estimated that this occurs less than once in 60,000 to 100,000 people with cancer and about 20 such cases have been reported annually ( 1 ,  2 ). Spontaneous regression of a malignant tumor has often noted in hypernephroma, neuroblastoma, malignant melanoma, choriocarcinoma, bladder cancer, but this is extremely rare for stomach cancer ( 3 ). We recently experienced a case of spontaneous regression of advanced gastric cancer, Borrmann type IV, and we report on our case along with a review of the related literature. An 84 yr-old male with a history of nausea and vomiting for the previous 3 weeks was admitted to our hospital. He used no medicine or herbal material except for an oral hypoglycemic agent for diabetes mellitus for 10 yr. His vital signs were stable and the physical examination revealed a palpable mass at the epigastrium. On laboratory examination, the WBC was 7,630/\\u00b5L, the hemoglobin level was 14.2 g/dL, platelet 194,000/\\u00b5L, fasting glucose 140 mg/dL, blood urea nitrogen 24.6 mg/dL, creatinine 1.25 mg/dL, total protein 6.62 g/dL, albumin 3.49 g/dL, AST 18 IU/L, ALT 13 IU/L, total bilirubin 0.41 mg/dL, direct bilirubin 0.18 mg/dL, alkaline phosphatase 126 IU/L, \\u03b3-GTP 11 IU/L, the prothrombin time (INR) was 1.0 and the activated partial thromboplastin time was 23.0 sec. Esophagogastroduodenoscopy (EGD) showed rugal fold thickening and widening along the whole length of the gastric body. The mucosa was congested and it looked dysplastic. When touched with biopsy forceps, the mucosa showed friability and bleeding to easy touch ( Fig. 1 ). Since the time of discharge, he took famotidine and itopride HCL without any other medication or herbal medication and he revisited our hospital for follow up 6 months later. EGD showed that the gastric folds were well distended by air inflation and the mucosa looked normal at the previously dysplastic and friable areas from the cardia to the lower body ( Fig. 2 ). Abdominal computerized tomography (CT) revealed diffuse irregular wall thickening and mucosal enhancement from the cardia to the lower body. Perigastric infiltration was present, yet lymph node enlargement or distant metastatic lesion was not identified ( Fig. 3A ). Multiple pieces of mucosal biopsy were obtained and the histological examination showed poorly differentiated adenocarcinoma ( Fig. 4A-C ). Malignant cells were stained at Periodic acid Schiff (PAS) stain and immunohistochemical stain was positive for cytokeratin AE1/AE3 and negative for LCA. Finally, we diagnosed the patient as suffering with advanced gastric cancer, Borrmann type IV, stage II (T3N0M0) and we planned surgical resection. But patient refused surgical resection and chemotherapy on consolation and then he was discharged. Abdominal CT revealed decreased wall thickening of the body compare to that of 6 months ago. The perigastric infiltration had regressed ( Fig. 3B ). The histology showed no evidence of malignancy ( Fig. 4D ). We examed DNA test (DNA fingerprinting) using autosomal short tandem repeat to know whether the two biopsy specimens were from the same patient. And the result of DNA test revealed that the DNA of two specimens were identical. The patient is alive without any digestive symptoms 14 months after the follow up exam. According to Everson's definition, spontaneous regression of malignant tumor is partial or complete disappearance of a malignant tumor in the absence of treatment that is ordinarily considered to be capable of producing regression. Further, the regression need not progress to complete disappearance of the tumor ( 4 ). This is very rare phenomenon. Of the 741 cases of spontaneous regression of malignant tumor from the year 1900 year to 1987, nine types of tumor (hypernephroma, malignant melanoma, neuroblastoma, retinoblastoma, leukemia, lymphoma, breast cancer, bladder cancer and choriocarcinoma) were noted in 511 cases (69%) and stomach cancer was noted in 17 cases (2.29%) ( 3 ). Regarding gastric cancer, 19 cases ( 5 ) of spontaneous regression have been reported. Spontaneous regression of malignant tumor is very rare and its cause is mostly ambiguous, so it is very difficult to determine the scientific mechanism. Yet the mechanisms suggested till now are a hormonal or immunological mechanism, operative trauma, necrosis or vascular insufficiency, infection and a psychological mechanism ( 1 ,  3 ). A hormonal mechanism was suggested because the onset of a pregnancy ( 6 ) or the cessation of ingestion of oral contraceptives was associated with the onset of a spontaneous regression of tumor ( 7 ,  8 ). In many cases of renal cell carcinoma, the remaining tumor mass regressed after the primary tumor or metastases had been surgically resected ( 9 ,  10 ), so we can presume that host immune system can destroy remaining tumor after the removal of the tumor burden ( 11 ). Other important mechanism is a febrile response associated with infection ( 12 ,  13 ). Of the patients with lung cancer, the 5-yr survival is increased in patients who are complicated with postoperative empyema ( 14 ). It was also reported that a patient with hepatocellular carcinoma developed severe cholangitis due to choledocholithiasis, and then hepatectomy was performed but the resected tumor demonstrated complete necrosis ( 15 ). In addition, some authors suggested a psychological factor ( 16 ) as the possible mechanism, but the evidence on this is still not clear. At first, we suspected that the biopsy specimens could be mistaken. But, it was proven that the DNA of two specimens were identical according to the results of DNA fingerprinting. And the patient had no twin brother. So we could conclude that it was the spontaneous regression of advanced gastric cancer. In this case, we could not identify any possible reason for the spontaneous regression of advanced gastric cancer. We herein report this very rare case, with a review of the related literature, in which advanced gastric cancer spontaneously regressed in 6 months without any clear cause. \"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 88
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import csv\n",
        "\n",
        "\n",
        "def append_csv_files(file1, file2, output_file):\n",
        "    # Increase the field size limit\n",
        "    csv.field_size_limit(100000000)  # Set to a large value, e.g., 100 MB\n",
        "\n",
        "    # Open the first file and read its contents\n",
        "    with open(file1, mode='r', newline='') as f1:\n",
        "        reader1 = csv.reader(f1)\n",
        "        data1 = list(reader1)\n",
        "\n",
        "    # Open the second file and read its contents\n",
        "    with open(file2, mode='r', newline='') as f2:\n",
        "        reader2 = csv.reader(f2)\n",
        "        data2 = list(reader2)\n",
        "\n",
        "    # Open the output file and write the combined contents\n",
        "    with open(output_file, mode='w', newline='') as f_out:\n",
        "        writer = csv.writer(f_out)\n",
        "        # Write the header from the first file\n",
        "        writer.writerow(data1[0])\n",
        "        # Write the data from the first file (excluding the header)\n",
        "        writer.writerows(data1[1:])\n",
        "        # Write the data from the second file (excluding the header)\n",
        "        writer.writerows(data2[1:])\n",
        "# Example usage\n",
        "file1 = '/content/data-for-clinical-note-summarizer/filtered_data.csv'\n",
        "file2 = '/content/data-for-clinical-note-summarizer/filtered_data1.csv'\n",
        "output_file = 'combined_output.csv'\n",
        "append_csv_files(file1, file2, output_file)\n"
      ],
      "metadata": {
        "id": "c5oYfC53Os-Q"
      },
      "execution_count": 90,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "x = pd.read_csv('/content/data-for-clinical-note-summarizer/combined_output.csv')"
      ],
      "metadata": {
        "id": "hT4dWXa6PfDV"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "x.head()"
      ],
      "metadata": {
        "id": "dn95YkzNPkkB"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [],
      "metadata": {
        "id": "1GqG2gKNPeyB"
      }
    }
  ]
}